<!-- Latest compiled and minified CSS & JS -->
<link href="https://cdn.bootcss.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">


<div class="container" style="margin-top: 20px">

    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>1. Genome-wide transcriptomic analysis of BR-deficient Micro-Tom reveals correlations between drought stress tolerance and brassinosteroid signaling in tomato.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29723826" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29723826" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29723826">
                    <p>Brassinosteroids (BRs) are plant steroid hormones that play crucial roles in a range of growth and developmental processes. Although BR signal transduction and biosynthetic pathways have been well characterized in model plants, their biological roles in an important crop, tomato (Solanum lycopersicum), remain unknown. Here, cultivated tomato (WT) and a BR synthesis mutant, Micro-Tom (MT), were compared using physiological and transcriptomic approaches. The cultivated tomato showed higher tolerance to drought and osmotic stresses than the MT tomato. However, BR-defective phenotypes of MT, including plant growth and stomatal closure defects, were completely recovered by application of exogenous BR or complementation with a SlDWARF gene. Using genome-wide transcriptome analysis, 619 significantly differentially expressed genes (DEGs) were identified between WT and MT plants. Several DEGs were linked to known signaling networks, including those related to biotic/abiotic stress responses, lignification, cell wall development, and hormone responses. Consistent with the higher susceptibility of MT to drought stress, several gene sets involved in responses to drought and osmotic stress were differentially regulated between the WT and MT tomato plants. Our data suggest that BR signaling pathways are involved in mediating the response to abiotic stress via fine-tuning of abiotic stress-related gene networks in tomato plants.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29723826">
                    <p>Brassinosteroids（BRs）是植物类固醇激素，在一系列生长和发育过程中发挥重要作用。尽管BR信号转导和生物合成途径在模式植物中得到了很好的表征，但它们在重要作物番茄（Solanum lycopersicum）中的生物学作用仍然未知。在这里，栽培番茄（WT）和BR合成突变体，微型汤姆（MT），使用生理和转录组学方法进行比较。与MT番茄相比，栽培番茄对干旱和渗透胁迫的耐受性更高。然而，通过施用外源BR或与SlDWARF基因互补，MT的BR缺陷表型（包括植物生长和气孔关闭缺陷）完全恢复。使用全基因组转录组分析，在WT和MT植物之间鉴定出619个显着差异表达的基因（DEGs）。几个DEG与已知的信号网络相关，包括与生物/非生物胁迫响应，木质化，细胞壁发育和激素响应相关的那些。与MT对干旱胁迫的较高易感性一致，涉及对干旱和渗透胁迫的响应的几个基因集在WT和MT番茄植物之间被差异调节。我们的数据表明，BR信号通路参与通过微调番茄植物中的非生物胁迫相关基因网络来介导对非生物胁迫的响应。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Plant Physiol. Biochem.">
                Journal: Plant Physiol. Biochem. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29723826" target="_blank">
                PMID: 29723826
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-27
            </a>

            <a class="btn btn-sm btn-danger" title="Lee J, Shim D, Moon S, Kim H, Bae W, Kim K, Kim YH, Rhee SK, Hong CP, Hong SY, Lee YJ, Sung J, Ryu H">
                Authors: Lee J, Shim D, Mo...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29723826">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>2. Sausage fermentation and starter cultures in the era of molecular biology methods.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29723706" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29723706" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29723706">
                    <p>Fermented sausages have a long tradition originating from Europe and they constitute a significant part of the Mediterranean diet. This kind of products has a specific microbiota that is typical of the region or area where they are produced. Therefore, in order to protect the traditional aspect of these products, it is essential to understand the microbial ecology during fermentation by studying the dynamic changes that occur and to select autochthonous starter cultures that can be used in the production. In this paper we summarize the state of the art concerning the selection and use of starter cultures and ecology aspects of naturally fermented sausages. We pay particular attention to the application of bacteriocinogenic strains as they could provide an additional tool in the prevention of foodborne pathogens as well as enhancing the competitiveness of the starter organisms. Microbial ecology of fermented sausages has been determined by traditional microbiological methods, but the introduction in food microbiology of new molecular techniques complements the studies carried out so far and allows scientists to overcome the limitations of traditional methods. Next Generation Sequencing (NGS) techniques represent a change in the way microbiologists address ecology and diversity in foods. Indeed the application of metataxonomics and metagenomics will permit a detailed understanding of microbial ecology. A thorough knowledge of the mechanisms behind the biological processes will enhance meat fermentation control and modulation to obtain products with desired organoleptic properties.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29723706">
                    <p>发酵香肠源自欧洲，传统悠久，是地中海饮食的重要组成部分。这种产品具有特定的微生物群，这是典型的生产地区或地区。因此，为了保护这些产品的传统方面，必须通过研究发酵过程中的动态变化来了解发酵过程中的微生物生态，并选择可用于生产的原始发酵剂培养物。在本文中，我们总结了关于天然发酵香肠的发酵剂和生态学方面的选择和使用的现状。我们特别关注产细菌菌株的应用，因为它们可以提供预防食源性病原体的另外工具，并提高起始生物体的竞争力。已经通过传统的微生物学方法确定了发酵香肠的微生物生态学，但是在食品微生物学中引入新的分子技术是迄今为止所进行的研究的补充，并且允许科学家克服传统方法的局限性。新一代测序（NGS）技术代表了微生物学家处理食物中生态和多样性的方式的变化。事实上，元代经济学和宏基因组学的应用将允许对微生物生态学的详细理解。对生物过程背后机制的全面了解将增强肉类发酵控制和调节，以获得具有所需感官特性的产品。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int. J. Food Microbiol.">
                Journal: Int. J. Food Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29723706" target="_blank">
                PMID: 29723706
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-25
            </a>

            <a class="btn btn-sm btn-danger" title="Franciosa I, Alessandria V, Dolci P, Rantsiou K, Cocolin L">
                Authors: Franciosa I, Ales...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29723706">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>3. CD44 targeting biocompatible and biodegradable hyaluronic acid cross-linked zein nanogels for curcumin delivery to cancer cells: In vitro and in vivo evaluation.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29723613" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29723613" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29723613">
                    <p>In this study, we developed novel hyaluronic acid cross-linked zein nanogels (HA-Zein NGs) to deliver the potential anticancer agent curcumin (CRC), a naturally occurring phytochemical drug in cancer cells. In vitro studies showed that they are highly compatible with the tested cell lines. They showed CD44 specific uptake in CT26 cell line more than by the CD44 receptor pre-inhibited CT26 cells. The CRC encapsulated HA-Zein NGs (HA-Zein-CRC NGs) found to exert a specific toxicity against CT26 sparing healthy normal fibroblast cells in vitro. The apoptotic effects were further confirmed with flow cytometry showing that the HA-Zein-CRC NGs exhibited high anticancer activity against the CT26 cells. The in vivo bio-distribution with a CT26 tumor model showed their high tumor accumulation thereby improved antitumor efficacy with a low dosage of CRC, compared to the previous reports. - Thus, the preclinical studies clearly showed that these novel HA-Zein NGs would be highly beneficial in encapsulating hydrophobic drugs with improved pharmacokinetics thereby enhancing the therapeutic outcomes.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29723613">
                    <p>在这项研究中，我们开发了新型透明质酸交联玉米醇溶蛋白纳米凝胶（HA-Zein NGs），以在癌细胞中提供潜在的抗癌剂姜黄素（CRC），一种天然存在的植物化学药物。体外研究表明它们与测试的细胞系高度相容。它们在CT26细胞系中表现出比通过CD44受体预先抑制的CT26细胞更多的CD44特异性摄取。 CRC封装的HA-Zein NGs（HA-Zein-CRC NGs）发现对CT26发挥特异性毒性，体外保护健康的正常成纤维细胞。流式细胞术进一步证实凋亡效应，表明HA-玉米醇溶蛋白NGs对CT26细胞显示出高抗癌活性。与先前的报道相比，具有CT26肿瘤模型的体内生物分布显示它们的高肿瘤积累，从而用低剂量的CRC改善了抗肿瘤效力。因此，临床前研究清楚地表明，这些新颖的HA-玉米醇溶蛋白在包封具有改善的药代动力学的疏水性药物方面将是非常有益的，从而增强了治疗效果。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Control Release">
                Journal: J Control Release [IF=7.786]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29723613" target="_blank">
                PMID: 29723613
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-30
            </a>

            <a class="btn btn-sm btn-danger" title="Seok HY, Sanoj Rejinold N, Lekshmi KM, Cherukula K, Park IK, Kim YC">
                Authors: Seok HY, Sanoj Re...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29723613">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>4. Can trophectoderm morphology act as a predictor for euploidy?</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29722953" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29722953" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29722953">
                    <p>Euploid embryo transfers yield better implantation rates. In Brazil, morphological evaluation is performed to select the best embryos, since genetic analysis is still an expensive procedure. This study aimed to evaluate whether there is an association between trophectoderm morphology and ploidy status.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29722953">
                    <p>整倍体胚胎转移产生更好的着床率。在巴西，进行形态评估是为了选择最好的胚胎，因为遗传分析仍然是一个昂贵的过程。本研究旨在评估滋养外胚层形态与倍性状态之间是否存在关联。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=JBRA Assist Reprod">
                Journal: JBRA Assist Reprod [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29722953" target="_blank">
                PMID: 29722953
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May-03
            </a>

            <a class="btn btn-sm btn-danger" title="Yoshida IH, Santos M, Berton CZ, Chiarella CL, Tanada MS, BCordts E, Carvalho WP, Barbosa CP">
                Authors: Yoshida IH, Santo...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29722953">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>5. Circulating tumor DNA evaluated by Next-Generation Sequencing is predictive of tumor response and prolonged clinical benefit with nivolumab in advanced non-small cell lung cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29721388" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29721388" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29721388">
                    <p>Nivolumab is an anti-PD1 antibody, given in second-line or later treatment in advanced non-small cell lung cancer (NSCLC). The objective of this study was to describe the predictive value of circulating tumor DNA (ctDNA) on the efficacy of nivolumab in advanced NSCLC. We prospectively included all consecutive patients with advanced NSCLC treated with nivolumab in our Department between June 2015 and October 2016. Plasma samples were obtained before the first injection of nivolumab and at the first tumor evaluation with nivolumab. ctDNA was analyzed by Next-Generation Sequencing (NGS), and the predominant somatic mutation was followed for each patient and correlated with tumor response, clinical benefit (administration of nivolumab for more than 6 months), and progression-free survival (PFS). Of 23 patients, 15 had evaluable NGS results at both times of analysis. ctDNA concentration at the first tumor evaluation and ctDNA change correlated with tumor response, clinical benefit and PFS. ROC curve analyses showed good diagnostic performances for tumor response and clinical benefit, both for ctDNA concentration at the first tumor evaluation (tumor response: positive predictive value (PPV) at 100.0% and negative predictive value (NPV) at 71.0%; clinical benefit: PPV at 83.3% and NPV 77.8%) and the ctDNA change (tumor response: PPV 100.0% and NPV 62.5%; clinical benefit: PPV 100.0% and NPV 80.0%). Patients without ctDNA concentration increase &gt;9% at 2 months had a long-term benefit of nivolumab. In conclusion, NGS analysis of ctDNA allows the early detection of tumor response and long-term clinical benefit with nivolumab in NSCLC.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29721388">
                    <p>Nivolumab是一种抗PD1抗体，用于晚期非小细胞肺癌（NSCLC）的二线治疗或晚期治疗。本研究的目的是描述循环肿瘤DNA（ctDNA）对nivolumab在晚期非小细胞肺癌疗效的预测价值。我们前瞻性地纳入了2015年6月至2016年10月期间我们所有接受nivolumab治疗的晚期非小细胞肺癌患者的连续患者。首次注射nivolumab前和第一次使用nivolumab进行肿瘤评估时获得血浆样本。通过下一代测序（NGS）分析ctDNA，并且每个患者都遵循主要体细胞突变并且与肿瘤反应，临床益处（施用nivolumab超过6个月）和无进展存活（PFS）相关。在23名患者中，15名患者在两次分析中均有可评估的NGS结果。第一次肿瘤评估时的ctDNA浓度和ctDNA的变化与肿瘤反应，临床受益和PFS相关。 ROC曲线分析显示对肿瘤反应和临床益处的良好诊断性能，对于第一次肿瘤评估中的ctDNA浓度（肿瘤反应：100.0％处的阳性预测值（PPV）和71.0％处的阴性预测值（NPV））;临床益处： PPV为83.3％，NPV为77.8％）和ctDNA变化（肿瘤反应：PPV 100.0％和NPV 62.5％;临床获益：PPV 100.0％和NPV 80.0％）。 2个月时无ctDNA浓度增加&gt; 9％的患者具有nivolumab的长期益处。总之，ctDNA的NGS分析可以在NSCLC中用nivolumab早期发现肿瘤反应和长期临床获益。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncoimmunology">
                Journal: Oncoimmunology [IF=7.719]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29721388" target="_blank">
                PMID: 29721388
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Giroux Leprieur E, Herbretau G, Dumenil C, Julie C, Giraud V, Labrune S, Dumoulin J, Tisserand J, Emile JF, Blons H, Chinet T">
                Authors: Giroux Leprieur E...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5927532" target="_blank">
                PMC5927532
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29721388">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>6. Precision cancer therapy: profiting from tumor specific defects in the DNA damage tolerance system.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29721165" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29721165" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29721165">
                    <p>DNA damage tolerance (DDT) enables replication to continue in the presence of a damaged template and constitutes a key step in DNA interstrand crosslink repair. In this way DDT minimizes replication stress inflicted by a wide range of endogenous and exogenous agents, and provides a critical first line defense against alkylating and platinating chemotherapeutics. Effective DDT strongly depends on damage-induced, site-specific PCNA-ubiquitination at Lysine (K) 164 by the E2/E3 complex (RAD6/18). A survey of The Cancer Genome Atlas (TCGA) revealed a high frequency of tumors presents RAD6/RAD18 bi-allelic inactivating deletions. For instance, 11% of renal cell carcinoma and 5% of pancreatic tumors have inactivating RAD18-deletions and 7% of malignant peripheral nerve sheath tumors lack RAD6B. To determine the potential benefit for tumor-specific DDT defects, we followed a genetic approach by establishing unique sets of DDT-proficient Pcna-K164- and -defective Pcna-K164R- lymphoma and breast cancer cell lines. In the absence of exogenous DNA damage, Pcna-K164R- tumors grew comparably to their Pcna-K164- controls in vitro and in vivo. However, DDT-defective lymphomas and breast cancers were compared to their DDT-proficient controls hypersensitive to the chemotherapeutic drug cisplatin (CsPt), both in vitro and in vivo. CsPt strongly inhibited tumor growth and the overall survival of tumor bearing mice greatly improved in the DDT-defective condition. These insights open new therapeutic possibilities for precision cancer medicine with DNA damaging chemotherapeutics and optimize Next-Generation-Sequencing (NGS)-based cancer-diagnostics, -therapeutics, and -prognosis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29721165">
                    <p>DNA损伤耐受性（DDT）使复制能够在存在受损模板的情况下继续进行，并构成DNA链间交联修复的关键步骤。通过这种方式，滴滴涕将广泛的内源性和外源性药物的复制压力降至最低，并为化学治疗药物的烷化和铂化提供了重要的一线防护。有效的DDT强烈依赖E2 / E3复合物（RAD6 / 18）在赖氨酸（K）164处的损伤诱导的位点特异性PCNA泛素化。癌症基因组图谱（TCGA）的调查显示高频率的肿瘤呈现RAD6 / RAD18双等位基因失活缺失。例如，11％的肾细胞癌和5％的胰腺肿瘤已经使RAD18缺失失活，并且7％的恶性外周神经鞘瘤缺乏RAD6B。为了确定肿瘤特异性DDT缺陷的潜在益处，我们通过建立独特的DDT-熟练的Pcna-K164-和 - 缺陷Pcna-K164R-淋巴瘤和乳腺癌细胞系来遵循遗传方法。在没有外源DNA损伤的情况下，Pcna-K164R-肿瘤在体外和体内与其Pcna-K164-对照生长相当。然而，DDT缺陷性淋巴瘤和乳腺癌与体内和体外对化疗药物顺铂（CsPt）过敏的DDT熟练对照相比较。 CsPt强烈抑制肿瘤生长，并且荷瘤小鼠的总体存活率在DDT缺陷状况中大大改善。这些见解为DNA损伤化疗药物治疗精确癌症药物开创了新的治疗可能性，并优化了基于下一代测序（NGS）的癌症诊断，治疗和预后。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29721165" target="_blank">
                PMID: 29721165
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-10
            </a>

            <a class="btn btn-sm btn-danger" title="Buoninfante OA, Pilzecker B, Aslam MA, Zavrakidis I, van der Wiel R, van de Ven M, van den Berk PCM, Jacobs H">
                Authors: Buoninfante OA, P...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5922359" target="_blank">
                PMC5922359
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29721165">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>7. Validation and comparison of two NGS assays for the detection of EGFR T790M resistance mutation in liquid biopsies of NSCLC patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29719623" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29719623" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29719623">
                    <p>Analysis of circulating cell-free DNA (cfDNA) derived from peripheral blood (&quot;liquid biopsy&quot;) is an attractive alternative to identify non-small cell lung cancer (NSCLC) patients with the EGFR T790M mutation eligible for 3rd generation tyrosine kinase inhibitor therapy. We evaluated two PCR-based next generation sequencing (NGS) approaches, one including unique molecular identifiers (UMI), with focus on highly sensitive EGFR T790M mutation detection. Therefore, we extracted and sequenced cfDNA from synthetic plasma samples spiked with mutated DNA at decreasing allele frequencies and from 21 diagnostic NSCLC patients. Data evaluation was performed to determine the limit of detection (LoD), accuracy, specificity and sensitivity of both assays. Considering all tested reference dilutions and mutations the UMI assay performed best in terms of LoD (1% vs. 5%), sensitivity (95.8% vs. 81.3%), specificity (100% vs. 93.8%) and accuracy (96.9% vs. 84.4%). Comparing mutation status of diagnostic samples with both assays showed 81.3% concordance with primary mutation verifiable in 52% of cases. EGFR T790M was detected concordantly in 6/7 patients with allele frequencies from 0.1% to 27%. In one patient, the T790M mutation was exclusively detectable with the UMI assay. Our data demonstrate that both assays are applicable as multi-biomarker NGS tools enabling the simultaneous detection of primary EGFR driver and resistance mutations. However, for mutations with low allelic frequencies the use of NGS panels with UMI facilitates a more sensitive and reliable detection.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29719623">
                    <p>来自外周血的循环无细胞DNA（cfDNA）分析（“液体活组织检查”）是鉴定EGFR T790M突变符合第3代酪氨酸激酶抑制剂治疗的非小细胞肺癌（NSCLC）患者的有吸引力的替代方法。我们评估了两种基于PCR的新一代测序（NGS）方法，其中一种包括独特的分子标识符（UMI），重点是高度敏感的EGFR T790M突变检测。因此，我们从加入突变DNA的合成血浆样品中，以降低的等位基因频率和从21名诊断性NSCLC患者中提取并测序了cfDNA。进行数据评估以确定两种测定的检测限（LoD），准确度，特异性和灵敏度。考虑到所有测试的参考稀释度和突变，UMI测定在LoD（1％对5％），灵敏度（95.8％对81.3％），特异性（100％对93.8％）和准确性（96.9％对。84.4％）。两种方法的诊断样本的突变状态比较显示，在52％的病例中，可以验证81.3％的原发性突变。 EGFR T790M在6/7等位基因频率从0.1％至27％的患者中一致检测。在一名患者中，T790M突变仅在UMI检测中可检测到。我们的数据表明这两种检测方法都适用于多生物标志物NGS工具，可同时检测主要EGFR驱动因子和抗性突变。然而，对于低等位基因频率的突变，使用带有UMI的NGS板有助于更灵敏和可靠的检测。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29719623" target="_blank">
                PMID: 29719623
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-06
            </a>

            <a class="btn btn-sm btn-danger" title="Vollbrecht C, Lehmann A, Lenze D, Hummel M">
                Authors: Vollbrecht C, Leh...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5915090" target="_blank">
                PMC5915090
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29719623">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>8. Genome Sequencing of Ancient Plant Remains: Findings, Uses and Potential Applications for the Study and Improvement of Modern Crops.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29719544" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29719544" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29719544">
                    <p>The advent of new sequencing technologies is revolutionizing the studies of ancient DNA (aDNA). In the last 30 years, DNA extracted from the ancient remains of several plant species has been explored in small-scale studies, contributing to understand the adaptation, and migration patterns of important crops. More recently, NGS technologies applied on aDNA have opened up new avenues of research, allowing investigation of the domestication process on the whole-genome scale. Genomic approaches based on genome-wide and targeted sequencing have been shown to provide important information on crop evolution and on the history of agriculture. Huge amounts of next-generation sequencing (NGS) data offer various solutions to overcome problems related to the origin of the material, such as degradation, fragmentation of polynucleotides, and external contamination. Recent advances made in several crop domestication studies have boosted interest in this research area. Remains of any nature are potential candidates for aDNA recovery and almost all the analyses that can be made on fresh DNA can also be performed on aDNA. The analysis performed on aDNA can shed light on many phylogenetic questions concerning evolution, domestication, and improvement of plant species. It is a powerful instrument to reconstruct patterns of crop adaptation and migration. Information gathered can also be used in many fields of modern agriculture such as classical breeding, genome editing, pest management, and product promotion. Whilst unlocking the hidden genome of ancient crops offers great potential, the onus is now on the research community to use such information to gain new insight into agriculture.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29719544">
                    <p>新的测序技术的出现正在革新古DNA（aDNA）的研究。在过去的30年中，从小规模研究中探索了从几种植物物种的古遗存中提取的DNA，有助于理解重要作物的适应和迁徙模式。最近，应用于DNA的NGS技术开辟了新的研究途径，允许在整个基因组范围内研究驯化过程。已经显示基于全基因组和定向测序的基因组方法提供了有关作物进化和农业历史的重要信息。大量的下一代测序（NGS）数据提供了各种解决方案来克服与材料来源有关的问题，例如多核苷酸的降解，片段化和外部污染。几项作物驯化研究取得的最新进展提升了对这一研究领域的兴趣。任何性质的遗留物都是DNA回收的潜在候选物，几乎所有可以在新鲜DNA上进行的分析也可以在DNA上进行。对DNA进行的分析可以揭示许多关于植物物种进化，驯化和改良的系统发育问题。它是重建作物适应和迁移模式的有力工具。收集的信息也可用于现代农业的许多领域，如经典育种，基因组编辑，虫害管理和产品推广。在解开古老作物隐藏的基因组的同时提供了巨大的潜力，现在研究界有责任利用这些信息来获得对农业的新见解。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Plant Sci">
                Journal: Front Plant Sci [IF=4.298]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29719544" target="_blank">
                PMID: 29719544
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Di Donato A, Filippone E, Ercolano MR, Frusciante L">
                Authors: Di Donato A, Fili...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5914272" target="_blank">
                PMC5914272
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29719544">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>9. Prediction of tumor mutation burden in breast cancer based on the expression of ER, PR, HER-2, and Ki-67.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29719409" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29719409" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29719409">
                    <p>Cancer immunoediting is the process of eliminating highly immunogenic tumor cells by somatic evolution and protecting the host from tumor development in the host immune system. Frequencies of somatic mutations or tumor mutation burden (TMB) were associated with immunogenicity of breast cancer. This study aimed to predict the level of TMB in patients with breast cancer by the expression of estrogen (ER), progesterone (PR), HER-2, and Ki-67, thereby anticipating the prognosis of patients and the possible response to immunotherapy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29719409">
                    <p>癌症免疫编辑是通过体细胞进化消除高度免疫原性肿瘤细胞并保护宿主免受宿主免疫系统中肿瘤发展的过程。体细胞突变或肿瘤突变负荷（TMB）的频率与乳腺癌的免疫原性有关。本研究旨在通过雌激素（ER），孕激素（PR），HER-2和Ki-67的表达来预测乳腺癌患者的TMB水平，从而预测患者的预后和对免疫治疗的可能反应。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Onco Targets Ther">
                Journal: Onco Targets Ther [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29719409" target="_blank">
                PMID: 29719409
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Xu J, Guo X, Jing M, Sun T">
                Authors: Xu J, Guo X, Jing...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5914885" target="_blank">
                PMC5914885
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29719409">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>10. geno2pheno[ngs-freq]: a genotypic interpretation system for identifying viral drug resistance using next-generation sequencing data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29718426" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29718426" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29718426">
                    <p>Identifying resistance to antiretroviral drugs is crucial for ensuring the successful treatment of patients infected with viruses such as human immunodeficiency virus (HIV) or hepatitis C virus (HCV). In contrast to Sanger sequencing, next-generation sequencing (NGS) can detect resistance mutations in minority populations. Thus, genotypic resistance testing based on NGS data can offer novel, treatment-relevant insights. Since existing web services for analyzing resistance in NGS samples are subject to long processing times and follow strictly rules-based approaches, we developed geno2pheno[ngs-freq], a web service for rapidly identifying drug resistance in HIV-1 and HCV samples. By relying on frequency files that provide the read counts of nucleotides or codons along a viral genome, the time-intensive step of processing raw NGS data is eliminated. Once a frequency file has been uploaded, consensus sequences are generated for a set of user-defined prevalence cutoffs, such that the constructed sequences contain only those nucleotides whose codon prevalence exceeds a given cutoff. After locally aligning the sequences to a set of references, resistance is predicted using the well-established approaches of geno2pheno[resistance] and geno2pheno[hcv]. geno2pheno[ngs-freq] can assist clinical decision making by enabling users to explore resistance in viral populations with different abundances and is freely available at http://ngs.geno2pheno.org.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29718426">
                    <p>确定对抗逆转录病毒药物的耐药性对于确保成功治疗感染病毒（如人类免疫缺陷病毒（HIV）或丙型肝炎病毒（HCV））的患者至关重要。与Sanger测序相比，新一代测序（NGS）可以检测少数群体中的抗性突变。因此，基于NGS数据的基因型耐药性测试可提供新颖的治疗相关见解。由于现有的用于分析NGS样品中耐药性的网络服务需要很长的处理时间，并遵循严格的基于规则的方法，因此我们开发了用于快速鉴定HIV-1和HCV样品中耐药性的网络服务geno2pheno [ngs-freq]。通过依赖提供病毒基因组核苷酸或密码子读数的频率文件，消除了处理原始NGS数据的时间密集步骤。一旦上传了频率文件，就会产生一组用户定义的流行截断值的共有序列，从而构建的序列仅包含密码子流行率超过给定截止值的那些核苷酸。在将序列局部对齐到一组参考之后，使用已知的geno2pheno [抗性]和geno2pheno [hcv]方法预测抗性。 geno2pheno [ngs-freq]可以通过使用户探索具有不同丰度的病毒群体的抗性来帮助临床决策制定，可以在http://ngs.geno2pheno.org免费获取。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Nucleic Acids Res.">
                Journal: Nucleic Acids Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29718426" target="_blank">
                PMID: 29718426
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May-01
            </a>

            <a class="btn btn-sm btn-danger" title="Döring M, Büch J, Friedrich G, Pironti A, Kalaghatgi P, Knops E, Heger E, Obermeier M, Däumer M, Thielen A, Kaiser R, Lengauer T, Pfeifer N">
                Authors: Döring M, Büch J,...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29718426">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>11. Droplet Digital™ PCR Next-Generation Sequencing Library QC Assay.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29717460" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29717460" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29717460">
                    <p>Digital PCR is a valuable tool to quantify next-generation sequencing (NGS) libraries precisely and accurately. Accurately quantifying NGS libraries enable accurate loading of the libraries on to the sequencer and thus improve sequencing performance by reducing under and overloading error. Accurate quantification also benefits users by enabling uniform loading of indexed/barcoded libraries which in turn greatly improves sequencing uniformity of the indexed/barcoded samples. The advantages gained by employing the Droplet Digital PCR (ddPCR™) library QC assay includes the precise and accurate quantification in addition to size quality assessment, enabling users to QC their sequencing libraries with confidence.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29717460">
                    <p>数字PCR是精确而准确地定量下一代测序（NGS）文库的有用工具。精确量化NGS文库可以将文库精确加载到序列发生器，从而通过减少误差和超载错误来提高测序性能。准确定量也使用户受益，因为可以统一加载索引/条形码库，从而大大提高索引/条形码样本的测序一致性。采用Droplet Digital PCR（ddPCR™）文库QC检测方法所获得的优势除了尺寸质量评估外，还包括精确和准确的定量，使用户可以放心地对其测序文库进行质量控制。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Methods Mol. Biol.">
                Journal: Methods Mol. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29717460" target="_blank">
                PMID: 29717460
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Heredia NJ">
                Authors: Heredia NJ
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29717460">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>12. [Next Generation Sequencing and ADPKD].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29710438" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29710438" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29710438">
                    <p>Autosomal Dominant Polycistic Kidney Disease (ADPKD) is the most common inherited genetic disorder in the word, caused by mutations in PKD1 gene in 85% of cases and PKD 2 gene in the remaining 15%. Although diagnosis is usually based on ultrasound, MRI and CT scans, in some cases genetic testing is necessary, for example, in patients with atypical phenotype or with a negative family history, or in cases of donation from relatives. The presence of pseudogenes in PKD1, the size of the gene, the costs of the Sanger sequencing and genetic heterogeneity underlying kidney disease make genetic analysis particularly difficult to be performed. Next Generation Sequencing (NGS) represents the last frontier of innovation among diagnostic tools for molecular diagnosis of inherited cystic kidney disease thanks to the ability to analyze several genes at the same time. In this regard, we have developed a NGS platform, called Nephroplex, with the aim of identifying variations in 115 genes responsible for numerous kidney diseases, including cystic and polycystic disease, achieving, overall, a target region of 338.8 kbps. The technology used for the enrichment is HaloPlex system, based on the digestion of genomic DNA with restriction enzymes and the capture of the regions of interest with specific hybridization probes. With our platform, we have analyzed 9 patients with clinical diagnosis of ADPKD. We have obtained a depth coverage of 100x for 96.5% of the target, while the region not covered accounted for only 3% of the region of interest. In 6 patients, we found causative mutations in the genes PKD1 and PKD2, achieving a detection rate of 66%. In conclusion, the NephroPlex platform has proved to be an excellent device for molecular diagnosis of kidney disease and could clarify the mechanisms underlying genetic heterogeneity observed in kidney disease.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29710438">
                    <p>常染色体显性多发性肾病（ADPKD）是该词中最常见的遗传性遗传病，其由85％的病例中的PKD1基因突变和剩余的15％中的PKD 2基因突变引起。虽然诊断通常基于超声波，MRI和CT扫描，但在某些情况下，基因检测是必要的，例如，对于非典型表型或阴性家族史患者，或亲属捐献的情况。 PKD1中假基因的存在，基因的大小，Sanger测序的成本以及潜在肾病的遗传异质性使得遗传分析特别难以进行。由于能够同时分析几种基因，下一代测序（NGS）代表了遗传性囊性肾病分子诊断诊断工具的最新创新前沿。在这方面，我们开发了一个名为Nephroplex的NGS平台，目的是鉴定115个基因变异，负责包括囊性和多囊性疾病在内的众多肾脏疾病，总体上达到338.8 kbps的目标区域。用于富集的技术是HaloPlex系统，基于用限制性内切酶消化基因组DNA并用特异性杂交探针捕获感兴趣的区域。通过我们的平台，我们分析了9例临床诊断为ADPKD的患者。我们已经获得了96.5％的目标100倍的深度覆盖率，而未覆盖的区域仅占该区域的3％。在6例患者中，我们发现PKD1和PKD2基因的致病突变，检出率达66％。总之，NephroPlex平台已被证明是肾脏疾病分子诊断的优秀设备，可以阐明肾脏疾病中遗传异质性的机制。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=G Ital Nefrol">
                Journal: G Ital Nefrol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29710438" target="_blank">
                PMID: 29710438
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2015-Dec
            </a>

            <a class="btn btn-sm btn-danger" title="Restivo A, Di Fruscio G, Masella C, Rinaldi L, Capolongo G, Raiola I, Capasso G">
                Authors: Restivo A, Di Fru...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29710438">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>13. CRISPR-Cas9: A Precise Approach to Genome Engineering.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29714581" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29714581" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29714581">
                    <p>In the last few decades, genomic manipulation has made significant progress as a result of the development of recombinant DNA technologies; however, more often than not, these techniques have been costly and labor intensive. In contrast, recently developed next-generation sequencing (NGS) technologies have provided a cheaper, faster, and easier process to study genomics. In particular, an NGS technique emerged from bacterial CRISPR-associated protein-9 nuclease (Cas9) as a revolutionary method to modify, regulate, or mark specific genomic sequences on virtually any organism. A later adaptation of this bacterial defense mechanism that successfully and permanently edits dysfunctional genes and corrects missing proteins has resulted in a new era for disease genetic engineering. Clinical trials using this technique are already being performed, and the applicability of CRISPR-Cas9 techniques is actively being investigated using in vivo studies. However, the concept of genome correction poses great concerns from a regulatory perspective, especially in terms of security, so principles for the regulation of these methodologies are being established. We delved into CRISPR-Cas9 from its natural and ortholog origins to its engineered variants and behaviors to present its notable and diverse applications in the fields of biotechnology and human therapeutics.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29714581">
                    <p>在过去的几十年中，由于重组DNA技术的发展，基因组操作已经取得了重大进展;然而，更多的时候，这些技术成本高昂且劳动强度大。相比之下，最近开发的新一代测序（NGS）技术为研究基因组学提供了更便宜，更快，更简单的过程。具体而言，NGS技术从细菌CRISPR相关蛋白-9核酸酶（Cas9）出现，作为修改，调节或标记几乎任何生物体上的特定基因组序列的革命性方法。后来改编的这种细菌防御机制成功并永久地编辑了功能失调的基因并纠正了缺失的蛋白质，这为疾病基因工程带来了新的时代。使用这种技术的临床试验已经开始进行，CRISPR-Cas9技术的适用性正在积极使用体内研究进行研究。然而，从监管的角度来看，基因组矫正的概念引起了很大的关注，特别是在安全性方面，因此正在制定这些方法的监管原则。我们研究了CRISPR-Cas9从其天然和直系同源起源到其工程变体和行为，以展示其在生物技术和人类治疗学领域的显着和多样化应用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ther Innov Regul Sci">
                Journal: Ther Innov Regul Sci [IF=0.442]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29714581" target="_blank">
                PMID: 29714581
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jan-01
            </a>

            <a class="btn btn-sm btn-danger" title="Simón JE, Rodríguez ÁS, Santiago Vispo N">
                Authors: Simón JE, Rodrígu...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29714581">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>14. Astragalus affects fecal microbial composition of young hens as determined by 16S rRNA sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29713833" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29713833" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29713833">
                    <p>The gut microbiota play important roles in the degradation of chemical compounds of herbal medicines (HMs). However, little information regarding the interplay between HMs and the gut microbiota is available. Thus, the aim of this study was to investigate the composition of the fecal microbiota of young (age, 11 weeks) hens fed a conventional diet containing a crude Astragalus (0.5%) additive for 21 days (group A) vs. controls (group B) that were fed only conventional feed. The fecal contents of 14-week-old hens were collected for DNA extraction, and then the V3 and V4 hyper-variable regions of the 16S rRNA gene were amplified and analyzed using high-throughput sequencing technology. A distinctive difference in microbial diversity was observed between the two groups. The microbial composition of hens fed a diet supplemented with Astragalus was greater than that of the control group. At the genus level, Lactobacillus was more abundant in group A than group B (p &lt; 0.05). Importantly, this study is the first to report the observation of a novel Romboutsia sp. in the feces of hens. However, Romboutsia was less abundant in group A than group B (17.94 vs. 33.98%, respectively, p &lt; 0.05). The microbial community differed significantly between the two groups at the genus level, suggesting that Astragalus modulates the composition of the fecal microbiota. Based on these differences, these findings provide fresh insights into the application of Astragalus in the poultry industry, as well as a better understanding of the interplay between HMs and the gut microbiota.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29713833">
                    <p>肠道微生物群在草药（HM）化合物降解中起着重要作用。然而，关于HM和肠道微生物群之间相互作用的信息很少。因此，本研究的目的是调查饲喂包含粗黄芪（0.5％）添加剂的常规饮食21天（A组）与对照（组（对照组））的年轻（年龄11周）母鸡的粪便微生物群的组成B）只喂入常规饲料。收集14周龄母鸡的粪便内容物用于DNA提取，然后使用高通量测序技术扩增和分析16S rRNA基因的V3和V4高变区。两组之间观察到微生物多样性的显着差异。饲喂黄芪膳食的母鸡的微生物组成大于对照组。在属类水平上，A组乳酸菌含量高于B组（p &lt;0.05）。重要的是，这项研究是第一次报道观察一种新型Romboutsia sp。在母鸡的粪便中。然而，R组的Romboutsia较B组少（17.94比33.98％，P &lt;0.05）。在属水平上，两组之间的微生物群落显着不同，表明黄芪调节粪便微生物群的组成。基于这些差异，这些发现为黄芪在家禽业的应用提供了新的见解，并且更好地理解了HM和肠道微生物群之间的相互作用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=AMB Express">
                Journal: AMB Express [IF=1.825]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29713833" target="_blank">
                PMID: 29713833
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-30
            </a>

            <a class="btn btn-sm btn-danger" title="Qiao H, Zhang L, Shi H, Song Y, Bian C">
                Authors: Qiao H, Zhang L, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29713833">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>15. Mutational Profiling of Non-Small-Cell Lung Cancer Resistant to Osimertinib Using Next-Generation Sequencing in Chinese Patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29713646" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29713646" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29713646">
                    <p>To identify the somatic mutated genes for optimal targets of non-small-cell lung cancer after resistance to osimertinib treatment.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29713646">
                    <p>为了确定osimertinib治疗后对非小细胞肺癌最佳靶点的体细胞突变基因。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Biomed Res Int">
                Journal: Biomed Res Int [IF=2.476]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29713646" target="_blank">
                PMID: 29713646
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Nie K, Jiang H, Zhang C, Geng C, Xu X, Zhang L, Zhang H, Zhang Z, Lan K, Ji Y">
                Authors: Nie K, Jiang H, Z...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5866881" target="_blank">
                PMC5866881
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29713646">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>16. A systematic review on the genetics of male infertility in the era of next-generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29713536" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29713536" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29713536">
                    <p>To identify the role of next-generation sequencing (NGS) in male infertility, as advances in NGS technologies have contributed to the identification of novel genes responsible for a wide variety of human conditions and recently has been applied to male infertility, allowing new genetic factors to be discovered.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29713536">
                    <p>为了确定新一代测序（NGS）在男性不育症中的作用，随着NGS技术的进步，有助于鉴定引起多种人类疾病的新基因，并且最近已应用于男性不育症，从而允许新的遗传因素被发现。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Arab J Urol">
                Journal: Arab J Urol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29713536" target="_blank">
                PMID: 29713536
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar
            </a>

            <a class="btn btn-sm btn-danger" title="Robay A, Abbasi S, Akil A, El-Bardisi H, Arafa M, Crystal RG, Fakhro KA">
                Authors: Robay A, Abbasi S...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5922186" target="_blank">
                PMC5922186
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29713536">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>17. Epilepsy syndromes during the first year of life and the usefulness of an epilepsy gene panel.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29713355" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29713355" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29713355">
                    <p>Recent advances in genetics have determined that a number of epilepsy syndromes that occur in the first year of life are associated with genetic etiologies. These syndromes range from benign familial epilepsy syndromes to early-onset epileptic encephalopathies that lead to poor prognoses and severe psychomotor retardation. An early genetic diagnosis can save time and overall cost by reducing the amount of time and resources expended to reach a diagnosis. Furthermore, a genetic diagnosis can provide accurate prognostic information and, in certain cases, enable targeted therapy. Here, several early infantile epilepsy syndromes with strong genetic associations are briefly reviewed, and their genotype-phenotype correlations are summarized. Because the clinical presentations of these disorders frequently overlap and have heterogeneous genetic causes, next-generation sequencing (NGS)-based gene panel testing represents a more powerful diagnostic tool than single gene testing. As genetic information accumulates, genetic testing will likely play an increasingly important role in diagnosing pediatric epilepsy. However, the efforts of clinicians to classify phenotypes in nondiagnosed patients and improve their ability to interpret genetic variants remain important in the NGS era.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29713355">
                    <p>遗传学方面的最新进展已经确定，在生命的第一年发生的一些癫痫综合征与遗传病因有关。这些综合征的范围从良性家族性癫痫综合症到导致预后不良和严重精神运动性迟缓的早发性癫痫性脑病。早期基因诊断可以通过减少花费在诊断上的时间和资源来节省时间和整体成本。此外，基因诊断可以提供准确的预后信息，并且在某些情况下，可以实现靶向治疗。在这里，简要回顾几种具有强烈遗传关联的早期婴儿癫痫综合征，并总结它们的基因型 - 表型相关性。因为这些疾病的临床表现经常重叠并且具有不同的遗传原因，所以基于下一代测序（NGS）的基因组测试代表比单基因测试更强大的诊断工具。随着遗传信息的积累，基因检测可能在诊断儿童癫痫中发挥越来越重要的作用。然而，在NGS时代，临床医生为未诊断的患者分类表型并提高其解读遗传变异体能力的努力仍然很重要。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Korean J Pediatr">
                Journal: Korean J Pediatr [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29713355" target="_blank">
                PMID: 29713355
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Lee EH">
                Authors: Lee EH
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5924840" target="_blank">
                PMC5924840
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29713355">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>18. High Incidence of Severe Combined Immunodeficiency Disease in Saudi Arabia Detected Through Combined T Cell Receptor Excision Circle and Next Generation Sequencing of Newborn Dried Blood Spots.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29713328" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29713328" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29713328">
                    <p>Severe combined immunodeficiency disease (SCID) is the most severe form of primary immunodeficiency disorders (PID). T-cell receptor excision circle (TREC) copy number analysis is an efficient tool for population-based newborn screening (NBS) for SCID and other T cell lymphopenias. We sought to assess the incidence of SCID among Saudi newborn population and examine the feasibility of using targeted next generation sequencing PID gene panel (T-NGS PID) on DNA isolated from dried blood spots (DBSs) in routine NBS programs as a mutation screening tool for samples with low TREC count. Punches from 8,718 DBS collected on Guthrie cards were processed anonymously for the TREC assay. DNA was extracted from samples with confirmed low TREC count, then screened for 22q11.2 deletion syndrome by real-time polymerase chain reaction and for mutations in PID-related genes by T-NGS PID panel. Detected mutations were confirmed by Sanger sequencing. Sixteen out of the 8,718 samples were confirmed to have low TREC copy number. Autosomal recessive mutations in AK2, JAK3, and MTHFD1 were confirmed in three samples. Two additional samples were positive for the 22q11.2 deletion syndrome. In this study, we provide evidence for high incidence of SCID among Saudi population (1/2,906 live births) and demonstrate the feasibility of using T-NGS PID panel on DNA extracted from DBSs as a new reliable, rapid, and cost-effective mutation screening method for newborns with low TREC assay, which can be implemented as part of NBS programs for SCID.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29713328">
                    <p>严重联合免疫缺陷病（SCID）是原发性免疫缺陷病（PID）中最严重的一种。 T细胞受体切除环（TREC）拷贝数分析是用于SCID和其他T细胞淋巴瘤的基于群体的新生儿筛查（NBS）的有效工具。我们试图评估沙特新生儿人群中SCID的发病率，并检查在常规NBS项目中使用从干血斑（DBS）中分离的DNA的靶向下一代测序PID基因组（T-NGS PID）作为突变筛选工具的可行性对于低TREC计数的样品。来自8,718位在Guthrie卡片上收集的DBS的冲床进行匿名处理以进行TREC分析。从确认低TREC计数的样品中提取DNA，然后通过实时聚合酶链反应筛选22q11.2缺失综合征，并通过T-NGS PID组对PID相关基因的突变进行筛选。通过Sanger测序证实检测到的突变。在8,718个样本中，有16个被证实具有低TREC拷贝数。在三个样品中确认了AK2，JAK3和MTHFD1的常染色体隐性突变。另外两个样品对于22q11.2缺失综合征是阳性的。在这项研究中，我们为沙特人群（1 / 2,906名新生儿）中SCID的高发病率提供了证据，并证明了使用T-NGS PID小组从DBS提取的DNA作为一种新的可靠，快速且具有成本效益的突变的可行性用于低TREC测定的新生儿的筛查方法，其可以作为用于SCID的NBS程序的一部分来实施。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Immunol">
                Journal: Front Immunol [IF=6.429]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29713328" target="_blank">
                PMID: 29713328
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Al-Mousa H, Al-Dakheel G, Jabr A, Elbadaoui F, Abouelhoda M, Baig M, Monies D, Meyer B, Hawwari A, Dasouki M">
                Authors: Al-Mousa H, Al-Da...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5911483" target="_blank">
                PMC5911483
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29713328">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>19. Molecular dissection of a natural transposable element invasion.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29712752" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29712752" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29712752">
                    <p>The first tracking of the dynamics of a natural invasion by a transposable element (TE) provides unprecedented details on the establishment of host defense mechanisms against TEs. We captured a D. simulans population at an early stage of a P-element invasion and studied the spread of the TE in replicated experimentally evolving populations kept under hot and cold conditions. We analysed the factors controlling the invasion by NGS, RNA-FISH and gonadal dysgenesis assays. Under hot conditions, the P-element spread rapidly for 20 generations but no further spread was noted later on. This plateauing of the invasion was mediated by the rapid emergence of P-element specific piRNAs. Under cold conditions we observe a lower expression of the P-element and a slower emergence of the piRNA defense, resulting in a three times slower invasion which continued beyond 40 generations. We conclude that the environment is a major factor determining the evolution of TEs in their host.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29712752">
                    <p>对转座因子（TE）自然入侵动态的首次跟踪为建立针对TE的主机防御机制提供了前所未有的细节。我们在P元素入侵的早期阶段捕获了拟似物种群，并研究了在热和冷条件下复制的实验演变群体中TE的扩散。我们分析了控制NGS，RNA-FISH和性腺发育不全试验侵入的因素。在炎热的条件下，P元素迅速传播20代，但后来没有进一步传播。这种入侵的高原期是由P元件特异性piRNA的快速出现所介导的。在寒冷条件下，我们观察到P元素的表达较低，piRNA防御的出现较慢，导致三倍以上的入侵持续超过40代。我们得出结论认为，环境是决定东道国TE发展的主要因素。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genome Res.">
                Journal: Genome Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29712752" target="_blank">
                PMID: 29712752
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-30
            </a>

            <a class="btn btn-sm btn-danger" title="Kofler R, Senti KA, Nolte V, Tobler R, Schlötterer C">
                Authors: Kofler R, Senti K...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29712752">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>20. The Genomics of Prostate Cancer: A Historic Perspective.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29712681" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29712681" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29712681">
                    <p>The genomics of prostate cancer (PCA) has been difficult to study compared with some other cancer types for a multitude of reasons, despite significant efforts since the early 1980s. Overcoming some of these obstacles has paved the way for greater insight into the genomics of PCA. The advent of high-throughput technologies coming from the initial use of microsatellite and oligonucleotide probes gave rise to techniques like comparative genomic hybridization (CGH). With the introduction of massively parallel genomic sequencing, referred to as next-generation sequencing (NGS), a deeper understanding of cancer genomics in general has occurred. Along with these technologic advances, there has been the development of computational biology and statistical approaches to address novel large data sets characterized by single base resolution. This review will provide a historic perspective of PCA genomics with an emphasis on the cardinal mutations and alterations observed to be consistently seen in PCA for both hormone-naïve localized PCA and castration-resistant prostate cancer (CRPC). There will be a focus on alterations that have the greatest potential to play a role in disease progression and therapy management.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29712681">
                    <p>尽管自20世纪80年代初期以来付出了巨大的努力，但前列腺癌（PCA）的基因组学与众多其他癌症类型相比，由于多种原因而很难进行研究。克服这些障碍中的一些已经为更深入地了解PCA的基因组铺平了道路。来自最初使用微卫星和寡核苷酸探针的高通量技术的出现带来了比较基因组杂交（CGH）等技术。随着大规模并行基因组测序的推出，被称为下一代测序（NGS），通常会对癌症基因组学有更深入的了解。随着这些技术的进步，已经有计算生物学和统计方法的发展，以解决以单碱基分辨率为特征的新型大数据集。本综述将提供PCA基因组学的历史性视角，重点强调观察到的主要突变和改变在PCA中始终见于激素未初始局部PCA和去势抵抗性前列腺癌（CRPC）。将重点关注在疾病进展和治疗管理中发挥最大作用的改变。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cold Spring Harb Perspect Med">
                Journal: Cold Spring Harb Perspect Med [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29712681" target="_blank">
                PMID: 29712681
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-30
            </a>

            <a class="btn btn-sm btn-danger" title="Rubin MA, Demichelis F">
                Authors: Rubin MA, Demiche...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29712681">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>21. Genetic screening of male patients with primary hypogammaglobulinemia can guide diagnosis and clinical management.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29709555" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29709555" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29709555">
                    <p>The precise diagnosis of an immunodeficiency is sometimes difficult to assess, especially due to the large spectrum of phenotypic variation reported among patients. Common variable immunodeficiency disorders (CVID) do not have, for a large part, an identified genetic cause. The identification of a causal genetic mutation is important to confirm, or in some cases correct, the diagnosis. We screened &gt;150 male patients with hypogammaglobulinemia for mutations in three genes involved in pediatric X-linked primary immunoglobulin deficiency: CD40LG, SH2D1A and BTK. The SH2D1A screening allowed to reclassify two individuals with an initial CVID presentation as XLP after mutations identification. All these mutations were associated with a lack of protein expression. In addition, 4 patients with a primary diagnosis of CVID and one with a primary IgG subclass deficiency were requalified as XLA after identifying BTK mutations. Interestingly, two out of these 5 patients carried a damaging coding BTK mutation associated with a lower, but detectable, BTK expression in monocytes, suggesting that a dysfunctional protein explains the disease phenotype in these patients. In conclusion, our results advocate to include SH2D1A and BTK in newly developed targeted NGS genetic testing, to contribute to providing the most appropriate medical treatment and genetic counselling.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29709555">
                    <p>精确的免疫缺陷诊断有时难以评估，特别是由于患者报告的大量表型变异。常见的变异性免疫缺陷病（CVID）很大程度上没有确定的遗传原因。确定原因性遗传突变对于确认或在某些情况下纠正诊断很重要。我们筛查了&gt; 150名男性低丙种球蛋白血症患者，其中涉及儿科X连接初级免疫球蛋白缺陷症的三个基因突变：CD40LG，SH2D1A和BTK。在突变鉴定后，SH2D1A筛查允许将初始CVID呈现的两个人重新分类为XLP。所有这些突变都与缺乏蛋白质表达有关。此外，4例初步诊断CVID和1例患有初级IgG亚类缺陷的患者在鉴定出BTK突变后重新鉴定为XLA。有趣的是，这5例患者中有2例携带了一种破坏性编码BTK突变，与单核细胞中较低但可检测到的BTK表达相关，提示功能失调蛋白可解释这些患者的疾病表型。总之，我们的研究结果主张将SH2D1A和BTK纳入新开发的靶向NGS基因检测，以提供最合适的医疗和遗传咨询。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Hum. Immunol.">
                Journal: Hum. Immunol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29709555" target="_blank">
                PMID: 29709555
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-27
            </a>

            <a class="btn btn-sm btn-danger" title="Vince N, Mouillot G, Malphettes M, Limou S, Boutboul D, Guignet A, Bertrand V, Pellet P, Gourraud PA, Debré P, Oksenhendler E, Théodorou I, Fieschi C">
                Authors: Vince N, Mouillot...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29709555">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>22. ABCC6 mutations and early onset stroke: Two cases of a typical Pseudoxanthoma Elasticum.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29709427" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29709427" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29709427">
                    <p>Pseudoxanthoma elasticum (PXE) is a rare genetic disorder characterized by fragmented and mineralized elastic fibers in the mid-dermis of the skin, eye, digestive tract and cardiovascular system. Clinical presentation includes typical skin lesions, ocular angioid streaks, and multisystem vasculopathy. The age of onset varies considerably from infancy to old age, but the diagnosis is usually made in young adults due to frequent absence of pathognomonic skin and ocular manifestations in early childhood. We report two children with PXE presenting with isolated multisystem vasculopathy and early-onset stroke. In the first patient, diagnosis was delayed until typical dermatologic alterations appeared; in the second patient, next-generation sequencing (NGS) study led to early diagnosis and specific follow-up, underlying the crucial role in idiopathic pediatric stroke of early genetic testing using NGS-based panels.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29709427">
                    <p>松毛刺假单胞菌（Pseudoxanthoma elasticum，PXE）是一种罕见的遗传疾病，其特征是在皮肤，眼睛，消化道和心血管系统的中部真皮中具有碎裂和矿化的弹性纤维。临床表现包括典型的皮肤损伤，眼部血管样条纹和多系统血管病变。发病年龄从婴幼儿到老年有很大差异，但诊断通常是在年轻人身上进行的，因为儿童早期经常缺乏特征性的皮肤和眼部表现。我们报告2例PXE患儿出现孤立性多系统血管病变和早发性中风。在第一例患者中，诊断延迟至出现典型的皮肤病学改变;在第二例患者中，新一代测序（NGS）研究导致早期诊断和特定随访，这是早期基因检测中使用NGS的专家组在特发性小儿中风中发挥关键作用的基础。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Paediatr. Neurol.">
                Journal: Eur. J. Paediatr. Neurol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29709427" target="_blank">
                PMID: 29709427
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-12
            </a>

            <a class="btn btn-sm btn-danger" title="Bertamino M, Severino M, Grossi A, Rusmini M, Tortora D, Gandolfo C, Pederzoli S, Malattia C, Picco P, Striano P, Ceccherini I, Di Rocco M">
                Authors: Bertamino M, Seve...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29709427">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>23. Analysis of immune-related genes during Nora virus infection of Drosophila melanogaster using next generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29707694" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29707694" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29707694">
                    <p>Drosophila melanogaster depends upon the innate immune system to regulate and combat viral infection. This is a complex, yet widely conserved process that involves a number of immune pathways and gene interactions. In addition, expression of genes involved in immunity are differentially regulated as the organism ages. This is particularly true for viruses that demonstrate chronic infection, as is seen with Nora virus. Nora virus is a persistent non-pathogenic virus that replicates in a horizontal manner in D. melanogaster. The genes involved in the regulation of the immune response to Nora virus infection are largely unknown. In addition, the temporal response of immune response genes as a result of infection has not been examined. In this study, D. melanogaster either infected with Nora virus or left uninfected were aged for 2, 10, 20 and 30 days. The RNA from these samples was analyzed by next generation sequencing (NGS) and the resulting immune-related genes evaluated by utilizing both the PANTHER and DAVID databases, as well as comparison to lists of immune related genes and FlyBase. The data demonstrate that Nora virus infected D. melanogaster exhibit an increase in immune related gene expression over time. In addition, at day 30, the data demonstrate that a persistent immune response may occur leading to an upregulation of specific immune response genes. These results demonstrate the utility of NGS in determining the potential immune system genes involved in Nora virus replication, chronic infection and involvement of antiviral pathways.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29707694">
                    <p>果蝇取决于先天免疫系统来调控和对抗病毒感染。这是一个复杂而又广泛保守的过程，涉及许多免疫途径和基因相互作用。另外，随着机体老化，参与免疫的基因的表达被差异调节。对于表现出慢性感染的病毒尤其如此，正如诺拉病毒所见。诺拉病毒是持久的非致病性病毒，在黑腹果蝇中以水平方式复制。参与调节对诺拉病毒感染的免疫应答的基因在很大程度上是未知的。另外，由于感染引起的免疫应答基因的时间反应尚未被检测。在这项研究中，感染诺拉病毒或未感染的黑腹果蝇年龄分别为2,10,20和30天。通过下一代测序（NGS）分析这些样品的RNA，并通过利用PANTHER和DAVID数据库评估得到的免疫相关基因，以及与免疫相关基因和FlyBase列表进行比较。数据表明，感染诺拉病毒的黑腹果蝇随着时间的推移表现出免疫相关基因表达的增加。另外，在第30天，数据表明可能发生持续免疫应答，导致特异性免疫应答基因的上调。这些结果证明了NGS在确定涉及诺拉病毒复制，慢性感染和抗病毒途径参与的潜在免疫系统基因中的实用性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=AIMS Microbiol">
                Journal: AIMS Microbiol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29707694" target="_blank">
                PMID: 29707694
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Lopez W, Page AM, Carlson DJ, Ericson BL, Cserhati MF, Guda C, Carlson KA">
                Authors: Lopez W, Page AM,...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5915338" target="_blank">
                PMC5915338
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29707694">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>24. RAS/BRAF Circulating Tumor DNA Mutations as a Predictor of Response to First-Line Chemotherapy in Metastatic Colorectal Cancer Patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29707525" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29707525" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29707525">
                    <p>Since circulating tumor DNA (ctDNA) offers clear advantages as a minimally invasive method for tumor monitoring compared with tumor tissue, we aimed to evaluate genotyping ctDNA using a next-generation sequencing- (NGS-) based panel to identify the prognostic value of mutation status in metastatic colorectal cancer (mCRC) patients with primary tumor resected and with subsequent lines of treatment in this study.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29707525">
                    <p>由于与肿瘤组织相比，循环肿瘤DNA（ctDNA）提供了作为肿瘤监测的微创方法的明显优势，我们旨在使用基于下一代测序（NGS）的小组来评估基因分型ctDNA以鉴定突变状态的预后价值在本研究中切除原发性肿瘤和随后的治疗线的转移性结直肠癌（mCRC）患者中。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Can J Gastroenterol Hepatol">
                Journal: Can J Gastroenterol Hepatol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29707525" target="_blank">
                PMID: 29707525
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Yao J, Zang W, Ge Y, Weygant N, Yu P, Li L, Rao G, Jiang Z, Yan R, He L, Yu Y, Jin M, Cheng G, An G">
                Authors: Yao J, Zang W, Ge...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5863340" target="_blank">
                PMC5863340
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29707525">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>25. Identification of variants in pleiotropic genes causing &quot;isolated&quot; premature ovarian insufficiency: implications for medical practice.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29706645" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29706645" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29706645">
                    <p>Next-generation sequencing (NGS) is increasingly being used in a clinical setting for the molecular diagnosis of patients with heterogeneous disorders, such as premature ovarian insufficiency (POI). We performed NGS of ~1000 candidate genes in four unrelated patients with POI. We discovered the genetic cause of &quot;isolated&quot; POI in two cases, both of which had causative variants in surprising genes. In the first case, a homozygous nonsense variant in NBN was causative. Recessive function-altering NBN variants typically cause Nijmegen breakage syndrome characterized by microcephaly, cancer predisposition, and immunodeficiency, none of which are evident in the patient. At a cellular level, we found evidence of chromosomal instability. In the second case, compound heterozygous variants in EIF2B2 were causative. Recessive EIF2B2 function-altering variants usually cause leukoencephalopathy with episodic decline. Subsequent MRI revealed subclinical neurological abnormalities. These cases demonstrate that variants in NBN and EIF2B2, which usually cause severe syndromes, can cause apparently isolated POI, and that (1) NGS can precede clinical diagnosis and guide patient management, (2) NGS can redefine the phenotypic spectrum of syndromes, and (3) NGS may make unanticipated diagnoses that must be sensitively communicated to patients. Although there is rigorous debate about the handling of secondary/incidental findings using NGS, there is little discussion of the management of causative pleiotropic gene variants that have broader implications than that for which genetic studies were sought.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29706645">
                    <p>新一代测序（NGS）越来越多地用于临床环境，用于异质性疾病患者的分子诊断，如卵巢早衰（POI）。我们在四名无关POI患者中进行了〜1000个候选基因的NGS。我们在两个病例中发现了“孤立”POI的遗传原因，这两个病例都有惊人基因的致病变异。在第一种情况下，NBN中的纯合无义变体是致病的。隐性功能改变的NBN变体通常导致以小头畸形，癌症易感性和免疫缺陷为特征的奈梅亨断裂综合症，其中任何一种在患者中都不明显。在细胞水平上，我们发现染色体不稳定的证据。在第二种情况下，EIF2B2中的复合杂合变异体是致病的。隐性EIF2B2功能改变变体通常会导致白细胞脑病发作性下降。随后的MRI显示亚临床神经异常。这些病例表明，NBN和EIF2B2变异通常引起严重综合征，可导致明显孤立的POI，并且（1）NGS可以在临床诊断之前并指导患者管理;（2）NGS可以重新定义综合征的表型谱;以及（3）NGS可能会做出意想不到的诊断，必须将这些诊断明确地传达给患者。尽管关于使用NGS处理次级/偶然发现存在严格的争论，但关于致病性多效性基因变体的管理的讨论很少，其具有比寻求基因研究更广泛含义的管理。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Hum. Genet.">
                Journal: Eur. J. Hum. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29706645" target="_blank">
                PMID: 29706645
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-30
            </a>

            <a class="btn btn-sm btn-danger" title="Tucker EJ, Grover SR, Robevska G, van den Bergen J, Hanna C, Sinclair AH">
                Authors: Tucker EJ, Grover...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29706645">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>26. Tubulinopathies continued: refining the phenotypic spectrum associated with variants in TUBG1.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29706637" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29706637" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29706637">
                    <p>Tubulinopathies are a heterogeneous group of conditions with a wide spectrum of clinical severity resulting from variants in genes of the tubulin superfamily. Variants in TUBG1 have been described in three patients with posterior predominant pachygyria and microcephaly. We here report eight additional patients with four novel heterozygous variants in TUBG1 identified by next-generation sequencing (NGS) analysis. All had severe motor and cognitive impairment and all except one developed seizures in early life. The core imaging features included a pachygyric cortex with posterior to anterior gradient, enlarged lateral ventricles most pronounced over the posterior horns, and variable degrees of reduced white matter volume. Basal ganglia, corpus callosum, brainstem, and cerebellum were often normal, in contrast to patients with variants in other tubulin genes where these structures are frequently malformed. The imaging phenotype associated with variants in TUBG1 is therefore more in line with the phenotype resulting from variants in LIS1 (a.k.a. PAFAH1B1). This difference may, at least in part, be explained by gamma-tubulin&#39;s physiological function in microtubule nucleation, which differs from that of alpha and beta-tubulin.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29706637">
                    <p>微管蛋白病是由微管蛋白超家族的基因变体引起的广泛临床严重程度的异质性组群。 TUBG1变异已被描述为3例后部占主导地位的小鼠和小头畸形患者。我们在这里报告了另外8例患者，其中有4个新的杂合子变异体在TUBG1中通过下一代测序（NGS）分析鉴定。所有人都有严重的运动和认知功能障碍，除了早期发生癫痫发作之外的所有人都有。核心成像特征包括一个前后梯度变大的硬皮质皮质，后脑角最明显的侧脑室增大，白质体积减小程度不同。基底神经节，胼um体，脑干和小脑通常是正常的，与其他微管蛋白基因变异的患者相比，这些基因结构常常畸形。因此，与TUBG1中的变体相关的成像表型与LIS1（又名PAFAH1B1）中的变体所导致的表型更加一致。这种差异至少部分可以通过γ-微管蛋白在微管成核中的生理功能来解释，其不同于α和β-微管蛋白的生理功能。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Hum. Genet.">
                Journal: Eur. J. Hum. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29706637" target="_blank">
                PMID: 29706637
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-30
            </a>

            <a class="btn btn-sm btn-danger" title="Brock S, Stouffs K, Scalais E, D&#39;Hooghe M, Keymolen K, Guerrini R, Dobyns WB, Di Donato N, Jansen AC">
                Authors: Brock S, Stouffs ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29706637">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>27. Advances in the Molecular Pathophysiology, Genetics, and Treatment of Primary Ovarian Insufficiency.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29706485" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29706485" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29706485">
                    <p>Primary ovarian insufficiency (POI) affects ∼1% of women before 40 years of age. The recent leap in genetic knowledge obtained by next generation sequencing (NGS) together with animal models has further elucidated its molecular pathogenesis, identifying novel genes/pathways. Mutations of &gt;60 genes emphasize high genetic heterogeneity. Genome-wide association studies have revealed a shared genetic background between POI and reproductive aging. NGS will provide a genetic diagnosis leading to genetic/therapeutic counseling: first, defects in meiosis or DNA repair genes may predispose to tumors; and second, specific gene defects may predict the risk of rapid loss of a persistent ovarian reserve, an important determinant in fertility preservation. Indeed, a recent innovative treatment of POI by in vitro activation of dormant follicles proved to be successful.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29706485">
                    <p>原发性卵巢功能不全（POI）影响40岁前约1％的女性。新一代测序（NGS）与动物模型一起获得的基因知识最近的飞跃进一步阐明了其分子发病机制，鉴定了新的基因/途径。 &gt; 60个基因的突变强调高度遗传异质性。全基因组关联研究揭示了POI与生殖老化之间的共同遗传背景。 NGS将提供导致遗传/治疗咨询的基因诊断：首先，减数分裂或DNA修复基因缺陷可能易于发生肿瘤;其次，特定的基因缺陷可以预测持续性卵巢储备迅速丧失的风险，这是保留生育力的重要决定因素。事实上，最近通过体外激活休眠卵泡而创新治疗POI被证明是成功的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Trends Endocrinol. Metab.">
                Journal: Trends Endocrinol. Metab. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29706485" target="_blank">
                PMID: 29706485
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-26
            </a>

            <a class="btn btn-sm btn-danger" title="Huhtaniemi I, Hovatta O, La Marca A, Livera G, Monniaux D, Persani L, Heddar A, Jarzabek K, Laisk-Podar T, Salumets A, Tapanainen JS, Veitia RA, Visser JA, Wieacker P, Wolczynski S, Misrahi M">
                Authors: Huhtaniemi I, Hov...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29706485">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>28. Missense Variants in HIF1A and LACC1 Contribute to Leprosy Risk in Han Chinese.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29706348" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29706348" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29706348">
                    <p>Genome-wide association studies (GWASs) and genome-wide linkage studies (GWLSs) have identified numerous risk genes affecting the susceptibility to leprosy. However, most of the reported GWAS hits are noncoding variants and account for only part of the estimated heritability for this disease. In order to identify additional risk genes and map the potentially functional variants within the GWAS loci, we performed a three-stage study combining whole-exome sequencing (WES; discovery stage), targeted next-generation sequencing (NGS; screening stage), and refined validation of risk missense variants in 1,433 individuals with leprosy and 1,625 healthy control individuals from Yunnan Province, Southwest China. We identified and validated a rare damaging variant, rs142179458 (c.1045G&gt;A [p.Asp349Asn]) in HIF1A, as contributing to leprosy risk (p = 4.95 × 10-9, odds ratio [OR] = 2.266). We were able to show that affected individuals harboring the risk allele presented with multibacillary leprosy at an earlier age (p = 0.025). We also confirmed the association between missense variant rs3764147 (c.760A&gt;G [p.Ile254Val]) in the GWAS hit LACC1 (formerly C13orf31) and leprosy (p = 6.11 × 10-18, OR = 1.605). By using the population attributable fraction, we have shown that HIF1A and LACC1 are the major genes with missense variants contributing to leprosy risk in our study groups. Consistently, mRNA expression levels of both HIF1A and LACC1 were upregulated in the skin lesions of individuals with leprosy and in Mycobacterium leprae-stimulated cells, indicating an active role of HIF1A and LACC1 in leprosy pathogenesis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29706348">
                    <p>全基因组关联研究（GWASs）和全基因组连锁研究（GWLSs）已经确定了影响麻风易感性的众多风险基因。然而，大多数报道的GWAS命中是非编码变体，并且仅占该疾病估计遗传力的一部分。为了鉴定额外的风险基因并在GWAS基因座内定位潜在的功能性变异，我们进行了一个将全外显子组测序（WES;发现阶段），靶向下一代测序（NGS;筛选阶段）和对中国西南地区云南省14333例麻风病患者和1625例健康对照者的风险错义突变进行了精确验证。我们在HIF1A中鉴定并验证了一种罕见的破坏性变体rs142179458（c.1045G&gt; A [p.Asp349Asn]），其导致麻风风险（p = 4.95×10-9，比值比[OR] = 2.266）。我们能够证明携带风险等位基因的受影响个体在较早年龄时出现多发性麻风（p = 0.025）。我们还证实了GWAS命中LACC1（以前为C13orf31）和麻风（p = 6.11×10-18，OR = 1.605）中错义变异rs3764147（c.760A&gt; G [p.Ile254Val]）之间的关联。通过使用人口归因分数，我们发现HIF1A和LACC1是我们研究组中具有引起麻风风险的错义变体的主要基因。一致地，HIF1A和LACC1的mRNA表达水平在麻风病人和麻风分枝杆菌刺激的细胞的皮肤损伤中上调，表明HIF1A和LACC1在麻风病发病机制中的活性作用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Am. J. Hum. Genet.">
                Journal: Am. J. Hum. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29706348" target="_blank">
                PMID: 29706348
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Wang D, Fan Y, Malhi M, Bi R, Wu Y, Xu M, Yu XF, Long H, Li YY, Zhang DF, Yao YG">
                Authors: Wang D, Fan Y, Ma...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29706348">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>29. Performance validation of an amplicon-based targeted next-generation sequencing assay and mutation profiling of 648 Chinese colorectal cancer patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29705968" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29705968" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29705968">
                    <p>Next-generation sequencing (NGS) has become a promising approach for tumor somatic mutation detection. However, stringent validation is required for its application on clinical specimens, especially for low-quality formalin-fixed paraffin-embedded (FFPE) tissues. Here, we validated the performance of an amplicon-based targeted NGS assay, OncoAim™ DNA panel, on both commercial reference FFPE samples and clinical FFPE samples of Chinese colorectal cancer (CRC) patients. Then we profiled the mutation spectrum of 648 Chinese CRC patients in a multicenter study to explore its clinical utility. This NGS assay achieved 100% test specificity and 95-100% test sensitivity for variants with mutant allele frequency (MAF) ≥ 5% when median read depth ≥ 500×. The orthogonal methods including amplification refractory mutation system (ARMS)-PCR and Sanger sequencing validated that NGS generated three false negatives (FNs) but no false positives (FPs) among 516 clinical samples for KRAS aberration detection. Genomic profiling of Chinese CRC patients with this assay revealed that 63.3% of the tumors harbored clinically actionable alterations. Besides the commonly mutated genes including TP53 (52.82%), KRAS (46.68%), APC (24.09%), PIK3CA (18.94%), SMAD4 (9.47%), BRAF (6.15%), FBXW7 (5.32%), and NRAS (4.15%), other less frequently mutated genes were also identified. Statistically significant association of specific mutated genes with certain clinicopathological features was detected, e.g., both BRAF and PIK3CA were more prevalent in right-side CRC (p &lt; 0.001 and p = 0.002, respectively). We concluded this targeted NGS assay is qualified for clinical practice, and our findings could help the diagnosis and prognosis of Chinese CRC patients.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29705968">
                    <p>新一代测序（NGS）已经成为肿瘤体细胞突变检测的有希望的方法。然而，对临床标本的应用需要严格的验证，特别是对于低质量的福尔马林固定石蜡包埋（FFPE）组织。在这里，我们对中国结直肠癌（CRC）患者的商业参考FFPE样品和临床FFPE样品的基于扩增子的靶向NGS测定OncoAim™DNA板的性能进行了验证。然后，我们通过一项多中心研究，对648名华人CRC患者的突变谱进行了分析，以探索其临床应用。当中值读取深度≥500×时，该突变等位基因频率（MAF）≥5％的变体具有100％的测试特异性和95-100％的测试敏感性。包括扩增难治突变系统（ARMS）-PCR和Sanger测序在内的正交方法验证了NGS在516例临床样品中产生了三种假阴性（FNs）但没有假阳性（FPs）用于KRAS畸变检测。用该测定法对中国CRC患者的基因组谱进行分析表明，63.3％的肿瘤具有临床可行的改变。除52.52％，KRAS（46.68％），APC（24.09％），PIK3CA（18.94％），SMAD4（9.47％），BRAF（6.15％），FBXW7（5.32％）和NRAS （4.15％），还发现了其他较少突变的基因。检测到特定突变基因与某些临床病理学特征的统计学显着关联，例如BRAF和PIK3CA在右侧CRC中更普遍（分别为p &lt;0.001和p = 0.002）。我们得出的结论是，这种有针对性的NGS检测方法是符合临床实践的，我们的发现可以帮助中国CRC患者的诊断和预后。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Virchows Arch.">
                Journal: Virchows Arch. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29705968" target="_blank">
                PMID: 29705968
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-28
            </a>

            <a class="btn btn-sm btn-danger" title="Wang Y, Liu H, Hou Y, Zhou X, Liang L, Zhang Z, Shi H, Xu S, Hu P, Zheng Z, Liu R, Tang T, Ye F, Liang Z, Bu H">
                Authors: Wang Y, Liu H, Ho...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29705968">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>30. Transcriptional profiling of the CAM plant Agave salmiana reveals conservation of a genetic program for regeneration.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29705332" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29705332" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29705332">
                    <p>In plants, the best characterized plant regeneration process is de novo organogenesis. This type of regeneration is characterized by the formation of a multicellular structure called callus. Calli are induced via phytohormone treatment of plant sections. The callus formation in plants like Agave species with Crassulacean Acid Metabolism (CAM) is poorly studied. In this study, we induced callus formation from Agave salmiana leaves and describe cell arrangement in this tissue. Moreover, we determined and analyzed the transcriptional program of calli, as well as those of differentiated root and leaf tissues, by using RNA-seq. We were able to reconstruct 170,844 transcripts of which 40,644 have a full Open Reading Frame (ORF). The global profile obtained by Next Generation Sequencing (NGS) reveals that several callus-enriched protein coding transcripts are orthologs of previously reported factors highly expressed in Arabidopsis calli. At least 62 genes were differentially expressed in Agave calli, 50 of which were up-regulated. Several of these are actively involved in the perception of, and response to, auxin and cytokinin. Not only are these the first results for the A. salmiana callus, but they provide novel data from roots and leaves of this Agave species, one of the largest non-tree plants in nature.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29705332">
                    <p>在植物中，最好的特征化植物再生过程是从头发器官发生。这种再生的特点是形成称为愈伤组织的多细胞结构。通过植物切片的植物激素处理诱导愈伤组织。像龙舌兰酸代谢（CAM）的植物如龙舌兰的愈伤组织形成研究很少。在这项研究中，我们从龙舌兰salmiana叶诱导愈伤组织形成并描述了这个组织中的细胞排列。此外，我们通过使用RNA-seq来确定和分析愈伤组织的转录程序，以及分化的根和叶组织的转录程序。我们能够重建170,844个转录本，其中40,644个具有完整的开放阅读框（ORF）。通过下一代测序（NGS）获得的全局概况揭示了几种富含愈伤组织的蛋白质编码转录物是先前报道的在拟南芥愈伤组织中高表达的因子的直系同源物。在龙舌兰愈伤组织中至少有62个基因差异表达，其中50个基因上调。其中有几个积极参与对生长素和细胞分裂素的感知和反应。这不仅是沙门氏菌愈伤组织的第一个结果，但它们提供了这种龙舌兰物种的根和叶的新数据，它是自然界中最大的非树木植物之一。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Dev. Biol.">
                Journal: Dev. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29705332" target="_blank">
                PMID: 29705332
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-26
            </a>

            <a class="btn btn-sm btn-danger" title="Cervantes-Pérez SA, Espinal-Centeno A, Oropeza-Aburto A, Caballero-Pérez J, Falcon F, Aragón-Raygoza A, Sánchez-Segura L, Herrera-Estrella L, Cruz-Hernández A, Cruz-Ramírez A">
                Authors: Cervantes-Pérez S...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29705332">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>31. Is it research or is it clinical? Revisiting an old frontier through the lens of next-generation sequencing technologies.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29704685" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29704685" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29704685">
                    <p>As next-generation sequencing technologies (NGS) are increasingly used in the clinic, one issue often pointed out in the literature is the fact that their implementation &quot;blurs the line&quot; between research and healthcare. Indeed, NGS data obtained through research study may have clinical significance, and patients may consent that their data is shared in international databases used in research. This blurred line may increase the risk of therapeutic misconception, or that of over-reporting incidental findings. The law has been used to impose a distinction between the two contexts, but this distinction may not always be as clear in the practice of clinical genomics. To illustrate this, we reviewed the legal frameworks in France and Quebec on the matter, and asked the opinion of stakeholders who use NGS to help cancer and rare disease patients in practice. We found that while there are clear legal distinctions between research and clinical care, bridges between the two contexts exist, and the law focuses on providing appropriate protections to persons, whether they are patients or research participants. The technology users we interviewed expressed that their use of NGS was designed to help patients, but harbored elements pertaining to research as well as care. We hence saw that NGS technologies are often used with a double objective, both individual care and the creation of collective knowledge. Our results highlight the importance of moving towards research-based care, where clinical information can be progressively enriched with evolutive research results. We also found that there can be a misalignment between scientific experts&#39; views and legal norms of what constitutes research or care, which should be addressed. Our method allowed us to shed light on a grey zone at the edge between research and care, where the full benefits of NGS can be yielded. We believe that this and other evidence from the realities of clinical research practice can be used to design more stable and responsible personalized medicine policies.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29704685">
                    <p>随着下一代测序技术（NGS）越来越多地应用于临床，文献中经常指出的一个问题是，它们的实施“模糊了研究与医疗保健之间的界限”。事实上，通过研究获得的NGS数据可能具有临床意义，并且患者可能会同意他们的数据在研究中使用的国际数据库中共享。这种模糊的界限可能会增加治疗误解的风险，或者过度报告偶然发现的风险。法律已被用来对两种情况加以区分，但这种区别在临床基因组学实践中可能并不总是那么明显。为了说明这一点，我们回顾了法国和魁北克在这个问题上的法律框架，并询问了利用NGS在实践中帮助癌症和罕见疾病患者的利益相关者的意见。我们发现，尽管研究和临床护理之间存在明确的法律区别，但存在两种背景之间的桥梁，法律侧重于为患者提供适当的保护，无论他们是患者还是研究参与者。我们采访的技术用户表示，他们使用NGS的目的是为了帮助患者，但其中包含与研究和护理有关的元素。因此，我们看到NGS技术经常被用于双重目标，无论是个人护理还是创造集体知识。我们的研究结果强调了转向基于研究的护理的重要性，临床信息可以逐步丰富进化研究结果。我们还发现，科学专家的观点与构成研究或关怀的法律规范之间可能存在错位，应该加以解决。我们的方法使我们能够在研究和护理之间的灰色区域发现亮点，从而可以获得NGS的全部好处。我们相信，临床研究实践中的这些和其他证据可以用来设计更稳定和负责任的个性化药品政策。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur J Med Genet">
                Journal: Eur J Med Genet [IF=2.137]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29704685" target="_blank">
                PMID: 29704685
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-25
            </a>

            <a class="btn btn-sm btn-danger" title="Bertier G, Cambon-Thomsen A, Joly Y">
                Authors: Bertier G, Cambon...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29704685">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>32. Comprehensive viral enrichment enables sensitive respiratory virus genomic identification and analysis by next generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29703817" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29703817" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29703817">
                    <p>Next generation sequencing (NGS) technologies have revolutionized the genomics field and are becoming more commonplace for identification of human infectious diseases. However, due to the low abundance of viral nucleic acids (NAs) in relation to host, viral identification using direct NGS technologies often lacks sufficient sensitivity. Here, we describe an approach based on two complementary enrichment strategies that significantly improves the sensitivity of NGS-based virus identification. To start, we developed two sets of DNA probes to enrich virus NAs associated with respiratory diseases. The first set of probes spans the genomes, allowing for identification of known viruses and full genome sequencing, while the second set targets regions conserved among viral families or genera, providing the ability to detect both known and potentially novel members of those virus groups. Efficiency of enrichment was assessed by NGS testing reference virus and clinical samples with known infection. We show significant improvement in viral identification using enriched NGS compared to unenriched NGS. Without enrichment, we observed an average of 0.3% targeted viral reads per sample. However, after enrichment, 50%-99% of the reads per sample were the targeted viral reads for both the reference isolates and clinical specimens using both probe sets. Importantly, dramatic improvements on genome coverage were also observed following virus-specific probe enrichment. The methods described here provide improved sensitivity for virus identification by NGS, allowing for a more comprehensive analysis of disease etiology.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29703817">
                    <p>新一代测序（NGS）技术已经彻底革新了基因组学领域，并且正变得越来越普遍用于鉴定人类传染病。然而，由于与宿主相关的病毒核酸（NAs）的丰度低，使用直接NGS技术的病毒鉴定通常缺乏足够的灵敏度。在这里，我们描述了一种基于两种互补富集策略的方法，其显着提高了基于NGS的病毒鉴定的灵敏度。首先，我们开发了两套DNA探针来丰富与呼吸系统疾病有关的病毒NA。第一组探针跨越基因组，允许识别已知病毒和全基因组测序，而第二组探针针对病毒科或属中保守的区域，提供检测这些病毒组的已知和潜在新成员的能力。 NGS检测参考病毒和已知感染的临床样品评估富集效率。与未富集的NGS相比，我们使用富集NGS显示病毒鉴定显着改善。如果不进行富集，我们观察到每个样本平均有0.3％的靶向病毒读数。然而，富集后，每个样品的50％-99％读数是使用两种探针组的参考分离物和临床样品的靶向病毒读数。重要的是，在病毒特异性探针富集后也观察到基因组覆盖的显着改善。这里描述的方法提高了NGS对病毒鉴定的灵敏度，允许对疾病病因学进行更全面的分析。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genome Res.">
                Journal: Genome Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29703817" target="_blank">
                PMID: 29703817
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-27
            </a>

            <a class="btn btn-sm btn-danger" title="O&#39;Flaherty BM, Li Y, Tao Y, Paden CR, Queen K, Zhang J, Dinwiddie DL, Gross SM, Schroth GP, Tong S">
                Authors: O&#39;Flaherty BM, Li...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29703817">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>33. Immuno-Oncology Biomarkers for Gastric and Gastroesophageal Junction Adenocarcinoma: Why PD-L1 Testing May Not Be Enough.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29703766" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29703766" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29703766">
                    <p>The treatment of patients with advanced gastric and gastroesophageal junction (G/GEJ) adenocarcinomas has been transformed by the U.S. Food and Drug Administration approval of pembrolizumab. Tumor and adjacent tissue must stain positively for the programmed cell death ligand 1 (PD-L1) protein by companion diagnostic testing. However, some patients with PD-L1-negative tumors also benefit from pembrolizumab. High microsatellite instability (MSI) and tumor mutational load (TML) are positive predictive biomarkers for immune checkpoint inhibition (ICI) in other tumors. We sought to identify more patients who could benefit from ICI using alternative PD-L1 thresholds, MSI, and TML.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29703766">
                    <p>美国食品和药物管理局批准pembrolizumab改变了进展性胃和胃食管连接（G / GEJ）腺癌患者的治疗方法。通过陪伴诊断测试，肿瘤和邻近组织必须为程序性细胞死亡配体1（PD-L1）蛋白阳性染色。但是，一些PD-L1阴性肿瘤患者也可以从哌布利珠单抗中受益。高度微卫星不稳定性（MSI）和肿瘤突变负荷（TML）是其他肿瘤中免疫检查点抑制（ICI）的阳性预测生物标志物。我们试图找出更多可以使用替代PD-L1阈值，MSI和TML从ICI中受益的患者。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncologist">
                Journal: Oncologist [IF=4.962]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29703766" target="_blank">
                PMID: 29703766
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-27
            </a>

            <a class="btn btn-sm btn-danger" title="Weinberg BA, Xiu J, Hwang JJ, Shields AF, Salem ME, Marshall JL">
                Authors: Weinberg BA, Xiu ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29703766">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>34. SMAD4 and NF1 mutations as potential biomarkers for poor prognosis to cetuximab-based therapy in Chinese metastatic colorectal cancer patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29703253" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29703253" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29703253">
                    <p>Cetuximab, an anti-EGFR monoclonal antibody, is used in combination with chemotherapy in clinic to enhance the outcome in metastatic colorectal cancer (mCRC) patients with only ~ 20% response rate. To date only activating mutations in KRAS and NRAS have been identified as poor prognosis biomarkers in cetuximab-based treatment, which makes an urgent need for identification of novel prognosis biomarkers to precisely predict patients&#39; response in order to maximize the benefit.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29703253">
                    <p>西妥昔单抗是一种抗EGFR单克隆抗体，与临床化疗联合使用可提高转移性​​结直肠癌（mCRC）患者的预后，仅有〜20％的缓解率。迄今为止，仅在KRAS和NRAS中激活突变已被确定为基于西妥昔单抗的治疗中预后不良的生物标志物，这迫切需要鉴定新的预后生物标志物以精确预测患者的反应以最大化益处。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Cancer">
                Journal: BMC Cancer [IF=3.288]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29703253" target="_blank">
                PMID: 29703253
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-27
            </a>

            <a class="btn btn-sm btn-danger" title="Mei Z, Shao YW, Lin P, Cai X, Wang B, Ding Y, Ma X, Wu X, Xia Y, Zhu D, Shu Y, Fu Z, Gu Y">
                Authors: Mei Z, Shao YW, L...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5921972" target="_blank">
                PMC5921972
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29703253">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>35. Cross-site comparison of ribosomal depletion kits for Illumina RNAseq library construction.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29703133" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29703133" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29703133">
                    <p>Ribosomal RNA (rRNA) comprises at least 90% of total RNA extracted from mammalian tissue or cell line samples. Informative transcriptional profiling using massively parallel sequencing technologies requires either enrichment of mature poly-adenylated transcripts or targeted depletion of the rRNA fraction. The latter method is of particular interest because it is compatible with degraded samples such as those extracted from FFPE and also captures transcripts that are not poly-adenylated such as some non-coding RNAs. Here we provide a cross-site study that evaluates the performance of ribosomal RNA removal kits from Illumina, Takara/Clontech, Kapa Biosystems, Lexogen, New England Biolabs and Qiagen on intact and degraded RNA samples.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29703133">
                    <p>核糖体RNA（rRNA）包含从哺乳动物组织或细胞系样品中提取的总RNA的至少90％。使用大规模平行测序技术的信息性转录分析需要富集成熟的多腺苷酸化转录物或靶向耗尽rRNA部分。后一种方法特别令人感兴趣，因为它与降解样品如从FFPE中提取的样品兼容，并且还捕获非多腺苷酸化的转录物，例如一些非编码RNA。在这里，我们提供了一个跨站点研究，评估来自Illumina，Takara / Clontech，Kapa Biosystems，Lexogen，New England Biolabs和Qiagen的核糖体RNA去除试剂盒在完整和降解RNA样品上的性能。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Genomics">
                Journal: BMC Genomics [IF=3.729]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29703133" target="_blank">
                PMID: 29703133
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-15
            </a>

            <a class="btn btn-sm btn-danger" title="Herbert ZT, Kershner JP, Butty VL, Thimmapuram J, Choudhari S, Alekseyev YO, Fan J, Podnar JW, Wilcox E, Gipson J, Gillaspy A, Jepsen K, BonDurant SS, Morris K, Berkeley M, LeClerc A, Simpson SD, Sommerville G, Grimmett L, Adams M, Levine SS">
                Authors: Herbert ZT, Kersh...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29703133">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>36. Performance of next-generation sequencing on small tumor specimens and/or low tumor content samples using a commercially available platform.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29702695" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29702695" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29702695">
                    <p>Next generation sequencing tests (NGS) are usually performed on relatively small core biopsy or fine needle aspiration (FNA) samples. Data is limited on what amount of tumor by volume or minimum number of FNA passes are needed to yield sufficient material for running NGS. We sought to identify the amount of tumor for running the PCDx NGS platform.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29702695">
                    <p>新一代测序测试（NGS）通常在相对较小的核心活组织检查或细针抽吸（FNA）样本上进行。数据受限于肿瘤体积的数量或最小数量的FNA通道是否需要足够的材料来运行NGS。我们试图确定运行PCDx NGS平台的肿瘤数量。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29702695" target="_blank">
                PMID: 29702695
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Morris S, Subramanian J, Gel E, Runger G, Thompson E, Mallery D, Weiss G">
                Authors: Morris S, Subrama...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29702695">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>37. A remark on copy number variation detection methods.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29702671" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29702671" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29702671">
                    <p>Copy number variations (CNVs) are gain and loss of DNA sequence of a genome. High throughput platforms such as microarrays and next generation sequencing technologies (NGS) have been applied for genome wide copy number losses. Although progress has been made in both approaches, the accuracy and consistency of CNV calling from the two platforms remain in dispute. In this study, we perform a deep analysis on copy number losses on 254 human DNA samples, which have both SNP microarray data and NGS data publicly available from Hapmap Project and 1000 Genomes Project respectively. We show that the copy number losses reported from Hapmap Project and 1000 Genome Project only have &lt; 30% overlap, while these reports are required to have cross-platform (e.g. PCR, microarray and high-throughput sequencing) experimental supporting by their corresponding projects, even though state-of-art calling methods were employed. On the other hand, copy number losses are found directly from HapMap microarray data by an accurate algorithm, i.e. CNVhac, almost all of which have lower read mapping depth in NGS data; furthermore, 88% of which can be supported by the sequences with breakpoint in NGS data. Our results suggest the ability of microarray calling CNVs and the possible introduction of false negatives from the unessential requirement of the additional cross-platform supporting. The inconsistency of CNV reports from Hapmap Project and 1000 Genomes Project might result from the inadequate information containing in microarray data, the inconsistent detection criteria, or the filtration effect of cross-platform supporting. The statistical test on CNVs called from CNVhac show that the microarray data can offer reliable CNV reports, and majority of CNV candidates can be confirmed by raw sequences. Therefore, the CNV candidates given by a good caller could be highly reliable without cross-platform supporting, so additional experimental information should be applied in need instead of necessarily.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29702671">
                    <p>拷贝数变异（CNV）是基因组DNA序列的获得和丧失。高通量平台如微阵列和下一代测序技术（NGS）已应用于全基因组拷贝数丢失。虽然两种方法都取得了进展，但两个平台的CNV呼叫的准确性和一致性仍然存在争议。在这项研究中，我们对254份人类DNA样本的拷贝数损失进行了深入分析，这些样本既包含SNP芯片数据，也包含分别从Hapmap Project和1000 Genomes Project公布的NGS数据。我们发现从Hapmap Project和1000 Genome Project报道的拷贝数损失仅有&lt;30％的重叠，而这些报告需要通过其相应的项目进行跨平台（如PCR，微阵列和高通量测序）实验支持，尽管采用了最先进的呼叫方法。另一方面，通过精确算法（即CNVhac）直接从HapMap微阵列数据中找到拷贝数量损失，其几乎全部在NGS数据中具有较低的读取映射深度;此外，其中88％可以通过NGS数据中具有断点的序列来支持。我们的研究结果表明微阵列调用CNV的能力以及从额外跨平台支持的基本要求中引入假阴性的可能性。 Hapmap项目和1000基因组项目的CNV报告不一致可能源于微阵列数据中包含的信息不足，检测标准不一致或跨平台支持的过滤效应。从CNVhac调用的CNV的统计测试显示，微阵列数据可以提供可靠的CNV报告，并且大部分CNV候选者可以通过原始序列确认。因此，如果没有跨平台的支持，优秀呼叫者提供的CNV候选人可能具有高度的可靠性，因此需要应用更多的实验信息而不是必要的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29702671" target="_blank">
                PMID: 29702671
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Li S, Dou X, Gao R, Ge X, Qian M, Wan L">
                Authors: Li S, Dou X, Gao ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29702671">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>38. Comprehensive Molecular Characterization of Young Chinese Patients with Lung Adenocarcinoma Identified a Distinctive Genetic Profile.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29700208" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29700208" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29700208">
                    <p>Occurrence at a younger age has been demonstrated to be associated with a distinct biology in non-small cell lung cancer. However, genomics and clinical characteristics among younger patients with lung adenocarcinoma remain to be determined. Here we studied the potentially targetable genetic alterations by next-generation sequencing (NGS) assay in young Chinese patients with lung adenocarcinoma.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29700208">
                    <p>已经证实，年轻时发生的事件与非小细胞肺癌中的独特生物学有关。然而，年轻肺腺癌患者的基因组学和临床特征仍有待确定。在这里，我们研究了新一代测序法（NGS）在年轻中国肺腺癌患者中的潜在靶向基因改变。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncologist">
                Journal: Oncologist [IF=4.962]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29700208" target="_blank">
                PMID: 29700208
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-26
            </a>

            <a class="btn btn-sm btn-danger" title="Hou H, Zhu H, Zhao H, Yan W, Wang Y, Jiang M, Liu B, Liu D, Zhou N, Zhang C, Li P, Chang L, Guan Y, Wang Z, Zhang X, Li Z, Fang B, Zhang X">
                Authors: Hou H, Zhu H, Zha...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29700208">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>39. Ioncopy: an R Shiny app to call copy number alterations in targeted NGS data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29699497" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29699497" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29699497">
                    <p>Somatic copy number alterations (CNAs) contribute to the clinically targetable aberrations in the tumor genome. For both routine diagnostics and biomarkers research, CNA analysis in a single assay together with somatic mutations is highly desirable.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29699497">
                    <p>体细胞拷贝数改变（CNA）有助于肿瘤基因组中的临床可靶向异常。对于常规诊断和生物标志物研究，单次测定中的CNA分析以及体细胞突变是非常需要的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Bioinformatics">
                Journal: BMC Bioinformatics [IF=2.448]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29699497" target="_blank">
                PMID: 29699497
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-24
            </a>

            <a class="btn btn-sm btn-danger" title="Budczies J, Pfarr N, Romanovsky E, Endris V, Stenzinger A, Denkert C">
                Authors: Budczies J, Pfarr...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5921540" target="_blank">
                PMC5921540
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29699497">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>40. A meta-barcoding approach to assess and compare the storage temperature-dependent bacterial diversity of gilt-head sea bream (Sparus aurata) originating from fish farms from two geographically distinct areas of Greece.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29698856" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29698856" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29698856">
                    <p>Bacterial diversity of whole gilt-head sea bream (Sparus aurata L. 1758) originating from Ionian and Aegean Sea aquaculture farms and stored at 0 (ice), 4 and 8 °C was determined by 16S rRNA gene amplicon sequencing method using the Illumina&#39;s MiSeq platform. The composition of Aerobic Plate Counts (APC) was also monitored by 16S rRNA gene sequencing. The rejection time point of sea bream from either area, as determined by sensory evaluation, was about 14, 6 and 3 days at 0, 4 and 8 °C, respectively. APC was approximately 4.5 log cfu/g at day 0 and ranged from 7.5 to 8.5 log cfu/g at sensory rejection. Culture-depended analysis showed that Pseudomonas and Shewanella were the most abundant microorganisms grown on plates for both seas. Moreover, culture-independent analysis of DNA extracted directly from fish flesh showed that sea bream originating from different geographical areas exhibited different bacterial diversity. Pseudomonas and Psychrobacter were the dominant microorganisms of chill-stored fish from Ionian (apart from 8 °C, where Carnobacterium dominated) and Aegean Sea, respectively. In addition, small changes of storage temperature greatly affected bacterial microbiota of stored fish. Various bacterial species, not detected by conventional microbiological methods, were also revealed through 16S amplicon sequencing. In conclusion, the use of NGS approach is a promising methodology for assessing bacterial diversity of sea bream originating from different geographical areas and stored at various temperatures.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29698856">
                    <p>使用Illumina的MiSeq通过16S rRNA基因扩增子测序方法测定来自爱奥尼亚海和爱琴海水产养殖场且储存在0（冰），4和8℃下的整头金头鲷（Sparus aurata L. 1758）的细菌多样性平台。通过16S rRNA基因测序也监测好氧板计数（APC）的组成。通过感官评估确定的来自任一区域的鲷鱼的排斥时间点分别在0℃，4℃和8℃约14,6和3d天。 APC在第0天约为4.5logcfu / g，在感官排斥时为7.5至8.5logcfu / g。依赖于培养的分析表明，假单胞菌和希瓦氏菌是在海洋上生长在板上的最丰富的微生物。此外，直接从鱼肉中提取的DNA的不依赖于培养物的分析表明，源自不同地理区域的鲷鱼表现出不同的细菌多样性。假单胞菌和嗜碱杆菌是爱奥尼亚冷藏鱼的主要微生物（除8℃以外，肉鸡占优势）和爱琴海。另外，存储温度的小变化极大地影响了存储的鱼的细菌微生物群。通过16S扩增子测序也揭示了通过常规微生物学方法未检测到的各种细菌种类。总之，使用NGS方法是评估来自不同地理区域并储存在不同温度的海鲷的细菌多样性的有前景的方法。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int. J. Food Microbiol.">
                Journal: Int. J. Food Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29698856" target="_blank">
                PMID: 29698856
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-19
            </a>

            <a class="btn btn-sm btn-danger" title="Parlapani FF, Michailidou S, Pasentsis K, Argiriou A, Krey G, Boziaris IS">
                Authors: Parlapani FF, Mic...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29698856">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>41. System for Informatics in the Molecular Pathology Laboratory (SIMPL): An Open-Source End-to-End Solution for Next-Generation Sequencing Clinical Data Management.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29698836" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29698836" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29698836">
                    <p>Next-generation sequencing (NGS) diagnostic assays are increasingly becoming the standard of care in oncology practice. As the scale of an NGS laboratory grows, management of these assays requires organizing large amounts of information, including patient data, laboratory processes, genomic data, as well as variant interpretation and reporting. Although several Laboratory Information Systems (LIS) and/or Laboratory Information Management Systems (LIMS) are commercially available, they may not meet all of the needs of a given laboratory, in addition to being frequently cost prohibitive. Herein, we present System for Informatics in the Molecular Pathology Laboratory (SIMPL), a free and open-source LIS/LIMS system for academic and nonprofit molecular pathology NGS laboratories, developed at the Genomic and Molecular Pathology division at the University of Chicago Medicine (UCM-GMP). SIMPL was designed as a modular end-to-end information system to handle all stages of the NGS laboratory workload from test order to reporting. We describe its features, clinical validation of the system at UCM-GMP, its installation and testing within a different academic center laboratory (University of Colorado), and propose a platform for future community co-development and interlaboratory data sharing.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29698836">
                    <p>新一代测序（NGS）诊断分析正日益成为肿瘤学实践中的护理标准。随着NGS实验室规模的扩大，这些检测的管理需要组织大量的信息，包括患者数据，实验室流程，基因组数据以及不同的解释和报告。尽管多个实验室信息系统（LIS）和/或实验室信息管理系统（LIMS）可以在市场上买到，但它们可能无法满足给定实验室的所有需求，除此之外，这些系统往往成本过高。在此，我们介绍了分子病理学实验室（SIMPL）中的信息系统，SIMPL是芝加哥大学医学院基因组和分子病理学部门开发的一个免费和开源的LIS / LIMS系统，用于学术和非营利性分子病理学NGS实验室。 UCM-GMP）。 SIMPL被设计为一个模块化的端到端信息系统，用于处理从测试订单到报告的NGS实验室工作负载的所有阶段。我们描述其功能，UCM-GMP系统的临床验证，在不同学术中心实验室（科罗拉多大学）内的安装和测试，并为未来社区共同开发和实验室间数据共享提供平台。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Mol Diagn">
                Journal: J Mol Diagn [IF=4.526]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29698836" target="_blank">
                PMID: 29698836
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-23
            </a>

            <a class="btn btn-sm btn-danger" title="Kang W, Kadri S, Puranik R, Wurst MN, Patil SA, Mujacic I, Benhamed S, Niu N, Zhen CJ, Ameti B, Long BC, Galbo F, Montes D, Iracheta C, Gamboa VL, Lopez D, Yourshaw M, Lawrence CA, Aisner DL, Fitzpatrick C, McNerney ME, Wang YL, Andrade J, Volchenboum SL, Furtado LV, Ritterhouse LL, Segal JP">
                Authors: Kang W, Kadri S, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29698836">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>42. Exact association test for small size sequencing data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29697368" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29697368" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29697368">
                    <p>Recent statistical methods for next generation sequencing (NGS) data have been successfully applied to identifying rare genetic variants associated with certain diseases. However, most commonly used methods (e.g., burden tests and variance-component tests) rely on large sample sizes. Notwithstanding, due to its-still high cost, NGS data is generally restricted to small sample sizes, that cannot be analyzed by most existing methods.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29697368">
                    <p>近期的下一代测序（NGS）数据统计方法已成功应用于鉴定与某些疾病相关的罕见遗传变异。然而，最常用的方法（例如负荷测试和方差分量测试）依赖于大样本量。尽管如此，由于其成本仍然很高，NGS数据通常仅限于小样本量，大多数现有方法无法对其进行分析。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Med Genomics">
                Journal: BMC Med Genomics [IF=2.848]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29697368" target="_blank">
                PMID: 29697368
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Lee J, Lee S, Jang JY, Park T">
                Authors: Lee J, Lee S, Jan...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918458" target="_blank">
                PMC5918458
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29697368">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>43. An integrated clinical and genomic information system for cancer precision medicine.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29697362" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29697362" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29697362">
                    <p>Increasing affordability of next-generation sequencing (NGS) has created an opportunity for realizing genomically-informed personalized cancer therapy as a path to precision oncology. However, the complex nature of genomic information presents a huge challenge for clinicians in interpreting the patient&#39;s genomic alterations and selecting the optimum approved or investigational therapy. An elaborate and practical information system is urgently needed to support clinical decision as well as to test clinical hypotheses quickly.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29697362">
                    <p>越来越多的下一代测序（NGS）的可负担性已经为实现基因组化的个性化癌症治疗作为精准肿瘤学的途径创造了机会。然而，基因组信息的复杂性质给临床医生解释患者基因组改变和选择最佳批准或研究治疗带来巨大挑战。迫切需要精心设计的实用信息系统来支持临床决策并快速测试临床假设。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Med Genomics">
                Journal: BMC Med Genomics [IF=2.848]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29697362" target="_blank">
                PMID: 29697362
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Jang Y, Choi T, Kim J, Park J, Seo J, Kim S, Kwon Y, Lee S, Lee S">
                Authors: Jang Y, Choi T, K...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5918454" target="_blank">
                PMC5918454
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29697362">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>44. NGS testing for cardiomyopathy: Utility of adding RASopathy-associated genes.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29696744" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29696744" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29696744">
                    <p>RASopathies include a group of syndromes caused by pathogenic germline variants in RAS-MAPK pathway genes and typically present with facial dysmorphology, cardiovascular disease, and musculoskeletal anomalies. Recently, variants in RASopathy-associated genes have been reported in individuals with apparently non-syndromic cardiomyopathy, suggesting that subtle features may be overlooked. To determine the utility and burden of adding RASopathy-associated genes to cardiomyopathy panels, we tested 11 RASopathy-associated genes by next generation sequencing (NGS), including NGS-based copy number variant assessment, in 1,111 individuals referred for genetic testing for hypertrophic cardiomyopathy (HCM) or dilated cardiomyopathy (DCM). Disease-causing variants were identified in 0.6% (4/692) of individuals with HCM, including three missense variants in the PTPN11, SOS1, and BRAF genes. Overall, 36 variants of uncertain significance (VUSs) were identified, averaging ∼3VUSs/100 cases. This study demonstrates that adding a subset of the RASopathy-associated genes to cardiomyopathy panels will increase clinical diagnoses without significantly increasing the number of VUSs/case. This article is protected by copyright. All rights reserved.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29696744">
                    <p>RAS病变包括由RAS-MAPK途径基因中致病性种系变异体引起的一组综合征，并且典型地伴有面部畸形，心血管疾病和肌肉骨骼异常。最近，RAS病相关基因的变异已在有明显非综合征性心肌病的个体中报道，表明可能忽略了细微的特征。为了确定将RAS病相关基因添加到心肌病组中的效用和负担，我们通过下一代测序（NGS）测试了11个RAS病相关基因，包括基于NGS的拷贝数变异评估，共纳入1,111位参与肥厚型心肌病基因检测的个体（HCM）或扩张型心肌病（DCM）。 0.6％（4/692）的HCM患者发现了致病变异株，包括PTPN11，SOS1和BRAF基因中的3种错义突变。总体而言，确定了36个不确定意义变体（VUSs），平均值为〜3VUSs / 100例。这项研究表明，将一部分RAS病相关基因添加到心肌病组将增加临床诊断，但不会显着增加VUS /病例的数量。本文受版权保护。版权所有。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Hum. Mutat.">
                Journal: Hum. Mutat. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29696744" target="_blank">
                PMID: 29696744
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-25
            </a>

            <a class="btn btn-sm btn-danger" title="Ceyhan-Birsoy O, Miatkowski MM, Hynes E, Funke BH, Mason-Suares H">
                Authors: Ceyhan-Birsoy O, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29696744">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>45. Searching for new biomarkers in ovarian cancer patients: Rationale and design of a retrospective study under the Mermaid III project.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29696206" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29696206" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29696206">
                    <p>Ovarian cancer is a silent killer and, due to late diagnosis, the primary cause of death amongst gynecological cancers, killing approximately 376 women annually in Denmark. The discovery of a specific and sensitive biomarker for ovarian cancer could improve early diagnosis, but also treatment, by predicting which patients will benefit from specific treatment strategies. The Mermaid III project is consisting of 3 parts including &quot;Early detection, screening and long-term survival,&quot; &quot;Biomarkers and/or prognostic markers&quot; and &quot;The infection theory.&quot; The present paper gives an overview of the part regarding biomarkers and/or prognostic markers, with a focus on rationale and design. The study described has 3 major branches: microRNAs, epigenetics and Next Generation Sequencing. Tissue and blood from ovarian cancer patients, already enrolled in the prospective ongoing pelvic mass cohort, will be examined. Relevant microRNAs and DNA methylation patterns will be investigated using array technology. Patient exomes will be fully sequenced, and identified genetic variations will be validated with Next Generation Sequencing. In all cases, data will be correlated with clinical information on the patient, in order to identify possible biomarkers. A thorough investigation of biomarkers in ovarian cancer, including large numbers of different markers, has never been done before. Besides from improving diagnosis and treatment, other outcomes could be markers for screening, knowledge of the molecular aspects of cancer and the discovery of new drugs. Moreover, biomarkers are a prerequisite for the development of precision medicine. This study will attack the ovarian cancer problem from several angles, thereby increasing the chance of successfully contributing to saving lives.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29696206">
                    <p>卵巢癌是一种无声的杀手，由于延误诊断，是妇科癌症死亡的主要原因，每年在丹麦造成约376名妇女死亡。通过预测哪些患者将从特定治疗策略中受益，卵巢癌的特异性和敏感性生物标志物的发现可以改善早期诊断，还可以改善治疗。美人鱼III项目由3部分组成，包括“早期发现，筛选和长期存活”，“生物标记和/或预后标记”和“感染理论”。本文件概述了关于生物标记和/或预后标记的部分，重点在于理论和设计。这项研究描述了3个主要分支：microRNAs，表观遗传学和下一代测序。将检查已经登记在未来的正在进行的盆腔肿块群中的卵巢癌患者的组织和血液。使用阵列技术将研究相关的microRNA和DNA甲基化模式。患者外显子组将被完全测序，并且鉴定出的遗传变异将通过下一代测序进行验证。在所有情况下，数据将与患者的临床信息相关联，以识别可能的生物标志物。卵巢癌中的生物标志物的彻底调查，包括大量不同的标志物，以前从未做过。除了改善诊断和治疗外，其他结果可能成为筛查的标记，癌症分子方面的知识和新药的发现。而且，生物标志物是精密医学发展的先决条件。这项研究将从多个角度攻击卵巢癌问题，从而增加挽救生命的成功机会。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Contemp Clin Trials Commun">
                Journal: Contemp Clin Trials Commun [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29696206" target="_blank">
                PMID: 29696206
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2017-Dec
            </a>

            <a class="btn btn-sm btn-danger" title="Hentze JL, Høgdall C, Kjær SK, Blaakær J, Høgdall E">
                Authors: Hentze JL, Høgdal...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5898550" target="_blank">
                PMC5898550
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29696206">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>46. Deletion at the 5&#39;-end of Estonian ASFV strains associated with an attenuated phenotype.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29695831" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29695831" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29695831">
                    <p>African swine fever (ASF) was introduced into the Eastern European Union in 2014 and led to considerable mortality among wild boar. In contrast, unexpected high antibody prevalence was reported in hunted wild boar in north-eastern Estonia. One of the causative virus strains was recently characterized. While it still showed rather high virulence in the majority of experimentally infected animals, one animal survived and recovered completely. Here, we report on the follow-up characterization of the isolate obtained from the survivor in the acute phase of infection. As a first step, three in vivo experiments were performed with different types of pigs: twelve minipigs (trial A), five domestic pigs (trial B), and five wild boar (trial C) were inoculated. 75% of the minipigs and all domestic pigs recovered after an acute course of disease. However, all wild boar succumbed to infection within 17 days. Representative samples were sequenced using NGS-technologies, and whole-genomes were compared to ASFV &quot;Georgia 2007/1&quot;. The alignments indicated a deletion of 14560 base pairs at the 5&#39; end, and genome reorganization by duplication. The characteristic deletion was confirmed in all trial samples and local field samples. In conclusion, an ASFV variant was found in Estonia that showed reduced virulence.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29695831">
                    <p>非洲猪瘟（ASF）于2014年引入东欧联盟，导致野猪死亡率相当高。相比之下，爱沙尼亚东北部发现野猪的意外高流行率。最近表征了致病性病毒株之一。虽然它在大多数实验感染的动物中仍然表现出相当高的毒力，但一只动物存活并完全恢复。在这里，我们报道了在感染急性期从幸存者获得的分离株的后续表征。作为第一步，用不同类型的猪进行三项体内实验：接种12只小型猪（试验A），5只家猪（试验B）和5只野猪（试验C）。 75％的小型猪和所有家猪在急性病程后康复。然而，所有野猪在17天内都感染了感染。使用NGS技术对代表性样品进行测序，并将全基因组与ASFV“Georgia 2007/1”进行比较。比对表明在5&#39;末端删除了14560个碱基对，并通过重复进行基因组重组。所有试验样品和当地现场样品都证实了特征性缺失。总之，在爱沙尼亚发现了一种ASFV变体，其毒力降低。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29695831" target="_blank">
                PMID: 29695831
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-25
            </a>

            <a class="btn btn-sm btn-danger" title="Zani L, Forth JH, Forth L, Nurmoja I, Leidenberger S, Henke J, Carlson J, Breidenstein C, Viltrop A, Höper D, Sauter-Louis C, Beer M, Blome S">
                Authors: Zani L, Forth JH,...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5916933" target="_blank">
                PMC5916933
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29695831">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>47. Molecular Profile of Advanced Thyroid Carcinomas by Next-Generation Sequencing: Characterizing tumors beyond diagnosis for targeted therapy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29695638" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29695638" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29695638">
                    <p>Next generation sequencing (NGS) for molecular diagnostics allows simultaneous testing of activating oncogenes and tumor suppressor mutations in multiple signal pathways. Extended mutational profiling of advanced thyroid cancers may enhance considerations for targeted therapies. We analyzed clinically derived molecular profiling of 216 patients with advanced thyroid carcinoma using NGS (Ion Torrent Personal Genome Machine) from 4/2012-2/2014. We examined substitutions and small indels in 46/50 cancer-related genes using Ampliseq Cancer Hotspot panel in respect to tumor diagnosis and clinical correlations. Mutations were common in advanced thyroid carcinomas 154 (71%) predominately in targetable MAPK pathway (146/216, 68%), and several PI3K/AKT pathway (8, 4%; 6 as co-mutations). BRAF V600E mutation associated with papillary (94/139, 68%), poorly differentiated (4/39, 10%) and anaplastic (3/12, 25%) carcinomas. NRAS mutations occurred in follicular (5/12, 40%) and poorly differentiated thyroid carcinoma (12/39, 31%). Tumor suppressor mutations (16, 7%) occurred predominantly in TP53 in Hurthle cell (2/5, 40%, the only mutation), in anaplastic (3/12, 25%), and poorly differentiated thyroid carcinoma (4/39, 10%) some co-mutations and in papillary thyroid carcinoma (5/139, 4%) always a co-mutation. Kaplan-Meier analysis showed patients with poorly differentiated thyroid carcinoma containing activating mutations who received targeted therapeutics showed improved survival compared to similarly treated patients without mutations in targetable pathways (p=0.02). In conclusion, MAPK pathway is the predominant target for therapy in advance thyroid carcinomas; adding NGS enables the identification of co-mutations associated with resistance (PI3K/AKT). Within poorly differentiated thyroid carcinoma, the molecular profile may hold prognostic value in the era of targeted therapy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29695638">
                    <p>用于分子诊断的新一代测序（NGS）允许同时检测多种信号通路中的活化癌基因和肿瘤抑制基因突变。晚期甲状腺癌的扩展突变分析可能会增强对靶向治疗的考虑。我们使用NGS（Ion Torrent Personal Genome Machine）从2012年4月4日至2014年4月分析了216例晚期甲状腺癌患者的临床衍生分子谱。我们使用Ampliseq Cancer Hotspot panel在肿瘤诊断和临床相关性方面研究了46/50癌症相关基因中的替换和小插入缺失。突变在晚期甲状腺癌中很常见154（71％）主要在可靶向的MAPK途径中（146/216,68％）和几种PI3K / AKT途径（8,4％; 6作为共同突变）。 BRAF V600E突变与乳头状癌（94 / 139,68％），低分化癌（4/39,10％）和间变性癌（3/12,25％）有关。 NRAS突变发生于滤泡型（5/12，40％）和低分化型甲状腺癌（12 / 39,31％）。肿瘤抑制基因突变（16,7％）主要发生在Hurthle细胞TP53中（2/5，仅有40％，唯一的突变），间变性（3/12,25％）和低分化甲状腺癌（4/39， 10％）一些共同突变和甲状腺乳头状癌（5/139,4％）总是共同突变。 Kaplan-Meier分析显示，与未接受可靶向途径突变的类似治疗患者相比，接受靶向治疗的含有活化突变的低分化甲状腺癌患者的生存率显着提高（p = 0.02）。总之，MAPK通路是预先甲状腺癌治疗的主要靶点;添加NGS可以鉴定与抗性相关的共同突变（PI3K / AKT）。在低分化甲状腺癌中，分子特征可能在靶向治疗时代具有预后价值。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Mol. Cancer Ther.">
                Journal: Mol. Cancer Ther. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29695638" target="_blank">
                PMID: 29695638
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-25
            </a>

            <a class="btn btn-sm btn-danger" title="Chen H, Luthra R, Routbort MJ, Patel KP, Cabanillas ME, Broaddus RR, Williams MD">
                Authors: Chen H, Luthra R,...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29695638">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>48. Evaluation of Targeted Next Generation Sequencing for Detection of Bovine Pathogens in Clinical Samples.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29695524" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29695524" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29695524">
                    <p>The laboratory diagnosis of infectious diseases, especially those caused by mixed infections, is challenging. Routinely, this requires submission of multiple samples to separate laboratories. Advances in next-generation sequencing (NGS) have provided the opportunity for development of a comprehensive method to identify infectious agents. This study describes the use of target-specific primers for PCR-mediated amplification with the NGS technology, in which pathogen genomic regions of interest are enriched and selectively sequenced from clinical samples. In the study, 198 primers were designed to target 43 common bovine and small ruminant bacterial, fungal, viral, and parasitic pathogens and a bioinformatics tool was specifically constructed for the detection of targeted pathogens. The primers were confirmed to detect the intended pathogens by testing reference strains and isolates. The method was then validated using 60 clinical samples (including tissues, feces and milk) that were also tested with other routine diagnostic techniques. Detection limits of the targeted NGS method were evaluated using 10 representative pathogens that were also tested by qPCR and the NGS method was able to detect the organisms from samples with qPCR Ct values in the 30s. The method was successful for the detection of multiple pathogens in the clinical samples, including some additional pathogens missed by the routine techniques because the specific tests needed for these particular organisms were not performed. These results demonstrate the feasibility of the approach and indicate that it is possible to incorporate NGS as a diagnostic tool in a cost effective manner into a veterinary diagnostic laboratory.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29695524">
                    <p>传染病的实验室诊断，特别是由混合感染引起的传染病，具有挑战性。通常，这需要将多个样品提交给不同的实验室。新一代测序（NGS）的进展为开发鉴定传染因子的综合方法提供了机会。这项研究描述了用NGS技术进行PCR介导扩增的目标特异性引物的使用，其中感兴趣的病原体基因组区域被富集并且从临床样品中选择性地测序。在该研究中，设计了198个引物，用于靶向43种常见的牛和小反刍动物的细菌，真菌，病毒和寄生虫病原体，并且特异性构建了用于检测靶向病原体的生物信息学工具。通过检测参考菌株和分离株确认引物检测预期的病原体。然后使用60种临床样品（包括组织，粪便和牛奶）验证该方法，这些样品也用其他常规诊断技术进行了测试。使用10种代表性病原体评估了目标NGS方法的检测限，这些代表性病原体也通过qPCR进行了测试，并且NGS方法能够在30s内检测到具有qPCR Ct值的样品中的生物体。该方法成功地用于检测临床样品中的多种病原体，包括常规技术漏检的一些其他病原体，因为这些特定生物体所需的特定测试未被执行。这些结果证明了该方法的可行性，并表明可以将NGS作为诊断工具以具有成本效益的方式纳入兽医诊断实验室。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Clin. Microbiol.">
                Journal: J. Clin. Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29695524" target="_blank">
                PMID: 29695524
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-25
            </a>

            <a class="btn btn-sm btn-danger" title="Anis E, Hawkins IK, Ilha MRS, Woldemeskel MW, Saliki JT, Wilkes RP">
                Authors: Anis E, Hawkins I...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29695524">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>49. Targeted next-generation sequencing reveals that a compound heterozygous mutation in phosphodiesterase 6a gene leads to retinitis pigmentosa in a Chinese family.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29693493" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29693493" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29693493">
                    <p>Retinitis pigmentosa (RP) is a genetically heterogeneous disease with over 70 causative genes identified to date. However, approximately 40% of RP cases remain genetically unsolved, suggesting that many novel disease-causing mutations are yet to be identified. The purpose of this study is to identify the causative mutations of a Chinese RP family.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29693493">
                    <p>色素性视网膜炎（RP）是一种遗传异质性疾病，迄今已鉴定出70多种致病基因。然而，大约40％的RP病例遗传性尚未解决，这表明许多新的致病性突变尚未确定。本研究的目的是鉴别中国RP家族的致病突变。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ophthalmic Genet.">
                Journal: Ophthalmic Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29693493" target="_blank">
                PMID: 29693493
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-25
            </a>

            <a class="btn btn-sm btn-danger" title="Zhang S, Li J, Li S, Yang Y, Yang M, Yang Z, Zhu X, Zhang L">
                Authors: Zhang S, Li J, Li...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29693493">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>50. Whole genome shotgun sequence of Bacillus paralicheniformis strain KMS 80, a rhizobacterial endophyte isolated from rice (Oryza sativa L.).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29692960" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29692960" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29692960">
                    <p>Bacillus paralicheniformis strain KMS 80 (MTCC No. 12704) is an isolate from the root tissues of rice (Oryza sativa L.) that displays biological nitrogen fixation and plant growth promoting abilities. Here, we report the complete genome sequence of this strain, which contains 4,566,040 bp, 4424 protein-coding genes, 8692 promoter sequences, 67 tRNAs, 20 rRNA genes with six copies of 5S rRNAs along with a single copy of 16S-23S rRNA and genome average GC-content of 45.50%. Twenty one genes involved in nitrogen metabolism pathway and two main transcriptional factor genes, glnR and tnrA responsible for regulation of nitrogen fixation in Bacillus sp. were predicted from the whole genome of strain KMS 80. Analysis of the ~ 4.57 Mb genome sequence will give support to understand the genetic determinants of host range, endophytic colonization behaviour as well as to enhance endophytic nitrogen fixation and other plant beneficial role of B. paralicheniformis in rice.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29692960">
                    <p>Bacillus paralicheniformis菌株KMS 80（MTCC编号12704）是显示生物固氮和植物生长促进能力的水稻（Oryza sativa L.）根组织的分离物。在这里，我们报告了该菌株的全基因组序列，其包含4,566,040bp，4424个蛋白质编码基因，8692个启动子序列，67个tRNA，20个rRNA基因和6个5S rRNAs拷贝以及16S-23S rRNA的单个拷贝，以及基因组平均GC含量为45.50％。二十一个参与氮代谢途径的基因和两个主要转录因子基因glnR和tnrA负责调控芽孢杆菌固氮。是从菌株KMS 80的全基因组预测的。对〜4.57 Mb基因组序列的分析将有助于理解宿主范围，内生菌定殖行为的遗传决定因素以及增强内生固氮和其他植物对B的有益作用。稻瘟病菌。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=3 Biotech">
                Journal: 3 Biotech [IF=1.361]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29692960" target="_blank">
                PMID: 29692960
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Annapurna K, Govindasamy V, Sharma M, Ghosh A, Chikara SK">
                Authors: Annapurna K, Govi...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5910378" target="_blank">
                PMC5910378
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29692960">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>51. Co-expression of the calcitonin receptor gene in the hypothalamic kisspeptin neurons in female rats.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29692674" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29692674" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29692674">
                    <p>Hypothalamic kisspeptin neurons are considered to play a critical role in regulating mammalian reproduction and integrating humoral and neuronal inputs that control gonadotropin-releasing hormone (GnRH)/gonadotropin release. The present study aimed to investigate the upstream regulator candidates for kisspeptin neurons.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29692674">
                    <p>认为下丘脑kisspeptin神经元在调节哺乳动物生殖和整合控制促性腺激素释放激素（GnRH）/促性腺激素释放的体液和神经元输入方面发挥关键作用。目前的研究旨在调查Kisspeptin神经元的上游调节器候选者。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Reprod. Med. Biol.">
                Journal: Reprod. Med. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29692674" target="_blank">
                PMID: 29692674
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Assadullah N, Ieda N, Kawai H, Ishii K, Ihara N, Inoue Y, Uenoyama H">
                Authors: Assadullah N, Ied...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5902471" target="_blank">
                PMC5902471
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29692674">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>52. Small ncRNA-Seq Results of Human Tissues: Variations Depending on Sample Integrity.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29691221" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29691221" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29691221">
                    <p>Although mature miRNAs are relatively stable in vivo, RNA degradation can have a substantial influence on small noncoding RNA (sncRNA) profiles.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29691221">
                    <p>虽然成熟的miRNA在体内相对稳定，但RNA降解可能对小的非编码RNA（sncRNA）谱具有实质影响。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Chem.">
                Journal: Clin. Chem. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29691221" target="_blank">
                PMID: 29691221
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-24
            </a>

            <a class="btn btn-sm btn-danger" title="Ludwig N, Fehlmann T, Galata V, Franke A, Backes C, Meese E, Keller A">
                Authors: Ludwig N, Fehlman...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29691221">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>53. Landscape of Actionable Genetic Alterations Profiled from 1,071 Tumor Samples in Korean Cancer Patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29690749" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29690749" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29690749">
                    <p>With the emergence of next-generation sequencing (NGS) technology, profiling a wide range of genomic alterations has become a possibility resulting in improved implementation of targeted cancer therapy. In Asian populations, the prevalence and spectrum of clinically actionable genetic alterations has not yet been determined because of a lack of studies examining high-throughput cancer genomic data.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29690749">
                    <p>随着新一代测序（NGS）技术的出现，分析广泛的基因组改变已成为改善靶向癌症治疗实施的可能性。在亚洲人群中，由于缺乏研究高通量癌症基因组数据的研究，尚未确定临床可行的基因改变的流行性和范围。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Res Treat">
                Journal: Cancer Res Treat [IF=3.772]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29690749" target="_blank">
                PMID: 29690749
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-23
            </a>

            <a class="btn btn-sm btn-danger" title="Lee SH, Lee B, Shim JH, Lee KW, Yun JW, Kim SY, Kim TY, Kim YH, Ko YH, Chung HC, Yu CS, Lee J, Rha SY, Kim TW, Jung KH, Im SA, Moon HG, Cho S, Kang JH, Kim J, Kim SK, Ryu HS, Ha SY, Kim JI, Chung YJ, Kim C, Kim HL, Park WY, Noh DY, Park K">
                Authors: Lee SH, Lee B, Sh...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29690749">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>54. [Clinical experience and next-generation sequencing analysis of encephalitis caused by pseudorabies virus].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29690727" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29690727" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29690727">
                    <p>Objective: To detect potential pathogens including pseudorabies virus in patients with encephalitis of unknown etiology in China and describe novel encephalitic entities. Methods: Patients with clinically suspected infectious encephalitis were enrolled in a multicenter study to identify the pathogens in PUMCH Encephalitis Program.Next-generation sequencing(NGS) of cerebrospinal fluid (CSF) was used in patients with encephalitis of unknown etiology enrolled from 2016 to 2017.The patients diagnosed as PRV encephalitis were studied to describe this novel entity. Results: The four patients(3 male, 1 male, 38-54 years old) had occupational exposure to raw park when working in the production or marketing of pork and at least one got injured during pork-cutting.Two of them were confirmed with NGS of CSF, and anti-PRV antibodies were positive in 3 patients whose serum was available for serological analysis.They all presented with an acute onset of fever, convulsion, loss of consciousness and respiratory failure within 1 to 4 days and rapidly deteriorated even on extensive treatment.All the patients needed ICU admission and 3 needed mechanical ventilation.Two patients also had bilateral retinitis.Neuroimaging revealed symmetric gray matter lesions including limbic system, basal ganglia and midbrain without obvious hemorrhage.Lumbar puncture revealed elevated intracranial pressure and lymphocytic pleocytosis [(6-64)×10(6)/L] of CSF.The patients failed to response to the treatment of acyclovir combined with intravenous immunoglobulin and steroids.Modified Rankin Score was 3, 5, 5 and 6 (died) for the 4 patients respectively on last follow-up. Conclusions: PRV could be a cause of severe encephalitis.The patients with suspected pseudorabies encephalitis (PRE) need to be tested for PRV DNA timely.Severe encephalitis with bilateral involvement of limbic system, basal ganglion, thalamus and midbrain in patient with occupational exposure indicate this emerging infectious encephalitis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29690727">
                    <p>目的：检测中国病因不明的脑炎病毒包括伪狂犬病病毒，并描述新的脑炎病例。方法：将临床疑似感染性脑炎患者纳入多中心研究，以确定PUMCH脑炎计划中的病原体。脑脊液（CSF）的下一代测序（NGS）用于2016  -  2017年登记的病因不明的脑炎患者。被诊断为PRV脑炎的患者被研究以描述这种新型实体。结果：4名患者（男性3名，男性1名，男性38-54岁）在猪肉生产或销售工作期间曾接触过原始园区，至少有一人在切猪时受伤。其中两人3例血清可用于血清学分析的CSF患者和抗PRV抗体阳性，均在1〜4天内出现急性发热，惊厥，意识丧失和呼吸衰竭，甚至在1〜4天内迅速恶化所有患者均需ICU入院，3例需机械通气。2例患者也有双侧视网膜炎。神经影像学显示对称性灰质病变包括边缘系统，基底节和中脑，无明显出血。腰椎穿刺显示颅内压升高和淋巴细胞增多[ （6-64）×10（6）/ L]。患者对阿昔洛韦联合静脉注射免疫球蛋白和类固醇治疗无效。在最后的随访中，4名患者分别为3,5,5和6名（死亡）。结论：PRV可能是导致严重脑炎的原因。疑似伪狂犬病病毒（PRE）患者需及时检测PRV DNA。职业暴露患者中存在边缘系统，基底节，丘脑和中脑双侧累及的严重脑炎这种新兴的传染性脑炎。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Zhonghua Yi Xue Za Zhi">
                Journal: Zhonghua Yi Xue Za Zhi [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29690727" target="_blank">
                PMID: 29690727
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Zhao WL, Wu YH, Li HF, Li SY, Fan SY, Wu HL, Li YJ, Lü YL, Han J, Zhang WC, Zhao Y, Li GL, Qiao XD, Ren HT, Zhu YC, Peng B, Cui LY, Guan HZ">
                Authors: Zhao WL, Wu YH, L...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29690727">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>55. Prostate Cancer Genomics: Recent Advances and the Prevailing Underrepresentation from Racial and Ethnic Minorities.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29690565" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29690565" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29690565">
                    <p>Prostate cancer (CaP) is the most commonly diagnosed non-cutaneous cancer and the second leading cause of male cancer deaths in the United States. Among African American (AA) men, CaP is the most prevalent malignancy, with disproportionately higher incidence and mortality rates. Even after discounting the influence of socioeconomic factors, the effect of molecular and genetic factors on racial disparity of CaP is evident. Earlier studies on the molecular basis for CaP disparity have focused on the influence of heritable mutations and single-nucleotide polymorphisms (SNPs). Most CaP susceptibility alleles identified based on genome-wide association studies (GWAS) were common, low-penetrance variants. Germline CaP-associated mutations that are highly penetrant, such as those found in HOXB13 and BRCA2, are usually rare. More recently, genomic studies enabled by Next-Gen Sequencing (NGS) technologies have focused on the identification of somatic mutations that contribute to CaP tumorigenesis. These studies confirmed the high prevalence of ERG gene fusions and PTEN deletions among Caucasian Americans and identified novel somatic alterations in SPOP and FOXA1 genes in early stages of CaP. Individuals with African ancestry and other minorities are often underrepresented in these large-scale genomic studies, which are performed primarily using tumors from men of European ancestry. The insufficient number of specimens from AA men and other minority populations, together with the heterogeneity in the molecular etiology of CaP across populations, challenge the generalizability of findings from these projects. Efforts to close this gap by sequencing larger numbers of tumor specimens from more diverse populations, although still at an early stage, have discovered distinct genomic alterations. These research findings can have a direct impact on the diagnosis of CaP, the stratification of patients for treatment, and can help to address the disparity in incidence and mortality of CaP. This review examines the progress of understanding in CaP genetics and genomics and highlight the need to increase the representation from minority populations.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29690565">
                    <p>前列腺癌（CaP）是美国最常诊断的非皮肤癌，也是男性癌症死亡的第二大原因。在非洲裔美国人（AA）男性中，CaP是最普遍的恶性肿瘤，其发病率和死亡率高得不成比例。即使扣除社会经济因素的影响，分子和遗传因素对CaP种族差异的影响也是显而易见的。早期关于CaP差异分子基础的研究集中在可遗传突变和单核苷酸多态性（SNPs）的影响上。基于全基因组关联研究（GWAS）鉴定的大多数CaP易感性等位基因是常见的低外显率变体。具有高渗透性的种系CaP相关突变（例如HOXB13和BRCA2中发现的突变）通常很少见。最近，由下一代测序（NGS）技术启用的基因组研究集中在识别有助于CaP肿瘤发生的体细胞突变。这些研究证实了高加索人美国人ERG基因融合和PTEN缺失的高发病率，并且在CaP早期阶段发现了SPOP和FOXA1基因的新型体细胞改变。在这些大规模的基因组研究中，具有非洲血统和其他少数民族的个体往往代表不足，主要使用欧洲血统男性的肿瘤。来自AA男性和其他少数民族的标本数量不足，以及人群中CaP分子病因学的异质性，挑战了这些项目发现的普遍性。尽管仍处于早期阶段，但通过对来自更多不同人群的大量肿瘤标本进行测序来弥补这一差距，但仍发现了不同的基因组改变。这些研究结果可以直接影响CaP的诊断，治疗患者的分层，并有助于解决CaP发病率和死亡率的差异。这篇综述考察了CaP遗传学和基因组学理解的进展，并强调需要增加少数族群的代表性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Mol Sci">
                Journal: Int J Mol Sci [IF=3.226]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29690565" target="_blank">
                PMID: 29690565
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-22
            </a>

            <a class="btn btn-sm btn-danger" title="Tan SH, Petrovics G, Srivastava S">
                Authors: Tan SH, Petrovics...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29690565">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>56. Metagenomics Analysis of Bacterial Population of Tympanosclerotic Plaques and Cholesteatomas.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29688828" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29688828" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29688828">
                    <p>Objective Chronic otitis media can cause cholesteatomas or tympanosclerosis; however, the pathophysiology of such conditions is not completely known. The aim was to identify a bacterial genome that might be present in tympanosclerotic plaques and cholesteatomas using sequence analysis of the gene responsible for the transcription of 16 ribosomal RNA (rRNA). Study Design Metagenomics analysis of the samples. Setting Samples were collected and evaluated at tertiary care centers. Subjects and Methods Sixty-five tympanosclerotic plaques and 37 cholesteatomas were evaluated. The polymerase chain reaction (PCR) was performed using primers designed for the amplification of the gene responsible for the transcription of bacterial 16 rRNA. The PCR-positive samples were sequenced via Sanger method, and 46 selected samples were analyzed with next-generation sequencing (NGS). Results Sanger sequencing revealed the presence of bacterial genomes in a total of 18 of the 102 samples tested. Sequencing of these genomes indicated the presence of Alloiococcus otitis, Staphylococcus aureus, Achromobacter xylosoxidans, Escherichia coli, Staphylococcus sciuri, Staphylococcus caprae, Parvimonas spp., and Bacillus sp. in the tested samples. The NGS showed 1 or more different bacterial genomes in 44 (95.7%) of the 46 samples tested. Predominately, genome of Clostridiales (27 samples), Staphylococcaceae (24 samples), Peptoniphilaceae (12 samples), and Turicella otitidis (9 samples) were identified. Conclusion The middle ear is inhabited by a diverse microbial community than that previously known. With the use of molecular biology, it has become easier to identify the bacterial genomes and improve our understanding of the role of middle ear microbiota in the pathogenesis of chronic inflammatory ear diseases.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29688828">
                    <p>目的慢性中耳炎可引起胆脂瘤或鼓室硬化症;然而，这种病症的病理生理学尚不完全清楚。其目的是使用负责16核糖体RNA（rRNA）转录的基因的序列分析来鉴定可能存在于鼓室硬化斑块和胆脂瘤中的细菌基因组。研究设计样本的宏基因组学分析。设置样本在三级护理中心收集和评估。受试者和方法评估65个鼓室硬化斑块和37个胆脂瘤。使用设计用于扩增负责细菌16rRNA转录的基因的引物进行聚合酶链式反应（PCR）。通过Sanger方法对PCR阳性样品进行测序，并用新一代测序（NGS）分析46个选择的样品。结果Sanger测序显示在所测试的102个样品中的总共18个中存在细菌基因组。对这些基因组进行测序表明存在Alloriococcus中耳炎，金黄色葡萄球菌，木糖氧化无色杆菌，大肠杆菌，金黄色葡萄球菌，金黄色葡萄球菌，Parvimonas spp。和芽孢杆菌属。在测试样品中。 NGS在46个样本中有44个（95.7％）显示1个或多个不同的细菌基因组。主要检测了梭状芽孢杆菌（27个样品），葡萄球菌科（24个样品），Pe ae科（12个样品）和水痘螈（9个样品）的基因组。结论中耳居住的微生物群落多于先前已知的微生物群落。随着分子生物学的应用，识别细菌基因组和提高我们对中耳微生物在慢性炎症性耳病发病机制中作用的理解变得更加容易。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Otolaryngol Head Neck Surg">
                Journal: Otolaryngol Head Neck Surg [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29688828" target="_blank">
                PMID: 29688828
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-01
            </a>

            <a class="btn btn-sm btn-danger" title="Kalcioglu MT, Guldemir D, Unaldi O, Egilmez OK, Celebi B, Durmaz R">
                Authors: Kalcioglu MT, Gul...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29688828">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>57. Comparing Rock-inhabiting Microbial Communities in Different Rock Types from a High Arctic Polar Desert.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29688499" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29688499" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29688499">
                    <p>Although rocks are habitable places for microbes in extreme environments, microbial diversity in these lithic environments is still poorly understood. The diversity and abundance of rock-inhabiting microbial communities in different types of rock in Svalbard, Norwegian High Arctic were examined using NGS sequencing of bacterial 16S rRNA genes and fungal 28S rRNA genes. Compositions of both bacterial and fungal communities varied across different rock types: sandstone, limestone, basalt, granite, and travertine. Bacterial communities were dominated by Actinobacteria, Proteobacteria, Chloroflexi, Bacteroidetes, and Acidobacteria. Fungal communities consisted of Eurotiomycetes, Lecanoromycetes, Dothideomycetes, and Leotiomycetes. Both bacterial and fungal community compositions were significantly correlated with the geochemical characteristics of rocks. Bacterial communities were considerably correlated with the rock elements such as Mg and Ca. Fungal communities were considerably correlated with Fe. Interestingly, many dominant bacterial and fungal operational taxonomic units (OTUs) in the investigated rocks from the study area were closely affiliated to those found in other cold regions such as Alpine area, Arctic and Antarctica, suggesting that environmental constraints such as cold temperature may lead to convergence in microbial community composition. These results confirm that rocks in cold environments act as reservoirs of diverse bacteria and fungi, which may improve our understanding of lithic microbial ecology in the cold desert.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29688499">
                    <p>尽管在极端环境中岩石是适合微生物居住的地方，但在这些岩石环境中的微生物多样性仍然知之甚少。利用细菌16S rRNA基因和真菌28S rRNA基因的NGS测序，检测了挪威北极斯瓦尔巴群岛不同类型岩石中岩石栖息微生物群落的多样性和丰度。细菌和真菌群落的成分在不同的岩石类型中有所不同：砂岩，石灰石，玄武岩，花岗岩和石灰华。细菌群落以放线菌，变形菌，Chloroflexi，拟杆菌和酸杆菌为主。真菌群落由Eurotiomycetes，Lecanoromycetes，Dothideomycetes和Leotiomycetes组成。细菌和真菌群落组成与岩石的地球化学特征显着相关。细菌群落与镁和钙等岩石元素显着相关。真菌群落与Fe显着相关。有趣的是，研究区岩石中的许多优势细菌和真菌操作分类单元（OTU）与其他寒冷地区如阿尔卑斯地区，北极和南极等发现的那些密切相关，表明环境约束如低温可能导致微生物群落组成的趋同。这些结果证实，寒冷环境中的岩石充当了各种细菌和真菌的储层，这可以提高我们对寒冷沙漠岩石微生物生态学的理解。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=FEMS Microbiol. Ecol.">
                Journal: FEMS Microbiol. Ecol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29688499" target="_blank">
                PMID: 29688499
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-21
            </a>

            <a class="btn btn-sm btn-danger" title="Choe YH, Kim M, Woo J, Lee MJ, Lee JI, Lee EJ, Lee YK">
                Authors: Choe YH, Kim M, W...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29688499">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>58. Dynamic Variation and Reversion in the Signature Amino Acids of H7N9 Virus During Human Infection.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29688498" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29688498" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29688498">
                    <p>Signature amino acids of H7N9 influenza virus play critical roles in human adaption and pathogenesis, but their dynamic variation is unknown during disease development.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29688498">
                    <p>H7N9流感病毒的特征性氨基酸在人类适应和发病机制中起关键作用，但在疾病发展过程中它们的动态变化是未知的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Infect. Dis.">
                Journal: J. Infect. Dis. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29688498" target="_blank">
                PMID: 29688498
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-24
            </a>

            <a class="btn btn-sm btn-danger" title="Zou X, Guo Q, Zhang W, Chen H, Bai W, Lu B, Zhang W, Fan Y, Liu C, Wang Y, Zhou F, Cao B">
                Authors: Zou X, Guo Q, Zha...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29688498">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>59. Maser: one-stop platform for NGS big data from analysis to visualization.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29688385" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29688385" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29688385">
                    <p>http://cell-innovation.nig.ac.jp/maser/.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29688385">
                    <p>HTTP://cell-innovation.你个.AC.键盘/马绍尔/.</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Database (Oxford)">
                Journal: Database (Oxford) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29688385" target="_blank">
                PMID: 29688385
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jan-01
            </a>

            <a class="btn btn-sm btn-danger" title="Kinjo S, Monma N, Misu S, Kitamura N, Imoto J, Yoshitake K, Gojobori T, Ikeo K">
                Authors: Kinjo S, Monma N,...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5905357" target="_blank">
                PMC5905357
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29688385">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>60. Impact of substrate stiffness on dermal papilla aggregates in microgels.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29687797" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29687797" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29687797">
                    <p>Interaction between cells and the extracellular environment plays a vital role in cellular development. The mechanical property of a 3-dimensional (3D) culture can be modified to mimic in vivo conditions. Dermal papilla (DP) cells are shown to gradually lose their inductivity in hair cycle development in a 2-dimensional culture. They are shown to partially restore their inductivity when transferred into a 3D microenvironment. In this study, a microarray fabricated from three different concentrations of poly-ethylene-glycol-diacrylate 3500, namely 5%, 10% and 15% w/v, yielded increasing substrate stiffness. The impact of varying substrate stiffness was tested for DP cell viability, attachment, and selected hair inductive markers. DP aggregates were shown to be viable and exhibited greater spreading with increasing substrate stiffness. Moreover, DP aggregates cultured on a softer substrate showed a greater fold change of gene and protein expressions than those cultured on a harder substrate.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29687797">
                    <p>细胞与细胞外环境之间的相互作用在细胞发育中起着至关重要的作用。可以修改三维（3D）培养物的机械性质以模拟体内条件。显示真皮乳头（DP）细胞在二维培养中逐渐失去它们在毛发周期发育中的诱导性。当它们转移到3D微环境中时，它们显示出部分恢复其诱导性。在这项研究中，由三种不同浓度的聚乙二醇二丙烯酸酯3500（即5％，10％和15％w / v）制成的微阵列产生了增加的基质硬度。测试了不同基质硬度对DP细胞活力，附着和选定的头发诱导标记物的影响。 DP聚集体被证明是可行的，并且随着基质硬度的增加展现出更大的扩散。此外，在较软底物上培养的DP聚集体显示出比在较硬底物上培养的那些更大的基因和蛋白质表达的倍数变化。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Biomater Sci">
                Journal: Biomater Sci [IF=4.210]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29687797" target="_blank">
                PMID: 29687797
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-24
            </a>

            <a class="btn btn-sm btn-danger" title="Tan JJY, Tee JK, Chou KO, Yong SYA, Pan J, Ho HK, Ho PCL, Kang L">
                Authors: Tan JJY, Tee JK, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29687797">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>61. Ultradeep single-molecule real-time sequencing of HIV envelope reveals complete compartmentalization of highly macrophage-tropic R5 proviral variants in brain and CXCR4-using variants in immune and peripheral tissues.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29687407" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29687407" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29687407">
                    <p>Despite combined antiretroviral therapy (cART), HIV+ patients still develop neurological disorders, which may be due to persistent HIV infection and selective evolution in brain tissues. Single-molecule real-time (SMRT) sequencing technology offers an improved opportunity to study the relationship among HIV isolates in the brain and lymphoid tissues because it is capable of generating thousands of long sequence reads in a single run. Here, we used SMRT sequencing to generate ~ 50,000 high-quality full-length HIV envelope sequences (&gt; 2200 bp) from seven autopsy tissues from an HIV+/cART+ subject, including three brain and four non-brain sites. Sanger sequencing was used for comparison with SMRT data and to clone functional pseudoviruses for in vitro tropism assays. Phylogenetic analysis demonstrated that brain-derived HIV was compartmentalized from HIV outside the brain and that the variants from each of the three brain tissues grouped independently. Variants from all peripheral tissues were intermixed on the tree but independent of the brain clades. Due to the large number of sequences, a clustering analysis at three similarity thresholds (99, 99.5, and 99.9%) was also performed. All brain sequences clustered exclusive of any non-brain sequences at all thresholds; however, frontal lobe sequences clustered independently of occipital and parietal lobes. Translated sequences revealed potentially functional differences between brain and non-brain sequences in the location of putative N-linked glycosylation sites (N-sites), V1 length, V3 charge, and the number of V4 N-sites. All brain sequences were predicted to use the CCR5 co-receptor, while most non-brain sequences were predicted to use CXCR4 co-receptor. Tropism results were confirmed by in vitro infection assays. The study is the first to use a SMRT sequencing approach to study HIV compartmentalization in tissues and supports other reports of limited trafficking between brain and non-brain sequences during cART. Due to the long sequence length, we could observe changes along the entire envelope gene, likely caused by differential selective pressure in the brain that may contribute to neurological disease.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29687407">
                    <p>尽管联合使用了抗逆转录病毒疗法（cART），艾滋病病毒+患者仍然患有神经系统疾病，这可能是由于持续的艾滋病毒感染和脑组织中的选择性进化。单分子实时（SMRT）测序技术为研究HIV分离物在大脑和淋巴组织中的关系提供了更好的机会，因为它能够在单次运行中产生数千个长序列读数。在这里，我们使用SMRT测序从HIV + / cART +受试者的七个尸检组织中产生约50,000个高质量的全长HIV包膜序列（&gt; 2200bp），包括三个脑和四个非脑部位。 Sanger测序用于与SMRT数据进行比较并克隆用于体外嗜性测定的功能性假病毒。系统发育分析表明，大脑派生的艾滋病毒从大脑外的艾滋病病毒中分离出来，并且三种脑组织中的每一种的变体都是独立分组的。来自所有外围组织的变体在树上混合，但独立于脑进化枝。由于大量的序列，在三个相似性阈值（99,99.5和99.9％）下也进行了聚类分析。所有脑部序列都聚集在任何阈值以外的任何非脑部序列中;然而，额叶序列聚集独立于枕叶和顶叶。翻译序列揭示了假定的N-连接糖基化位点（N位点），V1长度，V3电荷和V4N位点的数目在脑和非脑序列之间的潜在功能差异。预测所有脑部序列使用CCR5共同受体，而大多数非脑部序列预测使用CXCR4共同受体。取向性结果通过体外感染测定来确认。这项研究是第一个使用SMRT测序方法研究组织中的HIV区室化的方法，并支持其他有关cART期间脑和非脑序列之间有限运输的报告。由于序列长度很长，我们可以观察整个包膜基因的变化，这可能是由大脑中可能导致神经疾病的差异选择性压力引起的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Neurovirol.">
                Journal: J. Neurovirol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29687407" target="_blank">
                PMID: 29687407
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-23
            </a>

            <a class="btn btn-sm btn-danger" title="Brese RL, Gonzalez-Perez MP, Koch M, O&#39;Connell O, Luzuriaga K, Somasundaran M, Clapham PR, Dollar JJ, Nolan DJ, Rose R, Lamers SL">
                Authors: Brese RL, Gonzale...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29687407">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>62. Complete sequence and diversity of a maize-associated Polerovirus in East Africa.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29687187" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29687187" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29687187">
                    <p>Since 2011-2012, Maize lethal necrosis (MLN) has emerged in East Africa, causing massive yield loss and propelling research to identify viruses and virus populations present in maize. As expected, next generation sequencing (NGS) has revealed diverse and abundant viruses from the family Potyviridae, primarily sugarcane mosaic virus (SCMV), and maize chlorotic mottle virus (MCMV) (Tombusviridae), which are known to cause MLN by synergistic co-infection. In addition to these expected viruses, we identified a virus in the genus Polerovirus (family Luteoviridae) in 104/172 samples selected for MLN or other potential virus symptoms from Kenya, Uganda, Rwanda, and Tanzania. This polerovirus (MF974579) nucleotide sequence is 97% identical to maize-associated viruses recently reported in China, termed &#39;maize yellow mosaic virus&#39; (MaYMV) and maize yellow dwarf virus (MaYMV; KU291101, KU291107, MYDV-RMV2; KT992824); and 99% identical to MaYMV (KY684356) infecting sugarcane and itch grass in Nigeria; 83% identical to a barley-associated polerovirus recently identified in Korea (BVG; KT962089); and 79% identical to the U.S. maize-infecting polerovirus maize yellow dwarf virus (MYDV-RMV; KT992824). Nucleotide sequences from ORF0 of 20 individual East African isolates collected from Kenya, Uganda, Rwanda, and Tanzania shared 98% or higher identity, and were detected in 104/172 (60.5%) of samples collected for virus-like symptoms, indicating extensive prevalence but limited diversity of this virus in East Africa. We refer to this virus as &quot;MYDV-like polerovirus&quot; until symptoms of the virus in maize are known.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29687187">
                    <p>自2011年至2012年以来，东非出现了玉米致死性坏死（MLN），造成巨大的产量损失，推动了玉米中存在病毒和病毒种群的研究。正如预期的那样，下一代测序（NGS）已经揭示了来自家庭Potyviridae的主要是甘蔗花叶病毒（SCMV）和玉米褪绿斑驳病毒（MCMV）（Tombusviridae）的多种和丰富的病毒，其已知通过协同协同作用导致MLN，感染。除了这些预期的病毒之外，我们在来自肯尼亚，乌干达，卢旺达和坦桑尼亚的104/172个样品中鉴定了Polerovirus（Luteoviridae家族）病毒，选择MLN或其他潜在病毒症状。这种极柱病毒（MF974579）核苷酸序列与中国最近报道的称为“玉米黄花叶病毒”（MaYMV）和玉米黄矮病毒（MaYMV; KU291101，KU291107，MYDV-RMV2; KT992824）的玉米相关病毒有97％相同;和MaYMV（KY684356）在尼日利亚感染甘蔗和痒草的99％相同;最近在韩国发现的与大麦相关的杆状病毒有83％相同（BVG; KT962089）;和79％与美国玉米感染极性病毒玉米黄矮病毒（MYDV-RMV; KT992824）相同。来自肯尼亚，乌干达，卢旺达和坦桑尼亚的20个单独东非分离株的ORF0核苷酸序列的同一性为98％或更高，104/172（60.5％）个样本中检测到病毒样症状，表明广泛流行但这种病毒在东非的多样性有限。我们称这种病毒为“MYDV-like polerovirus”，直到玉米病毒的症状已知。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Virus Genes">
                Journal: Virus Genes [IF=1.431]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29687187" target="_blank">
                PMID: 29687187
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-23
            </a>

            <a class="btn btn-sm btn-danger" title="Massawe DP, Stewart LR, Kamatenesi J, Asiimwe T, Redinbaugh MG">
                Authors: Massawe DP, Stewa...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29687187">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>63. Computational Methods for Assessing Chromatin Hierarchy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29686798" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29686798" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29686798">
                    <p>The hierarchical organization of chromatin is known to associate with diverse cellular functions; however, the precise mechanisms and the 3D structure remain to be determined. With recent advances in high-throughput next generation sequencing (NGS) techniques, genome-wide profiling of chromatin structures is made possible. Here, we provide a comprehensive overview of NGS-based methods for profiling &quot;higher-order&quot; and &quot;primary-order&quot; chromatin structures from both experimental and computational aspects. Experimental requirements and considerations specific for each method were highlighted. For computational analysis, we summarized a common analysis strategy for both levels of chromatin assessment, focusing on the characteristic computing steps and the tools. The recently developed single-cell level techniques based on Hi-C and ATAC-seq present great potential to reveal cell-to-cell variability in chromosome architecture. A brief discussion on these methods in terms of experimental and data analysis features is included. We also touch upon the biological relevance of chromatin organization and how the combination with other techniques uncovers the underlying mechanisms. We conclude with a summary and our prospects on necessary improvements of currently available methods in order to advance understanding of chromatin hierarchy. Our review brings together the analyses of both higher- and primary-order chromatin structures, and serves as a roadmap when choosing appropriate experimental and computational methods for assessing chromatin hierarchy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29686798">
                    <p>已知染色质的等级组织与不同的细胞功能有关;然而，精确的机制和3D结构仍有待确定。随着近期高通量下一代测序（NGS）技术的进步，染色质结构的全基因组谱分析成为可能。在这里，我们全面概述了基于NGS的方法，用于从实验和计算两个方面分析“高阶”和“初级”染色质结构。强调了每种方法的特定实验要求和注意事项。对于计算分析，我们总结了两种染色质评估水平的通用分析策略，重点关注特征计算步骤和工具。最近开发的基于Hi-C和ATAC-seq的单细胞水平技术在揭示染色体结构中细胞间变异方面具有巨大的潜力。包括对这些方法在实验和数据分析功能方面的简要讨论。我们还涉及染色质组织的生物学相关性以及与其他技术的结合如何揭示基础机制。我们总结总结以及我们对当前可用方法的必要改进的前景，以加深对染色质层次的理解。我们的综述汇集了高阶和初级染色质结构的分析，并且在选择适当的实验和计算方法评估染色质层级时作为路线图。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Comput Struct Biotechnol J">
                Journal: Comput Struct Biotechnol J [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29686798" target="_blank">
                PMID: 29686798
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Chang P, Gohain M, Yen MR, Chen PY">
                Authors: Chang P, Gohain M...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5910504" target="_blank">
                PMC5910504
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29686798">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>64. Can genomics deliver climate-change ready crops?</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29685733" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29685733" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29685733">
                    <p>Development of climate resilient crops with accelerating genetic gains in crops will require integration of different disciplines/technologies, to see the impact in the farmer&#39;s field. In this review, we summarize how we are utilizing our germplasm collections to identify superior alleles/haplotypes through NGS based sequencing approaches and how genomics-enabled technologies together with precise phenotyping are being used in crop breeding. Pre-breeding and genomics-assisted breeding approaches are contributing to the more efficient development of climate-resilient crops. It is anticipated that the integration of several disciplines/technologies will result in the delivery of climate change ready crops in less time.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29685733">
                    <p>开发具有加速的作物遗传增益的气候适应性作物需要整合不同的学科/技术，才能看到农民田间的影响。在这篇综述中，我们总结了我们如何利用我们的种质资源库，通过基于NGS的测序方法来鉴定优良的等位基因/单倍型，以及基因组技术和精确表型如何用于作物育种。预育和基因组辅助育种方法有助于提高气候适应性作物的发育效率。预计若干学科/技术的整合将导致在较短时间内提供适应气候变化的作物。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Curr. Opin. Plant Biol.">
                Journal: Curr. Opin. Plant Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29685733" target="_blank">
                PMID: 29685733
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Varshney RK, Singh VK, Kumar A, Powell W, Sorrells ME">
                Authors: Varshney RK, Sing...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29685733">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>65. Deep NPM1 Sequencing Following Allogeneic Hematopoietic Cell Transplantation Improves Risk Assessment in Adults with NPM1-Mutated AML.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29684564" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29684564" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29684564">
                    <p>Relapse is the major cause of death in patients with acute myeloid leukemia (AML) after allogeneic hematopoietic cell transplant (HCT). Measurable residual disease (MRD) detected by multiparameter flow cytometry (MFC) before and after HCT is a strong, independent risk factor for relapse. As next generation sequencing (NGS) is increasingly applied in AML MRD detection, it remains to be determined if NGS can improve prediction of post-HCT relapse. Herein, we investigated pre-HCT MRD detected by MFC and NGS in 59 adult patients with NPM1-mutated AML in morphologic remission; 45 of the 59 had post-HCT MRD determined by MFC and NGS around day 28. Before HCT, MRD detected by MFC was the most significant risk factor for relapse (HR=4.63, p&lt;0.001), whereas MRD detected only by NGS was not. After HCT, MRD detected by either MFC or NGS was significant risk factor for relapse (HR= 4.96, p=0.004 and HR= 4.36, p=0.002, respectively). Combining pre- and post-HCT MRD provided the best prediction for relapse (HR=5.25, p&lt;0.001), with a sensitivity at 83%. We conclude that NGS testing of mutated NPM1 post-HCT improves the risk assessment for relapse, whereas pre-HCT MFC testing identifies a subset of high-risk patients in whom additional therapy should be tested.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29684564">
                    <p>在异基因造血细胞移植（HCT）后，复发是急性髓性白血病（AML）患者死亡的主要原因。 HCT前后多参数流式细胞仪（MFC）检测到的可测量残留病（MRD）是复发的强大独立危险因素。随着下一代测序（NGS）在AML MRD检测中的应用越来越多，NGS是否能够改善HCT后复发的预测仍有待确定。在此，我们调查了59例用NPM1突变的AML形态学缓解的成年患者中MFC和NGS检测到的前HCT MRD;在HCT之前，MFC检测到的MRD是复发的最重要危险因素（HR = 4.63，p &lt;0.001），而仅由NGS检测到的MRD为不。 HCT后，MFC或NGS检测到的MRD是复发的显着危险因素（HR分别为4.96，p = 0.004和HR = 4.36，p = 0.002）。 HCT前后的MRD提供了复发的最佳预测（HR = 5.25，p &lt;0.001），敏感性为83％。我们得出结论：HCT后突变NPM1的NGS检测改善了复发的风险评估，而HCT前的MFC检测确定了一部分高风险患者，其中应进行额外的治疗。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Biol. Blood Marrow Transplant.">
                Journal: Biol. Blood Marrow Transplant. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29684564" target="_blank">
                PMID: 29684564
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Zhou Y, Othus M, Walter RB, Estey EH, Wu D, Wood BL">
                Authors: Zhou Y, Othus M, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29684564">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>66. In vivo insertion pool sequencing identifies virulence factors in a complex fungal-host interaction.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29684023" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29684023" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29684023">
                    <p>Large-scale insertional mutagenesis screens can be powerful genome-wide tools if they are streamlined with efficient downstream analysis, which is a serious bottleneck in complex biological systems. A major impediment to the success of next-generation sequencing (NGS)-based screens for virulence factors is that the genetic material of pathogens is often underrepresented within the eukaryotic host, making detection extremely challenging. We therefore established insertion Pool-Sequencing (iPool-Seq) on maize infected with the biotrophic fungus U. maydis. iPool-Seq features tagmentation, unique molecular barcodes, and affinity purification of pathogen insertion mutant DNA from in vivo-infected tissues. In a proof of concept using iPool-Seq, we identified 28 virulence factors, including 23 that were previously uncharacterized, from an initial pool of 195 candidate effector mutants. Because of its sensitivity and quantitative nature, iPool-Seq can be applied to any insertional mutagenesis library and is especially suitable for genetically complex setups like pooled infections of eukaryotic hosts.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29684023">
                    <p>大规模的插入诱变筛选如果通过有效的下游分析进行简化，可能是功能强大的全基因组工具，这是复杂生物系统中的严重瓶颈。基于新一代测序（NGS）的毒力因子筛选成功的一个主要障碍是病原体的遗传物质常常在真核宿主内代表性不足，使得检测极具挑战性。因此，我们在感染了营养型真菌U. maydis的玉米上建立了插入池测序（iPool-Seq）。 iPool-Seq具有标记，独特的分子条形码，以及来自体内感染组织的病原体插入突变体DNA的亲和纯化。在使用iPool-Seq的概念验证中，我们从195个候选效应突变体的初始库中鉴定了28个毒力因子，其中包括23个以前未表征的毒力因子。由于其灵敏度和定量性，iPool-Seq可以应用于任何插入诱变文库，尤其适用于基因复杂的设置，如真核宿主的混合感染。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS Biol.">
                Journal: PLoS Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29684023" target="_blank">
                PMID: 29684023
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Uhse S, Pflug FG, Stirnberg A, Ehrlinger K, von Haeseler A, Djamei A">
                Authors: Uhse S, Pflug FG,...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29684023">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>67. Immunohistochemical Detection and Molecular Characterization of IDH-mutant Sinonasal Undifferentiated Carcinomas.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29683816" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29683816" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29683816">
                    <p>Recent studies have identified recurrent isocitrate dehydrogenase 2 (IDH2) mutations in a subset of sinonasal undifferentiated carcinomas (SNUCs); however, the true frequency of IDH mutations in SNUC is unknown. We evaluated the utility of mutation-specific IDH1/2 immunohistochemistry (IHC) in a large multi-institutional cohort of SNUC and morphologic mimics. IHC using a multispecific antibody for IDH1/2 (R132/R172) mutant protein was performed on 193 sinonasal tumors including: 53 SNUCs, 8 poorly differentiated carcinomas (PDCARs) and 132 histologic mimics. Mutant IDH1/2 IHC was positive in 26/53 SNUCs (49%; 20 strongly positive and 6 weak) and 3/8 PDCARs (37.5%; 2 strong; 1 weak) but was absent in all other tumor types (0/132). Targeted next-generation sequencing (NGS) on a subset of SNUC/PDCAR (6 strong and 3 weak positive for IDH1/2 IHC; 7 negative) showed frequent IDH2 R172X mutations (10/16) and a single IDH1 R132C mutation. All 6 cases with strong positive mutant IDH1/2 staining and NGS had IDH2 R172S/G mutations. The 3 IHC-weak cases all had IDH2 R172T mutations. Among the 7 tested cases that were negative for mutant IDH1/2 IHC, NGS detected 1 case each with IDH2 R172T and IDH1 R132C mutation. IDH-mutant carcinomas also had frequent mutations in TP53 (55%) and activating mutations in KIT (45%) or the PI3K pathway (36%). Mutation-specific IDH1/2 IHC identifies IDH mutations in SNUC, however, it lacks sensitivity for the full range of IDH mutations. These findings suggest that IDH-mutant sinonasal carcinoma may represent a distinct pathobiological entity with therapeutic implications that can be identified by a combined approach of multispecific IDH1/2 IHC and sequencing.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29683816">
                    <p>最近的研究已经确定了鼻窦未分化癌（SNUCs）子集中的异常柠檬酸脱氢酶2（IDH2）突变;然而，SNUC中IDH突变的真实频率是未知的。我们评估突变特异性IDH1 / 2免疫组织化学（IHC）在SNUC和形态学模拟物的大型多机构队列中的效用。对193例鼻窦肿瘤包括：53例SNUCs，8例低分化癌（PDCARs）和132例组织学模拟物进行了使用多特异性抗体用于IDH1 / 2（R132 / R172）突变蛋白的IHC。突变IDH1 / 2 IHC在26/53 SNUC中为阳性（49％; 20强阳性和6弱）和3/8 PDCAR（37.5％; 2强; 1弱），但在所有其他肿瘤类型（0/132 ）。针对SNUC / PDCAR亚组（6个强和3个IDH1 / 2 IHC弱阳性; 7个阴性）的靶向下一代测序（NGS）显示频繁的IDH2 R172X突变（10/16）和单个IDH1 R132C突变。所有6例强阳性突变IDH1 / 2染色和NGS均有IDH2 R172S / G突变。 3例IHC弱病例均有IDH2 R172T突变。在突变IDH1 / 2 IHC阴性的7个测试病例中，NGS检测到1例各自具有IDH2 R172T和IDH1 R132C突变。 IDH突变型癌在TP53（55％）中也有频繁突变，并且在KIT（45％）或PI3K途径（36％）中激活突变。突变特异性IDH1 / 2 IHC识别SNUC中的IDH突变，然而，它缺乏对全部IDH突变的敏感性。这些发现表明，IDH突变型鼻腔鼻窦癌可能是一种独特的病理生物学实体，具有治疗意义，可通过多特异性IDH1 / 2 IHC和测序的组合方法进行鉴定。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Am. J. Surg. Pathol.">
                Journal: Am. J. Surg. Pathol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29683816" target="_blank">
                PMID: 29683816
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Mito JK, Bishop JA, Sadow PM, Stelow EB, Faquin WC, Mills SE, Krane JF, French CA, Fletcher CDM, Hornick JL, Sholl LM, Jo VY">
                Authors: Mito JK, Bishop J...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29683816">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>68. Targeted Next-generation Sequencing and Bioinformatics Pipeline to Evaluate Genetic Determinants of Constitutional Disease.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29683450" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29683450" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29683450">
                    <p>Next-generation sequencing (NGS) is quickly revolutionizing how research into the genetic determinants of constitutional disease is performed. The technique is highly efficient with millions of sequencing reads being produced in a short time span and at relatively low cost. Specifically, targeted NGS is able to focus investigations to genomic regions of particular interest based on the disease of study. Not only does this further reduce costs and increase the speed of the process, but it lessens the computational burden that often accompanies NGS. Although targeted NGS is restricted to certain regions of the genome, preventing identification of potential novel loci of interest, it can be an excellent technique when faced with a phenotypically and genetically heterogeneous disease, for which there are previously known genetic associations. Because of the complex nature of the sequencing technique, it is important to closely adhere to protocols and methodologies in order to achieve sequencing reads of high coverage and quality. Further, once sequencing reads are obtained, a sophisticated bioinformatics workflow is utilized to accurately map reads to a reference genome, to call variants, and to ensure the variants pass quality metrics. Variants must also be annotated and curated based on their clinical significance, which can be standardized by applying the American College of Medical Genetics and Genomics Pathogenicity Guidelines. The methods presented herein will display the steps involved in generating and analyzing NGS data from a targeted sequencing panel, using the ONDRISeq neurodegenerative disease panel as a model, to identify variants that may be of clinical significance.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29683450">
                    <p>新一代测序（NGS）正在迅速革新如何研究构成性疾病的遗传决定因素。该技术非常高效，数百万次测序读数在短时间内以相对较低的成本生产。具体来说，有针对性的NGS能够根据研究疾病将调查集中在特别感兴趣的基因组区域。这不仅进一步降低了成本并提高了流程速度，而且减轻了NGS常常伴随的计算负担。虽然有针对性的NGS仅限于基因组的某些区域，从而阻止了潜在的新兴感兴趣基因座的识别，但当面对表型和遗传异质性疾病时，它可能是一种很好的技术，对此之前已知有遗传相关性。由于测序技术的复杂性，为了实现高覆盖率和高质量的测序读数，密切遵守方案和方法非常重要。此外，一旦获得测序读数，就使用复杂的生物信息学工作流程来精确地将读数映射到参考基因组，以调用变体，并确保变体通过质量度量。变异体还必须根据它们的临床意义进行注释和策划，这可以通过应用美国医学遗传学和基因组学致病性指南来标准化。本文介绍的方法将显示使用ONDRISeq神经变性疾病小组作为模型生成和分析来自靶向测序小组的NGS数据所涉及的步骤，以鉴定可能具有临床意义的变体。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Vis Exp">
                Journal: J Vis Exp [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29683450" target="_blank">
                PMID: 29683450
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-04
            </a>

            <a class="btn btn-sm btn-danger" title="Dilliott AA, Farhan SMK, Ghani M, Sato C, Liang E, Zhang M, McIntyre AD, Cao H, Racacho L, Robinson JF, Strong MJ, Masellis M, Bulman DE, Rogaeva E, Lang A, Tartaglia C, Finger E, Zinman L, Turnbull J, Freedman M, Swartz R, Black SE, Hegele RA">
                Authors: Dilliott AA, Farh...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29683450">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>69. Clinical experience with multigene carrier panels in the reproductive setting.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29683194" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29683194" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29683194">
                    <p>Expanded carrier testing is acknowledged as an acceptable strategy for carrier testing by the American College of Obstetrics and Gynecology. Limited studies have investigated positivity rates of expanded carrier panels. We describe our experience with three commercial laboratory panels varying in size from 3 to 218 disorders.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29683194">
                    <p>扩大的运营商测试被美国妇产科学院承认为运营商测试的可接受策略。有限的研究调查了扩展载体板的阳性率。我们描述了我们的经验与三个商业实验室面板大小从3到218混乱。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Prenat. Diagn.">
                Journal: Prenat. Diagn. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29683194" target="_blank">
                PMID: 29683194
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-23
            </a>

            <a class="btn btn-sm btn-danger" title="Terhaar C, Teed N, Allen R, Dohany L, Settler C, Holland C, Longman RE">
                Authors: Terhaar C, Teed N...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29683194">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>70. Improving the detection of patients with inherited predispositions to hematologic malignancies using next-generation sequencing-based leukemia prognostication panels.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29682723" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29682723" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29682723">
                    <p>Recognizing and referring patients with possible inherited cancer predisposition syndromes for appropriate genetic evaluation and testing provides insights into optimal patient treatment approaches and also can provide education and testing opportunities for family members. Next-generation sequencing (NGS)-based, targeted genotyping for somatic mutations is increasingly used in the diagnosis, prognostication, and treatment selection for patients with hematologic malignancies. However, certain mutations that may be somatically acquired can also be present as germline mutations in some individuals and families. Whether the results of NGS-based leukemia panels can be used to inform decisions and subsequent evaluation of patients with possible inherited cancer predispositions has not been described previously. Because a normal control often is not available when using NGS panels in patients with hematologic malignancies, NGS panel results offer both an opportunity and a challenge to determine the origin and pathogenicity of identified mutations. In the absence of a matched germline control, variant allele frequency (VAF) estimation and data from publically available data sets provide important clues to the possible germline origin of a variant. Careful annotation and review of NGS panels in patients with hematologic malignancies can provide a useful screening tool to systematically improve upon the detection of potentially pathogenic germline variants. Cancer 2018. © 2018 American Cancer Society.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29682723">
                    <p>认识并转诊可能遗传的癌症易感性综合征患者进行适当的遗传评估和测试，可以为最佳患者治疗方法提供见解，并且可以为家庭成员提供教育和检测机会。基于新一代测序（NGS）的体细胞突变靶向基因分型越来越多地用于血液恶性肿瘤患者的诊断，预后和治疗选择。然而，某些个体和家族中也可能存在某些可能是体细胞获得的突变，作为种系突变。先前没有描述基于NGS的白血病专家组的结果是否可用于通知决定并随后评估可能遗传的癌症易感性患者。由于在恶性血液病患者中使用NGS评分时通常无法获得正常的对照，因此NGS专家组结果为确定已确定突变的起源和致病性提供了机会和挑战。在缺乏匹配的种系对照的情况下，变异等位基因频率（VAF）估计和来自公开可用数据集的数据提供了变体可能的种系起源的重要线索。对血液恶性肿瘤患者的NGS评估小组进行仔细的注释和评估可以提供有用的筛查工具，系统地改善潜在致病性种系变异的检测。癌症2018.©2018年美国癌症协会。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer">
                Journal: Cancer [IF=6.072]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29682723" target="_blank">
                PMID: 29682723
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-06
            </a>

            <a class="btn btn-sm btn-danger" title="DiNardo CD, Routbort MJ, Bannon SA, Benton CB, Takahashi K, Kornblau SM, Luthra R, Kanagal-Shamanna R, Medeiros LJ, Garcia-Manero G, Kantarjian H, Futreal PA, Meric-Bernstam F, Patel KP">
                Authors: DiNardo CD, Routb...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29682723">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>71. Myelodysplastic Syndrome/Acute Myeloid Leukemia Arising in Idiopathic Erythrocytosis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29682367" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29682367" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29682367">
                    <p>The term &quot;idiopathic erythrocytosis (IE)&quot; is applied to those cases where a causal clinical or pathological event cannot be elucidated and likely reflects a spectrum of underlying medical and molecular abnormalities. The clinical course of a patient with IE is described manifesting as a persistent erythrocytosis with a low serum erythropoietin level, mild eosinophilia, and with evidence of a thrombotic event. The patient subsequently developed a myelodysplasic syndrome (MDS) and acute myeloid leukemia (AML), an event not observed in erythrocytosis patients other than those with polycythemia vera (PV). Application of a next-generation sequencing (NGS) approach targeted for myeloid malignancies confirmed wild-type JAK2 exons 12-15 and identified a common SH2B3 W262R single-nucleotide polymorphism associated with the development of hematological features of myeloproliferative neoplasms (MPNs). Further NGS analysis detected a CBL L380P mutated clone expanding in parallel with the development of MDS and subsequent AML. Despite the absence of JAK2, MPL exon 10, or CALR exon 9 mutations, a similarity with the disease course of PV/MPN was evident. A clonal link between the erythrocytosis and AML could be neither confirmed nor excluded. Future molecular identification of the mechanisms underlying IE is likely to provide a more refined therapeutic approach.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29682367">
                    <p>术语“特发性红细胞增多症（IE）”适用于那些不能阐明因果临床或病理事件并可能反映潜在医学和分子异常谱的病例。具有IE的患者的临床过程被描述为表现为持续性红细胞增多症，血清促红细胞生成素水平低，嗜酸性粒细胞增多轻，并且有血栓事件的证据。该患者随后发展出骨髓增生异常综合征（MDS）和急性髓性白血病（AML），这是红细胞增多症患者（真性红细胞增多症患者除外）未观察到的事件。应用针对骨髓恶性肿瘤的新一代测序（NGS）方法证实了野生型JAK2外显子12-15，并鉴定了与骨髓增殖性肿瘤（MPN）的血液学特征的发展相关的常见SH2B3 W262R单核苷酸多态性。进一步的NGS分析检测到CBL L380P突变克隆与MDS和随后的AML的发展并行地扩大。尽管缺乏JAK2，MPL外显子10或CALR外显子9突变，但与PV / MPN疾病过程的相似性是明显的。红细胞增多症和AML之间的克隆联系既不能被确认也不能被排除。未来对IE基础机制的分子鉴定可能提供更精确的治疗方法。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Case Rep Hematol">
                Journal: Case Rep Hematol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29682367" target="_blank">
                PMID: 29682367
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Langabeer SE, Conneally E, Flynn CM">
                Authors: Langabeer SE, Con...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5842720" target="_blank">
                PMC5842720
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29682367">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>72. Recent progress in whole genome sequencing, high-density linkage maps, and genomic databases of ornamental plants.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29681748" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29681748" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29681748">
                    <p>Genome information is useful for functional analysis of genes, comparative genomic analysis, breeding of new varieties by marker-assisted selection, and map-based gene isolation. Genome-related research in ornamentals plants has been relatively slow to develop because of their heterozygosity or polyploidy. Advances in analytical instruments, such as next-generation sequencers and information processing technologies have revolutionized biology, and have been applied in a large number and variety of species, including ornamental plants. Recently, high-quality whole genome sequences have been reported in plant genetics and physiology studies of model ornamentals, such as those in genus Petunia and Japanese morning glory (Ipomoea nil). In this review, whole genome sequencing and construction of high-density genetic linkage maps based on SNP markers of ornamentals will be discussed. The databases that store this information for ornamentals are also described.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29681748">
                    <p>基因组信息对于基因功能分析，比较基因组分析，通过标记辅助选择育种新品种以及基于图谱的基因分离是有用的。观赏植物中的基因组相关研究由于其杂合性或多倍性而相对较慢地发展。诸如下一代测序仪和信息处理技术等分析仪器的进步已经使生物学发生了革命性的变化，并且已经应用​​于大量多种物种，包括观赏植物。最近，在模式观赏植物的植物遗传学和生理学研究中已经报道了高质量的全基因组序列，例如矮牵牛属和日本牵牛花属（Ipomoea nil）。在此综述中，将讨论基于SNP标记的全基因组测序和高密度遗传连锁图的构建。还描述了存储用于装饰物的这些信息的数据库。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Breed. Sci.">
                Journal: Breed. Sci. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29681748" target="_blank">
                PMID: 29681748
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jan
            </a>

            <a class="btn btn-sm btn-danger" title="Yagi M">
                Authors: Yagi M
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5903975" target="_blank">
                PMC5903975
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29681748">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>73. Prevalence and Spectrum of BRCA1/2 Germline Mutations in Women with Breast Cancer in China Based on Next-Generation Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29681614" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29681614" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29681614">
                    <p>BACKGROUND BRCA1 and BRCA2 (BRCA1/2) play important roles in the development of breast cancer, but information regarding BRCA1/2 mutations in Chinese females remains limited. The aim of this study was to investigate the prevalence and spectrum of BRCA1/2 mutations in China. MATERIAL AND METHODS In total, 595 breast cancer patients in China were screened with an amplicon-based panel for the detection of BRCA1/2 mutations in coding regions using next-generation sequencing (NGS) with a Personal Genome Machine. Every pathogenic mutation detected was confirmed by Sanger sequencing. The disease-causing potential of variants of uncertain significance (VUS) was predicted using PolyPhen-2, SIFT, PhyloP, and Grantham. RESULTS The prevalence of BRCA1/2 mutations was 8.07% in the Chinese population. Forty-two pathogenic mutations were identified in 48 cases (17 BRCA1 cases and 31 BRCA2 cases), including 19 novel mutations. Nine VUS were predicted to be deleterious by PolyPhen-2 and SIFT and subsequently predicted by PhyloP and Grantham for the evolutionary conservation. CONCLUSIONS These results suggest that NGS is useful as a rapid, high-throughput, and cost-effective screening tool for the analysis of BRCA1/2 mutations. Based on this panel, we found that BRCA1/2 germline mutations in China exhibit distinct characteristics compared to those in Western populations.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29681614">
                    <p>背景BRCA1和BRCA2（BRCA1 / 2）在乳腺癌的发展中发挥重要作用，但有关中国女性BRCA1 / 2突变的信息仍然有限。本研究的目的是调查中国BRCA1 / 2突变的发病率和频谱。材料与方法总共有595名中国乳腺癌患者使用基于扩增子的小组进行筛查，该小组使用个人基因组测序仪检测编码区BRCA1 / 2突变，并使用下一代测序（NGS）。 Sanger测序证实所有检测到的致病突变。使用PolyPhen-2，SIFT，PhyloP和Grantham预测了不确定意义变体（VUS）的致病潜力。结果中国人BRCA1 / 2突变的患病率为8.07％。在48例（17例BRCA1例和31例BRCA2例）中共鉴定出42种致病性突变，其中包括19种新型突变。预测9种VUS会对PolyPhen-2和SIFT有害，并随后由PhyloP和Grantham预测进化保守性。结论这些结果表明，NGS可用作BRCA1 / 2突变分析的快速，高通量和高成本效益的筛查工具。基于这个小组，我们发现中国的BRCA1 / 2种系突变与西方种群相比具有明显的特征。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Med. Sci. Monit.">
                Journal: Med. Sci. Monit. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29681614" target="_blank">
                PMID: 29681614
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-23
            </a>

            <a class="btn btn-sm btn-danger" title="Liang Y, Yang X, Li H, Zhu A, Guo Z, Li M">
                Authors: Liang Y, Yang X, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29681614">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>74. Characterizing the ecology of Vibrio in the Neuse River Estuary, North Carolina using heat shock protein 60 (hsp60) next-generation amplicon sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29678912" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29678912" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29678912">
                    <p>Of marine eubacteria, the genus Vibrio is intriguing because member species are relevant to both marine ecology and human health. Many have touted the relationships of Vibrio to environmental factors, especially temperature and salinity, to predict total Vibrio abundance, but these studies lacked the taxonomic resolution to identify relationships among species and key drivers of Vibrio dynamics. In order to improve NGS surveys of Vibrio, we have conducted both 16S small subunit ribosomal RNA (rRNA) and heat shock protein 60 (hsp60) amplicon sequencing of water samples collected at two well-studied locations in the Neuse River Estuary, NC. Samples were collected between May and December 2016 with enhanced sampling efforts in response to two named storms. Using hsp60 sequences, 21 Vibrio species were identified, including the potential human pathogens V. cholerae, V. parahaemolyticus, and V. vulnificus. Changes in the Vibrio community mirrored seasonal and storm-related changes in the water column, especially in response to an influx of nutrient-rich freshwater to the estuary after Hurricane Matthew which initiated dramatic changes in the overall Vibrio community. Individual species dynamics were wide-ranging, indicating that individual Vibrio taxa have unique ecologies and that total Vibrio predictors are insufficient for risk assessments of potentially pathogenic species. Positive relationships between Vibrio, dinoflagellates, and Cyanobacteria were identified, as were intra-species associations which further illuminated the interactions of co-occurring Vibrio taxa along environmental gradients.IMPORTANCE The objective of this research was to utilize a novel approach to improve sequence-based surveys of Vibrio communities and to demonstrate the usefulness of this approach by presenting an analysis of Vibrio dynamics in context of environmental conditions, with a particular focus on species which cause disease in humans and storm effects. The methods presented here allowed for the analysis of Vibrio dynamics with excellent taxonomic resolution, and could be incorporated into future ecological studies and risk prediction strategies for potentially pathogenic species. Next-generation sequencing of hsp60 and other innovative sequence-based approaches are valuable tools and show great promise for studying Vibrio ecology and associated public health risks.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29678912">
                    <p>在海洋真细菌中，弧菌属很吸引人，因为成员物种与海洋生态和人类健康都有关系。许多人已经吹捧了弧菌与环境因素（特别是温度和盐度）之间的关系，以预测总弧菌丰度，但是这些研究缺乏分类学的方法来鉴定物种之间的关系以及弧菌动力学的关键驱动因素。为了改善对弧菌的NGS调查，我们对位于北卡罗来纳州Neuse河河口两个研究区域的水样进行了16S小亚基核糖体RNA（rRNA）和热休克蛋白60（hsp60）扩增测序。 2016年5月至12月期间收集了样本，并针对两次指定的风暴加强了抽样工作。使用hsp60序列，鉴定了21种弧菌，包括潜在的人类病原体霍乱弧菌，副溶血性弧菌和创伤弧菌。弧菌群的变化反映了水体中季节性和风暴相关的变化，特别是在飓风Matthew飓风引发整个弧菌群发生剧烈变化后，营养丰富的淡水涌入河口的反应。个体物种动态范围广泛，表明单个弧菌分类群具有独特的生态学特征，并且总的弧菌预测因子不足以用于潜在致病物种的风险评估。确定了弧菌，双鞭毛藻和蓝藻之间的正相关关系，以及进一步阐明了共生弧菌分类群沿环境梯度的相互作用的种内关联。研究目标本研究的目的是利用一种新方法来改进基于序列的通过对环境条件下的弧菌动态进行分析，特别关注引起人类疾病和风暴影响的物种，从而证明这种方法的有效性。这里介绍的方法允许以极好的分类学分辨率分析弧菌动态，并且可以将其纳入未来的生态学研究和潜在致病物种的风险预测策略中。 hsp60的新一代测序和其他创新的基于序列的方法是有价值的工具，并且对研究弧菌生态学和相关的公共健康风险显示出巨大的希望。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Appl. Environ. Microbiol.">
                Journal: Appl. Environ. Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29678912" target="_blank">
                PMID: 29678912
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Jesser KJ, Noble RT">
                Authors: Jesser KJ, Noble RT
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29678912">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>75. Determinants of Health-Related Quality of Life 1 Year after Pulmonary Thromboendarterectomy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29678648" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29678648" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29678648">
                    <p>Pulmonary thromboendarterectomy (PTE) is a recognized treatment for patients with chronic thromboembolic pulmonary hypertension (CTEPH), with good long-term haemodynamic results. The aim of this study was to test the hypothesis that the dynamics of health-related quality of life (QoL), 1 year after PTE, depends both on the clinical determinants associated with the underlying disease and the adverse events occurring in the early postoperative period.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29678648">
                    <p>肺血栓内膜剥脱术（PTE）是慢性血栓栓塞性肺动脉高压（CTEPH）患者的公认治疗方法，具有良好的长期血流动力学结果。这项研究的目的是为了检验这样的假设，即PTE后1年的健康相关生活质量（QoL）的动态取决于与潜在疾病相关的临床决定因素和术后早期发生的不良事件。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ann Vasc Surg">
                Journal: Ann Vasc Surg [IF=1.145]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29678648" target="_blank">
                PMID: 29678648
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Kamenskaya O, Klinkova A, Loginova I, Chernyavskiy A, Edemskiy A, Lomivorotov VV, Karaskov A">
                Authors: Kamenskaya O, Kli...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29678648">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>76. What Does This Mutation Mean? The Tools and Pitfalls of Variant Interpretation in Lymphoid Malignancies.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29677173" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29677173" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29677173">
                    <p>High throughput sequencing (HTS) is increasingly important in determining cancer diagnoses, with subsequent prognostic and therapeutic implications. The biology of cancer is becoming increasingly deciphered and it is clear that therapy needs to be individually tailored. Whilst translational research plays an important role in lymphoid malignancies, few guidelines exist to guide biologists and routine laboratories through this constantly evolving field. In this article, we review the challenges of interpreting HTS in lymphoid malignancies and provide a toolkit to interpret single nucleotide variants obtained from HTS. We define the pre-analytical issues such as sequencing DNA obtained from formalin-fixed and paraffin-embedded tissue (FFPE), the acquisition of germline DNA, or the bioinformatic pitfalls, the analytical issues encountered and how to manage them. We describe the main constitutional and cancer databases, their characteristics and limitations, with an emphasis on variant interpretation in lymphoid malignancies. Finally, we discuss the challenges of predictions that one can make using in silico or in vitro modelling, pharmacogenomic screening, and the limits of those prediction tools. This description of the current status in genomic interpretation highlights the need for new large databases and international collaboration in the lymphoma field.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29677173">
                    <p>高通量测序（HTS）在确定癌症诊断方面越来越重要，随后的预后和治疗影响。癌症的生物学正在变得越来越破译，很明显，治疗需要单独定制。虽然平移研究在淋巴恶性肿瘤中发挥重要作用，但是通过这个不断发展的领域，很少有指导指导生物学家和常规实验室。在本文中，我们回顾了在淋巴恶性肿瘤中解释HTS的挑战，并提供了解释从HTS获得的单核苷酸变体的工具包。我们定义了预分析问题，例如从福尔马林固定和石蜡包埋组织（FFPE）获得的DNA测序，获取生殖细胞DNA或生物信息缺陷，遇到的分析问题以及如何管理它们。我们描述了主要的宪法和癌症数据库及其特点和局限性，重点介绍了淋巴恶性肿瘤中的变异解读。最后，我们讨论人们可以用硅片或体外模型，药物基因组筛查以及这些预测工具的局限性进行预测的挑战。对基因组解读当前状况的这种描述突出了在淋巴瘤领域需要新的大型数据库和国际合作。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Mol Sci">
                Journal: Int J Mol Sci [IF=3.226]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29677173" target="_blank">
                PMID: 29677173
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-20
            </a>

            <a class="btn btn-sm btn-danger" title="Yann G, Jonathan L, Kaddour C, Sandrine H, Claire B, Gilles S, Pierre S, Sarah H">
                Authors: Yann G, Jonathan ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29677173">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>77. Interest of next-generation sequencing in BCG-treated high-risk bladder cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29673906" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29673906" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29673906">
                    <p>There are only few predictive factors for response of non-musculo-invasive bladder cancer (NMIBC) to Bacillus Calmette-Guérin (BCG) therapy. Our study analyzed the results of the sequencing of new generation (NGS) targeted on 50 genes of oncological interest obtained on bladder resection parts in high-risk NMIBC patients treated with BCG, to describe this population from a molecular point of view and try to correlate these results in patients who present or not recurrence after BCG.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29673906">
                    <p>非肌层浸润性膀胱癌（NMIBC）对卡介苗（BCG）治疗的反应仅有少数预测因素。我们的研究分析了以BCG治疗的高危NMIBC患者的膀胱切除部位获得的50个肿瘤感兴趣基因为目标的新一代（NGS）测序结果，以从分子角度描述该群体并试图关联这些结果表明在BCG后存在或不存在复发的患者。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Prog. Urol.">
                Journal: Prog. Urol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29673906" target="_blank">
                PMID: 29673906
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-16
            </a>

            <a class="btn btn-sm btn-danger" title="Jungels C, Martinez Chanza N, Albisinni S, Mercier M, d&#39;Haene N, Rorive S, Roumeguère T">
                Authors: Jungels C, Martin...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29673906">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>78. A study on fast calling variants from next-generation sequencing data using decision tree.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29673316" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29673316" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29673316">
                    <p>The rapid development of next-generation sequencing (NGS) technology has continuously been refreshing the throughput of sequencing data. However, due to the lack of a smart tool that is both fast and accurate, the analysis task for NGS data, especially those with low-coverage, remains challenging.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29673316">
                    <p>下一代测序（NGS）技术的快速发展不断刷新测序数据的吞吐量。然而，由于缺乏快速准确的智能工具，NGS数据的分析任务，尤其是低覆盖率的数据仍然具有挑战性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Bioinformatics">
                Journal: BMC Bioinformatics [IF=2.448]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29673316" target="_blank">
                PMID: 29673316
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-19
            </a>

            <a class="btn btn-sm btn-danger" title="Li Z, Wang Y, Wang F">
                Authors: Li Z, Wang Y, Wang F
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5907718" target="_blank">
                PMC5907718
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29673316">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>79. A MinION-based pipeline for fast and cost-effective DNA barcoding.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29673082" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29673082" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29673082">
                    <p>DNA barcodes are useful for species discovery and species identification, but obtaining barcodes currently requires a well-equipped molecular laboratory, is time-consuming, and/or expensive. We here address these issues by developing a barcoding pipeline for Oxford Nanopore MinION™ and demonstrate that one flowcell can generate barcodes for ~500 specimens despite the high base-call error rates of MinION™ reads. The pipeline overcomes these errors by first summarizing all reads for the same tagged amplicon as a consensus barcode. Consensus barcodes are overall mismatch-free but retain indel errors that are concentrated in homopolymeric regions. They are addressed with an optional error correction pipeline that are corrected based on conserved amino-acid motifs from publicly available barcodes. The effectiveness of this pipeline is documented by analysing reads from three MinION™ runs that represent three different stages of MinION™ development. They generated data for (1) 511 specimens of a mixed Diptera sample, (2) 575 specimens of ants, and (3) 50 specimens of Chironomidae. The run based on the latest chemistry yielded MinION barcodes for 490 of the 511 specimens which were assessed against reference Sanger barcodes (N=471). Overall, the MinION barcodes have an accuracy of 99.3%-100% with the number of ambiguous bases after correction ranging from &lt;0.01-1.5% depending on which correction pipeline is used. We demonstrate that it requires ~2 hours of sequencing to gather all information needed for obtaining reliable barcodes for most specimens (&gt;90%). We estimate that up to 1000 barcodes can be generated in one flowcell and that the cost per barcode can be &lt;USD 2. This article is protected by copyright. All rights reserved.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29673082">
                    <p>DNA条形码可用于物种发现和物种鉴定，但目前获取条形码需要设备良好的分子实验室，费时且/或昂贵。我们在这里通过为Oxford Nanopore MinION™开发条形码流水线来解决这些问题，并证明尽管MinION™读数的碱基错误率很高，但一个流动池可以生成约500个样品的条形码。该流水线通过首先将同一标记的扩增子的所有读数总结为共识条形码来克服这些错误。共有条形码总体上不具有错配，但保留集中在均聚区域的indel错误。它们通过可选的纠错管道进行处理，这些管道根据公开可用条形码的保守氨基酸基序进行校正。通过分析代表MinION™开发的三个不同阶段的三个MinION™运行的读取，可以记录此管道的有效性。他们为（1）混合双翅目样本的511个样本，（2）575个蚂蚁样本，和（3）50个摇蚊科样本产生了数据。根据最新的化学分析得出的Minion条形码为511份标本中的490份进行了评估，参考Sanger条形码（N = 471）。总的来说，MinION条形码的精确度为99.3％-100％，修正后的歧义基数为&lt;0.01-1.5％，具体取决于使用的校正流水线。我们证明它需要约2小时的测序来收集获得大多数标本（&gt; 90％）可靠条形码所需的全部信息。我们估计在一个流通池中可以生成多达1000个条码，并且每个条码的成本可能&lt;2美元。本文受版权保护。版权所有。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Mol Ecol Resour">
                Journal: Mol Ecol Resour [IF=7.332]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29673082" target="_blank">
                PMID: 29673082
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-19
            </a>

            <a class="btn btn-sm btn-danger" title="Srivathsan A, Baloğlu B, Wang W, Tan WX, Bertrand D, Ng AHQ, Boey EJH, Koh JJY, Nagarajan N, Meier R">
                Authors: Srivathsan A, Bal...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29673082">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>80. TRUmiCount: Correctly counting absolute numbers of molecules using unique molecular identifiers.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29672674" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29672674" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29672674">
                    <p>Counting molecules using next-generation sequencing (NGS) suffers from PCR amplification bias, which reduces the accuracy of many quantitative NGS-based experimental methods such as RNASeq. This is true even if molecules are made distinguishable using unique molecular identifiers (UMIs) before PCR amplification, and distinct UMIs are counted instead of reads: Molecules that are lost entirely during the sequencing process will still cause underestimation of the molecule count, and amplification artifacts like PCR chimeras create phantom UMIs and thus cause over-estimation.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29672674">
                    <p>使用新一代测序（NGS）计数分子会受到PCR扩增偏倚的影响，这会降低许多基于定量NGS的实验方法（如RNASeq）的准确性。即使在PCR扩增之前使用独特的分子标识符（UMI）使得分子可区分，并且不同的UMI被计数而不是读数：即使在测序过程中完全丢失的分子仍然会导致分子计数低估，并且放大假象像PCR嵌合体创建幻影UMIs，从而导致高估。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Bioinformatics">
                Journal: Bioinformatics [IF=7.307]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29672674" target="_blank">
                PMID: 29672674
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-16
            </a>

            <a class="btn btn-sm btn-danger" title="Pflug FG, von Haeseler A">
                Authors: Pflug FG, von Hae...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29672674">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>81. Rucaparib: a new treatment option for ovarian cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29672168" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29672168" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29672168">
                    <p>Approximately 50% of high-grade serous ovarian cancers present a deficiency in the pathways involved in homologous recombination (HR). PARP inhibitors prevent single-strand DNA damage repair and determine a progression of the defect towards double-strand breaks, which results in a process known as &#39;synthetic lethality&#39;. Areas covered: In this review, the authors discuss the efficacy and toxicity of rucaparib either as a single agent or as a maintenance treatment for ovarian cancer. This includes the NGS Foundation Medicine evaluation of the role of this drug in the treatment algorithm of ovarian cancer. Moreover, perspectives on the future development of this drug are presented. Expert opinion: The FDA has approved this drug for the treatment of recurrent BRCA-mutated ovarian cancers, which were previously treated with at least two chemotherapies and has accepted the supplemental new drug application for maintenance use in patients who respond to platinum-based chemotherapy via the Prescription Drug User Fee Act (PDUFA) on 6 April 2018. European Medicines Agency (EMA) approval in the same setting is awaited. The possibility of using PARP inhibitors as a maintenance therapy, as a front-line therapy to combat recurrence, and in combination with anti-angiogenic agents and immune-therapies appears to be of particular interest.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29672168">
                    <p>大约50％的高级浆液性卵巢癌在涉及同源重组（HR）的途径中存在缺陷。 PARP抑制剂阻止单链DNA损伤修复并确定缺陷向双链断裂的进展，这导致称为“合成致死性”的过程。覆盖范围：在这篇综述中，作者讨论了rucaparib作为单一药物或作为卵巢癌的维持治疗的功效和毒性。这包括NGS基金会医学评估这种药物在卵巢癌治疗算法中的作用。此外，还介绍了这种药物未来发展的观点。专家意见：FDA已批准用于治疗复发性BRCA突变卵巢癌的药物，这种药物以前曾接受过至少两种化疗药物治疗，并接受了补充性新药申请，用于对通过铂类化疗作出反应的患者的维持治疗2018年4月6日的“处方药使用者费用法”（PDUFA）。欧洲药品管理局（EMA）在同一环境下的批准正在等待中。使用PARP抑制剂作为维持治疗的可能性，作为抗复发的一线疗法，以及与抗血管生成剂和免疫疗法的组合似乎特别令人感兴趣。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Expert Opin Pharmacother">
                Journal: Expert Opin Pharmacother [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29672168" target="_blank">
                PMID: 29672168
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Sabatucci I, Maltese G, Lepori S, Tripodi E, Bogani G, Lorusso D">
                Authors: Sabatucci I, Malt...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29672168">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>82. Identification of a cold-tolerant locus in rice (Oryza sativa L.) using bulked segregant analysis with a next-generation sequencing strategy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29671148" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29671148" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29671148">
                    <p>Cold stress can cause serious abiotic damage that limits the growth, development and yield of rice. Cold tolerance during the booting stage of rice is a key factor that can guarantee a high and stable yield under cold stress. The cold tolerance of rice is controlled by quantitative trait loci (QTLs). Based on the complex genetic basis of cold tolerance in rice, additional efforts are needed to detect reliable QTLs and identify candidate genes. In this study, recombinant inbred lines (RILs) derived from a cross between a cold sensitive variety, Dongnong422, and strongly cold-tolerant variety, Kongyu131, were used to screen for cold-tolerant loci at the booting stage of rice.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29671148">
                    <p>冷胁迫会导致严重的非生物损害，限制水稻的生长，发育和产量。水稻孕穗期耐冷性是保证冷胁迫下高产稳产的关键因素。水稻的耐冷性由数量性状基因座（QTL）控制。基于水稻耐冷性的复杂遗传基础，需要做更多的工作来检测可靠的QTL并鉴定候选基因。在本研究中，利用冷敏感品种东农422和抗冷性强的品种Kongyu131杂交获得的重组自交系RILs在水稻孕穗期进行抗寒基因筛选。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Rice (N Y)">
                Journal: Rice (N Y) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29671148" target="_blank">
                PMID: 29671148
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-18
            </a>

            <a class="btn btn-sm btn-danger" title="Sun J, Yang L, Wang J, Liu H, Zheng H, Xie D, Zhang M, Feng M, Jia Y, Zhao H, Zou D">
                Authors: Sun J, Yang L, Wa...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5906412" target="_blank">
                PMC5906412
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29671148">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>83. Aneuploid CTC and CEC.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29670052" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29670052" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29670052">
                    <p>Conventional circulating tumor cell (CTC) detection technologies are restricted to large tumor cells (&gt; white blood cells (WBCs)), or those unique carcinoma cells with double positive expression of surface epithelial cell adhesion molecule (EpCAM) for isolation, and intracellular structural protein cytokeratins (CKs) for identification. With respect to detecting the full spectrum of highly heterogeneous circulating rare cells (CRCs), including CTCs and circulating endothelial cells (CECs), it is imperative to develop a strategy systematically coordinating all tri-elements of nucleic acids, biomarker proteins, and cellular morphology, to effectively enrich and comprehensively identify CRCs. Accordingly, a novel strategy integrating subtraction enrichment and immunostaining-fluorescence in situ hybridization (SE-iFISH), independent of cell size variation and free of hypotonic damage as well as anti-EpCAM perturbing, has been demonstrated to enable in situ phenotyping multi-protein expression, karyotyping chromosome aneuploidy, and detecting cytogenetic rearrangements of the ALK gene in non-hematologic CRCs. Symbolic non-synonymous single nucleotide variants (SNVs) of both the TP53 gene (P33R) in each single aneuploid CTCs, and the cyclin-dependent kinase inhibitor 2A (CDKN2A) tumor suppressor gene in each examined aneuploid CECs, were identified for the first time across patients with diverse carcinomas. Comprehensive co-detecting observable aneuploid CTCs and CECs by SE-iFISH, along with applicable genomic and/or proteomic single cell molecular profiling, are anticipated to facilitate elucidating how those disparate categories of aneuploid CTCs and CECs cross-talk and functionally interplay with tumor angiogenesis, therapeutic drug resistance, tumor progression, and cancer metastasis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29670052">
                    <p>传统的循环肿瘤细胞（CTC）检测技术仅限于大肿瘤细胞（&gt;白细胞（WBC）），或具有表达上皮细胞粘附分子（EpCAM）的双重阳性表达的独特癌细胞用于分离，而细胞内结构蛋白细胞角蛋白（CK）进行鉴定。关于检测包括CTC和循环内皮细胞（CEC）在内的高度异质性循环稀有细胞（CRC）的全谱图，必须制定一个系统地协调核酸，生物标志物蛋白质和细胞形态学中所有三元素的策略，有效地丰富和全面识别社保资金。因此，已经证明了一种集消减富集和免疫染色 - 荧光原位杂交（SE-iFISH）于一体的独立于细胞大小变化且无低渗损伤以及抗EpCAM干扰的新策略能够实现原位表型多蛋白表达，核型分析染色体非整倍体，以及检测ALK基因在非血液学CRC中的细胞遗传学重排。首次鉴定了每个单倍体非整倍体CTC中TP53基因（P33R）和每个检测到的非整倍体CEC中的细胞周期蛋白依赖性激酶抑制剂2A（CDKN2A）肿瘤抑制基因的符号非同义单核苷酸变体（SNV）跨越不同癌症的患者。预计通过SE-iFISH全面共检测可观察到的非整倍体CTC和CEC以及适用的基因组和/或蛋白质组单细胞分子谱，有助于阐明这些不同类别的非整倍性CTC和CEC串扰并与肿瘤血管生成功能性相互作用，治疗性药物抗性，肿瘤进展和癌症转移。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Diagnostics (Basel)">
                Journal: Diagnostics (Basel) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29670052" target="_blank">
                PMID: 29670052
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-18
            </a>

            <a class="btn btn-sm btn-danger" title="Lin PP">
                Authors: Lin PP
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29670052">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>84. GARFIELD-NGS: Genomic vARiants FIltering by dEep Learning moDels in NGS.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29668842" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29668842" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29668842">
                    <p>Exome sequencing approach is extensively used in research and diagnostic laboratories to discover pathological variants and study genetic architecture of human diseases. However, a significant proportion of identified genetic variants are actually false positive calls, and this pose serious challenges for variants interpretation. Here, we propose a new tool named GARFIELD-NGS (Genomic vARiants FIltering by dEep Learning moDels in NGS), which rely on deep learning models to dissect false and true variants in exome sequencing experiments performed with Illumina or ION platforms. GARFIELD-NGS showed strong performances for both SNP and INDEL variants (AUC 0.71 - 0.98) and outperformed established hard filters. The method is robust also at low coverage down to 30X and can be applied on data generated with the recent Illumina two-colour chemistry. GARFIELD-NGS processes standard VCF file and produces a regular VCF output. Thus, it can be easily integrated in existing analysis pipeline, allowing application of different thresholds based on desired level of sensitivity and specificity.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29668842">
                    <p>外显子组测序方法广泛用于研究和诊断实验室以发现病理变体并研究人类疾病的遗传结构。然而，确定的遗传变异体中有相当一部分实际上是假阳性，这对变异体解释提出了严峻的挑战。在这里，我们提出了一个名为GARFIELD-NGS（基因组变异在NGS中通过dEep学习模式筛选）的新工具，它依靠深度学习模型来分析Illumina或ION平台进行的外显子组测序实验中的假和真变体。 GARFIELD-NGS在SNP和INDEL变体（AUC 0.71  -  0.98）方面表现出强劲的表现，并且超越了既定的硬性过滤器。该方法在低至30X的低覆盖范围内也可靠，可应用于最近采用Illumina双色化学技术生成的数据。 GARFIELD-NGS处理标准VCF文件并生成常规VCF输出。因此，它可以很容易地集成到现有的分析流程中，允许根据所需的灵敏度和特异性水平应用不同的阈值。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Bioinformatics">
                Journal: Bioinformatics [IF=7.307]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29668842" target="_blank">
                PMID: 29668842
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-14
            </a>

            <a class="btn btn-sm btn-danger" title="Ravasio V, Ritelli M, Legati A, Giacopuzzi E">
                Authors: Ravasio V, Ritell...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29668842">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>85. The 6th Rare Disease South Eastern Europe (See) Meeting, Skopje, Macedonia (November 11th, 2017).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29668480" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29668480" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29668480">
                    <p>The sixth SEE meeting on rare diseases (RDs) was held in MASA the November 10th, 2017. A block of lectures on rare renal diseases started the meeting: nephrotic syndrome, Alport syndrome, atypical HUS, hypophosphatemic rickets, CAKUT were presented in all complexities. Their molecular and genetic mechanisms were discussed. The discovery of a dozen of newly genes in CAKUT, congenital overgrowth, spodilocostal dysplasia, precocious puberty has been done with collaboration of Macedonian and foreign researchers. NGS and other molecular methods in diagnosis of RDs have been presented by several presenters. The mitochondrial diseases, the novelties and importance of early discovery were comprehensively presented and discussed. The genetics and treatment of persistent neonatal hypoglcaemia were of special interest. Dysmorphic syndromes (Klippel Feil) were also presented. A session of oral electronic posters was reach and inspiring. Several lectures dealt with mucopolisaccaridoses, glycogen storage diseases and the possibilities for their diagnosis and treatment. Enzyme replacement treatement (ERT), its availability, effects (or the lack of it on the brain), intratecal ERT administration and further prospects of eventual gene treatment were comprehensively exposed and discussed. The main purposes of this traditional meeting are hopefully fulfilled: increased number of patients with RDs treated and cutting edge presentations got.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29668480">
                    <p>第六届SEE罕见疾病会议于2017年11月10日在MASA举行。关于罕见肾脏疾病的一系列讲座开始了会议：肾病综合征，Alport综合征，非典型HUS，低磷血症性r，病，CAKUT在所有复杂性。他们的分子和遗传机制进行了讨论。马其顿和外国研究人员合作完成了CAKUT中十几种新基因，先天性过度生长，spodilocostal发育不良，性早熟的发现。几位主持人已经提出了诊断RD的NGS和其他分子方法。全面介绍和讨论了线粒体疾病，早期发现的新颖性和重要性。持续性新生儿低血糖的遗传学和治疗特别令人感兴趣。还提出了畸形综合征（Klippel Feil）。一个口头电子海报的会议是达到和鼓舞人心的。有几次讲座涉及粘多糖病，糖原贮积症以及诊断和治疗的可能性。酶替代治疗（ERT），其可用性，影响（或大脑缺乏），肠内ERT施用和最终基因治疗的进一步前景全面暴露和讨论。这次传统会议的主要目的有望实现：获得治疗的RD患者数量和尖端的演讲得到了增加。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Pril (Makedon Akad Nauk Umet Odd Med Nauki)">
                Journal: Pril (Makedon Akad Nauk Umet Odd Med Nauki) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29668480" target="_blank">
                PMID: 29668480
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2017-Dec-01
            </a>

            <a class="btn btn-sm btn-danger" title="Gucev Z, Tasic V, Polenakovic M">
                Authors: Gucev Z, Tasic V,...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29668480">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>86. Large Enhancement of 2D Electron Gases Mobility Induced by Interfacial Localized Electron Screening Effect.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29667241" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29667241" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29667241">
                    <p>The interactions between delocalized and localized charges play important roles in correlated electron systems. Here, using a combination of transport measurements, spectroscopic ellipsometry (SE), and X-ray absorption spectroscopy (XAS) supported by theoretical calculations, we reveal the important role of interfacial localized charges and their screening effects in determining the mobility of (La0.3 Sr0.7 )(Al0.65 Ta0.35 )O3 /SrTiO3 (LSAT/SrTiO3 ) interfaces. When the LSAT layer thickness reaches the critical value of 5 uc, the insulating interface abruptly becomes conducting, accompanied by the appearance of a new midgap state. This midgap state emerges at ≈1 eV below the Ti t2g band and shows a strong character of Ti 3dxy - O 2p hybridization. Increasing the LSAT layer from 5 to 18 uc, the number of localized charges increases, resulting in an enhanced screening effect and higher mobile electron mobility. This observation contradicts the traditional semiconductor interface where the localized charges always suppress the carrier mobility. These results demonstrate a new strategy to probe localized charges and mobile electrons in correlated electronic systems and highlight the important role of screening effects from localized charges in improving the mobile electron mobility at complex oxide interfaces.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29667241">
                    <p>离域化和局域化电荷之间的相互作用在相关电子系统中起着重要作用。在这里，使用传输测量，光谱椭偏仪（SE）和X射线吸收光谱（XAS）的理论计算支持的组合，我们揭示了界面局部电荷的重要作用和它们在确定（La0。 3 Sr0.7）（Al0.65 Ta0.35）O3 / SrTiO3（LSAT / SrTiO3）界面。当LSAT层厚度达到5 uc的临界值时，绝缘界面突然变为导电状态，伴随着新的中间隙状态的出现。这种中间态出现在Ti t2g带下约≈1eV，并表现出Ti 3dxy  -  O 2p杂交的强烈特性。将LSAT层从5增加到18 uc，局域电荷的数量增加，导致增强的屏蔽效应和更高的移动电子迁移率。这种观察与传统的半导体接口相矛盾，在这种接口中局部电荷总是抑制载流子迁移率。这些结果展示了一种探测相关电子系统中的局域电荷和移动电子的新策略，并且突出了局部电荷的屏蔽效应在改善复杂氧化物界面处的移动电子迁移率中的重要作用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Adv. Mater. Weinheim">
                Journal: Adv. Mater. Weinheim [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29667241" target="_blank">
                PMID: 29667241
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Chi X, Huang Z, Asmara TC, Han K, Yin X, Yu X, Diao C, Yang M, Schmidt D, Yang P, Trevisanutto PE, Whitcher TJ, Venkatesan T, Breese MBH, Ariando A">
                Authors: Chi X, Huang Z, A...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29667241">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>87. SNP-based pool genotyping and haplotype analysis accelerate fine-mapping of the wheat genomic region containing stripe rust resistance gene Yr26.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29666883" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29666883" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29666883">
                    <p>NGS-assisted super pooling emerging as powerful tool to accelerate gene mapping and haplotype association analysis within target region uncovering specific linkage SNPs or alleles for marker-assisted gene pyramiding. Conventional gene mapping methods to identify genes associated with important agronomic traits require significant amounts of financial support and time. Here, a single nucleotide polymorphism (SNP)-based mapping approach, RNA-Seq and SNP array assisted super pooling analysis, was used for rapid mining of a candidate genomic region for stripe rust resistance gene Yr26 that has been widely used in wheat breeding programs in China. Large DNA and RNA super-pools were genotyped by Wheat SNP Array and sequenced by Illumina HiSeq, respectively. Hundreds of thousands of SNPs were identified and then filtered by multiple filtering criteria. Among selected SNPs, over 900 were found within an overlapping interval of less than 30 Mb as the Yr26 candidate genomic region in the centromeric region of chromosome arm 1BL. The 235 chromosome-specific SNPs were converted into KASP assays to validate the Yr26 interval in different genetic populations. Using a high-resolution mapping population (&gt; 30,000 gametes), we confined Yr26 to a 0.003-cM interval. The Yr26 target region was anchored to the common wheat IWGSC RefSeq v1.0 and wild emmer WEWSeq v.1.0 sequences, from which 488 and 454 kb fragments were obtained. Several candidate genes were identified in the target genomic region, but there was no typical resistance gene in either genome region. Haplotype analysis identified specific SNPs linked to Yr26 and developed robust and breeder-friendly KASP markers. This integration strategy can be applied to accelerate generating many markers closely linked to target genes/QTL for a trait of interest in wheat and other polyploid species.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29666883">
                    <p>NGS辅助的超级池技术正成为加速目标区域内基因定位和单体型关联分析的有力工具，揭示了标记辅助基因聚合的特定连锁SNPs或等位基因。鉴定与重要农艺性状相关的基因的常规基因定位方法需要大量的财政支持和时间。在此，使用基于单核苷酸多态性（SNP）的定位方法，RNA-Seq和SNP阵列辅助的超级汇集分析，快速挖掘已广泛用于小麦育种程序的条锈病抗性基因Yr26的候选基因组区域在中国。通过小麦SNP阵列对大DNA和RNA超级库进行基因分型，并通过Illumina HiSeq分别进行测序。数十万个SNP被鉴定出来，然后通过多个过滤标准进行过滤。在所选的SNP中，在小于30Mb的重叠区间内发现超过900个，作为染色体臂1BL的着丝粒区域中的Yr26候选基因组区域。将235个染色体特异性SNP转化为KASP分析以验证不同遗传群体中的Yr26区间。使用高分辨率的作图群体（&gt; 30,000配子），我们将Yr26限制在0.003cM的范围内。将Yr26靶标区域锚定到普通小麦IWGSC RefSeq v1.0和野生二粒小麦WEWSeq v.1.0序列，从中获得488和454kb片段。在目标基因组区域鉴定了几个候选基因，但是在任一基因组区域都没有典型的抗性基因。单体型分析鉴定出与Yr26相关的特定SNP，并开发出健壮且易于培育的KASP标记。这种整合策略可以应用于加速产生与目标基因/ QTL紧密连锁的许多标记，以获得小麦和其他多倍体物种感兴趣的性状。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Theor. Appl. Genet.">
                Journal: Theor. Appl. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29666883" target="_blank">
                PMID: 29666883
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Wu J, Zeng Q, Wang Q, Liu S, Yu S, Mu J, Huang S, Sela H, Distelfeld A, Huang L, Han D, Kang Z">
                Authors: Wu J, Zeng Q, Wan...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29666883">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>88. Using biology to guide the treatment of sarcomas and aggressive connective-tissue tumours.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29666441" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29666441" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29666441">
                    <p>Sarcomas are a heterogeneous group of malignancies that arise from cells of a mesenchymal origin. Surgery forms the mainstay of the treatment of most patients with localized sarcoma and might be followed or preceded by chemotherapy and/or radiotherapy. In the metastatic setting, systemic treatments tend to improve survival and control symptoms. However, the adverse events and sometimes disappointing outcomes associated with these empirical approaches to treatment indicate a need for new approaches. The advent of next-generation sequencing (NGS) has enabled more targeted treatment of many malignancies based on the presence of specific alterations. NGS analyses of sarcomas have revealed the presence of many alterations that can be targeted using therapies that are already used in patients with other forms of cancer. In this Review, we describe the genomic alterations considered to define specific nosological subgroups of sarcoma and whose contribution to oncogenesis provides a biological rationale for the use of a specific targeted therapy. We also report several less successful examples that should guide researchers and clinicians to better define the extent to which the identification of driver molecular alterations should influence the development of novel treatments.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29666441">
                    <p>肉瘤是由间充质来源的细胞产生的异质性恶性肿瘤组。手术是大多数局限性肉瘤患者治疗的主要手段，并可能接受或先于化疗和/或放疗。在转移性环境中，全身治疗倾向于改善生存和控制症状。然而，与这些经验治疗方法相关的不良事件和有时令人失望的结果表明需要新的方法。新一代测序（NGS）的出现使得基于特定改变的存在能够更有针对性地治疗许多恶性肿瘤。肉瘤的NGS分析揭示了许多改变的存在，可以使用已经用于其他形式的癌症的患者的治疗来靶向。在这篇综述中，我们描述了被认为定义了肉瘤的特定鼻内亚群的基因组改变，并且其对肿瘤发生的贡献提供了使用特定靶向治疗的生物学原理。我们还报告了一些不太成功的例子，应该指导研究人员和临床医生更好地确定驱动分子改变的识别应该影响新疗法发展的程度。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Nat Rev Clin Oncol">
                Journal: Nat Rev Clin Oncol [IF=20.693]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29666441" target="_blank">
                PMID: 29666441
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Dufresne A, Brahmi M, Karanian M, Blay JY">
                Authors: Dufresne A, Brahm...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29666441">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>89. Novel insights into the pathogenesis of T-cell lymphomas.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29666117" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29666117" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29666117">
                    <p>T-cell lymphomas are a heterogeneous group of rare malignancies with overlapping clinical, immunologic and histologic features. Recent advances in our understanding of T-cell differentiation based on gene expression profiling, next-generation sequencing (NGS) and transgenic mouse modeling studies have better elucidated the pathogenetic mechanisms underlying the diverse biology of T-cell lymphomas. These studies show that while genetic alterations in epigenetic modifiers are implicated in all subtypes of T-cell lymphomas, specific subtypes demonstrate enrichment for particular recurrent alterations targeting specific genes. In this regard, RHOA and TET2 alterations are prevalent in nodal T-cell lymphomas particularly angioimmunoblastic T-cell lymphomas (AITL), peripheral T-cell lymphomas, not otherwise specified (PTCL, NOS), and nodal peripheral T-cell lymphoma with T-follicular helper phenotype (PTCL, TFH). JAK-STAT signaling pathways are mutationally activated in many extranodal T-cell lymphomas such as NK/T-cell and hepatosplenic T-cell lymphomas (HSTCL). The functional significance of many of these genetic alterations are becoming better understood. Altogether these advances will continue to refine diagnostic criteria, improve prognostication and identify novel therapeutic targets resulting in improved outcomes for patient with T-cell lymphomas.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29666117">
                    <p>T细胞淋巴瘤是罕见恶性肿瘤的异质组，具有重叠的临床，免疫和组织学特征。基于基因表达谱，新一代测序（NGS）和转基因小鼠模型研究，我们了解T细胞分化的最新进展更好地阐明了T细胞淋巴瘤多样生物学的发病机制。这些研究表明，尽管表观遗传修饰物中的遗传改变涉及T细胞淋巴瘤的所有亚型，但特定的亚型表现出富集针对特定基因的特定复发性改变。在这方面，RHOA和TET2改变在节点性T细胞淋巴瘤特别是血管免疫母细胞性T细胞淋巴瘤（AITL），外周T细胞淋巴瘤（未另外指定）（PTCL，NOS）和淋巴结周T细胞淋巴瘤中普遍存在 - 毛囊辅助表型（PTCL，TFH）。 JAK-STAT信号通路在许多结外T细胞淋巴瘤如NK / T细胞和肝脾T细胞淋巴瘤（HSTCL）中被突变激活。许多这些基因改变的功能意义正在变得更好理解。总之，这些进展将继续完善诊断标准，改善预后并确定新的治疗靶点，从而改善T细胞淋巴瘤患者的预后。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Blood">
                Journal: Blood [IF=13.164]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29666117" target="_blank">
                PMID: 29666117
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Van Arnam JS, Lim MS, Elenitoba-Johnson KSJ">
                Authors: Van Arnam JS, Lim...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29666117">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>90. Microsatellite instability in prostate cancer by PCR or next-generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29665853" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29665853" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29665853">
                    <p>Microsatellite instability (MSI) is now being used as a sole biomarker to guide immunotherapy treatment for men with advanced prostate cancer. Yet current molecular diagnostic tests for MSI have not been evaluated for use in prostate cancer.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29665853">
                    <p>微卫星不稳定性（MSI）现在被用作指导晚期前列腺癌男性免疫疗法治疗的唯一生物标志物。然而目前针对MSI的分子诊断测试尚未被评估用于前列腺癌。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Immunother Cancer">
                Journal: J Immunother Cancer [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29665853" target="_blank">
                PMID: 29665853
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Hempelmann JA, Lockwood CM, Konnick EQ, Schweizer MT, Antonarakis ES, Lotan TL, Montgomery B, Nelson PS, Klemfuss N, Salipante SJ, Pritchard CC">
                Authors: Hempelmann JA, Lo...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5904988" target="_blank">
                PMC5904988
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29665853">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>91. A machine learning model to determine the accuracy of variant calls in capture-based next generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29665779" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29665779" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29665779">
                    <p>Next generation sequencing (NGS) has become a common technology for clinical genetic tests. The quality of NGS calls varies widely and is influenced by features like reference sequence characteristics, read depth, and mapping accuracy. With recent advances in NGS technology and software tools, the majority of variants called using NGS alone are in fact accurate and reliable. However, a small subset of difficult-to-call variants that still do require orthogonal confirmation exist. For this reason, many clinical laboratories confirm NGS results using orthogonal technologies such as Sanger sequencing. Here, we report the development of a deterministic machine-learning-based model to differentiate between these two types of variant calls: those that do not require confirmation using an orthogonal technology (high confidence), and those that require additional quality testing (low confidence). This approach allows reliable NGS-based calling in a clinical setting by identifying the few important variant calls that require orthogonal confirmation.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29665779">
                    <p>新一代测序（NGS）已成为临床基因检测的常用技术。 NGS呼叫的质量变化很大，并受参考序列特征，读取深度和映射精度等特征的影响。随着NGS技术和软件工具的最新进展，单独称为NGS的大多数变体事实上准确可靠。然而，仍然需要正交确认的一小部分难以通话的变体存在。为此，许多临床实验室使用正交技术如Sanger测序确认NGS结果。在这里，我们报告了确定性基于机器学习的模型的发展，以区分这两种类型的变异调用：那些不需要使用正交技术进行确认（高置信度），以及那些需要额外质量测试（低置信度）。这种方法通过识别需要正交确认的少数重要变体呼叫，可以在临床环境中实现可靠的基于NGS的呼叫。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Genomics">
                Journal: BMC Genomics [IF=3.729]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29665779" target="_blank">
                PMID: 29665779
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="van den Akker J, Mishne G, Zimmer AD, Zhou AY">
                Authors: van den Akker J, ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5904977" target="_blank">
                PMC5904977
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29665779">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>92. Distinctiveness and Similarities Between Extended-Spectrum β-Lactamase-Producing Escherichia coli Isolated from Cattle and the Community in Israel.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29664687" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29664687" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29664687">
                    <p>The goal of this study was to compare the molecular features of bovine- and human community-acquired extended-spectrum β-lactamase (ESBL)-producing Escherichia coli in Israel. Bovine ESBL-producing E. coli were isolated during a point-prevalence study from the main farming locations throughout Israel. Human ESBL-producing E. coli isolates were collected from community-acquired urinary tract infection cases. Molecular typing was done initially by repetitive extragenic palindromic-PCR. Representative isolates were subjected to next-generation sequencing (NGS) and analyzed for multilocus sequence typing (MLST), core genome MLST (cgMLST), blaCTX-M gene allele, and mobile genetic elements (MGEs) surrounding it. Out of the 287 bovine- and 104 community-derived ESBL-producing E. coli isolates, 44 and 26 isolates were subjected to NGS, respectively. Both populations exhibited a diverse but distinct clonal structure with predominance of several sequence types (STs); two clones, ST-10/167 (n = 13) and ST-38 (n = 8), were present. cgMLST analysis of these clones revealed that the majority of isolates exhibited phylogenetic distance (PD) of &gt;178 gene difference from their closest isolate, with the exception of five isolates that exhibited PD of &lt;24 gene difference, including two bovine- to three community-derived isolates. Hence, clonal transmission of ESBL-producing E. coli between cattle and the community, although uncommon, is likely to have occurred. The blaCTX-M-15 gene was identified in 52/70 (74%) isolates from both cattle and the community and was surrounded by MGEs that were composed mostly of either the Tn3 or IS1380 families. Thus, MGEs are likely to play an important role in the exchange of resistance genes.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29664687">
                    <p>这项研究的目的是比较以色列牛和人类社区获得的超广谱β-内酰胺酶（ESBL）生产大肠埃希氏菌的分子特征。在整个以色列主要农业地点的流行病点研究中，分离出产ESBL的大肠杆菌。从社区获得性尿路感染病例中收集产ESBL的大肠杆菌分离株。分子分型最初由重复的基因外回文PCR完成。对代表性分离株进行下一代测序（NGS）并分析多位点序列分型（MLST），核心基因组MLST（cgMLST），blaCTX-M基因等位基因和围绕其的移动遗传元件（MGE）。在来自287株牛和104株社区产ESBL的大肠杆菌分离株中，分别有44株和26株分离物经历了NGS。这两个群体表现出多样但独特的克隆结构，其中多种序列类型（ST）占优势;存在两个克隆ST-10/167（n = 13）和ST-38（n = 8）。对这些克隆的cgMLST分析显示大多数分离株表现出与其最接近的分离株有&gt; 178个基因差异的系统发生距离（PD），除了表现出&lt;24个基因差异的PD的五个分离株（包括两个牛至三个群落 - 衍生的分离物。因此，牛和社区之间产ESBL的大肠杆菌的克隆传播虽然不常见，但很可能已经发生。 blaCTX-M-15基因在来自牛和社区的52/70（74％）分离株中鉴定，并被主要由Tn3或IS1380家族组成的MGE包围。因此，MGE可能在抗性基因交换中发挥重要作用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Microb. Drug Resist.">
                Journal: Microb. Drug Resist. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29664687" target="_blank">
                PMID: 29664687
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Lifshitz Z, Sturlesi N, Parizade M, Blum SE, Gordon M, Taran D, Adler A">
                Authors: Lifshitz Z, Sturl...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29664687">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>93. Unifying cancer and normal RNA sequencing data from different sources.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29664468" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29664468" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29664468">
                    <p>Driven by the recent advances of next generation sequencing (NGS) technologies and an urgent need to decode complex human diseases, a multitude of large-scale studies were conducted recently that have resulted in an unprecedented volume of whole transcriptome sequencing (RNA-seq) data, such as the Genotype Tissue Expression project (GTEx) and The Cancer Genome Atlas (TCGA). While these data offer new opportunities to identify the mechanisms underlying disease, the comparison of data from different sources remains challenging, due to differences in sample and data processing. Here, we developed a pipeline that processes and unifies RNA-seq data from different studies, which includes uniform realignment, gene expression quantification, and batch effect removal. We find that uniform alignment and quantification is not sufficient when combining RNA-seq data from different sources and that the removal of other batch effects is essential to facilitate data comparison. We have processed data from GTEx and TCGA and successfully corrected for study-specific biases, enabling comparative analysis between TCGA and GTEx. The normalized datasets are available for download on figshare.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29664468">
                    <p>在下一代测序（NGS）技术的最新进展以及迫切需要解码复杂的人类疾病的驱动下，最近进行了大量大规模研究，这些研究导致了前所未有的大量全转录组测序（RNA-seq）数据例如基因型组织表达项目（GTEx）和癌症基因组图谱（TCGA）。虽然这些数据为识别疾病的机制提供了新的机会，但由于样本和数据处理的差异，来自不同来源的数据的比较仍然具有挑战性。在这里，我们开发了一个流水线，用于处理和统一来自不同研究的RNA-seq数据，其中包括统一重新对齐，基因表达量化和批次效应去除。我们发现当组合不同来源的RNA-seq数据时，统一的比对和定量是不够的，并且为了便于数据比较，去除其他批次效应是必不可少的。我们处理了来自GTEx和TCGA的数据，并成功纠正了针对特定研究的偏差，从而可以对TCGA和GTEx进行比较分析。规范化的数据集可以在figshare上下载。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Data">
                Journal: Sci Data [IF=4.836]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29664468" target="_blank">
                PMID: 29664468
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Wang Q, Armenia J, Zhang C, Penson AV, Reznik E, Zhang L, Minet T, Ochoa A, Gross BE, Iacobuzio-Donahue CA, Betel D, Taylor BS, Gao J, Schultz N">
                Authors: Wang Q, Armenia J...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5903355" target="_blank">
                PMC5903355
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29664468">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>94. Equine Encephalosis Virus in India, 2008.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29664366" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29664366" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29664366">
                    <p>A virus isolated from a sick horse from India in 2008 was confirmed by next-generation sequencing analysis to be equine encephalosis virus (EEV). EEV in India is concerning because several species of Culicoides midge, which play a major role in EEV natural maintenance and transmission, are present in this country.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29664366">
                    <p>新一代测序分析证实，2008年从印度病马中分离出的病毒为马脑病病毒（EEV）。 EEV在印度是有关的，因为在这个国家出现了几种在EEV自然维护和传播中起主要作用的Culicoides mid蚊。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Emerging Infect. Dis.">
                Journal: Emerging Infect. Dis. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29664366" target="_blank">
                PMID: 29664366
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Yadav PD, Albariño CG, Nyayanit DA, Guerrero L, Jenks MH, Sarkale P, Nichol ST, Mourya DT">
                Authors: Yadav PD, Albariñ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29664366">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>95. Overlap at the molecular and immunohistochemical levels between angioimmunoblastic T-cell lymphoma and a subgroup of peripheral T-cell lymphomas without specific morphological features.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29662631" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29662631" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29662631">
                    <p>The overlap of morphology and immunophenotype between angioimmunoblastic T-cell lymphoma (AITL) and other nodal peripheral T-cell lymphomas (n-PTCLs) is a matter of current interest whose clinical relevance and pathogenic background have not been fully established. We studied a series of 98 n-PTCL samples (comprising 57 AITL and 41 PTCL-NOS) with five TFH antibodies (CD10, BCL-6, PD-1, CXCL13, ICOS), looked for mutations in five of the genes most frequently mutated in AITL (TET2, DNMT3A, IDH2, RHOA and PLCG1) using the Next-Generation-Sequencing Ion Torrent platform, and measured the correlations of these characteristics with morphology and clinical features. The percentage of mutations in the RHOA and TET2 genes was similar (23.5% of cases). PLCG1 was mutated in 14.3%, IDH2 in 11.2% and DNMT3A in 7.1% of cases, respectively. In the complete series, mutations in RHOA gene were associated with the presence of mutations in IDH2, TET2 and DNMT3A (p &lt; 0.001, p = 0.043, and p = 0.029, respectively). Fourteen cases featured RHOA mutations without TET2 mutations. A close relationship was found between the presence of these mutations and a TFH-phenotype in AITL and PTCL-NOS patients. Interestingly, BCL-6 expression was the only TFH marker differentially expressed between AITL and PTCL-NOS cases. There were many fewer mutated cases than there were cases with a TFH phenotype. Overall, these data suggest alternative ways by which neoplastic T-cells overexpress these proteins. On the other hand, no clinical or survival differences were found between any of the recognized subgroups of patients with respect to their immunohistochemistry or mutational profile.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29662631">
                    <p>血管免疫母细胞性T细胞淋巴瘤（AITL）与其他淋巴结外周T细胞淋巴瘤（n-PTCL）之间的形态学和免疫表型的重叠是目前感兴趣的问题，其临床相关性和致病背景尚未完全确立。我们用五种TFH抗体（CD10，BCL-6，PD-1，CXCL13，ICOS）研究了一系列98个n-PTCL样品（包含57个AITL和41个PTCL-NOS），最频繁地寻找5个基因中的突变使用Next-Generation-Sequencing Ion Torrent平台在AITL（TET2，DNMT3A，IDH2，RHOA和PLCG1）中进行突变，并测量这些特征与形态学和临床特征的相关性。 RHOA和TET2基因突变的百分比相似（23.5％的病例）。 PLCG1突变为14.3％，IDH2分别为11.2％和DNMT3A分别为7.1％。在完整系列中，RHOA基因突变与IDH2，TET2和DNMT3A突变的存在相关（分别为p &lt;0.001，p = 0.043和p = 0.029）。有14例病例表现为无TET2突变的RHOA突变。 AITL和PTCL-NOS患者中存在这些突变和TFH表型之间存在密切关系。有趣的是，BCL-6表达是AITL和PTCL-NOS病例之间差异表达的唯一TFH标记。发生突变的病例比具有TFH表型的病例少得多。总体而言，这些数据表明肿瘤性T细胞过度表达这些蛋白质的替代方式。另一方面，任何公认的亚组患者在免疫组化或突变方面均未发现临床或生存差异。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29662631" target="_blank">
                PMID: 29662631
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Manso R, González-Rincón J, Rodríguez-Justo M, Roncador G, Gómez S, Sánchez-Beato M, Piris MA, Rodríguez-Pinilla SM">
                Authors: Manso R, González...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5882322" target="_blank">
                PMC5882322
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29662631">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>96. Are we ready for routine precision medicine? Highlights from the Milan Summit on Precision Medicine, Milan, Italy, 8-9 February 2018.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29662530" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29662530" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29662530">
                    <p>On 8 and 9 February 2018, the IFOM-IEO campus in Milan hosted the Milan summit on Precision Medicine, which gathered clinical and translational research experts from academia, industry and regulatory bodies to discuss the state of the art of precision medicine in Europe. The meeting was pervaded by a generalised feeling of excitement for a field that is perceived to be technologically mature for the transition into clinical routine but still hampered by numerous obstacles of a methodological, ethical, regulatory and possibly cultural nature. Through lively discussions, the attendees tried to identify realistic ways to implement a technology-rich precision approach to cancer patients.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29662530">
                    <p>2018年2月8日和9日，米兰IFOM-IEO校园举办了精准医学米兰峰会，聚集了来自学术界，工业界和监管机构的临床和转化研究专家，讨论欧洲精准医学的最新发展状况。这次会议弥漫着一种普遍的兴奋感，这种场被认为对于过渡到临床常规而言在技术上已经成熟，但仍然受到方法论，伦理，监管和可能文化性质的无数障碍的阻碍。通过热烈的讨论，与会者试图找出切实可行的方法，为癌症患者实施技术丰富的精确方法。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ecancermedicalscience">
                Journal: Ecancermedicalscience [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29662530" target="_blank">
                PMID: 29662530
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Mazzarella L">
                Authors: Mazzarella L
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880225" target="_blank">
                PMC5880225
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29662530">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>97. Parent-of-origin effect of hypomorphic pathogenic variants and somatic mosaicism impact on phenotypic expression of retinoblastoma.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29662154" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29662154" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29662154">
                    <p>Retinoblastoma is the most common eye cancer in children. Numerous families have been described displaying reduced penetrance and expressivity. An extensive molecular characterization of seven families led us to characterize the two main mechanisms impacting on phenotypic expression, as follows: (i) mosaicism of amorphic pathogenic variants; and (ii) parent-of-origin-effect of hypomorphic pathogenic variants. Somatic mosaicism for RB1 splicing variants (c.1960+5G&gt;C and c.2106+2T&gt;C), leading to a complete loss of function was demonstrated by high-depth NGS in two families. In both cases, the healthy carrier parent (one with retinoma) showed a variant frequency lower than that expected for a heterozygous individual, indicating a 56-60% mosaicism level. Previous evidences of a ~3-fold excess of RB1 maternal canonical transcript led us to hypothesize that this differential allelic expression could influence phenotypic outcome in families at risk for RB onset. Accordingly, in five families, we identified a higher tumor risk associated with paternally inherited hypomorphic pathogenic variants, namely a deletion resulting in the loss of 37 amino acids at the N-terminus (c.608-16_608del), an exonic substitution with a &quot;leaky&quot; splicing effect (c.1331A&gt;G), a partially deleterious substitution (c.1981C&gt;T) and a truncating C-terminal variant (c.2663+2T&gt;C). The identification of these mechanisms changes the genetic/prenatal counseling and the clinical management of families, indicating a higher recurrence risk when the hypomorphic pathogenic variant is inherited from the father, and suggesting the need for second tumor surveillance in unaffected carriers at risk of developing adult-onset cancer such as osteosarcoma or leiomyosarcoma.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29662154">
                    <p>视网膜母细胞瘤是儿童中最常见的眼癌。已经描述了许多家族显示出降低的外显率和表达性。 7个家族的广泛分子特征使我们能够表征影响表型表达的两种主要机制，如下所述：（i）无定形致病变体的镶嵌性;和（ii）hypomorphic致病变异体的父母起源效应。 RB1剪接变体（c.1960 + 5G&gt; C和c.2106 + 2T&gt; C）的体细胞嵌合体导致功能的完全丧失，这表现为两个家族的高深度NGS。在这两种情况下，健康的载体母体（一个患有视网膜瘤）显示变异频率低于杂合子个体的预期频率，表明56-60％镶嵌性水平。以前的证据表明，RB1母体经典转录本的过度〜3倍过量导致我们推测这种差异性等位基因表达可能影响存在RB发病风险的家族的表型结果。因此，在五个家族中，我们发现与父系遗传的hypomorphic致病变异体相关的更高的肿瘤风险，即导致N-末端37个氨基酸缺失的缺失（c.608-16_608del），用“泄漏“剪接效应（c.1331A&gt; G），部分有害替代（c.1981C&gt; T）和截短C-末端变体（c.2663 + 2T&gt; C）。这些机制的鉴定改变了遗传/产前咨询和家属的临床管理，表明当遗传来自父亲的hypomorphic致病性变体时具有更高的复发风险，并且表明需要对处于发育成人风险的未受影响的携带者进行第二次肿瘤监测引发癌症，如骨肉瘤或平滑肌肉瘤。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Hum. Genet.">
                Journal: Eur. J. Hum. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29662154" target="_blank">
                PMID: 29662154
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Imperatore V, Pinto AM, Gelli E, Trevisson E, Morbidoni V, Frullanti E, Hadjistilianou T, De Francesco S, Toti P, Gusson E, Roversi G, Accogli A, Capra V, Mencarelli MA, Renieri A, Ariani F">
                Authors: Imperatore V, Pin...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29662154">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>98. Characterization of extensively- or pandrug-resistant ST147 and ST101 OXA-48-producing Klebsiella pneumoniae isolates causing bloodstream infections in ICU patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29661874" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29661874" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29661874">
                    <p>Carbapenem-resistant Klebsiella pneumoniae cause important healthcare-associated infections worldwide. An outbreak of ST11 OXA-48-producing K. pneumoniae isolates (OXA-48-Kp) occurred in Tzaneio Hospital in 2012 and was contained until 2014, when OXA-48-Kp reemerged. The present study involved 19 bloodstream infection (BSI) OXA-48-Kp recovered from 19 ICU patients hospitalized between August 2014 and July 2016. MICs were determined by broth microdilution. β-lactamase genes were detected by PCR. All isolates were typed by PFGE/MLST and 10 representative isolates by next generation sequencing (NGS). Of the 19 study patients, nine had previous hospitalizations and 10 carried OXA-48-Kp prior to BSI isolations; median time from ICU admission to BSI was 29 days. Four OXA-48-Kp belonged to PFGE profile A (ST147) and were pandrug-resistant (PDR), while 15 isolates exhibited PFGE profile B (ST101) and were extensively drug-resistant. NGS resistome analysis detected genes justifying resistances, except for tigecycline and fosfomycin. Insertional inactivation of mgrB (distinct per clone) conferred colistin resistance in all 19 isolates. NGS SNPs analysis validated the clonal relatedness of the ST147 and ST101 strains and revealed possible presence of two index ST147 strains and microevolution of ST101 strains. Distinct, but highly related, IncL OXA-48-encoding plasmid lineages were identified; plasmids of the ST147 strains were identical with that of ST11 OXA-48-Kp causing the 2012 outbreak. In conclusion, biclonal circulation of OXA-48-Kp and, alarmingly, emergence of a PDR clone are reported. These observations, along with the challenging phenotypic detection of OXA-48 producers and the high reported transmissibility of blaOXA-48, necessitate intensive efforts to prevent their further spread.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29661874">
                    <p>碳青霉烯抗性肺炎克雷伯菌引起全球重要的医疗相关感染。 2012年在Tzaneio医院发生ST11 OXA-48产肺炎克雷伯菌分离株（OXA-48-Kp），直到2014年OXA-48-Kp再次出现。本研究涉及从2014年8月至2016年7月期间住院的19例ICU患者中回收的19例血流感染（BSI）OXA-48-Kp。通过微量肉汤稀释法测定MIC。通过PCR检测β-内酰胺酶基因。通过下一代测序（NGS），PFGE / MLST和10个代表性分离株分型。在19名研究患者中，有9名曾接受过住院治疗，10名患者在BSI分离之前携带了OXA-48-Kp;从ICU入学到BSI的中位时间为29天。 4个OXA-48-Kp属于PFGE图谱A（ST147）并且是耐药性的（PDR），而15个分离株显示PFGE图谱B（ST101）并且是广泛耐药的。除替加环素和磷霉素外，NGS抗氧化剂分析检测到证明耐药性的基因。 mgrB的插入灭活（不同的每个克隆）在所有19个分离株中赋予粘菌素抗性。 NGS SNPs分析验证了ST147和ST101菌株的克隆相关性，并揭示了可能存在两种指标ST147菌株和ST101菌株的微进化。不同但高度相关的IncL OXA-48编码质粒谱系被鉴定; ST147菌株的质粒与ST11 OXA-48-Kp的质粒相同，导致2012年爆发。总之，报道了OXA-48-Kp的双克隆循环以及令人震惊的PDR克隆的出现。这些观察结果，以及OXA-48生产者具有挑战性的表型检测和blaOXA-48高报道的传播性，需要进行大量努力以防止其进一步传播。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Antimicrob. Agents Chemother.">
                Journal: Antimicrob. Agents Chemother. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29661874" target="_blank">
                PMID: 29661874
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-16
            </a>

            <a class="btn btn-sm btn-danger" title="Avgoulea K, Di Pilato V, Zarkotou O, Sennati S, Politi L, Cannatelli A, Themeli-Digalaki K, Giani T, Tsakris A, Rossolini GM, Pournaras S">
                Authors: Avgoulea K, Di Pi...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29661874">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>99. Cell-Free Plasma DNA for Disease Stratification and Prognosis in Head and Neck Cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29661793" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29661793" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29661793">
                    <p>Clinicians face many challenges in disease stratification and outcome prediction in head and neck squamous cancer cell (HNSCC) patients. Given the limitations of currently used clinical scoring, repetitive biopsies, and imaging techniques, liquid biopsy approaches may provide valuable additional diagnostic and prognostic information.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29661793">
                    <p>临床医生在头颈部鳞状细胞癌（HNSCC）患者的疾病分层和预后预测中面临着许多挑战。鉴于目前使用的临床评分，重复活检和成像技术的局限性，液体活检方法可提供有价值的额外诊断和预后信息。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Chem.">
                Journal: Clin. Chem. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29661793" target="_blank">
                PMID: 29661793
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-16
            </a>

            <a class="btn btn-sm btn-danger" title="Schirmer MA, Beck J, Leu M, Oellerich M, Rave-Fränk M, Walson PD, Schütz E, Canis M">
                Authors: Schirmer MA, Beck...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29661793">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>100. The first mitochondrial genome of the model echinoid Lytechinus variegatus and insights into Odontophoran phylogenetics.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29660476" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29660476" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29660476">
                    <p>Assembly of publically available next-generation sequence data facilitated the generation of three camarodont echinoid mitogenomes: two for the Green Urchin (Lytechinus variegatus) and one for the Red Urchin (Mesocentrotus franciscanus). The data generated are exploited in a phylogenomic analysis of the superfamily Odontophora, originally proposed for echinoids with tooth supports on the epiphyses of the jaw. The analysis highly supports this taxon and its current subdivision into three families: the Echinometridae, Toxopneustidae, and Strongylocentrotidae. The analysis furthermore implies that historical taxonomic issues between two members of the genus Strongylocentrotus (S. pallidus and S. droebachiensis) may have a genetic basis. The novel mitogenomes for the model species L. variegatus complements the draft genome available for this taxon, one of only three genome-enabled echinoid species. The assembly method applied herein, follows a divide-and-conquer approach that provides for reduced computational requirements and facilitates resolving assembly problems when processing ultra-high coverage next-generation sequence data.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29660476">
                    <p>大会公开提供的下一代序列数据促进了三个camarodont echinoid mitogenomes的产生：绿海胆（Lytechinus variegatus）两个，红海胆（Mesocentrotus franciscanus）一个。所产生的数据在Odontophora超家族的系统发育分析中被利用，最初被提出用于在颌骨骨骺上具有牙齿支持的echinoids。该分析极大地支持了该分类单元，并将其分为三个科：Echinometridae，Toxopneustidae和Strongylocentrotidae。分析还意味着Strongylocentrotus（S. pallidus和S. droebachiensis）两个成员之间的历史分类学问题可能具有遗传基础。模式物种L. variegatus的新型mitogenome补充了该分类群的可用基因组草图，这是仅有的三种基因组启用的echinoid物种之一。这里应用的组装方法遵循分而治之的方法，该方法提供了降低的计算要求并有助于在处理超高覆盖率的下一代序列数据时解决组装问题。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genomics">
                Journal: Genomics [IF=2.801]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29660476" target="_blank">
                PMID: 29660476
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-13
            </a>

            <a class="btn btn-sm btn-danger" title="Bronstein O, Kroh A">
                Authors: Bronstein O, Kroh A
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29660476">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>101. Feasibility study of cancer genome alterations identified by next generation sequencing: ABC study.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29659903" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29659903" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29659903">
                    <p>To confirm the feasibility and explore the clinical applicability of amplicon sequencing by next generation sequencing (NGS) of biopsy samples from patients with advanced solid tumors, we conducted a prospective study.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29659903">
                    <p>为证实可行性，并探索下一代测序（NGS）对来自晚期实体瘤患者的活检样本的扩增子测序的临床适用性，我们进行了前瞻性研究。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Jpn. J. Clin. Oncol.">
                Journal: Jpn. J. Clin. Oncol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29659903" target="_blank">
                PMID: 29659903
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-12
            </a>

            <a class="btn btn-sm btn-danger" title="Naito Y, Takahashi H, Shitara K, Okamoto W, Bando H, Kuwata T, Kuboki Y, Matsumoto S, Miki I, Yamanaka T, Watanabe A, Kojima M">
                Authors: Naito Y, Takahash...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29659903">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>102. Tracking the NGS revolution: managing life science research on shared high-performance computing clusters.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29659792" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29659792" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29659792">
                    <p>Next-generation sequencing (NGS) has transformed the life sciences, and many research groups are newly dependent upon computer clusters to store and analyze large datasets. This creates challenges for e-infrastructures accustomed to hosting computationally mature research in other sciences. Using data gathered from our own clusters at UPPMAX computing center at Uppsala University, Sweden, where core hour usage of ∼800 NGS and ∼200 non-NGS projects is now similar, we compare and contrast the growth, administrative burden, and cluster usage of NGS projects with projects from other sciences.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29659792">
                    <p>新一代测序（NGS）已经改变了生命科学，许多研究小组新近依赖计算机集群来存储和分析大型数据集。这对习惯于在其他科学领域进行计算机成熟研究的电子基础设施构成挑战。在瑞典乌普萨拉大学的UPPMAX计算中心使用我们自己的集群采集的数据，我们的核心小时使用约800个NGS和约200个非NGS项目现在类似，我们比较和比较了增长，管理负担和集群使用NGS项目来自其他科学项目。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Gigascience">
                Journal: Gigascience [IF=6.871]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29659792" target="_blank">
                PMID: 29659792
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May-01
            </a>

            <a class="btn btn-sm btn-danger" title="Dahlö M, Scofield DG, Schaal W, Spjuth O">
                Authors: Dahlö M, Scofield...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29659792">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>103. Reconstruction of a 10-mm-long median nerve gap in an ischemic environment using autologous conduits with different patterns of blood supply: A comparative study in the rat.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29659600" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29659600" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29659600">
                    <p>The aim of this study was to evaluate in the Wistar rat the efficacy of various autologous nerve conduits with various forms of blood supply in reconstructing a 10-mm-long gap in the median nerve (MN) under conditions of local ischemia. A 10-mm-long median nerve defect was created in the right arm. A loose silicone tube was placed around the nerve gap zone, in order to simulate a local ischemic environment. Rats were divided in the following experimental groups (each with 20 rats): the nerve Graft (NG) group, in which the excised MN segment was reattached; the conventional nerve flap (CNF) and the arterialized neurovenous flap (ANVF) groups in which the gap was bridged with homonymous median nerve flaps; the prefabricated nerve flap (PNF) group in which the gap was reconstructed with a fabricated flap created by leaving an arteriovenous fistula in contact with the sciatic nerve for 5 weeks; and the two control groups, Sham and Excision groups. In the latter group, the proximal stump of the MN nerve was ligated and no repair was performed. The rats were followed for 100 days. During this time, they did physiotherapy. Functional, electroneuromyographic and histological studies were performed. The CNF and ANVF groups presented better results than the NG group in the following assessments: grasping test, nociception, motor stimulation threshold, muscle weight, and histomorphometric evaluation. Radial deviation of the operated forepaw was more common in rats that presented worse results in the other outcome variables. Overall, CNFs and ANVFs produced a faster and more complete recovery than NGs in the reconstruction of a 10-mm-long median nerve gap in an ischemic environment in the Wistar rat. Although, results obtained with CNFs were in most cases were better than ANVFs, these differences were not statistically significant for most of the outcome variables.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29659600">
                    <p>本研究的目的是在Wistar大鼠中评估各种自体神经导管与各种形式的血液供应在重建局部缺血条件下的正中神经（MN）中10mm长的间隙中的功效。右臂产生10毫米长的正中神经缺损。将松散的硅胶管放置在神经间隙区域周围，以模拟局部缺血环境。将大鼠分成以下实验组（每组20只大鼠）：神经移植物（NG）组，其中切除的MN节段被重新连接;常规神经瓣（CNF）和动脉化神经静脉瓣（ANVF）组，其中间隙与同义正中神经瓣桥接;预制的神经瓣（PNF）组，其中间隙用通过使动静脉内瘘与坐骨神经接触5周而制造的制造皮瓣重建;和两个对照组Sham和Excision组。在后一组中，MN神经的近端残端结扎并且不进行修复。跟踪大鼠100天。在此期间，他们做了物理治疗。进行功能性，电神经营养和组织学研究。 CNF和ANVF组在下列评估中表现出比NG组更好的结果：抓握测试，伤害感受，运动刺激阈值，肌肉重量和组织形态学评估。操作前爪的径向偏差在大鼠中更常见，其在其他结果变量中表现出更差的结果。总体而言，CNS和ANVF在Wistar大鼠的缺血环境中重建10mm长的正中神经间隙比NGs产生更快且更完全的恢复。尽管用CNF获得的结果多数情况下优于ANVF，但这些差异在大多数结果变量中并不具有统计学意义。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29659600" target="_blank">
                PMID: 29659600
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Casal D, Mota-Silva E, Iria I, Alves S, Farinho A, Pen C, Lourenço-Silva N, Mascarenhas-Lemos L, Silva-Ferreira J, Ferraz-Oliveira M, Vassilenko V, Videira PA, Goyri-O&#39;Neill J, Pais D">
                Authors: Casal D, Mota-Sil...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29659600">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>104. Targeted next generation sequencing identifies functionally deleterious germline mutations in novel genes in early-onset/familial prostate cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29659569" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29659569" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29659569">
                    <p>Considering that mutations in known prostate cancer (PrCa) predisposition genes, including those responsible for hereditary breast/ovarian cancer and Lynch syndromes, explain less than 5% of early-onset/familial PrCa, we have sequenced 94 genes associated with cancer predisposition using next generation sequencing (NGS) in a series of 121 PrCa patients. We found monoallelic truncating/functionally deleterious mutations in seven genes, including ATM and CHEK2, which have previously been associated with PrCa predisposition, and five new candidate PrCa associated genes involved in cancer predisposing recessive disorders, namely RAD51C, FANCD2, FANCI, CEP57 and RECQL4. Furthermore, using in silico pathogenicity prediction of missense variants among 18 genes associated with breast/ovarian cancer and/or Lynch syndrome, followed by KASP genotyping in 710 healthy controls, we identified &quot;likely pathogenic&quot; missense variants in ATM, BRIP1, CHEK2 and TP53. In conclusion, this study has identified putative PrCa predisposing germline mutations in 14.9% of early-onset/familial PrCa patients. Further data will be necessary to confirm the genetic heterogeneity of inherited PrCa predisposition hinted in this study.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29659569">
                    <p>考虑到已知的前列腺癌（PrCa）易感基因（包括遗传性乳房/卵巢癌和Lynch综合征的那些）的突变解释了早发性/家族性PrCa的少于5％，我们使用下一个序列测序了94个与癌易感性相关的基因（NGS）在一系列121名PrCa患者中的表达。我们发现7个基因中的单等位基因截短/功能有害突变，包括先前已与PrCa易感性相关的ATM和CHEK2，以及与诱发隐性病症的癌症有关的五个新的候选PrCa相关基因，即RAD51C，FANCD2，FANCI，CEP57和RECQL4 。此外，我们在710名健康对照中使用硅胶致病性预测18个与乳腺癌/卵巢癌和/或Lynch综合征相关的基因中的错义突变，然后在710名健康对照中进行KASP基因分型，我们鉴定了ATM，BRIP1，CHEK2和TP53中“可能的致病性”错义变体。总之，这项研究已经确定了14.9％的早发性/家族性PrCa患者中假定的PrCa诱发性生殖细胞突变。进一步的数据将是必要的，以确认在这项研究中暗示的遗传PrCa易感性的遗传异质性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS Genet.">
                Journal: PLoS Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29659569" target="_blank">
                PMID: 29659569
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Paulo P, Maia S, Pinto C, Pinto P, Monteiro A, Peixoto A, Teixeira MR">
                Authors: Paulo P, Maia S, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29659569">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>105. Examining the Genetic Background of Porcine Muscle Growth and Development Based on Transcriptome and miRNAome Data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29659518" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29659518" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29659518">
                    <p>Recently, selection in pigs has been focused on improving the lean meat content in carcasses; this focus has been most evident in breeds constituting a paternal component in breeding. Such sire-breeds are used to improve the meat quantity of cross-breed pig lines. However, even in one breed, a significant variation in the meatiness level can be observed. In the present study, the comprehensive analysis of genes and microRNA expression profiles in porcine muscle tissue was applied to identify the genetic background of meat content. The comparison was performed between whole gene expression and miRNA profiles of muscle tissue collected from two sire-line pig breeds (Pietrain, Hampshire). The RNA-seq approach allowed the identification of 627 and 416 differentially expressed genes (DEGs) between pig groups differing in terms of loin weight between Pietrain and Hampshire breeds, respectively. The comparison of miRNA profiles showed differential expression of 57 microRNAs for Hampshire and 34 miRNAs for Pietrain pigs. Next, 43 genes and 18 miRNAs were selected as differentially expressed in both breeds and potentially related to muscle development. According to Gene Ontology analysis, identified DEGs and microRNAs were involved in the regulation of the cell cycle, fatty acid biosynthesis and regulation of the actin cytoskeleton. The most deregulated pathways dependent on muscle mass were the Hippo signalling pathway connected with the TGF-β signalling pathway and controlling organ size via the regulation of ubiquitin-mediated proteolysis, cell proliferation and apoptosis. The identified target genes were also involved in pathways such as the FoxO signalling pathway, signalling pathways regulating pluripotency of stem cells and the PI3K-Akt signalling pathway. The obtained results indicate molecular mechanisms controlling porcine muscle growth and development. Identified genes (SOX2, SIRT1, KLF4, PAX6 and genes belonging to the transforming growth factor beta superfamily) could be considered candidate genes for determining muscle mass in pigs.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29659518">
                    <p>最近，猪的选择一直集中在改善屠体中的瘦肉含量;这种关注在育种中构成父本组成的品种中最为明显。这种陛下品种被用于改善杂交猪生产线的肉量。然而，即使在一个品种中，也可以观察到肉质水平的显着变化。在本研究中，应用猪肌肉组织中基因和microRNA表达谱的综合分析来鉴定肉类的遗传背景。比较在两个公猪系（Pietrain，Hampshire）收集的肌肉组织的全基因表达和miRNA谱之间进行比较。 RNA-seq方法可以分别鉴定Pietrain品种和Hampshire品种之间在腰围方面不同的猪组之间差异表达的基因（DEGs）的差异表达基因（DEGs）。 miRNA谱的比较显示了Hampshire的57个微RNA和皮特兰猪的34个miRNA的差异表达。接下来，选择43个基因和18个miRNA作为在两个品种中差异表达并可能与肌肉发育相关的基因。根据Gene Ontology分析，鉴定的DEGs和microRNA参与细胞周期调控，脂肪酸生物合成和肌动蛋白细胞骨架的调节。依赖于肌肉质量的最不受控制的途径是与TGF-β信号传导途径相关的Hippo信号传导途径，并通过调节泛素介导的蛋白水解，细胞增殖和凋亡来控制器官大小。已鉴定的靶基因也参与途径，如FoxO信号通路，调节干细胞多能性的信号通路和PI3K-Akt信号通路。获得的结果表明控制猪肌肉生长和发育的分子机制。已鉴定的基因（SOX2，SIRT1，KLF4，PAX6和属于转化生长因子β超家族的基因）可被视为确定猪肌肉质量的候选基因。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Mol Sci">
                Journal: Int J Mol Sci [IF=3.226]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29659518" target="_blank">
                PMID: 29659518
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-16
            </a>

            <a class="btn btn-sm btn-danger" title="Ropka-Molik K, Pawlina-Tyszko K, Żukowski K, Piórkowska K, Żak G, Gurgul A, Derebecka N, Wesoły J">
                Authors: Ropka-Molik K, Pa...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29659518">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>106. ShRangeSim: Simulation of Single Nucleotide Polymorphism Clusters in Next-Generation Sequencing Data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29658778" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29658778" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29658778">
                    <p>Genomic variations are in the focus of research to uncover mechanisms of host-pathogen interactions and diseases such as cancer. Nowadays, next-generation sequencing (NGS) data are analyzed through dedicated pipelines to detect them. Surrogate NGS data in conjunction with genomic variations help to evaluate pipelines and validate their outcomes, fostering selection of proper tools for a given scientific question. I describe how existing approaches for simulating NGS data in conjunction with genomic variations fail to model local enrichments of single nucleotide polymorphisms (SNPs), so called SNP clusters. Two distributions for count data are applied to publicly available collections of genomic variations. The results suggest modeling of SNP cluster sizes by overdispersion-aware distributions.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29658778">
                    <p>基因组变异是研究的重点，以揭示宿主 - 病原体相互作用和癌症等疾病的机制。如今，通过专用管线分析下一代测序（NGS）数据以检测它们。将NGS数据与基因组变异联系起来有助于评估管道并验证其结果，从而促进为特定科学问题选择合适的工具。我描述了如何结合基因组变异模拟NGS数据的现有方法未能模拟局部富集单核苷酸多态性（SNPs），即所谓的SNP簇。计数数据的两个分布适用于公开可用的基因组变异集合。结果表明SNP簇大小由过分散感知分布建模。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Comput. Biol.">
                Journal: J. Comput. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29658778" target="_blank">
                PMID: 29658778
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-16
            </a>

            <a class="btn btn-sm btn-danger" title="Boenn M">
                Authors: Boenn M
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29658778">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>107. Expression of Bcl-2 Family Proteins in the Ovarian Follicular Apparatus in the Acute Period after Experimental Hyperthermia.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29658076" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29658076" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29658076">
                    <p>The expression of apoptosis regulators (proapoptotic protein Bad and anti-apoptotic protein Bcl-2) was analyzed and Bcl-2/Bad ratio in the follicular apparatus of the rat ovary was determined on day 3 after hyperthermia (rectal temperature 43.5°C). Hyperthermia in the catabolic phase leads to different degrees of activation of the molecular &quot;switches&quot; of apoptosis in cells of ovarian follicular epithelium. This was seen from increased intensity of immunohistochemical staining for Bad protein against the background of more pronounced expression of Bcl-2 protein. On day 3 after exposure to hyperthermia, Bcl-2/Bad ratio increased, which reflects antiapoptotic protection of cells and conditions for blockade of mitochondrial pathway of apoptosis in the follicular apparatus of the ovaries during the acute period after hyperthermia.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29658076">
                    <p>分析凋亡调节因子（促凋亡蛋白Bad和抗凋亡蛋白Bcl-2）的表达，并在高温（直肠温度43.5℃）后第3天测定大鼠卵巢滤泡器中的Bcl-2 / Bad比率。分解代谢阶段中的热疗导致卵巢滤泡上皮细胞中凋亡的分子“开关”的不同程度的活化。这可以从免疫组织化学染色Bad蛋白的强度更强的Bcl-2蛋白表达更明显的背景中看出。热疗后第3天Bcl-2 / Bad比值升高，这反映了热疗后急性期卵巢滤泡器中细胞的抗凋亡保护作用和细胞凋亡线粒体途径的阻断条件。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Bull. Exp. Biol. Med.">
                Journal: Bull. Exp. Biol. Med. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29658076" target="_blank">
                PMID: 29658076
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Michurina SV, Kolesnikov SI, Bochkareva AL, Arkhipov SA, Ishchenko IY">
                Authors: Michurina SV, Kol...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29658076">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>108. The role of Next-Generation Sequencing in tumoral radiosensitivity prediction.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29658008" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29658008" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29658008">
                    <p>Technological advances have led to more precise radiation delivery, which has resulted in significant clinical gains. A better understanding of tumoral radiosensitivity is still needed to develop strategies and further personalize radiation treatments. Next-Generation Sequencing (NGS) and system biology have significantly transformed the field of oncology in the last two decades, but have only a few clinical applications in radiation oncology. This review describes the technical aspects and evolutions of NGS and discusses the latest clinical applications of genomics to predict tumoral radiosensitivity.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29658008">
                    <p>技术进步已经导致了更精确的辐射传输，从而带来了显着的临床获益。仍然需要更好地了解肿瘤放射敏感性，以制定策略并进一步个性化放疗。新一代测序（NGS）和系统生物学在过去的二十年中已经显着改变了肿瘤学领域，但在放射肿瘤学领域仅有少数临床应用。这篇综述描述了NGS的技术方面和发展，并讨论了基因组学预测肿瘤放射敏感性的最新临床应用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin Transl Radiat Oncol">
                Journal: Clin Transl Radiat Oncol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29658008" target="_blank">
                PMID: 29658008
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2017-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Bibault JE, Tinhofer I">
                Authors: Bibault JE, Tinho...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5893518" target="_blank">
                PMC5893518
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29658008">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>109. Characteristics of the gut microbiota in adult patients with type 1 and 2 diabetes based on the analysis of a fragment of 16S rRNA gene using next-generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29657308" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29657308" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29657308">
                    <p>INTRODUCTION    Scientific data indicate  a possible influence of the gastrointestinal tract microbiota on the development of type 1 (T1DM) and 2 (T2DM) diabetes. Sequence analysis of 16S rRNA identified several hundred bacterial species of the intestinal ecosystem, most of which cannot be cultured. OBJECTIVES    We aimed at evaluating the composition of the gut microbiota in adult patients with T1DM and T2DM and establishing a link between data obtained from microbiological testing and the patients&#39; clinical data. PATIENTS AND METHODS    We examined DNA isolated from fecal samples within three groups: healthy volunteers (n=23), T1DM (n=22) and T2DM (n=23). The next-generation sequencing method (NGS) was performed on the MiSeq platform.  RESULTS    At the phylum level, in all groups, Firmicutes bacteria (&gt;77%) definitely prevail. At the taxonomic levels L2 (phylum) and L6 (genus), statistically significant differences were demonstrated in bacterial profiles, particularly in T2DM group. A correlation has been established between: 1) several genera of bacteria and the percentage of hemoglobin A1c  in T2DM group (negative correlation) and 2) bacteria belonging to the genus Bifidobacterium and high density lipoprotein cholesterol level in both T1DM and T2DM groups (positive correlation). CONCLUSIONS    Our results are a good reason to expand research in this field in order to develop future individualized therapy for patients with diabetes based on modifying the intestinal microbiota composition, as a new method for controlling glycemia. The NGS method allows rapid identification of the DNA of all bacteria present in the sample and their taxonomic classification.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29657308">
                    <p>引言科学数据表明胃肠道微生物菌群可能影响1型（T1DM）和2型（T2DM）糖尿病的发展。 16S rRNA的序列分析鉴定出几百种肠道生态系统的细菌种类，其中大部分细菌无法培养。目标我们旨在评估成人T1DM和T2DM患者肠道微生物群的组成，并建立微生物检测数据与患者临床资料之间的联系。患者和方法我们检测了三组健康志愿者（n = 23），T1DM（n = 22）和T2DM（n = 23）三组粪便样本中分离的DNA。新一代测序方法（NGS）在MiSeq平台上进行。结果在门的水平，在所有组中，厚壁菌（&gt; 77％）肯定占上风。在分类水平L2（门）和L6（属）中，在细菌谱中显示出统计学显着性差异，特别是在T2DM组中。在T1DM和T2DM组中，1）T2DM组中细菌的几个属和血红蛋白A1c的百分比（负相关）和2）属于双歧杆菌属的细菌和高密度脂蛋白胆固醇水平（正相关）。结论我们的研究结果是扩大该领域研究的一个很好的理由，旨在开发基于修改肠道微生物群组成的糖尿病患者未来的个体化治疗，作为一种控制血糖的新方法。 NGS方法可以快速鉴定样本中存在的所有细菌的DNA及其分类分类。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=.">
                Journal: . [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29657308" target="_blank">
                PMID: 29657308
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-15
            </a>

            <a class="btn btn-sm btn-danger" title="Salamon D, Sroka-Oleksiak A, Kapusta P, Szopa M, Mrozińska S, Ludwig-Słomczyńska AH, Wołkow PP, Bulanda M, Klupa T, Małecki MT, Gosiewski T">
                Authors: Salamon D, Sroka-...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29657308">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>110. HLA-DRB1 Shared Epitope Alleles and Disease Activity Are Correlated with Reduced T Cell Receptor Repertoire Diversity in CD4+ T Cells in Rheumatoid Arthritis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29657145" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29657145" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29657145">
                    <p>Shared epitope (SE) alleles are the most significant genetic susceptibility locus in rheumatoid arthritis (RA); however, their target populations in CD4+ T cells are not well elucidated. We analyzed the association between SE alleles and the T cell receptor (TCR) repertoire diversity of naive and memory CD4+ T cells using next-generation sequencing (NGS).</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29657145">
                    <p>共享表位（SE）等位基因是类风湿性关节炎（RA）中最重要的遗传易感性位点;然而，它们在CD4 + T细胞中的目标群体没有被很好地阐明。我们使用新一代测序（NGS）分析了SE等位基因与幼稚和记忆CD4 + T细胞的T细胞受体（TCR）库多样性之间的关联。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Rheumatol.">
                Journal: J. Rheumatol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29657145" target="_blank">
                PMID: 29657145
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-15
            </a>

            <a class="btn btn-sm btn-danger" title="Sakurai K, Ishigaki K, Shoda H, Nagafuchi Y, Tsuchida Y, Sumitomo S, Kanda H, Suzuki A, Kochi Y, Yamamoto K, Fujio K">
                Authors: Sakurai K, Ishiga...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29657145">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>111. Feasibility of endobronchial ultrasound transbronchial needle aspiration for massively parallel next-generation sequencing in thoracic cancer patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29656758" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29656758" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29656758">
                    <p>Next-generation sequencing (NGS) allows for the identification of a growing number of therapeutic and prognostic molecular targets. However, NGS typically requires greater quantities of DNA than traditional molecular testing does. Endobronchial ultrasound transbronchial needle aspiration (EBUS-TBNA) is a minimally invasive procedure used to sample central thoracic lesions, but it is not well established whether this technique provides sufficient material for NGS.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29656758">
                    <p>新一代测序（NGS）可以识别越来越多的治疗和预后分子靶点。然而，与传统分子检测相比，NGS通常需要更多的DNA。支气管内超声经支气管针吸活检（EBUS-TBNA）是一种微创手术，用于对中央胸部病变进行取样，但尚不能确定该技术是否为NGS提供足够的材料。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Lung Cancer">
                Journal: Lung Cancer [IF=4.294]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29656758" target="_blank">
                PMID: 29656758
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Turner SR, Buonocore D, Desmeules P, Rekhtman N, Dogan S, Lin O, Arcila ME, Jones DR, Huang J">
                Authors: Turner SR, Buonoc...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5905717" target="_blank">
                PMC5905717
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29656758">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>112. Diagnostic outcomes for genetic testing of 70 genes in 8565 patients with epilepsy and neurodevelopmental disorders.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29655203" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29655203" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29655203">
                    <p>We evaluated &gt;8500 consecutive, unselected patients with epilepsy and neurodevelopmental disorders who underwent multigene panel testing to determine the average age at molecular diagnosis and diagnostic yield of 70 genes.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29655203">
                    <p>我们评估了&gt; 8500个连续的未经选择的患有癫痫和神经发育障碍的患者，这些患者接受多基因小组测试以确定分子诊断的平均年龄和70个基因的诊断产量。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Epilepsia">
                Journal: Epilepsia [IF=5.295]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29655203" target="_blank">
                PMID: 29655203
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Lindy AS, Stosser MB, Butler E, Downtain-Pickersgill C, Shanmugham A, Retterer K, Brandt T, Richard G, McKnight DA">
                Authors: Lindy AS, Stosser...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29655203">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>113. Three-dimensional printed PLA scaffold and human gingival stem cell-derived extracellular vesicles: a new tool for bone defect repair.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29653587" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29653587" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29653587">
                    <p>The role of bone tissue engineering in the field of regenerative medicine has been a main research topic over the past few years. There has been much interest in the use of three-dimensional (3D) engineered scaffolds (PLA) complexed with human gingival mesenchymal stem cells (hGMSCs) as a new therapeutic strategy to improve bone tissue regeneration. These devices can mimic a more favorable endogenous microenvironment for cells in vivo by providing 3D substrates which are able to support cell survival, proliferation and differentiation. The present study evaluated the in vitro and in vivo capability of bone defect regeneration of 3D PLA, hGMSCs, extracellular vesicles (EVs), or polyethyleneimine (PEI)-engineered EVs (PEI-EVs) in the following experimental groups: 3D-PLA, 3D-PLA + hGMSCs, 3D-PLA + EVs, 3D-PLA + EVs + hGMSCs, 3D-PLA + PEI-EVs, 3D-PLA + PEI-EVs + hGMSCs.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29653587">
                    <p>骨组织工程在再生医学领域的作用一直是过去几年的主要研究课题。人们对使用与人类牙龈间充质干细胞（hGMSC）复合的三维（3D）工程支架（PLA）作为改善骨组织再生的新治疗策略一直备受关注。通过提供能够支持细胞存活，增殖和分化的3D底物，这些装置可以模拟体内细胞的更有利的内源性微环境。本研究评估了3D PLA，hGMSCs，细胞外囊泡（EVs）或聚乙烯亚胺（PEI）工程电动车（PEI-EVs）在以下实验组中的骨缺损再生的体外和体内能力：3D-PLA， 3D-PLA + hGMSC，3D-PLA + EV，3D-PLA + EV + hGMSC，3D-PLA + PEI-EV，3D-PLA + PEI-EV + hGMSC。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Stem Cell Res Ther">
                Journal: Stem Cell Res Ther [IF=4.211]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29653587" target="_blank">
                PMID: 29653587
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-13
            </a>

            <a class="btn btn-sm btn-danger" title="Diomede F, Gugliandolo A, Cardelli P, Merciaro I, Ettorre V, Traini T, Bedini R, Scionti D, Bramanti A, Nanci A, Caputi S, Fontana A, Mazzon E, Trubiani O">
                Authors: Diomede F, Guglia...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5899396" target="_blank">
                PMC5899396
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29653587">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>114. Genetic analysis of Charcot-Marie-Tooth disease in Denmark and the implementation of a next generation sequencing platform.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29653220" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29653220" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29653220">
                    <p>Charcot-Marie-Tooth disease (CMT) is a heterogeneous group of hereditary polyneuropathies. Variants in more than 80 different genes have been associated with the disorder. In recent years, the introduction of next generation sequencing (NGS) techniques have completely changed the genetic diagnostic approach from the analysis of a handful of genes to the analysis of all genes associated with CMT in a single run. In this study we describe the CMT diagnostics in Denmark in 1992-2012, prior to the implementation of NGS, by combining laboratory- and national registry data. We investigate the effect of implementing a targeted NGS approach of 63 genes associated with CMT in the diagnostic laboratory setting. This was performed by analyzing a cohort of 195 samples from patients previously analyzed by Sanger sequencing and quantitative analysis for the common causes of CMT without reaching a molecular diagnosis. A total of 1442 CMT analyses were performed in Denmark in the period 1992-2012; a disease-causing variant was detected in 21.6% of the cases. Interestingly, the diagnosis was genetically confirmed in significantly more women than men; 25.9% compared to18.5%. In our study cohort, we found a 5.6% increase in the diagnostic yield with the introduction of a targeted NGS approach.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29653220">
                    <p>Charcot-Marie-Tooth疾病（CMT）是遗传性多发性神经病的异质性群体。超过80种不同基因的变异与疾病有关。近年来，下一代测序（NGS）技术的引入已经彻底改变了从少数基因分析到在一次运行中分析与CMT相关的所有基因的基因诊断方法。在这项研究中，我们描述了在实施NGS之前，1992  -  2012年在丹麦的CMT诊断，通过结合实验室和国家登记数据。我们调查在诊断实验室环境中实施与CMT相关的63个基因的靶向NGS方法的效果。这是通过分析来自患者的195个样品的队列进行的，所述患者以前通过Sanger测序和CMT的常见原因的定量分析进行分析，而没有达到分子诊断。 1992  -  2012年期间在丹麦进行了1442次CMT分析;在21.6％的病例中检测到致病变体。有趣的是，这一诊断在女性明显多于男性中得到了遗传学证实; 25.9％，而18.5％。在我们的研究队列中，我们发现通过引入有针对性的NGS方法，诊断率提高了5.6％。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur J Med Genet">
                Journal: Eur J Med Genet [IF=2.137]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29653220" target="_blank">
                PMID: 29653220
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-11
            </a>

            <a class="btn btn-sm btn-danger" title="Vaeth S, Christensen R, Dunø M, Lildballe DL, Thorsen K, Vissing J, Svenstrup K, Hertz JM, Andersen H, Jensen UB">
                Authors: Vaeth S, Christen...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29653220">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>115. [Diagnosis of two neonates with galactosemia by using next generation sequencing].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29653003" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29653003" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29653003">
                    <p>To explore the genetic basis of two neonates suspected for galactosemia.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29653003">
                    <p>探讨怀疑半乳糖血症的两名新生儿的遗传基础。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Zhonghua Yi Xue Yi Chuan Xue Za Zhi">
                Journal: Zhonghua Yi Xue Yi Chuan Xue Za Zhi [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29653003" target="_blank">
                PMID: 29653003
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-10
            </a>

            <a class="btn btn-sm btn-danger" title="Zhang H, Chen D, Liu C, Liu X, Gai Z, Liu Y">
                Authors: Zhang H, Chen D, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29653003">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>116. [Analysis of pathological mutation in a Chinese pedigree affected with familial exudative vitreoretinopathy].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29652990" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29652990" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29652990">
                    <p>To detect potential mutation in a Chinese pedigree affected with familial exudative vitreoretinopathy (FEVR).</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29652990">
                    <p>检测受家族性渗出性玻璃体视网膜病变（FEVR）影响的中国家系中的潜在突变。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Zhonghua Yi Xue Yi Chuan Xue Za Zhi">
                Journal: Zhonghua Yi Xue Yi Chuan Xue Za Zhi [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29652990" target="_blank">
                PMID: 29652990
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-10
            </a>

            <a class="btn btn-sm btn-danger" title="Su N, Qin L, Wang H, Xiao H, Guo Q, Li T, Liao S">
                Authors: Su N, Qin L, Wang...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29652990">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>117. [Ataxia with oculomotor apraxia type 4 detected by next-generation sequencing].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29652299" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29652299" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29652299">
                    <p>Ataxias with oculomotor apraxia (AOA) belong to autosomal recessive ataxias. Their common feature is oculomotor apraxia: inability to coordinate eye movements not due to muscle weakness. Next-generation sequencing (NGS) gives unique opportunities of rare disorders diagnostics and discovering of new forms, including AOA. In 2015, AOA type 4 produced by PNKP mutations was delineated in a group of Portuguese patients. We diagnosed AOA4 in a 9-year-old boy from Byelorussian family. He presented with ataxia since 2 years and deterioration in 8 years, oculomotor apraxia, dystonic hyperkinesia, dysarthria, polyneuropathy, borderline/mildly impaired intelligence, cerebellar atrophy on MRI and moderate hypercholesterolemia. Panel NGS detected two PNKP mutations: c.1123G&gt;T (p.Gly375Trp) common in Portuguese patients, and novel c.1270_1283dupACAAACCCAGACGC (p.Ala429fs). This is one of a few world AOA4 cases and first non-Portuguese case with &#39;Portuguese&#39; common mutation. The case illustrates NGS diagnostic value, particularly in rare heterogeneous disorders like AOA.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29652299">
                    <p>伴有动眼神经失用症（AOA）的Ataxias属于常染色体隐性遗传性共济失调。它们的共同特征是动眼神经失用：无法协调眼肌运动，而不是由于肌肉无力。新一代测序（NGS）为罕见疾病诊断和发现新形式（包括AOA）提供了独特的机会。 2015年，由PNKP突变产生的AOA4型在一组葡萄牙患者中划定。我们从一个来自白俄罗斯家庭的9岁男孩中诊断出AOA4。 2年后出现共济失调，8年内恶化，动眼神经失用，肌张力过度运动障碍，构音障碍，多发性神经病，智力交界性/轻度智力障碍，MRI小脑萎缩和中度高胆固醇血症。 Panel NGS检测到两种PNKP突变：在葡萄牙患者中常见的c.1123G&gt; T（p.Gly375Trp）和新的c.1270_1283dupACAAACCCAGACGC（p.Ala429fs）。这是几起世界AOA4案件之一，也是第一起非葡萄牙案件，有&#39;葡萄牙&#39;常见突变。该案例说明了NGS的诊断价值，特别是在罕见的异质性疾病如AOA中。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Zh Nevrol Psikhiatr Im S S Korsakova">
                Journal: Zh Nevrol Psikhiatr Im S S Korsakova [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29652299" target="_blank">
                PMID: 29652299
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Rudenskaya GE, Surkova EI, Konovalov FA">
                Authors: Rudenskaya GE, Su...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29652299">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>118. Testosterone Deficiency, Weakness, and Multimorbidity in Men.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29651127" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29651127" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29651127">
                    <p>The purposes of this study were to evaluate the association between total testosterone (TT) deficiency and weakness on multimorbidity in men. Analyses were performed to examine the prevalence of multimobidity among young, middle-aged, and older men, with and without testosterone deficiency. Multivariate logistic models were also used to determine the association between age-specific TT tertiles and multimorbidity, adjusting for key sociodemographic variables, as well as a secondary analysis adjusted for grip strength. Multimorbidity was more prevalent among men with testosterone deficiency, compared to normal TT in the entire group (36.6% vs 55.2%; p &lt; 0.001); however, differences were only seen within young (testosterone deficiency: 36.4%; normal TT: 13.5%; p &lt; 0.001) and older men (testosterone deficiency: 75.0%; normal TT: 61.5%; p &lt; 0.001). Robust associations were found between the age-specific low-TT (OR: 2.87; 95%CI: 2.14-3.83) and moderate-TT (OR: 1.67; 95%CI: 1.27-2.20) tertiles (reference high-TT) and multimorbidity. Secondary analysis demonstrated that both low TT (OR: 1.82; 95%CI: 1.29-2.55) and moderate-TT (OR: 1.31; 95%CI: 1.01-1.69) were associated with multimorbidity, even after adjusting for obesity (OR: 1.75; 95%CI: 1.07-2.87) and NGS (OR: 1.21 per 0.05 unit lower NGS). Low TT and weakness in men were independently associated with multimorbidity at all ages; however, multimorbidity was more prevalent among young and older men with testosterone deficiency.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29651127">
                    <p>这项研究的目的是评估总睾酮（TT）缺乏与男性多发性软弱症之间的关系。进行分析以检查年轻人，中年人和老年人多动症的患病率，有无睾酮缺乏症。多变量logistic模型也用于确定年龄特异性TT三分位数与多发病率之间的关联，调整关键社会人口统计学变量，以及调整握力强度的二次分析。睾丸激素缺乏男性患者多发病率更高，与整个组中正常TT相比（36.6％vs 55.2％; p &lt;0.001）;然而，只有年轻人（睾酮缺乏症：36.4％;正常TT：13.5％; p &lt;0.001）和年长男性（睾酮缺乏症：75.0％;正常TT：61.5％; p &lt;0.001）中观察到差异。年龄特异性低TT（OR：2.87; 95％CI：2.14-3.83）和中等TT（OR：1.67; 95％CI：1.27-2.20）三分位数（参考高TT）和multimorbidity。二级分析显示即使调整肥胖（OR：1.82; 95％CI：1.29-2.55）和中度TT（OR：1.31; 95％CI：1.01-1.69）也与多发性相关1.75; 95％CI：1.07-2.87）和NGS（OR：每降低0.05单位NGS为1.21）。男性低TT和无力与所有年龄段的多发病独立相关;然而，睾丸激素缺乏的青年和老年男性多发病率更高。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29651127" target="_blank">
                PMID: 29651127
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-12
            </a>

            <a class="btn btn-sm btn-danger" title="Peterson MD, Belakovskiy A, McGrath R, Yarrow JF">
                Authors: Peterson MD, Bela...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5897450" target="_blank">
                PMC5897450
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29651127">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>119. Impact of sequencing depth on the characterization of the microbiome and resistome.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29651035" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29651035" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29651035">
                    <p>Developments in high-throughput next generation sequencing (NGS) technology have rapidly advanced the understanding of overall microbial ecology as well as occurrence and diversity of specific genes within diverse environments. In the present study, we compared the ability of varying sequencing depths to generate meaningful information about the taxonomic structure and prevalence of antimicrobial resistance genes (ARGs) in the bovine fecal microbial community. Metagenomic sequencing was conducted on eight composite fecal samples originating from four beef cattle feedlots. Metagenomic DNA was sequenced to various depths, D1, D0.5 and D0.25, with average sample read counts of 117, 59 and 26 million, respectively. A comparative analysis of the relative abundance of reads aligning to different phyla and antimicrobial classes indicated that the relative proportions of read assignments remained fairly constant regardless of depth. However, the number of reads being assigned to ARGs as well as to microbial taxa increased significantly with increasing depth. We found a depth of D0.5 was suitable to describe the microbiome and resistome of cattle fecal samples. This study helps define a balance between cost and required sequencing depth to acquire meaningful results.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29651035">
                    <p>高通量下一代测序（NGS）技术的发展已经迅速提高了对整体微生物生态学的理解以及不同环境中特定基因的发生和多样性。在本研究中，我们比较了不同测序深度的能力，以产生关于牛粪便微生物群落中的抗生素抗性基因（ARG）的分类结构和普遍性的有意义的信息。宏基因组测序是在来自四个肉牛饲养场的八个复合粪便样品上进行的。将宏基因组DNA测序至各种深度D1，D0.5和D0.25，平均样品读数分别为117,59和2600万。对齐不同门类和抗菌类的读数相对丰度的比较分析表明，无论深度如何，阅读任务的相对比例保持相当恒定。然而，随着深度的增加，分配给ARG以及微生物分类单元的读数显着增加。我们发现D0.5的深度适合描述牛粪便样品的微生物组和抗生物组织。这项研究有助于定义成本与所需测序深度之间的平衡，以获得有意义的结果。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29651035" target="_blank">
                PMID: 29651035
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-12
            </a>

            <a class="btn btn-sm btn-danger" title="Zaheer R, Noyes N, Ortega Polo R, Cook SR, Marinier E, Van Domselaar G, Belk KE, Morley PS, McAllister TA">
                Authors: Zaheer R, Noyes N...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5897366" target="_blank">
                PMC5897366
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29651035">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>120. [Towards Development of Innovative Cancer Therapies - Trans-OMICS Approach].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29650893" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29650893" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29650893">
                    <p>Comprehensive genomic and transcriptome analyses using next-generation sequencing(NGS)analysis has lead a discovery of a variety of novel driver gene mutations and new therapeutic targets for cancer patients, and has remarkably improved outcome of the patients through the development novel molecular targeting drugs. Even so, in so-called intractable or refractory cancers, those &quot;druggable&quot;alterations common to the diseases are rarely found due to the high diversity of the tumor. Furthermore, most of molecular target therapy is known to acquire the resistance to the drug by means of multiple factors such as up-regulation of the partially inhibited pathway, mutation of the target, activation of alternative pathways, histological translocation, and oncogene de-addiction. Understanding of intra-tumoral heterogeneity and tumor-stromal crosstalk in tumor microenvironment with consequence of biological network re-construction are also of key importance to overcome the resistance. These suggest the limitation of mono-layer OMIC approach focusing on genome and/or cancer cell alone to identify truly effective therapeutic target and biomarker. Under these circumstances,&quot;Trans-OMICS&quot; concept has emerged as a novel approach to clarify a global biochemical network across multiple omics layers(eg genome, transcriptome, proteome, and metabolome)directly correct with a variable phenotype by use of both multi-omic measurements and computational data integration. This approach has great potential for drug discovery and clinical implementation of omics-based cancer medicine. We introduce here the outline of technologies and analysis for Trans-OMICS approach, and review for the recent studies in oncology research with showing our recent attempt.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29650893">
                    <p>使用新一代测序（NGS）分析的全面基因组和转录组分析已经导致发现各种新的驱动基因突变和癌症患者的新治疗靶标，并且通过开发新的分子靶向药物显着改善了患者的结果。即使如此，在所谓的难治性或难治性癌症中，由于肿瘤的高度多样性，很少发现这些疾病常见的“可药代”改变。此外，已知大多数分子靶向疗法通过多种因素获得对药物的抗性，所述多种因素例如上调部分抑制的途径，靶标的突变，替代途径的激活，组织学易位和癌基因去沉积。了解肿瘤微环境中的肿瘤内异质性和肿瘤 - 基质串扰以及生物网络重建的结果对于克服耐药性也是至关重要的。这些表明单层OMIC方法的局限性集中在仅基因组和/或癌细胞上以确定真正有效的治疗靶标和生物标志物。在这些情况下，“跨OMICS”的概念已经出现，作为一种新颖的方法来阐明跨多个组学层（如基因组，转录组，蛋白质组和代谢组）的全球生物化学网络，通过使用多元组学测量和计算数据集成。这种方法在基于组学的癌症药物的药物发现和临床实施方面具有巨大的潜力。我们在这里介绍Trans-OMICS方法的技术和分析概要，并回顾最近在肿瘤研究方面的研究，并展示我们最近的尝试。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Gan To Kagaku Ryoho">
                Journal: Gan To Kagaku Ryoho [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29650893" target="_blank">
                PMID: 29650893
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar
            </a>

            <a class="btn btn-sm btn-danger" title="Kawabata-Iwakawa R, Nishiyama M">
                Authors: Kawabata-Iwakawa ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29650893">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>121. Next-Generation Sequencing of the Complete Mitochondrial Genome of the Endangered Species Black Lion Tamarin Leontopithecus chrysopygus (Primates) and Mitogenomic Phylogeny Focusing on the Callitrichidae Family.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29650540" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29650540" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29650540">
                    <p>We describe the complete mitochondrial genome sequence of the Black Lion Tamarin, an endangered primate species endemic to the Atlantic Rainforest of Brazil. We assembled the Leontopithecuschrysopygus mitogenome, through analysis of 523M base pairs (bp) of short reads produced by next-generation sequencing (NGS) on the Illumina Platform, and investigated the presence of nuclear mitochondrial pseudogenes and heteroplasmic sites. Additionally, we conducted phylogenetic analyses using all complete mitogenomes available for primates until June 2017. The single circular mitogenome of BLT showed organization and arrangement that are typical for other vertebrate species, with a total of 16618 bp, containing 13 protein-coding genes, 22 transfer RNA genes, 2 ribosomal RNA genes, and 1 non-coding region (D-loop region). Our full phylogenetic tree is based on the most comprehensive mitogenomic dataset for Callitrichidae species to date, adding new data for the Leontopithecus genus, and discussing previous studies performed on primates. Moreover, the mitochondrial genome reported here consists of a robust mitogenome with 3000X coverage, which certainly will be useful for further phylogenetic and evolutionary analyses of Callitrichidae and higher taxa.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29650540">
                    <p>我们描述了黑狮蜥蜴的完整线粒体基因组序列，这是巴西大西洋热带雨林特有的濒危灵长类物种。我们通过分析在Illumina平台上由下一代测序（NGS）产生的短读数的523M碱基对（bp），并且研究核线粒体假基因和异质性位点的存在来组装Leontopithecuschrysopygus有丝分裂原。此外，我们使用灵长类动物所有完整的有丝分裂基因进行系统发育分析，直到2017年6月.BLT的单环状有丝分裂组织显示出其他脊椎动物物种典型的组织和排列，总共16618bp，含有13种蛋白质编码基因，22种转移RNA基因，2个核糖体RNA基因和1个非编码区（D环区）。我们的完整系统发育树是基于迄今为止Callitrichidae物种的最全面的有丝分裂数据集，增加了Leontopithecus属的新数据，并讨论了以前对灵长类动物进行的研究。此外，这里报道的线粒体基因组由一个3000X覆盖率的强有力的有丝分裂原组成，这对于进一步对胼rich体科和更高级的分类群进行系统发育和进化分析当然是有用的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=G3 (Bethesda)">
                Journal: G3 (Bethesda) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29650540" target="_blank">
                PMID: 29650540
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-12
            </a>

            <a class="btn btn-sm btn-danger" title="de Freitas PD, Mendez FL, Chávez-Congrains K, Galetti PM, Coutinho LL, Pissinatti A, Bustamante CD">
                Authors: de Freitas PD, Me...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29650540">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>122. Preimplantation High Resolution HLA Sequencing Using Next Generation Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29649618" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29649618" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29649618">
                    <p>Hematopoietic Stem Cell Transplantation (HSCT) is the only therapeutic option in a number of heritable hematologic disorders as well as hematologic cancers. Many parents and families fail to find an HLA-identical donor for their affected family member. In such cases, conceiving for a savior baby remains the only option especially in countries without access to national registries. By means of Next Generation Sequencing (NGS) techniques, in a single experiment on single cell products of IVF, a healthy HLA-identical embryo can be implanted in the uterus of a concerned mother. The patient can therefore benefit from cord blood stem cell transplantation along with confirming that the fetuses are not suffering from the heritable disorder. This study is an attempt to study the feasibility of Preimplantation HLA Sequencing (PHS) on single blastomeres using Next Generation Sequencing. Two couples previously undergone preimplantation genetic diagnosis (PGD) of Beta Thalassemia and their overall 10 embryos are studied and their 5 HLA loci are typed in high resolution through multiple displacement amplification (MDA) and NGS of single cells. For 88.9% of the 90 HLA alleles, conclusive HLA typing in 4 digit sets was made. HLA alleles were typed; one ambiguity in allelic group and 4 ambiguities in protein level were observed which were then unraveled by haplotype analysis. Amplification efficiency was 93.3% with an allele drop-out (ADO) rate of 22.2% (6 alleles dropped from a maximum of 27 possible ADOs). In this study, the feasibility of a new method of Preimplantation HLA Sequencing (PHS) via combining the state-of-the-art techniques used in single-cell whole genome amplification, preimplantation genetic diagnosis, and high-resolution HLA typing by Next-Generation Sequencing has been shown. This method can make preimplantation HLA sequencing a practicable technique in families desperate for a HLA-matched donor.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29649618">
                    <p>造血干细胞移植（HSCT）是许多遗传性血液病和血液癌的唯一治疗选择。许多父母和家人未能找到与其受感染家庭成员HLA相同的捐赠者。在这种情况下，特别是在没有国家注册机构的国家，构想拯救婴儿仍然是唯一的选择。通过下一代测序（NGS）技术，在一次关于IVF单细胞产物的实验中，可将健康的HLA相同的胚胎植入有关母亲的子宫内。因此，患者可以从脐血干细胞移植中受益，同时确认胎儿没有患遗传性疾病。本研究旨在研究使用下一代测序技术对单个卵裂球进行Preimplantation HLA测序（PHS）的可行性。研究了两对以前接受β地中海贫血植入前基因诊断（PGD）及其总共10个胚胎的研究对象，并通过多重置换扩增（MDA）和单细胞NGS以高分辨率分型了5个HLA基因座。对于90个HLA等位基因中的88.9％，在4位数组中确定了HLA分型。 HLA等位基因分型;观察等位基因组中的一个模糊性和蛋白质水平的4个模糊性，然后通过单体型分析解开这些模糊性。扩增效率为93.3％，等位基因剔除率（ADO）为22.2％（最多27个可能的ADO中有6个等位基因下降）。在这项研究中，通过结合单细胞全基因组扩增，植入前遗传学诊断和高分辨率HLA分型的先进技术，通过Next-Cell技术结合了植入前HLA测序（PHS）的新方法的可行性，生成测序已被显示。这种方法可以使植入前的HLA测序成为绝望于HLA匹配供体的家族的一种可行技术。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Biol. Blood Marrow Transplant.">
                Journal: Biol. Blood Marrow Transplant. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29649618" target="_blank">
                PMID: 29649618
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-09
            </a>

            <a class="btn btn-sm btn-danger" title="Rafati M, Akhondi MM, Sadeghi MR, Tara SZ, Ghaffari SR">
                Authors: Rafati M, Akhondi...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29649618">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>123. Validation of CZECANCA (CZEch CAncer paNel for Clinical Application) for targeted NGS-based analysis of hereditary cancer syndromes.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29649263" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29649263" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29649263">
                    <p>Carriers of mutations in hereditary cancer predisposition genes represent a small but clinically important subgroup of oncology patients. The identification of causal germline mutations determines follow-up management, treatment options and genetic counselling in patients&#39; families. Targeted next-generation sequencing-based analyses using cancer-specific panels in high-risk individuals have been rapidly adopted by diagnostic laboratories. While the use of diagnosis-specific panels is straightforward in typical cases, individuals with unusual phenotypes from families with overlapping criteria require multiple panel testing. Moreover, narrow gene panels are limited by our currently incomplete knowledge about possible genetic dispositions.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29649263">
                    <p>遗传性癌症易感基因突变的携带者代表肿瘤患者的小但临床上重要的亚群。因果种系突变的鉴定决定了患者家属的随访管理，治疗选择和遗传咨询。诊断实验室已经迅速采用了针对高风险人群使用癌症特异性组的快速有针对性的下一代基于测序的分析。虽然在典型情况下使用诊断特异性小组很简单，但具有重叠标准的家族中具有不同表型的个体需要多次小组检测。此外，由于我们目前关于可能遗传倾向的不完整知识，狭义基因小组受到限制。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29649263" target="_blank">
                PMID: 29649263
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Soukupova J, Zemankova P, Lhotova K, Janatova M, Borecka M, Stolarova L, Lhota F, Foretova L, Machackova E, Stranecky V, Tavandzis S, Kleiblova P, Vocka M, Hartmannova H, Hodanova K, Kmoch S, Kleibl Z">
                Authors: Soukupova J, Zema...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29649263">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>124. Simultaneous VENTANA IHC and RT-PCR testing of ALK status in Chinese non-small cell lung cancer patients and response to crizotinib.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29642919" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29642919" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29642919">
                    <p>ALK rearrangement-advanced NSCLC patients respond to crizotinib. ALK rearrangement is currently determined with RT-PCR. VENTANA IHC is a standard method to identify ALK protein overexpression in NSCLC; however, VENTANA IHC has rarely been used to determine the response to crizotinib in Chinese patients with NSCLC and ALK overexpression. To better clarify the clinical implication of VENTANA IHC to detect ALK rearrangements, we conducted this study to analyze VENTANA IHC and RT-PCR in a large cohort of Chinese patients with NSCLC undergoing screening for ALK rearrangements.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29642919">
                    <p>ALK重排 - 晚期NSCLC患者对克唑替尼有反应。 ALK重排目前通过RT-PCR来确定。 VENTANA IHC是鉴定NSCLC中ALK蛋白过表达的标准方法;然而，VENTANA IHC很少用于确定中国NSCLC和ALK过表达患者对克唑替尼的反应。为了更好地阐明VENTANA IHC检测ALK重排的临床意义，我们开展了这项研究，以分析VENTANA IHC和RT-PCR在中国患有NSCLC的大型队列中进行ALK重排筛查。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Transl Med">
                Journal: J Transl Med [IF=3.786]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29642919" target="_blank">
                PMID: 29642919
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-11
            </a>

            <a class="btn btn-sm btn-danger" title="Xu CW, Wang WX, Chen YP, Chen Y, Liu W, Zhong LH, Chen FF, Zhuang W, Song ZB, Chen XH, Huang YJ, Guan YF, Yi X, Lv TF, Zhu WF, Lu JP, Wang XJ, Shi Y, Lin XD, Chen G, Song Y">
                Authors: Xu CW, Wang WX, C...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896026" target="_blank">
                PMC5896026
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29642919">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>125. Genomic Profiling on an Unselected Solid Tumor Population Reveals a Highly Mutated Wnt/β-Catenin Pathway Associated with Oncogenic EGFR Mutations.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29642553" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29642553" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29642553">
                    <p>Oncogenic epidermal growth factor receptors (EGFRs) can recruit key effectors in diverse cellular processes to propagate oncogenic signals. Targeted and combinational therapeutic strategies have been successfully applied for treating EGFR-driven cancers. However, a main challenge in EGFR therapies is drug resistance due to mutations, oncogenic shift, alternative signaling, and other potential mechanisms. To further understand the genetic alterations associated with oncogenic EGFRs and to provide further insight into optimal and personalized therapeutic strategies, we applied a proprietary comprehensive next-generation sequencing (NGS)-based assay of 435 genes to systematically study the genomic profiles of 1565 unselected solid cancer patient samples. We found that activating EGFR mutations were predominantly detected in lung cancer, particularly in non-small cell lung cancer (NSCLC). The mutational landscape of EGFR-driven tumors covered most key signaling pathways and biological processes. Strikingly, the Wnt/β-catenin pathway was highly mutated (48 variants detected in 46% of the EGFR-driven tumors), and its variant number topped that in the TP53/apoptosis and PI3K-AKT-mTOR pathways. Furthermore, an analysis of mutation distribution revealed a differential association pattern of gene mutations between EGFR exon 19del and EGFR L858R. Our results confirm the aggressive nature of the oncogenic EGFR-driven tumors and reassure that a combinational strategy should have advantages over an EGFR-targeted monotherapy and holds great promise for overcoming drug resistance.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29642553">
                    <p>致癌性表皮生长因子受体（EGFR）可以在不同的细胞过程中募集关键效应物以传播致癌信号。目标和组合治疗策略已成功应用于治疗EGFR驱动的癌症。然而，EGFR疗法中的主要挑战是由于突变，致癌转变，替代信号传导和其他潜在机制引起的耐药性。为了进一步了解与致癌EGFRs相关的基因改变，并为最佳和个性化的治疗策略提供进一步的见解，我们应用了专有的基于新一代测序（NGS）的435个基因检测系统研究了1565个未选择的固体癌症患者样品。我们发现激活EGFR突变主要在肺癌中检测到，特别是在非小细胞肺癌（NSCLC）中。 EGFR驱动的肿瘤的突变景观覆盖了大多数关键的信号传导途径和生物学过程。引人注目的是，Wnt /β-连环蛋白途径高度突变（在46％的EGFR驱动的肿瘤中检测到48个变体），其变体数在TP53 /细胞凋亡和PI3K-AKT-mTOR途径中突出。此外，突变分布分析揭示了EGFR外显子19del和EGFR L858R之间基因突变的差异性关联模式。我们的研究结果证实了致癌性EGFR驱动的肿瘤的侵袭性，并确保组合策略应该比EGFR靶向的单一疗法具有优势，并且具有克服抗药性的巨大前景。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Pers Med">
                Journal: J Pers Med [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29642553" target="_blank">
                PMID: 29642553
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-09
            </a>

            <a class="btn btn-sm btn-danger" title="Jiang J, Protopopov A, Sun R, Lyle S, Russell M">
                Authors: Jiang J, Protopop...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29642553">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>126. Monitoring of the Apple Fruit Moth: Detection of Genetic Variation and Structure Applying a Novel Multiplex Set of 19 STR Markers.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29642498" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29642498" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29642498">
                    <p>The apple fruit moth Argyresthia conjugella (Lepidoptera, Yponomeutidae) is a seed predator of rowan (Sorbus aucuparia) and is distributed in Europe and Asia. In Fennoscandia (Finland, Norway and Sweden), rowan fruit production is low every 2-4 years, and apple (Malus domestica) functions as an alternative host, resulting in economic loss in apple crops in inter-mast years. We have used Illumina MiSeq sequencing to identify a set of 19 novel tetra-nucleotide short tandem repeats (STRs) in Argyresthia conjugella. Such motifs are recommended for genetic monitoring, which may help to determine the eco-evolutionary processes acting on this pest insect. The 19 STRs were optimized and amplified into five multiplex PCR reactions. We tested individuals collected from Norway and Sweden (n = 64), and detected very high genetic variation (average 13.6 alleles, He = 0.75) compared to most other Lepidoptera species studied so far. Spatial genetic differentiation was low and gene flow was high in the test populations, although two non-spatial clusters could be detected. We conclude that this set of genetic markers may be a useful resource for population genetic monitoring of this economical important insect species.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29642498">
                    <p>苹果果蛾Argyresthia conjugella（鳞翅目，Yponomeutidae）是花楸（欧洲花楸）的种子捕食者，分布于欧洲和亚洲。在芬诺斯堪的亚（芬兰，挪威和瑞典），花楸果实产量每2  -  4年就会降低，而苹果（苹果）会作为替代寄主，导致蚜虫年间苹果作物的经济损失。我们使用Illumina MiSeq测序鉴定了Argyresthia conjugella中的一组19个新型四核苷酸短串联重复序列（STRs）。这种图案被推荐用于遗传监测，这可能有助于确定作用于这种害虫的生态进化过程。 19个STR被优化并扩增成5个多重PCR反应。我们测试了从挪威和瑞典（n = 64）收集的个体，并且与迄今研究的大多数其他鳞翅目物种相比，检测到非常高的遗传变异（平均13.6个等位基因，He = 0.75）。尽管可以检测到两个非空间簇，但空间遗传分化低并且测试群体中的基因流动高。我们得出这样的结论：这组遗传标记可能是这种经济重要昆虫种群的遗传监测的有用资源。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Molecules">
                Journal: Molecules [IF=2.861]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29642498" target="_blank">
                PMID: 29642498
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-08
            </a>

            <a class="btn btn-sm btn-danger" title="Elameen A, Eiken HG, Fløystad I, Knudsen G, Hagen SB">
                Authors: Elameen A, Eiken ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29642498">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>127. New Challenges in Targeting Signaling Pathways in Acute Lymphoblastic Leukemia by NGS Approaches: An Update.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29642462" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29642462" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29642462">
                    <p>The identification and study of genetic alterations involved in various signaling pathways associated with the pathogenesis of acute lymphoblastic leukemia (ALL) and the application of recent next-generation sequencing (NGS) in the identification of these lesions not only broaden our understanding of the involvement of various genetic alterations in the pathogenesis of the disease but also identify new therapeutic targets for future clinical trials. The present review describes the main deletions, amplifications, sequence mutations, epigenetic lesions, and new structural DNA rearrangements detected by NGS in B-ALL and T-ALL and their clinical importance for therapeutic procedures. We reviewed the molecular basis of pathways including transcriptional regulation, lymphoid differentiation and development, TP53 and the cell cycle, RAS signaling, JAK/STAT, NOTCH, PI3K/AKT/mTOR, Wnt/β-catenin signaling, chromatin structure modifiers, and epigenetic regulators. The implementation of NGS strategies has enabled important mutated genes in each pathway, their associations with the genetic subtypes of ALL, and their outcomes, which will be described further. We also discuss classic and new cryptic DNA rearrangements in ALL identified by mRNA-seq strategies. Novel cooperative abnormalities in ALL could be key prognostic and/or predictive biomarkers for selecting the best frontline treatment and for developing therapies after the first relapse or refractory disease.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29642462">
                    <p>涉及与急性淋巴细胞白血病（ALL）的发病机理相关的各种信号传导途径的基因改变的鉴定和研究以及最近的下一代测序（NGS）在识别这些病变中的应用不仅扩大了我们对疾病发病机制中的各种遗传改变，但也为未来的临床试验确定新的治疗靶点。本综述介绍了NGS在B-ALL和T-ALL中检测到的主要缺失，扩增，序列突变，表观遗传损伤和新的结构DNA重排及其对治疗过程的临床意义。我们回顾了包括转录调控，淋巴样分化和发育，TP53和细胞周期，RAS信号，JAK / STAT，NOTCH，PI3K / AKT / mTOR，Wnt /β-连环蛋白信号传导，染色质结构修饰和表观遗传的途径的分子基础监管机构。 NGS策略的实施使每条途径中的重要突变基因，它们与ALL的遗传亚型的关联及其结果，将进一步描述。我们还讨论了通过mRNA-seq策略识别的ALL中经典和新的隐性DNA重排。 ALL中的新型协同异常可能是选择最佳前线治疗和在第一次复发或难治性疾病后开发治疗的关键预后和/或预测性生物标志物。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancers (Basel)">
                Journal: Cancers (Basel) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29642462" target="_blank">
                PMID: 29642462
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-07
            </a>

            <a class="btn btn-sm btn-danger" title="Montaño A, Forero-Castro M, Marchena-Mendoza D, Benito R, Hernández-Rivas JM">
                Authors: Montaño A, Forero...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29642462">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>128. Pre-clinical validation of a next generation sequencing testing panel.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29641993" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29641993" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29641993">
                    <p>Next Generation Sequencing (NGS) has become a useful tool for gene mutation testing which is required for targeted therapies. The aim of this study was to validate the GeneRead QIAact Actionable Insights Tumor Panel (Qiagen) on the GeneReader System in a diagnostic laboratory setting.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29641993">
                    <p>新一代测序（NGS）已成为靶向治疗所需的基因突变检测的有用工具。本研究的目的是在诊断实验室环境中对GeneReader系统上的GeneRead QIAact Actionable Insights Tumor Panel（Qiagen）进行验证。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Exp. Mol. Pathol.">
                Journal: Exp. Mol. Pathol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29641993" target="_blank">
                PMID: 29641993
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-08
            </a>

            <a class="btn btn-sm btn-danger" title="Lüsebrink J, Pieper M, Tillmann RL, Brockmann M, Schildgen O, Schildgen V">
                Authors: Lüsebrink J, Piep...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29641993">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>129. Systematic evaluation of a targeted gene capture sequencing panel for molecular diagnosis of retinitis pigmentosa.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29641573" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29641573" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29641573">
                    <p>Inherited eye diseases are major causes of vision loss in both children and adults. Inherited eye diseases are characterized by clinical variability and pronounced genetic heterogeneity. Genetic testing may provide an accurate diagnosis for ophthalmic genetic disorders and allow gene therapy for specific diseases.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29641573">
                    <p>继承的眼病是儿童和成年人视力丧失的主要原因。遗传性眼病的特征是临床变异性和显着的遗传异质性。基因检测可以为眼科遗传疾病提供准确的诊断，并为特定疾病提供基因治疗。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29641573" target="_blank">
                PMID: 29641573
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Huang H, Chen Y, Chen H, Ma Y, Chiang PW, Zhong J, Liu X, Asan J, Wu Y, Su X, Li J, Deng Y, Huang X, Zhang Y, Li N, Fan Y, Wang L, Tang J, Shen M, Chen X, Zhang D, Te S, Banerjee H, Liu M, Qi X">
                Authors: Huang H, Chen Y, ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5894961" target="_blank">
                PMC5894961
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29641573">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>130. Acquired resistance to AZD9291 as an upfront treatment is dependent on ERK signaling in a preclinical model.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29641535" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29641535" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29641535">
                    <p>AZD9291 (osimertinib) is approved for standard care in patients with EGFR T790M-positive non-small cell lung cancer (NSCLC) after prior EGFR TKI progression. Furthermore, AZD9291 is now being evaluated as a first-line treatment for NSCLC patients with activation EGFR mutations. Based on previous experiments, resistance to AZD9291 as a first-line treatment may also emerge. Thus, identification and understanding of resistance mechanisms to AZD9291 as a first-line treatment can help direct development of future therapies. AZD9291-resistant cells (PC9/AZDR) were established using EGFR inhibitor-naïve PC9 cells. Resistance mechanisms were analyzed using next-generation sequencing (NGS) and a proteome profiler array. Resistance to AZD9291 developed through aberrant activation of ERK signaling by an EGFR-independent mechanism. The combination of a MEK inhibitor with AZD9291 restored the sensitivity of PC9/AZDR cells in vitro and in vivo. PC9/AZDR cells also showed increased MET expression and an HRAS G13R mutation. In addition, maspin expression was higher after AZD9291 treatment in PC9/AZDR cells. Sustained ERK activation confers resistance to AZD9291 as a first-line therapy. Thus, co-targeting EGFR and MEK may be an effective strategy to overcome resistance to AZD9291.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29641535">
                    <p>AZD9291（osimertinib）被批准用于标准护理患EGFR T790M阳性非小细胞肺癌（NSCLC）的患者，此前EGFR TKI进展。此外，AZD9291目前正被评估为EGFR突变激活的非小细胞肺癌患者的一线治疗。基于之前的实验，也可能出现对AZD9291作为一线治疗的耐药性。因此，识别和理解AZD9291作为一线治疗的耐药机制有助于直接开发未来的治疗方法。 AZD9291耐药细胞（PC9 / AZDR）建立使用EGFR抑制剂幼稚PC9细胞。使用下一代测序（NGS）和蛋白质组分析仪阵列分析抗性机制。通过EGFR非依赖性机制异常激活ERK信号而产生对AZD9291的抗性。 MEK抑制剂与AZD9291的组合在体外和体内恢复了PC9 / AZDR细胞的敏感性。 PC9 / AZDR细胞也显示MET表达增加和HRAS G13R突变。另外，在PC9 / AZDR细胞中AZD9291处理后maspin表达较高。持续的ERK激活赋予AZD9291作为一线治疗的耐药性。因此，共同定位EGFR和MEK可能是克服AZD9291耐药性的有效策略。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29641535" target="_blank">
                PMID: 29641535
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Ku BM, Choi MK, Sun JM, Lee SH, Ahn JS, Park K, Ahn MJ">
                Authors: Ku BM, Choi MK, S...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29641535">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>131. Delivery of Proapoptotic Agents in Glioma Cell Lines by TSPO Ligand-Dextran Nanogels.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29641449" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29641449" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29641449">
                    <p>Translocator protein 18-kDa (TSPO) is a versatile mitochondrial target for molecular imaging and therapy. Moreover, selective TSPO ligands have been widely investigated for diagnostic purposes and explored to target drug delivery systems directed to cancer cells overexpressing TSPO. Indeed, poly(d,l-lactic-co-glycolic acid (PLGA) polymers and nanocarriers decorated with TSPO ligands are capable of transporting TSPO ligands inside cancer cells, inducing survival inhibition in cancer cells and producing mitochondrial morphology modification. The aim of this work was to prepare nanogels (NGs) made with TSPO ligand dextran conjugates (TSPO-Dex) that are useful as potential delivery systems of two TSPO ligands as apoptotic agents. Synthesis and complete characterization of TSPO-dextran conjugates, an average molecular weights analysis, TSPO ligand release profiles, thermal behaviour and swelling studies were achieved. NG preparation, characterization and in vitro biological studies were also performed. The release of TSPO ligands released from dextran conjugates at 37 °C occurred in human serum at a faster rate than that detected in phosphate buffer. Cytotoxicity studies demonstrated that NGs produced from TSPO ligand-dextran conjugates induce survival inhibition in rat C6 glioma cell lines. Cellular uptake was also proven by fluorescence microscopy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29641449">
                    <p>易位蛋白18-kDa（TSPO）是用于分子成像和治疗的多功能线粒体靶标。此外，选择性TSPO配体已经被广泛研究用于诊断目的并且探索靶向针对过表达TSPO的癌细胞的药物递送系统。实际上，用TSPO配体修饰的聚（d，l-乳酸 - 共 - 乙醇酸（PLGA）聚合物和纳米载体能够在癌细胞内运输TSPO配体，诱导癌细胞的存活抑制和产生线粒体形态修饰。工作是制备用TSPO配体葡聚糖偶联物（TSPO-Dex）制备的纳米凝胶（NGs），其可用作两种TSPO配体的潜在递送系统作为凋亡剂.TSPO-葡聚糖偶联物的合成和完整表征，平均分子量分析， TSPO配体释放曲线，热行为和肿胀的研究实现了NG的制备，表征和体外生物学研究，在37°C释放的葡聚糖缀合物释放的TSPO配体在人血清中发生的速度比检测到的更快在细胞毒性研究中证明由TSPO配体 - 葡聚糖偶联物产生的NG在r中诱导存活抑制在C6胶质瘤细胞系。荧光显微镜也证实了细胞摄取。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Mol Sci">
                Journal: Int J Mol Sci [IF=3.226]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29641449" target="_blank">
                PMID: 29641449
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-11
            </a>

            <a class="btn btn-sm btn-danger" title="Lopalco A, Cutrignelli A, Denora N, Perrone M, Iacobazzi RM, Fanizza E, Lopedota A, Depalo N, de Candia M, Franco M, Laquintana V">
                Authors: Lopalco A, Cutrig...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29641449">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>132. Low prevalence of transmitted HIV-1 drug resistance detected by a dried blood spot (DBS)-based next-generation sequencing (NGS) method in newly diagnosed individuals in Cameroon in the years 2015-16.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29635462" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29635462" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29635462">
                    <p>To determine the most recent prevalence, transmission patterns and risk factors of transmitted drug-resistance mutations (TDRMs) in Cameroon, we initiated a multicentre study monitoring HIV-1 drug resistance in newly HIV-1-diagnosed individuals using a novel next-generation sequencing (NGS) assay applicable to fingerprick dried blood spot (DBS) samples.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29635462">
                    <p>为了确定喀麦隆最近传播的耐药突变（TDRMs）的流行情况，传播模式和风险因素，我们启动了一项多中心研究，使用新一代新一代测序技术监测HIV-1新诊断个体中的HIV-1耐药性（NGS）分析适用于fingerprick干血斑（DBS）样品。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Antimicrob. Chemother.">
                Journal: J. Antimicrob. Chemother. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29635462" target="_blank">
                PMID: 29635462
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-07
            </a>

            <a class="btn btn-sm btn-danger" title="Mbunkah HA, Marzel A, Schmutz S, Kok YL, Zagordi O, Shilaih M, Nsanwe NN, Mbu ET, Besong LM, Sama BA, Orock E, Kouyos RD, Günthard HF, Metzner KJ">
                Authors: Mbunkah HA, Marze...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29635462">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>133. Clinical pharmacogenomics testing in the era of next generation sequencing: challenges and opportunities for precision medicine.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29634383" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29634383" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29634383">
                    <p>The rapid development and dramatic decrease in cost of sequencing techniques have ushered the implementation of genomic testing in patient care. Next generation DNA sequencing (NGS) techniques have been used increasingly in clinical laboratories to scan the whole or part of the human genome in order to facilitate diagnosis and/or prognostics of genetic disease. Despite many hurdles and debates, pharmacogenomics (PGx) is believed to be an area of genomic medicine where precision medicine could have immediate impact in the near future. Areas covered: This review focuses on lessons learned through early attempts of clinically implementing PGx testing; the challenges and opportunities that PGx testing brings to precision medicine in the era of NGS. Expert commentary: Replacing targeted analysis approach with NGS for PGx testing is neither technically feasible nor necessary currently due to several technical limitations and uncertainty involved in interpreting variants of uncertain significance for PGx variants. However, reporting PGx variants out of clinical whole exome or whole genome sequencing (WES/WGS) might represent additional benefits for patients who are tested by WES/WGS.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29634383">
                    <p>测序技术的快速发展和成本的显着降低已经引发了患者护理中基因组测试的实施。新一代DNA测序（NGS）技术越来越多地应用于临床实验室，以扫描整个或部分人类基因组以便于遗传疾病的诊断和/或预后。尽管存在许多障碍和争议，药物基因组学（PGx）被认为是精准医学在不久的将来可能产生直接影响的基因组医学领域。涵盖的领域：本评论重点在于通过临床实施PGx测试的早期尝试获得的经验教训;在NGS时代，PGx测试给精准医学带来的挑战和机遇。专家评论：使用NGS代替PGx测试的目标分析方法在技术上既不现实也不必要，因为解释PGx变体的不确定重要变体的几个技术限制和不确定性。然而，从临床整体外显子组或全基因组测序（WES / WGS）报告PGx变异体可能对WES / WGS检测的患者具有额外的益处。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Expert Rev. Mol. Diagn.">
                Journal: Expert Rev. Mol. Diagn. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29634383" target="_blank">
                PMID: 29634383
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-23
            </a>

            <a class="btn btn-sm btn-danger" title="Ji Y, Si Y, McMillin GA, Lyon E">
                Authors: Ji Y, Si Y, McMil...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29634383">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>134. Identification of A Novel Compound Heterozygous Mutation in BBS12 in An Iranian Family with Bardet-Biedl Syndrome Using Targeted Next Generation Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29633607" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29633607" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29633607">
                    <p>Bardet-Biedl syndrome (BBS) is a pleiotropic and multisystemic disorder characterized by rod-cone dystrophy, polydactyly, learning difficulties, renal abnormalities, obesity and hypogonadism. This disorder is genetically heterogeneous. Until now, a total of nineteen genes have been identified for BBS whose mutations explain more than 80% of diagnosed cases. Recently, the development of next generation sequencing (NGS) technology has accelerated mutation screening of target genes, resulting in lower cost and less time consumption. Here, we screened the most common BBS genes (BBS1-BBS13) using NGS in an Iranian family of a proposita displaying symptoms of BBS. Among the 18 mutations identified in the proposita, one (BBS12 c.56T&gt;G and BBS12 c.1156C&gt;T) was novel. This compound heterozygosity was confirmed by Sanger sequencing in the proposita and her parents. Although our data were presented as a case report, however, we suggest a new probable genetic mechanism other than the conventional autosomal recessive inheritance of BBS. Additionally, given that in some Iranian provinces, like Khuzestan, consanguineous marriages are common, designing mutational panels for genetic diseases is strongly recommended, especially for those with an autosomal recessive inheritance pattern.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29633607">
                    <p>Bardet-Biedl综合征（BBS）是一种多向性和多系统性疾病，其特征为棒状营养不良，多指，学习困难，肾脏异常，肥胖和性腺机能减退。这种疾病是遗传异质性的。到目前为止，已经鉴定了19种基因，其中BBS的突变解释了超过80％的诊断病例​​。最近，下一代测序（NGS）技术的发展加速了目标基因的突变筛选，从而降低了成本，减少了时间消耗。在这里，我们在显示BBS症状的伊朗家庭中使用NGS筛选了最常见的BBS基因（BBS1-BBS13）。在提案中发现的18个突变中，一个（BBS12 c.56T&gt; G和BBS12 c.1156C&gt; T）是新的。这种复合杂合性在提索塔及其父母中通过Sanger测序证实。尽管我们的数据是作为病例报告呈现的，但是，我们提出了一种新的可能的遗传机制，而不是传统的常染色体隐性遗传BBS。此外，鉴于在伊朗一些省份（如胡齐斯坦省），血缘婚姻很常见，所以强烈建议设计遗传疾病突变小组，特别是那些常染色体隐性遗传模式的人。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cell J">
                Journal: Cell J [IF=1.636]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29633607" target="_blank">
                PMID: 29633607
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jul
            </a>

            <a class="btn btn-sm btn-danger" title="Nikkhah E, Safaralizadeh R, Mohammadiasl J, Tahmasebi Birgani M, Hosseinpour Feizi MA, Golchin N">
                Authors: Nikkhah E, Safara...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5893301" target="_blank">
                PMC5893301
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29633607">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>135. Simple and efficient germline copy number variant visualization method for the Ion AmpliSeq™ custom panel.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29633566" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29633566" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29633566">
                    <p>Recent advances in molecular genetic analysis using next-generation sequencing (NGS) have drastically accelerated the identification of disease-causing gene mutations. Most next-generation sequencing analyses of inherited diseases have mainly focused on single-nucleotide variants and short indels, although, recently, structure variations including copy number variations have come to be considered an important cause of many different diseases. However, only a limited number of tools are available for multiplex PCR-based target genome enrichment.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29633566">
                    <p>使用新一代测序（NGS）进行分子遗传分析的最新进展大大加速了致病基因突变的鉴定。大多数遗传疾病的新一代测序分析主要集中在单核苷酸变异和短插入缺失，尽管最近包括拷贝数变异在内的结构变异已被认为是许多不同疾病的重要原因。然而，只有有限数量的工具可用于基于多重PCR的目标基因组富集。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Mol Genet Genomic Med">
                Journal: Mol Genet Genomic Med [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29633566" target="_blank">
                PMID: 29633566
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-06
            </a>

            <a class="btn btn-sm btn-danger" title="Nishio SY, Moteki H, Usami SI">
                Authors: Nishio SY, Moteki...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29633566">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>136. Inferring the effect of genomic variation in the new era of genomics.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29633501" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29633501" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29633501">
                    <p>Accurate and detailed understanding of the effects of variants in the coding and noncoding regions of the genome is the next big challenge in the new genomic era of personalized medicine, especially to tackle newer findings of genetic and phenotypic heterogeneity of diseases. This is necessary to resolve the gene-variant-disease relationship, the pathogenic variant spectrum of genes, pathogenic variants with variable clinical consequences, and multiloci diseases. In turn, this will facilitate patient recruitment for relevant clinical trials. In this review, we describe the trends in research at the intersection of basic and clinical genomics aiming to (a) overcome molecular diagnostic challenges and increase the clinical utility of next-generation sequencing (NGS) platforms, (b) elucidate variants associated with disease, (c) determine overall genomic complexity including epistasis, complex inheritance patterns such as &quot;synergistic heterozygosity,&quot; digenic/multigenic inheritance, modifier effect, and rare variant load. We describe the newly emerging field of integrated functional genomics, in vivo or in vitro large-scale functional approaches, statistical bioinformatics algorithms that support NGS genomics data to interpret variants for timely clinical diagnostics and disease management. Thus, facilitating the discovery of new therapeutic or biomarker options, and their roles in the future of personalized medicine.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29633501">
                    <p>准确和详细地了解变异体在基因组编码和非编码区域中的作用是个体化医学新基因组时代的下一个重大挑战，特别是解决遗传和表型异质性疾病的新发现。这对于解决基因 - 变异 - 疾病关系，基因的致病变异谱，具有可变临床后果的致病变体和多重病害疾病是必需的。反过来，这将有助于患者招募相关的临床试验。在这篇综述中，我们描述了旨在（a）克服分子诊断挑战并提高下一代测序（NGS）平台的临床实用性的基础和临床基因组学交叉研究的趋势，（b）阐明与疾病相关的变体，（c）确定整体基因组复杂性，包括上位性，诸如“协同杂合性”，双基因/多基因遗传，修饰物效应和稀有变异负荷的复杂遗传模式。我们描述新兴领域的综合功能基因组学，体内或体外大规模功能方法，统计生物信息学算法，支持NGS基因组学数据来解释变体，以便及时进行临床诊断和疾病管理。因此，促进发现新的治疗或生物标志物选择，以及它们在个性化医学的未来中的角色。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Hum. Mutat.">
                Journal: Hum. Mutat. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29633501" target="_blank">
                PMID: 29633501
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-06
            </a>

            <a class="btn btn-sm btn-danger" title="Chakravorty S, Hegde M">
                Authors: Chakravorty S, He...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29633501">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>137. [Effective management of an outbreak with multiresistent Klebsiella pneumoniae in a neurorehabilitation unit].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29633038" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29633038" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29633038">
                    <p>In addition to acute care hospitals, rehabilitation centres are increasingly confronted with multi-resistant pathogens. Long durations of stay and intensive treatments impose special hygienic challenges.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29633038">
                    <p>除了急症医院外，康复中心也越来越多地面临多重耐药病原体。长时间的停留和强化治疗带来了特殊的卫生挑战。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz">
                Journal: Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29633038" target="_blank">
                PMID: 29633038
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Dohle C, Korr G, Friedrichs M, Kullmann V, Tung ML, Kaase M, Rüssmann H, Sissolak D, Werber D, Becker L, Fuchs S, Pfeifer Y, Semmler T, Widders G, Eckmanns T, Werner G, Zill E, Haller S">
                Authors: Dohle C, Korr G, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29633038">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>138. ERASE-Seq: Leveraging replicate measurements to enhance ultralow frequency variant detection in NGS data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29630678" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29630678" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29630678">
                    <p>The accurate detection of ultralow allele frequency variants in DNA samples is of interest in both research and medical settings, particularly in liquid biopsies where cancer mutational status is monitored from circulating DNA. Next-generation sequencing (NGS) technologies employing molecular barcoding have shown promise but significant sensitivity and specificity improvements are still needed to detect mutations in a majority of patients before the metastatic stage. To address this we present analytical validation data for ERASE-Seq (Elimination of Recurrent Artifacts and Stochastic Errors), a method for accurate and sensitive detection of ultralow frequency DNA variants in NGS data. ERASE-Seq differs from previous methods by creating a robust statistical framework to utilize technical replicates in conjunction with background error modeling, providing a 10 to 100-fold reduction in false positive rates compared to published molecular barcoding methods. ERASE-Seq was tested using spiked human DNA mixtures with clinically realistic DNA input quantities to detect SNVs and indels between 0.05% and 1% allele frequency, the range commonly found in liquid biopsy samples. Variants were detected with greater than 90% sensitivity and a false positive rate below 0.1 calls per 10,000 possible variants. The approach represents a significant performance improvement compared to molecular barcoding methods and does not require changing molecular reagents.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29630678">
                    <p>DNA样本中超低等位基因频率变异的准确检测在研究和医学环境中都很有意义，特别是在液体活检中，其中从循环DNA监测癌症突变状态。采用分子条形码技术的新一代测序（NGS）技术已显示出前景，但在转移阶段之前仍需要对大多数患者的突变进行显着的敏感性和特异性改进。为了解决这个问题，我们提供了ERASE-Seq（消除复发性伪影和随机错误）的分析验证数据，这是一种准确和灵敏地检测NGS数据中超低频DNA变体的方法。 ERASE-Seq与以前的方法有所不同，创建了一个强大的统计框架，利用技术重复和背景误差建模，与公布的分子条形码方法相比，假阳性率降低了10到100倍。使用具有临床现实DNA输入量的加标的人DNA混合物检测ERASE-Seq以检测SNV和插入缺失等位基因频率的0.05％至1％，这是液体活检样品中常见的范围。检测到变异的灵敏度高于90％，假阳性率低于每10,000个可能变体0.1个调用。与分子条形码方法相比，该方法表现出显着的性能改进，并且不需要改变分子试剂。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29630678" target="_blank">
                PMID: 29630678
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Kamps-Hughes N, McUsic A, Kurihara L, Harkins TT, Pal P, Ray C, Ionescu-Zanetti C">
                Authors: Kamps-Hughes N, M...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5890993" target="_blank">
                PMC5890993
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29630678">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>139. Ultrasound-guided lumbar puncture with a needle-guidance system: A prospective and controlled study to evaluate the learnability and feasibility of a newly developed approach.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29630646" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29630646" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29630646">
                    <p>To evaluate the learnability and feasibility of a new technique comprising a needle-guidance-system (NGS) for ultrasound-assisted lumbar puncture.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29630646">
                    <p>评估包括用于超声辅助腰椎穿刺的针引导系统（NGS）的新技术的可学性和可行性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29630646" target="_blank">
                PMID: 29630646
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Backhaus T, von Cranach M, Brich J">
                Authors: Backhaus T, von C...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5891015" target="_blank">
                PMC5891015
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29630646">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>140. Multiplex PCR Targeted Amplicon Sequencing (MTA-Seq): Simple, Flexible, and Versatile SNP Genotyping by Highly Multiplexed PCR Amplicon Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29628929" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29628929" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29628929">
                    <p>Next-generation sequencing (NGS) technologies have enabled genome re-sequencing for exploring genome-wide polymorphisms among individuals, as well as targeted re-sequencing for the rapid and simultaneous detection of polymorphisms in genes associated with various biological functions. Therefore, a simple and robust method for targeted re-sequencing should facilitate genotyping in a wide range of biological fields. In this study, we developed a simple, custom, targeted re-sequencing method, designated &quot;multiplex PCR targeted amplicon sequencing (MTA-seq),&quot; and applied it to the genotyping of the model grass Brachypodium distachyon. To assess the practical usability of MTA-seq, we applied it to the genotyping of genome-wide single-nucleotide polymorphisms (SNPs) identified in natural accessions (Bd1-1, Bd3-1, Bd21-3, Bd30-1, Koz-1, Koz-3, and Koz-4) by comparing the re-sequencing data with that of reference accession Bd21. Examination of SNP-genotyping accuracy in 443 amplicons from eight parental accessions and an F1 progeny derived by crossing of Bd21 and Bd3-1 revealed that ~95% of the SNPs were correctly called. The assessment suggested that the method provided an efficient framework for accurate and robust SNP genotyping. The method described here enables easy design of custom target SNP-marker panels in various organisms, facilitating a wide range of high-throughput genetic applications, such as genetic mapping, population analysis, molecular breeding, and genomic diagnostics.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29628929">
                    <p>新一代测序（NGS）技术已经实现了基因组重测序，以探索个体间的全基因组多态性，以及有针对性的重测序，以快速同时检测与各种生物功能相关基因的多态性。因此，一种简单而强大的靶向重测序方法应该有助于在广泛的生物学领域进行基因分型。在本研究中，我们开发了一种简单的，定制的，有针对性的重测序方法，命名为“多重PCR靶向扩增测序（MTA-seq）”，并将其应用于模型草Brachypodium distachyon的基因分型。为了评估MTA-seq的实际可用性，我们将其应用于在天然种质（Bd1-1，Bd3-1，Bd21-3，Bd30-1，Koz-1）中鉴定的全基因组单核苷酸多态性（SNP） 1，Koz-3和Koz-4）通过比较重排序数据与参考登录号Bd21。从8个亲本材料和通过Bd21和Bd3-1杂交衍生的F1后代的443个扩增子中检测SNP基因分型准确性揭示约95％的SNP被正确称为。评估表明，该方法为准确和强大的SNP基因分型提供了一个有效的框架。此处描述的方法可以轻松设计各种生物体中的定制目标SNP标记板，从而促进各种高通量遗传应用，例如遗传作图，群体分析，分子育种和基因组诊断。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Plant Sci">
                Journal: Front Plant Sci [IF=4.298]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29628929" target="_blank">
                PMID: 29628929
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Onda Y, Takahagi K, Shimizu M, Inoue K, Mochida K">
                Authors: Onda Y, Takahagi ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5876661" target="_blank">
                PMC5876661
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29628929">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>141. Targeted therapy according to next generation sequencing-based panel sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29628467" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29628467" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29628467">
                    <p>Targeted therapy against actionable gene mutations shows a significantly higher response rate as well as longer survival compared to conventional chemotherapy, and has become a standard therapy for many cancers. Recent progress in next-generation sequencing (NGS) has enabled to identify huge number of genetic aberrations. Based on sequencing results, patients recommend to undergo targeted therapy or immunotherapy. In cases where there are no available approved drugs for the genetic mutations detected in the patients, it is recommended to be facilitate the registration for the clinical trials. For that purpose, a NGS-based sequencing panel that can simultaneously target multiple genes in a single investigation has been used in daily clinical practice. To date, various types of sequencing panels have been developed to investigate genetic aberrations with tumor somatic genome variants (gain-of-function or loss-of-function mutations, high-level copy number alterations, and gene fusions) through comprehensive bioinformatics. Because sequencing panels are efficient and cost-effective, they are quickly being adopted outside the lab, in hospitals and clinics, in order to identify personal targeted therapy for individual cancer patients.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29628467">
                    <p>与常规化疗相比，针对可行性基因突变的靶向治疗显示出显着更高的应答率以及更长的存活率，并已成为许多癌症的标准疗法。新一代测序（NGS）方面的最新进展使得能够识别大量的遗传畸变。根据测序结果，患者推荐进行靶向治疗或免疫治疗。如果没有可用的批准药物用于检测患者的基因突变，建议为临床试验注册提供便利。为此目的，在日常临床实践中已经使用了基于NGS的测序小组，其可以在单个研究中同时靶向多个基因。迄今为止，已经开发了各种类型的测序板以通过全面的生物信息学研究与肿瘤体细胞基因组变体（功能获得或功能缺失突变，高水平拷贝数改变和基因融合）的遗传畸变。由于测序板是高效且具有成本效益的，因此它们很快在医院和诊所的实验室外被采用，以便为个体癌症患者确定个人靶向治疗。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Fukushima J Med Sci">
                Journal: Fukushima J Med Sci [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29628467" target="_blank">
                PMID: 29628467
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-17
            </a>

            <a class="btn btn-sm btn-danger" title="Saito M, Momma T, Kono K">
                Authors: Saito M, Momma T,...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29628467">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>142. New chimeric RNAs in acute myeloid leukemia.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29623188" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29623188" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29623188">
                    <p>Background: High-throughput next generation sequencing (NGS) technologies enable the detection of biomarkers used for tumor classification, disease monitoring and cancer therapy. Whole-transcriptome analysis using RNA-seq is important, not only as a means of understanding the mechanisms responsible for complex diseases but also to efficiently identify novel genes/exons, splice isoforms, RNA editing, allele-specific mutations, differential gene expression and fusion-transcripts or chimeric RNA (chRNA). Methods: We used Crac, a tool that uses genomic locations and local coverage to classify biological events and directly infer splice and chimeric junctions within a single read. Crac&#39;s algorithm extracts transcriptional chimeric events irrespective of annotation with a high sensitivity, and CracTools was used to aggregate, annotate and filter the chRNA reads. The selected chRNA candidates were validated by real time PCR and sequencing.  In order to check the tumor specific expression of chRNA, we analyzed a publicly available dataset using a new tag search approach. Results:  We present data related to acute myeloid leukemia (AML) RNA-seq analysis. We highlight novel biological cases of chRNA, in addition to previously well characterized leukemia chRNA. We have identified and validated 17 chRNAs among 3 AML patients: 10 from an AML patient with a translocation between chromosomes 15 and 17 (AML-t(15;17), 4  from patient with normal karyotype (AML-NK) 3 from a patient with chromosomal 16 inversion (AML-inv16). The new fusion transcripts can be classified into four groups according to the exon organization. Conclusions:  All groups suggest complex but distinct synthesis mechanisms involving either collinear exons of different genes, non-collinear exons, or exons of different chromosomes. Finally, we check tumor-specific expression in a larger RNA-seq AML cohort and identify new AML biomarkers that could improve diagnosis and prognosis of AML.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29623188">
                    <p>背景：高通量下一代测序（NGS）技术能够检测用于肿瘤分类，疾病监测和癌症治疗的生物标志物。使用RNA-seq进行全转录组分析非常重要，不仅可作为理解导致复杂疾病的机制的手段，还可有效鉴定新基因/外显子，剪接异构体，RNA编辑，等位基因特异性突变，差异基因表达和融合 - 转录物或嵌合RNA（chRNA）。方法：我们使用Crac，一种使用基因组位置和局部覆盖的工具来分类生物事件，并在单次读取中直接推断剪接和嵌合连接。 Crac的算法提取转录嵌合事件，而不考虑高灵敏度的注释，并且CracTools用于聚集，注释和过滤chRNA读数。选择的chRNA候选物通过实时PCR和测序进行验证。为了检查chRNA的肿瘤特异性表达，我们使用新的标签搜索方法分析了公开可用的数据集。结果：我们提供有关急性髓细胞性白血病（AML）RNA-seq分析的数据。我们突出了chRNA的新型生物学病例，以及以前很好表征的白血病chRNA。我们在3名AML患者中鉴定和验证了17种chRNA：10名来自患有正常核型（AML-NK）3患者的染色体15和17之间易位的AML患者（AML-t（15; 17），4 （AML-inv16），根据外显子组织可将新融合转录物分为四组。结论：所有组均提示了复杂而独特的合成机制，涉及不同基因的共线外显子，非共线外显子，或最后，我们检查了更大的RNA-seq AML队列中的肿瘤特异性表达，并鉴定了能够改善AML的诊断和预后的新的AML生物标志物。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=F1000Res">
                Journal: F1000Res [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29623188" target="_blank">
                PMID: 29623188
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2017
            </a>

            <a class="btn btn-sm btn-danger" title="Rufflé F, Audoux J, Boureux A, Beaumeunier S, Gaillard JB, Bou Samra E, Megarbane A, Cassinat B, Chomienne C, Alves R, Riquier S, Gilbert N, Lemaitre JM, Bacq-Daian D, Bougé AL, Philippe N, Commes T">
                Authors: Rufflé F, Audoux ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861515.2" target="_blank">
                PMC5861515.2
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29623188">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>143. NGS combined with phylogenetic analysis to detect HIV-1 dual infection in Romanian people who inject drugs.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29626632" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29626632" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29626632">
                    <p>Dual HIV infections are possible and likely in people who inject drugs (PWID). Thirty-eight newly diagnosed patients, 19 PWID and 19 heterosexually HIV infected were analyzed. V2V3 loop of HIV-1 env gene was sequenced on the NGS platform 454 GSJunior (Roche). HIV-1 dual/multiple infections were identified in five PWID. For three of these patients, the reconstructed variants belonged to pure F1 subtype and CRF14_BG strains according to phylogenetic analysis. New recombinant forms between these parental strains were identified in two PWID samples. NGS data can provide, with the help of phylogenetic analysis, important insights about the intra-host sub-population structure.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29626632">
                    <p>艾滋病毒双重感染是可能的，可能是注射吸毒者（PWID）。分析了38名新诊断的患者，19名PWID和19名异性HIV感染者。 HIV-1 env基因的V2V3环在NGS平台454 GSJunior（Roche）上测序。在五个PWID中确定了HIV-1双重/多重感染。根据系统发育分析，对于这些患者中的三名，重建的变体属于纯F1亚型和CRF14_BG菌株。在两个PWID样品中鉴定了这些亲本菌株之间的新重组形式。在系统发育分析的帮助下，NGS数据可以提供有关宿主内亚群结构的重要见解。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Microbes Infect.">
                Journal: Microbes Infect. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29626632" target="_blank">
                PMID: 29626632
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-04
            </a>

            <a class="btn btn-sm btn-danger" title="Popescu B, Banica L, Nicolae I, Radu E, Niculescu I, Abagiu A, Otelea D, Paraschiv S">
                Authors: Popescu B, Banica...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29626632">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>144. The viral structure of the Cuatro Ciénegas Basin, a unique oasis in Northern Mexico, reveals a highly diverse population at a small geographic scale.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29625974" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29625974" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29625974">
                    <p>The Cuatro Ciénegas Basin (CCB) is located in the Chihuahuan desert, in the Mexican state of Coahuila; it has been characterized as a site with a biological high diversity despite its extreme oligotrophic conditions. It has the greatest number of endemic species in North America, containing abundant living microbialites (including stromatolites and microbial mats), and diverse microbial communities. With the hypothesis that this large biodiversity and the geographic structure should be reflected in the virome, the viral communities in 11 different locations of three drainage systems: Churince, La Becerra, and Pozas Rojas and in the intestinal content of 3 different fish species were analyzed, looking for both eukaryotic and prokaryotic RNA and DNA viruses, using NGS methods. dsDNA virus families were the most abundant (72.5% of reads), followed by ssDNA viruses (2.9%), and ssRNA and dsRNA virus families (0.5%). Thirteen families had dsDNA genomes, five ssDNA, three dsRNA, and sixteen had ssRNA. A highly diverse viral community was found, with an ample range of hosts and a strong geographical structure with very even distributions and signals of endemicity in the phylogenetic trees from several different virus families. The majority of viruses found were bacteriophages, but eukaryotic viruses were also frequent, and the large diversity of viruses related to algae were a surprise, since algae are not evident in the previously analyzed aquatic systems of this ecosystem. Animal viruses were also frequently found, showing the large diversity of aquatic animals in this oasis, where plants, protozoa and archaea are rare.IMPORTANCE In this study, we tested whether the large biodiversity and geographic structure of CCB is reflected in its virome. CCB is an extraordinarily biodiverse oasis in the Chihuahuan desert, where a previous virome study suggested that viruses had followed the marine ancestry of the marine bacteria and as a result of their long isolation, became endemic to the site. In this study, which includes a larger sequencing coverage and water samples from other sites within the valley, we confirmed the virus large biodiversity and uniqueness, as well as the strong biogeographical diversification of the CCB. In addition, we also analyzed fish intestinal contents, finding that each fish species eat different preys and, as a result, they present different viral compositions even if they coexist in the same pond. These facts highlight the high and novel virus diversity of CCB and its &quot;lost world&quot; status.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29625974">
                    <p>CuatroCiénegasBasin（CCB）位于墨西哥Coahuila州的Chihuahuan沙漠;尽管其具有极端的营养缺陷条件，但其已被定性为具有生物高度多样性的地点。它在北美拥有最多的特有物种，含有丰富的活微生物（包括叠层石和微生物垫）以及不同的微生物群落。在假设这种大型生物多样性和地理结构应该反映在六流域中的情况下，分析了三种排水系统的11个不同位置的病毒群落：Churince，La Becerra和Pozas Rojas以及3种不同鱼类的肠内容物使用NGS方法寻找真核和原核RNA和DNA病毒。双链DNA病毒科是最丰富的（占72.5％），其次是ssDNA病毒（2.9％），ssRNA和dsRNA病毒科（0.5％）。 13个家庭有dsDNA基因组，5个ssDNA，3个dsRNA，16个有ssRNA。发现了一个高度多样化的病毒群落，具有多种宿主和强大的地理结构，在来自几个不同病毒科的系统发生树中具有非常均匀的分布和流行信号。发现的大多数病毒都是噬菌体，但真核病毒也很常见，与藻类有关的大量病毒令人惊讶，因为在先前分析的这个生态系统的水生系统中，藻类并不明显。动物病毒也经常被发现，显示这个绿洲中的水生动物种类繁多，植物，原生动物和古生物很少。在这项研究中，我们测试了CCB的大生物多样性和地理结构是否反映在它的三维图中。 CCB是Chihuahuan沙漠中一个非常具有生物多样性的绿洲，在此之前的一项研究表明，病毒已经跟随海洋细菌的海洋血统，并且由于长时间的隔离而成为该地的特有种。在这项包括更大测序覆盖率和来自谷内其他地点的水样的研究中，我们证实了该病毒具有很大的生物多样性和独特性，以及建设银行强大的生物地理多样性。此外，我们还分析了鱼的肠内容，发现每种鱼类都吃不同的猎物，因此，即使它们共存于同一个池塘中，它们也呈现出不同的病毒组成。这些事实凸显了建行的高新病毒多样性及其“失落的世界”地位。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Appl. Environ. Microbiol.">
                Journal: Appl. Environ. Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29625974" target="_blank">
                PMID: 29625974
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-06
            </a>

            <a class="btn btn-sm btn-danger" title="Taboada B, Isa P, Gutiérrez-Escolano AL, Del Ángel RM, Ludert JE, Vázquez N, Tapia-Palacios MA, Chávez P, Garrido E, Espinosa AC, Eguiarte LE, López S, Souza V, Arias CF">
                Authors: Taboada B, Isa P,...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29625974">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>145. NGS-based likelihood ratio for identifying contributors in two- and three-person DNA mixtures.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29625871" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29625871" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29625871">
                    <p>DNA fingerprinting, also known as DNA profiling, serves as a standard procedure in forensics to identify a person by the short tandem repeat (STR) loci in their DNA. By comparing the STR loci between DNA samples, practitioners can calculate a probability of match to identity the contributors of a DNA mixture. Most existing methods are based on 13 core STR loci which were identified by the Federal Bureau of Investigation (FBI). Analyses based on these loci of DNA mixture for forensic purposes are highly variable in procedures, and suffer from subjectivity as well as bias in complex mixture interpretation. With the emergence of next-generation sequencing (NGS) technologies, the sequencing of billions of DNA molecules can be parallelized, thus greatly increasing throughput and reducing the associated costs. This allows the creation of new techniques that incorporate more loci to enable complex mixture interpretation. In this paper, we propose a computation for likelihood ratio that uses NGS (next generation sequencing) data for DNA testing on mixed samples. We have applied the method to 4480 simulated DNA mixtures, which consist of various mixture proportions of 8 unrelated whole-genome sequencing data. The results confirm the feasibility of utilizing NGS data in DNA mixture interpretations. We observed an average likelihood ratio as high as 285,978 for two-person mixtures. Using our method, all 224 identity tests for two-person mixtures and three-person mixtures were correctly identified.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29625871">
                    <p>DNA指纹分析，也称为DNA分析，作为法医学的标准程序，通过其DNA中的短串联重复序列（STR）位点识别人。通过比较DNA样本之间的STR位点，从业人员可以计算匹配概率以鉴定DNA混合物的贡献者。现有的大多数方法都是基于联邦调查局（FBI）确定的13个核心STR基因座。基于这些DNA混合物位点的分析以用于法医目的在程序中是高度可变的，并且在复杂的混合物解释中存在主观性和偏见。随着新一代测序（NGS）技术的出现，数十亿DNA分子的测序可以并行进行，从而大大提高吞吐量并降低相关成本。这允许创建包含更多基因座的新技术以实现复杂的混合物解释。在本文中，我们提出了使用NGS（下一代测序）数据对混合样本进行DNA测试的似然比计算。我们已将该方法应用于4480个模拟DNA混合物，其由8个不相关的全基因组测序数据的各种混合比例组成。结果证实了在DNA混合物解释中利用NGS数据的可行性。我们观察到两人混合物的平均似然比高达285,978。使用我们的方法，对双人混合物和三人混合物的所有224个身份测试都被正确识别。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Comput Biol Chem">
                Journal: Comput Biol Chem [IF=1.331]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29625871" target="_blank">
                PMID: 29625871
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-28
            </a>

            <a class="btn btn-sm btn-danger" title="Chan Mun Wei J, Zhao Z, Li SC, Ng YK">
                Authors: Chan Mun Wei J, Z...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29625871">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>146. Targeted next generation sequencing with an extended gene panel does not impact variant detection in mitochondrial diseases.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29625556" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29625556" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29625556">
                    <p>Since the advent of next generation sequencing (NGS), several studies have tried to evaluate the relevance of targeted gene panel sequencing and whole exome sequencing for molecular diagnosis of mitochondrial diseases. The comparison between these different strategies is extremely difficult. A recent study analysed a cohort of patients affected by a mitochondrial disease using a NGS approach based on a targeted gene panel including 132 genes. This strategy led to identify the causative mutations in 15.2% of cases. The number of novel genes responsible for respiratory chain deficiency increases very rapidly.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29625556">
                    <p>自从下一代测序（NGS）出现以来，一些研究试图评估靶向基因组测序和整个外显子组测序在分子诊断线粒体疾病中的相关性。这些不同策略之间的比较是非常困难的。最近的一项研究使用基于包括132个基因的靶向基因组的NGS方法分析了受线粒体疾病影响的一组患者。这一策略导致在15.2％的病例中找出致病突变。负责呼吸链缺陷的新基因数量增长非常迅速。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Med. Genet.">
                Journal: BMC Med. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29625556" target="_blank">
                PMID: 29625556
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-07
            </a>

            <a class="btn btn-sm btn-danger" title="Plutino M, Chaussenot A, Rouzier C, Ait-El-Mkadem S, Fragaki K, Paquis-Flucklinger V, Bannwarth S">
                Authors: Plutino M, Chauss...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5889585" target="_blank">
                PMC5889585
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29625556">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>147. Retrospective genome-wide comparisons of Salmonella enterica serovar Enteritidis from suspected outbreaks in Singapore.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29625239" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29625239" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29625239">
                    <p>The number of salmonellosis cases in Singapore has increased over the years. Salmonella enterica serovar Enteritidis has always been the most predominant serovar in the last five years. The National Public Health Laboratory assisted outbreak investigations by performing multilocus variable number tandem repeat analysis (MLVA) on isolates that were collected at the time of the investigations. Isolates were defined as belonging to a particular cluster if they had identical MLVA patterns. Whilst MLVA has been instrumental in outbreak investigations, it may not be useful when outbreaks are caused by an endemic MLVA type. In this study, we analysed 67 isolates from 12 suspected outbreaks with known epidemiological links to explore the use of next-generation sequencing (NGS) for defining outbreaks. We found that NGS can confidently group isolates into their respective outbreaks. The isolates from each suspected outbreak were closely related and differed by a maximum of 3 single nucleotide polymorphisms (SNPs). They were also clearly separated from isolates that belonged to different suspected outbreaks. This study provides an important insight and further evidence on the value of NGS for routine surveillance and outbreak detection of S. Enteritidis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29625239">
                    <p>多年来，新加坡的沙门氏菌病病例数量有所增加。沙门氏菌肠炎血清型肠炎沙门氏菌始终是过去五年中最主要的血清型。国家公共卫生实验室通过对调查时收集的分离物进行多位点可变数量串联重复分析（MLVA）来协助疫情调查。如果分离物具有相同的MLVA模式，则将其定义为属于特定的聚类。虽然MLVA在爆发调查中起到了重要作用，但当爆发是由地方性MLVA类型引起时可能没有用处。在这项研究中，我们分析了来自12个怀疑爆发的67个分离株，并与已知的流行病学联系，以探索使用下一代测序（NGS）来确定爆发。我们发现NGS可以自信地将分离物分组到各自的暴发中。来自每个怀疑爆发的分离物密切相关，并且最多与3个单核苷酸多态性（SNP）不同。它们也清楚地与属于不同疑似爆发的分离株分开。本研究为NGS对肠炎沙门氏菌的常规监测和暴发检测提供了重要的见解和进一步的证据。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Infect. Genet. Evol.">
                Journal: Infect. Genet. Evol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29625239" target="_blank">
                PMID: 29625239
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-04
            </a>

            <a class="btn btn-sm btn-danger" title="Octavia S, Ang MLT, La MV, Zulaina S, Saat ZAAS, Tien WS, Han HK, Ooi PL, Cui L, Lin RTP">
                Authors: Octavia S, Ang ML...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29625239">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>148. Phylogeny, evolution and mitochondrial gene order rearrangement in scale worms (Aphroditiformia, Annelida).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29625228" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29625228" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29625228">
                    <p>Next-generation sequencing (NGS) has become a powerful tool in phylogenetic and evolutionary studies. Here we applied NGS to recover two ribosomal RNA genes (18S and 28S) from 16 species and 15 mitochondrial genomes from 16 species of scale worms representing six families in the suborder Aphroditiformia (Phyllodocida, Annelida), a complex group of polychaetes characterized by the presence of dorsal elytra or scales. The phylogenetic relationship of the several groups of scale worms remains unresolved due to insufficient taxon sampling and low resolution of individual gene markers. Phylogenetic tree topology based on mitochondrial genomes is comparable with that based on concatenated sequences from two mitochondrial genes (cox1 and 16S) and two ribosomal genes (18S and 28S) genes, but has higher statistical support for several clades. Our analyses show that Aphroditiformia is monophyletic, indicating the presence of elytra is an apomorphic trait. Eulepethidae and Aphroditidae together form the sister group to all other families in this suborder, whereas Acoetidae is sister to Iphionidae. Polynoidae is monophyletic, but within this family the deep-sea subfamilies Branchinotogluminae and Macellicephalinae are paraphyletic. Mitochondrial genomes in most scale-worm families have a conserved gene order, but within Polynoidae there are two novel arrangement patterns in the deep-sea clade. Mitochondrial protein-coding genes in polynoids as a whole have evolved under strong purifying selection, but substitution rates in deep-sea species are much higher than those in shallow-water species, indicating that purifying selection is relaxed in deep-sea polynoids. There are positive selected amino acids for some mitochondrial genes of the deep-sea clade, indicating they may involve in the adaption of deep-sea polynoids. Overall, our study (1) provided more evidence for reconstruction of the phylogeny of Aphroditiformia, (2) provided evidence to refute the assumption that mitochondrial gene order in Errantia is conserved, and (3) indicated that the deep-sea extreme environment may have affected the mitochondrial genome evolution rate and gene order arrangement in Polynoidae.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29625228">
                    <p>新一代测序（NGS）已成为系统发育和进化研究中的一个强大工具。在这里，我们应用NGS从16个物种中恢复了两个核糖体RNA基因（18S和28S），并从16个物种代表了6个家庭的15种线粒体基因组，这些物种代表了亚纲Aphroditiformia（Phyllodocida，Annelida），一个复杂的多毛类群，背e或鳞片。由于分类单元采样不足以及单个基因标记的分辨率低，几组尺度蠕虫的系统发育关系仍未解决。基于线粒体基因组的系统发生树拓扑结构与基于来自两个线粒体基因（cox1和16S）和两个核糖体基因（18S和28S）基因的连接序列的系统发育树拓扑结构相当，但对几个进化枝具有更高的统计学支持。我们的分析表明，Aphroditiformia是单系的，表明elytra的存在是一种无融合的特征。 Eulepethidae和Aphroditidae一起构成了这个亚目的所有其他家庭的姐妹群，而Acoetidae是Iphionidae的姐妹。 Polynoidae属单系，但在这个家族内，深海亚科Branchinotogluminae和Macellicephalinae属于同系类。大多数蚯蚓家系的线粒体基因组具有保守的基因顺序，但在Polynoidae中，深海进化枝中有两种新颖的排列模式。整个多角体中的线粒体蛋白质编码基因已经在强大的纯化选择下发展，但是深海物种的替代率远高于浅水物种，这表明在深海多角体中净化选择是放松的。对于深海进化枝的一些线粒体基因，存在正选择的氨基酸，表明它们可能涉及深海多角体的适应。总体而言，我们的研究（1）为重建Aphroditiformia的系统发育提供了更多证据，（2）提供了证据来驳斥Errantia线粒体基因顺序保守的假设，并且（3）表明深海极端环境可能具有影响了Polynoidae中线粒体基因组的进化速率和基因排列顺序。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Mol. Phylogenet. Evol.">
                Journal: Mol. Phylogenet. Evol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29625228" target="_blank">
                PMID: 29625228
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-03
            </a>

            <a class="btn btn-sm btn-danger" title="Zhang Y, Sun J, Rouse GW, Wiklund H, Pleijel F, Watanabe HK, Chen C, Qian PY, Qiu JW">
                Authors: Zhang Y, Sun J, R...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29625228">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>149. Characteristics of advanced methods used for typing bacterial isolates from mastitis with particular reference to Staphylococci.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29624017" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29624017" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29624017">
                    <p>Staphylococci are very common human and animal pathogens. A variety of staphylococcal virulence determinates leads to vast range of infections. One of them is mastitis which is a common disease of the mammary glands. The incidence of this disease is widespread all over the world and depends on bacterial virulence and on prevention programs. The influence of mastitis on human health is not globally evaluated, however, in veterinary fields loses in milk production caused by bovine mastitis are a constant economic problem. One of the most important parts of the mastitis control programs is accurate diagnosis of the inflammation and characterization of the etiological factors which leads to reduction of mastitis spread. Recent reports show that staphylococci are common bacterial etiological factors of mastitis, and this paper is an overview of the diagnostic typing methods used for characterization of staphylococcal isolates. A number of different techniques available to applicate is described. Phenotypic methods to identify and to differentiate isolates or discriminate virulence factors are still in use, however, some advanced genetic methods offering higher discriminatory power are reported as more accurate. In fact, nowadays the most powerful tool on that field is next generation sequencing (NGS) of the whole genome, but its high cost and requirement of special laboratory equipment makes it hard to use for routine diagnostics. That is why standard PCR techniques-based methods, and the sequencing of particular genes, are mostly used for typing bacterial isolates. Most of these techniques are characterized by a high discriminatory power, big epidemiological concordance, and repeatable results. The presented report describes the techniques used most frequent in mastitis diagnostics related to staphylococci typing and shows their advantages and disadvantages.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29624017">
                    <p>葡萄球菌是非常常见的人和动物病原体。各种葡萄球菌毒力决定因素导致广泛的感染。其中之一是乳腺炎，这是乳腺常见的疾病。这种疾病的发病率在全世界范围内广泛存在，并取决于细菌毒力和预防方案。乳腺炎对人类健康的影响尚未得到全球评估，但是，在兽医领域，由牛乳腺炎引起的牛奶产量损失是一个经常性的问题。乳腺炎控制项目最重要的部分之一是准确诊断导致乳腺炎扩散减少的病因因素的炎症和表征。最近的报道显示葡萄球菌是乳腺炎的常见细菌病因，本文概述了用于葡萄球菌分离物鉴定的诊断分型方法。描述了申请人可用的许多不同技术。鉴定和区分分离株或鉴别毒力因子的表型方法仍在使用中，然而，一些提供更高辨别力的先进遗传方法被报道为更准确。事实上，如今在该领域最强大的工具是全基因组的下一代测序（NGS），但其高成本和特殊实验室设备的要求使其难以用于常规诊断。这就是为什么基于标准PCR技术的方法和特定基因的测序主要用于分型细菌分离株的原因。大多数这些技术的特点是具有高度的歧视能力，流行病学的一致性和可重复的结果。本报告描述了与葡萄球菌分型相关的乳腺炎诊断中使用最频繁的技术，并展示了它们的优点和缺点。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Pol J Vet Sci">
                Journal: Pol J Vet Sci [IF=0.697]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29624017" target="_blank">
                PMID: 29624017
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar
            </a>

            <a class="btn btn-sm btn-danger" title="Lisowska-Łysiak K, Dudko P, Kosecka-Strojek M, Walczak J, Wójcik P, Międzobrodzki J">
                Authors: Lisowska-Łysiak K...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29624017">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>150. TrawlerWeb: an online de novo motif discovery tool for next-generation sequencing datasets.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29621972" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29621972" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29621972">
                    <p>A strong focus of the post-genomic era is mining of the non-coding regulatory genome in order to unravel the function of regulatory elements that coordinate gene expression (Nat 489:57-74, 2012; Nat 507:462-70, 2014; Nat 507:455-61, 2014; Nat 518:317-30, 2015). Whole-genome approaches based on next-generation sequencing (NGS) have provided insight into the genomic location of regulatory elements throughout different cell types, organs and organisms. These technologies are now widespread and commonly used in laboratories from various fields of research. This highlights the need for fast and user-friendly software tools dedicated to extracting cis-regulatory information contained in these regulatory regions; for instance transcription factor binding site (TFBS) composition. Ideally, such tools should not require prior programming knowledge to ensure they are accessible for all users.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29621972">
                    <p>后基因组时代的一个重点是挖掘非编码调控基因组，以便揭示协调基因表达的调节元件的功能（Nat 489：57-74，2012; Nat 507：462-70,2014; Nat 507：455-61,2014; Nat 518：317-30,2015）。基于新一代测序（NGS）的全基因组方法已经提供了对不同细胞类型，器官和生物体中调节元件的基因组位置的了解。这些技术现在已广泛应用于各个研究领域的实验室。这突出表明需要快速和用户友好的软件工具来提取这些监管区域中包含的顺式规则信息;例如转录因子结合位点（TFBS）组成。理想情况下，这些工具不需要事先编程知识来确保它们可供所有用户访问。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Genomics">
                Journal: BMC Genomics [IF=3.729]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29621972" target="_blank">
                PMID: 29621972
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-05
            </a>

            <a class="btn btn-sm btn-danger" title="Dang LT, Tondl M, Chiu MHH, Revote J, Paten B, Tano V, Tokolyi A, Besse F, Quaife-Ryan G, Cumming H, Drvodelic MJ, Eichenlaub MP, Hallab JC, Stolper JS, Rossello FJ, Bogoyevitch MA, Jans DA, Nim HT, Porrello ER, Hudson JE, Ramialison M">
                Authors: Dang LT, Tondl M,...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5887194" target="_blank">
                PMC5887194
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29621972">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>151. Identifying Associations between Somatic Mutations and Clinicopathologic Findings in Lung Cancer Pathology Reports.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29621832" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29621832" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29621832">
                    <p>We aim to build an informatics methodology capable of identifying statistically significant associations between the clinical findings of non-small cell lung cancer (NSCLC) recorded in patient pathology reports and the various clinically actionable genetic mutations identified from next-generation sequencing (NGS) of patient tumor samples.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29621832">
                    <p>我们的目标是构建一种信息学方法，能够识别患者病理报告中记录的非小细胞肺癌（NSCLC）的临床表现与患者下一代测序（NGS）中鉴定的各种临床可行的基因突变之间的统计学显着关联肿瘤样品。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Methods Inf Med">
                Journal: Methods Inf Med [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29621832" target="_blank">
                PMID: 29621832
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-02
            </a>

            <a class="btn btn-sm btn-danger" title="Kumar N, Tafe LJ, Higgins JH, Peterson JD, de Abreu FB, Deharvengt SJ, Tsongalis GJ, Amos CI, Hassanpour S">
                Authors: Kumar N, Tafe LJ,...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29621832">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>152. Performance comparison of three DNA extraction kits on human whole-exome data from formalin-fixed paraffin-embedded normal and tumor samples.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29621323" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29621323" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29621323">
                    <p>Next-generation sequencing (NGS) studies are becoming routinely used for the detection of novel and clinically actionable DNA variants at a pangenomic scale. Such analyses are now used in the clinical practice to enable precision medicine. Formalin-fixed paraffin-embedded (FFPE) tissues are still one of the most abundant source of cancer clinical specimen, unfortunately this method of preparation is known to degrade DNA and therefore compromise subsequent analysis. Some studies have reported that variant detection can be performed on FFPE samples sequenced with NGS techniques, but few or none have done an in-depth coverage analysis and compared the influence of different state-of-the-art FFPE DNA extraction kits on the quality of the variant calling. Here, we generated 42 human whole-exome sequencing data sets from fresh-frozen (FF) and FFPE samples. These samples include normal and tumor tissues from two different organs (liver and colon), that we extracted with three different FFPE extraction kits (QIAamp DNA FFPE Tissue kit and GeneRead DNA FFPE kit from Qiagen, Maxwell™ RSC DNA FFPE Kit from Promega). We determined the rate of concordance of called variants between matched FF and FFPE samples on all common variants (representing at least 86% of the total number of variants for SNVs). The concordance rate is very high between all matched FF / FFPE pairs, with equivalent values for the three kits we analyzed. On the other hand, when looking at the difference between the total number of variants in FF and FFPE, we find a significant variation for the three different FFPE DNA extraction kits. Coverage analysis shows that FFPE samples have less good indicators than FF samples, yet the coverage quality remains above accepted thresholds. We detect limited but statistically significant variations in coverage indicator values between the three FFPE extraction kits. Globally, the GeneRead and QIAamp kits have better variant calling and coverage indicators than the Maxwell kit on the samples used in this study, although this kit performs better on some indicators and has advantages in terms of practical usage. Taken together, our results confirm the potential of FFPE samples analysis for clinical genomic studies, but also indicate that the choice of a FFPE DNA extraction kit should be done with careful testing and analysis beforehand in order to maximize the accuracy of the results.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29621323">
                    <p>新一代测序（NGS）研究正在成为常规用于在pangenomic规模检测新型和临床可行的DNA变体。现在这种分析在临床实践中用于精确医学。福尔马林固定的石蜡包埋（FFPE）组织仍然是癌症临床标本中最丰富的来源之一，不幸的是，这种制备方法已知会降解DNA并因此影响随后的分析。一些研究报道可以在用NGS技术测序的FFPE样品上进行变异检测，但很少或没有做过深入的覆盖分析，并比较了不同的最先进的FFPE DNA提取试剂盒对质量的影响的变体调用。在这里，我们从新鲜冷冻（FF）和FFPE样品中产生了42个人类全外显子组测序数据集。这些样品包括来自两种不同器官（肝脏和结肠）的正常组织和肿瘤组织，我们用三种不同的FFPE提取试剂盒（来自Qiagen的QIAamp DNA FFPE组织试剂盒和GeneRead DNA FFPE试剂盒，来自Promega的Maxwell TM RSC DNA FFPE试剂盒）提取。我们确定了所有常见变体（相对于SNV变体总数的至少86％）匹配的FF和FFPE样本之间所称变体的一致性率。所有匹配的FF / FFPE对之间的一致率非常高，我们分析的三种试剂盒的等同值相同。另一方面，当查看FF和FFPE变体总数的差异时，我们发现三种不同的FFPE DNA提取试剂盒存在显着差异。覆盖率分析表明，FFPE样本的指标好于FF样本，但覆盖质量仍然高于公认的阈值。我们检测到三种FFPE提取试剂盒之间的覆盖率指标值存在有限但统计上显着的变化。在全球范围内，GeneRead和QIAamp试剂盒比本研究中使用的Maxwell试剂盒具有更好的变异检测和覆盖率指标，尽管此试剂盒在某些指标上表现更好，并且在实际使用方面具有优势。综上所述，我们的研究结果证实了FFPE样品分析用于临床基因组研究的潜力，但也表明FFPE DNA提取试剂盒的选择应事先进行仔细的检测和分析，以最大限度地提高结果的准确性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29621323" target="_blank">
                PMID: 29621323
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Bonnet E, Moutet ML, Baulard C, Bacq-Daian D, Sandron F, Mesrob L, Fin B, Delépine M, Palomares MA, Jubin C, Blanché H, Meyer V, Boland A, Olaso R, Deleuze JF">
                Authors: Bonnet E, Moutet ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5886566" target="_blank">
                PMC5886566
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29621323">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>153. iCopyDAV: Integrated platform for copy number variations-Detection, annotation and visualization.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29621297" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29621297" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29621297">
                    <p>Discovery of copy number variations (CNVs), a major category of structural variations, have dramatically changed our understanding of differences between individuals and provide an alternate paradigm for the genetic basis of human diseases. CNVs include both copy gain and copy loss events and their detection genome-wide is now possible using high-throughput, low-cost next generation sequencing (NGS) methods. However, accurate detection of CNVs from NGS data is not straightforward due to non-uniform coverage of reads resulting from various systemic biases. We have developed an integrated platform, iCopyDAV, to handle some of these issues in CNV detection in whole genome NGS data. It has a modular framework comprising five major modules: data pre-treatment, segmentation, variant calling, annotation and visualization. An important feature of iCopyDAV is the functional annotation module that enables the user to identify and prioritize CNVs encompassing various functional elements, genomic features and disease-associations. Parallelization of the segmentation algorithms makes the iCopyDAV platform even accessible on a desktop. Here we show the effect of sequencing coverage, read length, bin size, data pre-treatment and segmentation approaches on accurate detection of the complete spectrum of CNVs. Performance of iCopyDAV is evaluated on both simulated data and real data for different sequencing depths. It is an open-source integrated pipeline available at https://github.com/vogetihrsh/icopydav and as Docker&#39;s image at http://bioinf.iiit.ac.in/icopydav/.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29621297">
                    <p>发现拷贝数变异（CNV）是结构变异的一个主要类别，已经极大地改变了我们对个体间差异的理解，并为人类疾病的遗传基础提供了另一种范式。 CNV包括拷贝增益和拷贝丢失事件，并且它们的全基因组检测现在可以使用高通量，低成本的下一代测序（NGS）方法。然而，由于各种系统性偏倚引起的阅读不均匀覆盖，从NGS数据准确检测CNV并非易事。我们已经开发了一个集成平台iCopyDAV来处理全基因组NGS数据中CNV检测中的一些问题。它有一个由五个主要模块组成的模块化框架：数据预处理，分割，变体调用，注释和可视化。 iCopyDAV的一个重要功能是功能注释模块，它使用户能够识别和区分包含各种功能元素，基因组特征和疾病关联的CNV。分割算法的并行化使得iCopyDAV平台甚至可以在桌面上访问。在这里，我们将展示测序覆盖率，阅读长度，分子大小，数据预处理和分割方法对准确检测CNV完整谱的影响。 iCopyDAV的性能在模拟数据和实际数据上针对不同的测序深度进行评估。它是一个开源的集成管道，可以在https://github.com/vogetihrsh/icopydav和Docker的图片http://bioinf.iiit.ac.in/icopydav/上找到。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29621297" target="_blank">
                PMID: 29621297
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Dharanipragada P, Vogeti S, Parekh N">
                Authors: Dharanipragada P,...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5886540" target="_blank">
                PMC5886540
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29621297">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>154. Comparison of an in vitro Diagnostic Next-Generation Sequencing Assay with Sanger Sequencing for HIV-1 Genotypic Resistance Testing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29618499" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29618499" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29618499">
                    <p>The ability of next-generation sequencing (NGS) technologies to detect low frequency HIV-1 drug resistance mutations (DRMs) not detected by dideoxynucleotide Sanger sequencing has potential advantages for improved patient outcomes. We compared the performance of an in vitro diagnostic (IVD) NGS assay, the Sentosa® SQ HIV Genotyping Assay for HIV-1 genotypic resistance testing, with Sanger sequencing on 138 protease/reverse transcriptase (RT) and 39 integrase sequences. The NGS assay used a five percent threshold for reporting low-frequency variants. The level of complete plus partial nucleotide concordance between Sanger and NGS was 99.9%. Among the 138 protease/RT sequences, a mean of 6.4 DRMs were identified by both Sanger and NGS, 0.5 DRMs were detected by NGS alone, and 0.1 were detected by Sanger alone. Among the 39 integrase sequences, a mean of 1.6 DRMs were detected by both Sanger and NGS and 0.15 were detected by NGS alone. Compared with Sanger sequencing, NGS estimated higher levels of drug resistance to one or more antiretroviral drugs for 18.2% of protease/RT sequences and 5.1% of integrase sequences. There was little evidence for technical artifact in the NGS sequences but G-to-A hypermutation was detected in three samples. In conclusion, the IVD NGS assay evaluated in this study was highly concordant with Sanger sequencing. At the five percent threshold for reporting minority variants, NGS appeared to attain a modest increased sensitivity for detecting low-frequency DRMs without compromising sequence accuracy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29618499">
                    <p>新一代测序（NGS）技术检测未被双脱氧核苷酸Sanger测序检测到的低频HIV-1耐药突变（DRM）的能力对于改善患者结果具有潜在优势。我们比较了体外诊断（IVD）NGS检测，用于HIV-1基因型耐药性检测的Sentosa®SQ HIV基因分型检测，以及138个蛋白酶/逆转录酶（RT）和39个整合酶序列上的Sanger测序的性能。 NGS分析使用5％的阈值报告低频变异。 Sanger和NGS之间完全加上部分核苷酸一致性的水平为99.9％。在138个蛋白酶/ RT序列中，Sanger和NGS均确定了6.4个DRMs的平均值，单独NGS检测到0.5个DRMs，仅Sanger检测到0.1个DRMs。在39个整合酶序列中，Sanger和NGS均检测到平均1.6个DRM，单独NGS检测到0.15个DRM。与Sanger测序相比，NGS估计18.2％的蛋白酶/ RT序列和5.1％的整合酶序列对一种或多种抗逆转录病毒药物具有较高的耐药性水平。在NGS序列中几乎没有证据表明技术伪影，但在三个样品中检测到G到A超突变。总之，本研究中评估的IVD NGS测定与Sanger测序高度一致。在报告少数变体的百分之五阈值时，NGS似乎在不影响序列准确性的情况下检测低频DRM的敏感度略有提高。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Clin. Microbiol.">
                Journal: J. Clin. Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29618499" target="_blank">
                PMID: 29618499
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-04
            </a>

            <a class="btn btn-sm btn-danger" title="Tzou PL, Ariyaratne P, Varghese V, Lee C, Rakhmanaliev E, Villy C, Yee M, Tan K, Michel G, Pinsky BA, Shafer RW">
                Authors: Tzou PL, Ariyarat...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29618499">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>155. Genetic analysis of Wnt/PCP genes in neural tube defects.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29618362" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29618362" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29618362">
                    <p>Mouse homozygous mutants in Wnt/planar cell polarity (PCP) pathway genes have been shown to cause neural tube defects (NTDs) through the disruption of normal morphogenetic processes critical to neural tube closure (NTC). Knockout mice that are heterozygotes of single PCP genes likely fail to produce NTD phenotypes, yet damaging variants detected in human NTDs are almost always heterozygous, suggesting that other deleterious interacting variants are likely to be present. Nonetheless, the Wnt/PCP pathway remains a genetic hotspot. Addressing these issues is essential for understanding the genetic etiology of human NTDs.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29618362">
                    <p>已经显示Wnt /平面细胞极性（PCP）途径基因中的小鼠纯合突变体通过破坏对神经管闭合（NTC）关键的正常形态发生过程而导致神经管缺陷（NTD）。单基因PCP基因杂合子的敲除小鼠可能不能产生NTD表型，但在人NTD中检测到的破坏性变体几乎总是杂合的，这表明其他有害的相互作用变体可能存在。尽管如此，Wnt / PCP途径仍然是一个遗传热点。解决这些问题对于理解人类NTDs的遗传病因至关重要。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Med Genomics">
                Journal: BMC Med Genomics [IF=2.848]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29618362" target="_blank">
                PMID: 29618362
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-04
            </a>

            <a class="btn btn-sm btn-danger" title="Chen Z, Lei Y, Cao X, Zheng Y, Wang F, Bao Y, Peng R, Finnell RH, Zhang T, Wang H">
                Authors: Chen Z, Lei Y, Ca...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5885375" target="_blank">
                PMC5885375
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29618362">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>156. nQuire: a statistical framework for ploidy estimation using next generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29618319" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29618319" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29618319">
                    <p>Intraspecific variation in ploidy occurs in a wide range of species including pathogenic and nonpathogenic eukaryotes such as yeasts and oomycetes. Ploidy can be inferred indirectly - without measuring DNA content - from experiments using next-generation sequencing (NGS). We present nQuire, a statistical framework that distinguishes between diploids, triploids and tetraploids using NGS. The command-line tool models the distribution of base frequencies at variable sites using a Gaussian Mixture Model, and uses maximum likelihood to select the most plausible ploidy model. nQuire handles large genomes at high coverage efficiently and uses standard input file formats.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29618319">
                    <p>倍性的种内变异发生在各种物种中，包括致病性和非致病性真核生物如酵母和卵菌。通过使用下一代测序（NGS）的实验，可以间接推断倍性（无需测量DNA含量）。我们介绍nQuire，一种统计框架，使用NGS区分二倍体，三倍体和四倍体。命令行工具使用高斯混合模型模拟可变点处基频的分布，并使用最大似然来选择最可能的倍性模型。 nQuire能够高效处理高覆盖率的大型基因组，并使用标准输入文件格式。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Bioinformatics">
                Journal: BMC Bioinformatics [IF=2.448]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29618319" target="_blank">
                PMID: 29618319
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-04
            </a>

            <a class="btn btn-sm btn-danger" title="Weiß CL, Pais M, Cano LM, Kamoun S, Burbano HA">
                Authors: Weiß CL, Pais M, ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5885312" target="_blank">
                PMC5885312
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29618319">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>157. Estimation of the Genetic Substitution Rate of Hanwoo and Holstein Cattle Using Whole Genome Sequencing Data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29618185" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29618185" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29618185">
                    <p>Despite the importance of mutation rate, some difficulties exist in estimating it. Next-generation sequencing (NGS) data yields large numbers of single-nucleotide polymorphisms, which can make it feasible to estimate substitution rates. The genetic substitution rates of Hanwoo and Holstein cattle were estimated using NGS data. Our main findings was to calculate the gene&#39;s substitution rates. Through estimation of genetic substitution rates, we found: diving region of altered substitution density exists. This region may indicate a boundary between protected and unprotected genes. The protected region is mainly associated with the gene ontology terms of regulatory genes. The genes that distinguish Hanwoo from Holstein in terms of substitution rate predominantly have gene ontology terms related to blood and circulatory system. This might imply that Hanwoo and Holstein evolved with dissimilar mutation rates and processes after domestication. The difference in meat quality between Hanwoo and Holstein could originate from differential evolution of the genes related to these blood and circulatory system ontology terms.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29618185">
                    <p>尽管突变率的重要性，估计它存在一些困难。新一代测序（NGS）数据产生大量单核苷酸多态性，这可以使估计替代率成为可能。用NGS数据估算了韩窝和荷斯坦牛的遗传替代率。我们的主要研究结果是计算基因的替代率。通过遗传替代率的估计，我们发现存在改变替代密度的潜水区域。该区域可能表示受保护和未受保护基因之间的界限。受保护的区域主要与调控基因的基因本体术语有关。根据替代率区分Hanwoo和Holstein的基因主要有与血液和循环系统相关的基因本体术语。这可能意味着Hanwoo和Holstein在驯化后发生了不同的突变率和过程。 Hanwoo和Holstein之间的肉品质差异可能源于与这些血液和循环系统本体术语有关的基因的差异进化。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genomics Inform">
                Journal: Genomics Inform [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29618185" target="_blank">
                PMID: 29618185
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar
            </a>

            <a class="btn btn-sm btn-danger" title="Lee YS, Shin D">
                Authors: Lee YS, Shin D
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5903062" target="_blank">
                PMC5903062
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29618185">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>158. AGORA : Organellar genome annotation from the amino acid and nucleotide references.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29617954" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29617954" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29617954">
                    <p>Next-generation sequencing (NGS) technologies have led to the accumulation of highthroughput sequence data from various organisms in biology. To apply gene annotation of organellar genomes for various organisms, more optimized tools for functional gene annotation are required. Almost all gene annotation tools are mainly focused on the chloroplast genome of land plants or the mitochondrial genome of animals.We have developed a web application AGORA for the fast, user-friendly, and improved annotations of organellar genomes. AGORA annotates genes based on a BLAST-based homology search and clustering with selected reference sequences from the NCBI database or user-defined uploaded data. AGORA can annotate the functional genes in almost all mitochondrion and plastid genomes of eukaryotes. The gene annotation of a genome with an exon-intron structure within a gene or inverted repeat region is also available. It provides information of start and end positions of each gene, BLAST results compared with the reference sequence, and visualization of gene map by OGDRAW.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29617954">
                    <p>新一代测序（NGS）技术已经导致来自生物体中各种生物体的高通量序列数据的积累。为了将细胞器基因组的基因注释应用于各种生物体，需要用于功能基因注释的更优化的工具。几乎所有的基因注释工具都主要集中在陆地植物的叶绿体基因组或动物的线粒体基因组。我们开发了一个网络应用程序AGORA，用于快速，方便用户和改进的细胞器基因组注释。 AGORA基于基于BLAST的同源性搜索和来自NCBI数据库的选定参考序列或用户定义的上传数据进行聚类来标注基因。 AGORA可以在真核生物的几乎所有线粒体和质体基因组中注释功能基因。在基因或反向重复区域内具有外显子 - 内含子结构的基因组的基因注释也是可用的。它提供了每个基因的起始和终止位置信息，与参考序列比较的BLAST结果以及OGDRAW可视化基因图。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Bioinformatics">
                Journal: Bioinformatics [IF=7.307]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29617954" target="_blank">
                PMID: 29617954
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-29
            </a>

            <a class="btn btn-sm btn-danger" title="Jung J, Kim JI, Jeong YS, Yi G">
                Authors: Jung J, Kim JI, J...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29617954">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>159. Characterization of two novel mycoviruses from Penicillium digitatum and the related fungicide resistance analysis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29615698" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29615698" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29615698">
                    <p>Pathogenic fungi including Penicillium digitatum and Penicillium italicum are the main destructive pathogens in the citrus industry, causing great losses during postharvest process. To our knowledge, only one mycovirus from P. digitatum has been reported, and the prevalence of such mycoviruses against citrus postharvest pathogenic fungi and their genotyping were still under investigation. In the present study, we showed that 39 of 152 Penicillium isolates from main citrus-growing areas in China were infected with various mycoviruses belonging to polymycoviruses, Narna-like viruses, and families Totiviridae, Partitivirdae and Chrysoviridae. The next generation sequencing (NGS) towards virus genome library and the following molecular analysis revealed two novel mycoviruses Penicillium digitatum polymycovirus 1 (PdPmV1) and Penicillium digitatum Narna-like virus 1 (PdNLV1), coexisting in P. digitatum strain HS-RH2. The fungicide-resistant P. digitatum strains HS-F6 and HS-E9 coinfected by PdPmV1 and PdNLV1 exhibited obvious reduction in triazole drug prochloraz resistance by mycelial growth analysis on both PDA plates and citrus fruit epidermis with given prochloraz concentration. This report at the first time characterized two novel mycoviruses from P. digitatum and revealed the mycovirus-induced reduction of fungicide resistance.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29615698">
                    <p>包括Penicillium digitatum和Penicillium italicum在内的病原真菌是柑橘产业的主要破坏性病原体，在采后过程中造成很大的损失。据我们所知，只有一种来自P. digitatum的真菌病毒已被报道，此类真菌病毒对柑橘采后病原真菌及其基因分型的发病率仍在调查中。在本研究中，我们发现来自中国主要柑橘生长地区的152株青霉分离株中有39株感染了多种属于多粘病毒，类Narna病毒以及Totiviridae，Partitivirdae和Chrysoviridae家族的真菌病毒。针对病毒基因组文库的下一代测序（NGS）和以下分子分析揭示了两种新型真菌病毒指状青霉多粘病毒1（PdPmV1）和指状青霉病毒Narna样病毒1（PdNLV1），共存于指状青霉菌菌株HS-RH2中。用PdPmV1和PdNLV1共同感染​​的抗真菌的P. digitatum菌株HS-F6和HS-E9通过菌丝生长分析显示在给定咪鲜胺浓度的PDA平板和柑橘果实表皮上三唑药物丙氯灵抗性显着降低。该报告首次描述了来自P. digitatum的两种新型真菌病毒，并揭示了真菌病毒引起的对杀真菌剂抗性的降低。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29615698" target="_blank">
                PMID: 29615698
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-03
            </a>

            <a class="btn btn-sm btn-danger" title="Niu Y, Yuan Y, Mao J, Yang Z, Cao Q, Zhang T, Wang S, Liu D">
                Authors: Niu Y, Yuan Y, Ma...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5882929" target="_blank">
                PMC5882929
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29615698">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>160. Clinical and pathological characteristics of KEAP1- and NFE2L2- mutated non-small cell lung carcinoma (NSCLC).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29615460" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29615460" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29615460">
                    <p>KEAP1 and NFE2L2 mutations are associated with impaired prognosis in many cancers and with squamous cell carcinoma formation in non-small cell lung cancer (NSCLC). However, few is known about frequency, histology-dependence, molecular and clinical presentation as well as response to systemic treatment in NSCLC.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29615460">
                    <p>KEAP1和NFE2L2突变与许多癌症的预后受损以及非小细胞肺癌（NSCLC）中的鳞状细胞癌形成相关。然而，关于频率，组织学依赖性，分子和临床表现以及对NSCLC系统治疗的反应知之甚少。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Cancer Res.">
                Journal: Clin. Cancer Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29615460" target="_blank">
                PMID: 29615460
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-03
            </a>

            <a class="btn btn-sm btn-danger" title="Frank R, Scheffler M, Merkelbach-Bruse S, Ihle MA, Kron A, Rauer M, Ueckeroth F, Koenig K, Michels S, Fischer R, Eisert A, Fassunke J, Heydt C, Serke M, Ko YD, Gerigk U, Geist T, Kaminsky B, Heukamp L, Clement-Ziza M, Buettner R, Wolf J">
                Authors: Frank R, Scheffle...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29615460">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>161. CMS to Cover NGS Companion Diagnostics.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29615401" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29615401" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29615401">
                    <p>The Centers for Medicare &amp; Medicaid Services recently announced that next-generation sequencing-based companion diagnostics will be covered for beneficiaries with advanced cancer. This announcement takes coverage decisions out of the hands of local contractors, providing more uniform access to such tests.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29615401">
                    <p>医疗保险和医疗补助服务中心最近宣布，为患有晚期癌症的受益人提供下一代基于测序的伴侣诊断。本公告将覆盖决策置于当地承包商手中，为此类测试提供更加统一的访问权限。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Discov">
                Journal: Cancer Discov [IF=20.011]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29615401" target="_blank">
                PMID: 29615401
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="">
                Authors: 
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29615401">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>162. Use of the Ion AmpliSeq Cancer Hotspot Panel in clinical molecular pathology laboratories for analysis of solid tumours: With emphasis on validation with relevant single molecular pathology tests and the Oncomine Focus Assay.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29615338" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29615338" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29615338">
                    <p>Targeted application of next-generation sequencing (NGS) technology allows detection of specific mutations that can provide treatment opportunities for cancer patients. We evaluated the applicability of the Ion AmpliSeq Cancer Hotspot Panel V2 (CHV2) using formalin-fixed, paraffin-embedded (FFPE) tissue of clinical specimens. Thirty-five FFPE tumour samples with known mutational status were collected from four different hospitals and sequenced with CHV2 using an Ion Chef System and Ion S5 XL system. Out of 35 cases, seven were sequenced with Oncomine focus Assay Panel for comparison. For the limit of detection test, we used an FFPE reference standard, a cell line that included an engineered 50% EGFR T790 M in an RKO cell line background. Coverage analysis results including number of mapped reads, on target percent, mean depth, and uniformity were not different according to hospitals. Sensitivity for mutation detection down to 3% was demonstrated. NGS results showed 100% concordance with the results from single molecular pathology tests Assay in 30 cases with 24 known positive mutations and 14 known negative mutations, and another NGS panel of the Oncomine focus in seven cases. The CHV2 NGS test for solid tumours using Ion chef system and S5 XL system in clinical molecular pathology laboratories for analysis of solid tumours could be routinely used and could replace some single molecular pathology tests after a stringent and thorough validation process.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29615338">
                    <p>新一代测序（NGS）技术的靶向应用可以检测出可为癌症患者提供治疗机会的特定突变。我们使用福尔马林固定的石蜡包埋（FFPE）临床标本组织评估了Ion AmpliSeq Cancer Hotspot Panel V2（CHV2）的适用性。从四家不同的医院收集了35个具有已知突变状态的FFPE肿瘤样品，并使用Ion Chef System和Ion S5 XL系统用CHV2测序。在35个病例中，7个用Oncomine重点检测小组进行测序以作比较。对于检测限的检测，我们使用了FFPE参考标准，这是一种在RKO细胞系背景中包含工程化的50％EGFR T790M的细胞系。根据医院的覆盖率分析结果，包括映射读数的数量，目标百分比，平均深度和均匀性没有差异。展示了低至3％的突变检测灵敏度。 NGS结果显示与30例24个已知阳性突变和14个已知阴性突变的单个分子病理学测试结果的100％一致性，以及另一个Oncomine聚焦7个NGS小组的结果。在临床分子病理学实验室中使用Ion chef系统和S5 XL系统进行固体肿瘤的CHV2 NGS检测可用于常规分析实体瘤，并可在经过严格和彻底的验证过程后替代某些单分子病理学检测。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Pathol. Res. Pract.">
                Journal: Pathol. Res. Pract. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29615338" target="_blank">
                PMID: 29615338
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-29
            </a>

            <a class="btn btn-sm btn-danger" title="Lee A, Lee SH, Jung CK, Park G, Lee KY, Choi HJ, Min KO, Kim TJ, Lee EJ, Lee YS">
                Authors: Lee A, Lee SH, Ju...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29615338">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>163. Early Stage HER2-Positive Breast Cancers Not Achieving a pCR From Neoadjuvant Trastuzumab- or Pertuzumab-Based Regimens Have an Immunosuppressive Phenotype.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29615305" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29615305" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29615305">
                    <p>Stromal tumor-infiltrating lymphocytes (TILs) might predict pathologic complete response (pCR) in patients with HER2-positive (HER2+) breast cancer treated with trastuzumab (H). Docetaxel (T), carboplatin (C), H, and pertuzumab (P) have immune-modulating effects. Pre- and post-treatment immune biomarkers in cancers treated with neoadjuvant TCH with or without P are lacking. In this study we quantified baseline and changes in TILs, cluster of differentiation (CD) 4+, CD8+, FoxP3+, and PD-L1+ cells using immunohistochemistry (IHC) and quantified productive T-cell receptor β (TCRβ) rearrangements and TCRβ clonality using next-generation sequencing (NGS) in 30 HER2+ breast cancer tissues treated with neoadjuvant H with or without P regimens.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29615305">
                    <p>基质细胞瘤浸润淋巴细胞（TILs）可能预测曲妥珠单抗（H）治疗的HER2阳性（HER2 +）乳腺癌患者的病理完全缓解（pCR）。多西他赛（T），卡铂（C），H和帕妥珠单抗（P）具有免疫调节作用。缺乏用新辅助TCH治疗或不治疗P的癌症治疗前和治疗后的免疫生物标志物。在本研究中，我们使用免疫组织化学（IHC）和量化的生产性T细胞受体β（TCRβ）重排和TCRβ克隆来量化TIL，分化簇（CD）4+，CD8 +，FoxP3 +和PD-L1 +细胞的基线和变化新一代测序（NGS）在用新辅助治疗H或不用P方案治疗的30例HER2 +乳腺癌组织中的表达。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Breast Cancer">
                Journal: Clin. Breast Cancer [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29615305" target="_blank">
                PMID: 29615305
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb-24
            </a>

            <a class="btn btn-sm btn-danger" title="Force J, Howie LJ, Abbott SE, Bentley R, Marcom PK, Kimmick G, Westbrook K, Sammons SL, Parks M, Topping DL, Emerson R, Broadwater G, Hyslop T, Blackwell KL, Nair SK">
                Authors: Force J, Howie LJ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29615305">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>164. Phylogenomics and barcoding of Panax: toward the identification of ginseng species.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29614961" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29614961" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29614961">
                    <p>The economic value of ginseng in the global medicinal plant trade is estimated to be in excess of US$2.1 billion. At the same time, the evolutionary placement of ginseng (Panax ginseng) and the complex evolutionary history of the genus is poorly understood despite several molecular phylogenetic studies. In this study, we use a full plastome phylogenomic framework to resolve relationships in Panax and to identify molecular markers for species discrimination.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29614961">
                    <p>据估计，全球药用植物贸易中人参的经济价值超过21亿美元。同时，尽管进行了几项分子系统发育研究，人参（Panax ginseng）的进化放置和该属的复杂进化史仍不甚了解。在这项研究中，我们使用完整的plastome phylogenomic框架来解决人参中的关系，并确定物种鉴别的分子标记。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Evol. Biol.">
                Journal: BMC Evol. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29614961" target="_blank">
                PMID: 29614961
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-03
            </a>

            <a class="btn btn-sm btn-danger" title="Manzanilla V, Kool A, Nguyen Nhat L, Nong Van H, Le Thi Thu H, de Boer HJ">
                Authors: Manzanilla V, Koo...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5883351" target="_blank">
                PMC5883351
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29614961">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>165. Discovery of novel transcripts of the human tissue kallikrein (KLK1) and kallikrein-related peptidase 2 (KLK2) in human cancer cells, exploiting Next-Generation Sequencing technology.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29614347" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29614347" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29614347">
                    <p>Tissue kallikrein, kallikrein-related peptidases (KLKs), and plasma kallikrein form the largest group of serine proteases in the human genome, sharing many structural and functional properties. Several KLK transcripts have been found aberrantly expressed in numerous human malignancies, confirming their prognostic or/and diagnostic values. However, the process of alternative splicing can now be studied in-depth due to the development of Next-Generation Sequencing (NGS). In the present study, we used NGS to discover novel transcripts of the KLK1 and KLK2 genes, after nested touchdown PCR. Bioinformatics analysis and PCR experiments revealed a total of eleven novel KLK transcripts (two KLK1 and nine KLK2 transcripts). In addition, the expression profiles of each novel transcript were investigated with nested PCR experiments using variant-specific primers. Since KLKs are implicated in human malignancies, qualifying as potential biomarkers, the quantification of the presented novel transcripts in human samples may have clinical applications in different types of cancer.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29614347">
                    <p>组织激肽释放酶，激肽释放酶相关肽酶（KLKs）和血浆激肽释放酶是人类基因组中最大的一组丝氨酸蛋白酶，具有许多结构和功能特性。已经发现几种KLK转录物在许多人类恶性肿瘤中异常表达，证实了它们的预后或/和诊断价值。然而，由于下一代测序（NGS）的发展，现在可以深入研究选择性剪接的过程。在本研究中，我们使用NGS发现嵌套接触式PCR后KLK1和KLK2基因的新转录物。生物信息学分析和PCR实验共揭示了十一种新型KLK转录物（两种KLK1和九种KLK2转录物）。此外，使用变体特异性引物，使用巢式PCR实验研究每种新转录物的表达谱。由于KLK与人类恶性肿瘤相关，因此有资格成为潜在的生物标志物，所提供的人类样本中新转录物的定量可能在不同类型的癌症中具有临床应用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genomics">
                Journal: Genomics [IF=2.801]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29614347" target="_blank">
                PMID: 29614347
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-31
            </a>

            <a class="btn btn-sm btn-danger" title="Adamopoulos PG, Kontos CK, Scorilas A">
                Authors: Adamopoulos PG, K...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29614347">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>166. Equine-like G3 rotavirus strains as predominant strains among children in Indonesia in 2015-2016.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29614325" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29614325" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29614325">
                    <p>Rotavirus A (RVA) is a major cause of acute gastroenteritis in humans and animals worldwide. As a result of the segmented nature of the rotavirus genome, genetic reassortment commonly occurs. This study aims to clarify the genetic characteristics of RVAs circulating in Indonesia. From June 2015 through August 2016, stool samples were collected from 134 children aged &lt;5 years (71 male and 63 female) with acute gastroenteritis who were inpatients at a private hospital in Surabaya, Indonesia. All stool samples were screened for RVA antigen using immunochromatography. Forty-two samples (31.3%, 42/134) were RVA antigen-positive. All RVA positive samples tested showed the unusual combinations of G3P[8] (n = 36) and G3P[6] (n = 3) with a short RNA pattern by G/P typing and polyacrylamide gel electrophoresis (PAGE). Whole genome analysis by next-generation sequencing (NGS) was performed for 11 strains to determine the RVA genotypes. Eleven rotavirus strains were found to carry a DS-like genetic backbone; nine strains showed a G3-P[8]-I2-R2-C2-M2-A2-N2-T2-E2-H2 genome constellation, which was recently reported in Australia, Hungary, Spain and Brazil; as well, two strains showed a G3-P[6]-I2-R2-C2-M2-A2-N2-T2-E2-H2 genome constellation. The phylogenetic tree based on the VP7 gene showed that all 11 strains were classified as equine-like G3, which is genetically distinct and different in origin from typical human G3 strains. The phylogenetic tree based on the NSP4 gene showed that six strains were classified as bovine-like strain and the remaining five were classified as human strain. In conclusion, we identified the strains which are intergenogroup reassortants containing an equine-like G3 VP7, a P[8])/P[6] VP4, with a DS-1-like genetic backbone. These findings suggest that equine-like G3P[8] and P[6] RVA strains have been circulating in the Indonesian population for at least 1 year and probably longer, indicating a diversity of RVAs in this area.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29614325">
                    <p>轮状病毒A（RVA）是全球人类和动物急性胃肠炎的主要原因。由于轮状病毒基因组的分段性质，通常会发生遗传重配。本研究旨在阐明印度尼西亚流行的RVAs的遗传特征。从2015年6月至2016年8月，从印度尼西亚泗水一家私人医院住院的&lt;134岁&lt;5岁儿童（71男性和63女性）患有急性胃肠炎的儿童收集粪便样本。使用免疫色谱法筛选所有粪便样品的RVA抗原。 42份样本（31.3％，42/134）为RVA抗原阳性。所有RVA阳性样本通过G / P分型和聚丙烯酰胺凝胶电泳（PAGE）显示G3P [8]（n = 36）和G3P [6]（n = 3）具有短RNA模式的不寻常组合。对11个菌株进行通过下一代测序（NGS）进行的全基因组分析以确定RVA基因型。发现11个轮状病毒株携带DS样遗传主链;最近在澳大利亚，匈牙利，西班牙和巴西报道了9个菌株显示G3-P [8] -I2-R2-C2-M2-A2-N2-T2-E2-H2基因组星座。以及两个菌株显示出G3-P [6] -I2-R2-C2-M2-A2-N2-T2-E2-H2基因组星座。基于VP7基因的系统发育树表明，所有11个菌株均被归类为马类G3，其来源于典型的人类G3菌株，其遗传性明显不同。基于NSP4基因的系统发育树显示6个菌株被分类为牛样菌株，其余5个被分类为人类菌株。总之，我们确定了含有类似G3 VP7，P [8]）/ P [6] VP4的基因间重组株与具有DS-1样遗传骨架的菌株。这些研究结果表明马类G3P [8]和P [6] RVA菌株在印度尼西亚人群中至少流行了1年，可能更长，表明该区域有多种RVAs。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Infect. Genet. Evol.">
                Journal: Infect. Genet. Evol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29614325" target="_blank">
                PMID: 29614325
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-31
            </a>

            <a class="btn btn-sm btn-danger" title="Utsumi T, Wahyuni RM, Doan YH, Dinana Z, Soegijanto S, Fujii Y, Juniastuti LN, Yamani C, Matsui L, Deng T, Abe MI, Soetjipto K, Lusida H, Ishii K, Shimizu I">
                Authors: Utsumi T, Wahyuni...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29614325">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>167. Determining Genotypic Drug Resistance by Ion Semiconductor Sequencing With the Ion AmpliSeq™ TB Panel in Multidrug-Resistant Mycobacterium tuberculosis Isolates.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29611381" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29611381" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29611381">
                    <p>We examined the feasibility of a full-length gene analysis for the drug resistance-related genes inhA, katG, rpoB, pncA, rpsL, embB, eis, and gyrA using ion semiconductor next-generation sequencing (NGS) and compared the results with those obtained from conventional phenotypic drug susceptibility testing (DST) in multidrug-resistant Mycobacterium tuberculosis (MDR-TB) isolates.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29611381">
                    <p>我们使用离子半导体下一代测序（NGS）研究了耐药相关基因inhA，katG，rpoB，pncA，rpsL，embB，eis和gyrA的全长基因分析的可行性，并将结果与​​那些从耐多药结核分枝杆菌（MDR-TB）分离株的常规表型药敏试验（DST）获得。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ann Lab Med">
                Journal: Ann Lab Med [IF=2.174]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29611381" target="_blank">
                PMID: 29611381
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jul
            </a>

            <a class="btn btn-sm btn-danger" title="Park J, Shin SY, Kim K, Park K, Shin S, Ihm C">
                Authors: Park J, Shin SY, ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5895860" target="_blank">
                PMC5895860
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29611381">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>168. Principles of Genetic Counseling in the Era of Next-Generation Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29611378" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29611378" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29611378">
                    <p>Traditional genetic counseling has focused on the target gene and its natural progress with respect to disease risk. Next-generation sequencing (NGS) can produce information on several genetic variants simultaneously, with different functions and consequences for each. Accordingly, determining the status of the patient or consultant and interpreting sequencing results from many genes can largely increase the complexity of genetic counseling. Moreover, the current environment of big data that can be readily shared via the internet and a ubiquitous network provides many different avenues for which a consultant must handle the traditional principle of genetic counseling in different ways. Thus, further consideration and rethinking of genetic counseling principles are necessary in the era of NGS. In this review, we discuss several aspects of genetic counseling that one can encounter when faced with NGS data.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29611378">
                    <p>传统的遗传咨询专注于目标基因及其在疾病风险方面的自然发展。新一代测序（NGS）可以同时产生关于几种遗传变异的信息，对每种变异都有不同的功能和结果。因此，确定患者或顾问的状态并解释来自许多基因的测序结果可以大大增加遗传咨询的复杂性。而且，目前可通过互联网和无处不在的网络共享的大数据环境提供了许多不同的途径，顾问必须以不同的方式处理传统的遗传咨询原则。因此，在NGS时代，进一步考虑和重新思考遗传咨询原则是必要的。在这篇综述中，我们讨论了面对NGS数据时可能遇到的遗传咨询的几个方面。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ann Lab Med">
                Journal: Ann Lab Med [IF=2.174]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29611378" target="_blank">
                PMID: 29611378
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jul
            </a>

            <a class="btn btn-sm btn-danger" title="Yang M, Kim JW">
                Authors: Yang M, Kim JW
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5895857" target="_blank">
                PMC5895857
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29611378">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>169. Clinically actionable mutation profiles in patients with cancer identified by whole-genome sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29610388" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29610388" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29610388">
                    <p>Next-generation sequencing (NGS) efforts have established catalogs of mutations relevant to cancer development. However, the clinical utility of this information remains largely unexplored. Here, we present the results of the first eight patients recruited into a clinical whole-genome sequencing (WGS) program in the United Kingdom. We performed PCR-free WGS of fresh frozen tumors and germline DNA at 75× and 30×, respectively, using the HiSeq2500 HTv4. Subtracted tumor VCFs and paired germlines were subjected to comprehensive analysis of coding and noncoding regions, integration of germline with somatically acquired variants, and global mutation signatures and pathway analyses. Results were classified into tiers and presented to a multidisciplinary tumor board. WGS results helped to clarify an uncertain histopathological diagnosis in one case, led to informed or supported prognosis in two cases, leading to de-escalation of therapy in one, and indicated potential treatments in all eight. Overall 26 different tier 1 potentially clinically actionable findings were identified using WGS compared with six SNVs/indels using routine targeted NGS. These initial results demonstrate the potential of WGS to inform future diagnosis, prognosis, and treatment choice in cancer and justify the systematic evaluation of the clinical utility of WGS in larger cohorts of patients with cancer.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29610388">
                    <p>新一代测序（NGS）努力已经建立了与癌症发展相关的突变目录。然而，这一信息的临床实用性仍然在很大程度上尚未探索。在这里，我们介绍了在英国招募到临床全基因组测序（WGS）项目的前八名患者的结果。我们使用HiSeq2500 HTv4分别以75×和30×进行新鲜冷冻肿瘤和生殖细胞DNA的无PCR WGS。减数肿瘤VCF和配对生殖系受到编码和非编码区域的综合分析，种系与体细胞获得变体的整合，以及全局突变特征和通路分析。结果被分为多个等级并呈现给多学科肿瘤委员会。 WGS的结果有助于澄清一例病例不确定的组织病理学诊断，导致两例患者获知或支持预后，导致治疗逐步升级，并指出所有八例中的潜在治疗。使用WGS比较了使用常规靶向NGS的6个SNV / indels总共26个不同的1级潜在临床可行性研究结果。这些初步结果证明了WGS可能为癌症的未来诊断，预后和治疗选择提供信息，并证明WGS在更大的癌症患者队列中的临床实用性的系统性评估是合理的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cold Spring Harb Mol Case Stud">
                Journal: Cold Spring Harb Mol Case Stud [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29610388" target="_blank">
                PMID: 29610388
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Schuh A, Dreau H, Knight SJL, Ridout K, Mizani T, Vavoulis D, Colling R, Antoniou P, Kvikstad EM, Pentony MM, Hamblin A, Protheroe A, Parton M, Shah KA, Orosz Z, Athanasou N, Hassan B, Flanagan AM, Ahmed A, Winter S, Harris A, Tomlinson I, Popitsch N, Church D, Taylor JC">
                Authors: Schuh A, Dreau H,...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880257" target="_blank">
                PMC5880257
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29610388">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>170. Optimal Fault Detection and Diagnosis in Transcriptional Circuits Using Next-Generation Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29610100" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29610100" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29610100">
                    <p>We propose a methodology for model-based fault detection and diagnosis for stochastic Boolean dynamical systems indirectly observed through a single time series of transcriptomic measurements using Next Generation Sequencing (NGS) data. The fault detection consists of an innovations filter followed by a fault certification step, and requires no knowledge about the possible system faults. The innovations filter uses the optimal Boolean state estimator, called the Boolean Kalman Filter (BKF). In the presence of knowledge about the possible system faults, we propose an additional step of fault diagnosis based on a multiple model adaptive estimation (MMAE) method consisting of a bank of BKFs running in parallel. Performance is assessed by means of false detection and misdiagnosis rates, as well as average times until correct detection and diagnosis. The efficacy of the proposed methodology is demonstrated via numerical experiments using a p53-MDM2 negative feedback loop Boolean network with stuck-at faults that model molecular events commonly found in cancer.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29610100">
                    <p>我们提出了一种基于模型的故障检测和诊断方法，用于通过使用下一代测序（NGS）数据的单个时间序列转录组测量间接观察的随机布尔动力系统。故障检测由创新过滤器和故障认证步骤组成，并且不需要关于可能的系统故障的知识。创新滤波器使用布尔状态最优估计器，称为布尔卡尔曼滤波器（BKF）。在存在关于可能的系统故障的知识的情况下，我们基于由并行运行的BKF库组成的多模型自适应估计（MMAE）方法提出额外的故障诊断步骤。通过错误检测和误诊率以及平均时间来评估表现，直到正确的检测和诊断。所提出的方法的功效通过使用p53-MDM2负反馈环路布尔网络的数值实验来证明，该网络具有模拟癌症中常见的分子事件的卡住故障。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=IEEE/ACM Trans Comput Biol Bioinform">
                Journal: IEEE/ACM Trans Comput Biol Bioinform [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29610100" target="_blank">
                PMID: 29610100
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018 Mar-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Bahadorinejad A, Braga-Neto UM">
                Authors: Bahadorinejad A, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29610100">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>171. Next generation sequencing assisted in establishing the diagnosis and treatment for a Chinese patient with breast and lung multiple primary malignancies.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29608968" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29608968" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29608968">
                    <p>A case of primary lung cancer was reported, which was misdiagnosed as breast cancer metastasis to the lung based on pathology in combination with a history of breast cancer.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29608968">
                    <p>报道了一例原发性肺癌，根据病理结合乳腺癌病史将其误诊为乳腺癌肺转移。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Thorac Oncol">
                Journal: J Thorac Oncol [IF=6.595]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29608968" target="_blank">
                PMID: 29608968
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-30
            </a>

            <a class="btn btn-sm btn-danger" title="Song C, Gao Z, Zong J, Shi J, Yao M, Lu G, Liu X, Wang K, Han L">
                Authors: Song C, Gao Z, Zo...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29608968">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>172. Integrated massively parallel sequencing of 15 autosomal STRs and Amelogenin using a simplified library preparation approach.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29608791" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29608791" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29608791">
                    <p>Massively parallel sequencing (MPS) technologies, also termed as next-generation sequencing (NGS), are becoming increasingly popular in study of short tandem repeats (STR). However, current library preparation methods are usually based on ligation or two-round PCR that requires more steps, making it time-consuming (about 2 days), laborious and expensive. In this study, a 16-plex STR typing system was designed with fusion primer strategy based on the Ion Torrent S5 XL platform which could effectively resolve the above challenges for forensic DNA database-type samples (bloodstains, saliva stains, etc.). The efficiency of this system was tested in 253 Han Chinese participants. The libraries were prepared without DNA isolation and adapter ligation, and the whole process only required approximately 5 h. The proportion of thoroughly genotyped samples in which all the 16 loci were successfully genotyped was 86% (220/256). Of the samples, 99.7% showed 100% concordance between NGS-based STR typing and capillary electrophoresis (CE)-based STR typing. The inconsistency might have been caused by off-ladder alleles and mutations in primer binding sites. Overall, this panel enabled the large-scale genotyping of the DNA samples with controlled quality and quantity because it is a simple, operation-friendly process flow that saves labor, time and costs.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29608791">
                    <p>大规模并行测序（MPS）技术也被称为新一代测序（NGS），在研究短串联重复序列（STR）方面越来越受欢迎。然而，目前的文库制备方法通常基于连接或双轮PCR，这需要更多的步骤，使其耗时（约2天），费力且昂贵。本研究采用基于Ion Torrent S5 XL平台的融合引物策略设计了一种16重STR分型系统，可有效解决法医DNA数据库样本（血迹，唾液污渍等）面临的上述挑战。该系统的效率在253名汉族参与者中进行了测试。这些文库是在没有DNA分离和接头连接的情况下制备的，整个过程只需要约5小时。全部16个基因座成功基因分型的完全基因分型样品的比例为86％（220/256）。在样本中，99.7％显示基于NGS的STR分型与基于毛细管电泳（CE）的STR分型之间的100％一致性。不一致可能是由于梯子外等位基因和引物结合位点的突变造成的。总体而言，该专家组能够对受控质量和数量的DNA样品进行大规模基因分型，因为它是一种简单，操作友好的工艺流程，可节省人力，时间和成本。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Electrophoresis">
                Journal: Electrophoresis [IF=2.744]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29608791" target="_blank">
                PMID: 29608791
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-02
            </a>

            <a class="btn btn-sm btn-danger" title="Xue J, Wu R, Pan Y, Wang S, Qu B, Qin Y, Shi Y, Zhang C, Li R, Zhang L, Zhou C, Sun H">
                Authors: Xue J, Wu R, Pan ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29608791">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>173. Exploring the Feasibility and Utility of Exome-scale Tumour Sequencing in a Clinical Setting.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29607586" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29607586" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29607586">
                    <p>Technology has progressed from single gene panel to large-scale genomic sequencing. This is raising expectations from clinicians and patients alike. The utility and performance of this technology in a clinical setting needs to be evaluated.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29607586">
                    <p>技术从单基因组板块发展到大规模基因组测序。这提高了临床医生和患者的期望。这种技术在临床上的实用性和性能需要进行评估。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Intern Med J">
                Journal: Intern Med J [IF=1.902]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29607586" target="_blank">
                PMID: 29607586
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-01
            </a>

            <a class="btn btn-sm btn-danger" title="Lee B, Tran B, Hsu AL, Taylor GR, Fox SB, Fellowes A, Marquis R, Mooi J, Desai J, Doig K, Ekert P, Gaff C, Herath D, Hamilton A, James P, Roberts A, Snyder R, Waring P, McArthur G">
                Authors: Lee B, Tran B, Hs...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29607586">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>174. Clinical implication of MEN1 mutation in surgically resected thymic carcinoid patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29607201" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29607201" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29607201">
                    <p>Thymic carcinoid is a rare but very aggressive neuroendocrine tumour derived from the neuroendocrine system. Here we report a male patient with thymic atypical carcinoid. Though thymic carcinoid is relatively common, the gene sequencing profile was performed and the gene sequencing result indicated germline multiple endocrine neoplasia type 1 (MEN1) mutation and two somatic mutations on MEN1 gene and no copy number variation or fusion events were detected. It is well-known that the mutation of MEN1 is the typical manifestation of MEN1 syndrome, which is an autosome dominant disease that includes varying combinations of more than 20 endocrine and non-endocrine tumors. In the English literature, 7 cases of solitary thymic carcinoid harboring somatic variants in MEN1 are reported in the absence of other organs involvement as MEN1 syndrome presents. We summarized the clinical features and prognosis of this rare thymic tumor.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29607201">
                    <p>胸腺类癌是由神经内分泌系统衍生的罕见但非常侵略性的神经内分泌肿瘤。在这里，我们报告一名男性患者胸腺非典型类癌。虽然胸腺类癌相对较常见，但进行基因测序分析，基因测序结果显示MEN1基因生殖系多发性内分泌肿瘤1型（MEN1）突变和两个体细胞突变，未检测到拷贝数变异或融合事件。众所周知，MEN1突变是MEN1综合征的典型表现，MEN1综合征是一种常染色体显性疾病，包括20多种内分泌和非内分泌肿瘤的不同组合。在英国文献中，MEN1综合征中没有其他器官受累的报道，在MEN1中有7例携带体细胞变异的孤立性胸腺类癌。我们总结了这种罕见的胸腺肿瘤的临床特征和预后。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Thorac Dis">
                Journal: J Thorac Dis [IF=2.365]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29607201" target="_blank">
                PMID: 29607201
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb
            </a>

            <a class="btn btn-sm btn-danger" title="Li X, Li M, Shi T, Liu R, Ren D, Yang F, Wei S, Chen G, Chen J, Xu S">
                Authors: Li X, Li M, Shi T...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5864637" target="_blank">
                PMC5864637
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29607201">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>175. Sogatella furcifera hepe-like virus: First member of a novel Hepeviridae clade identified in an insect.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29605729" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29605729" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29605729">
                    <p>A novel virus was identified in the white-backed planthopper (Sogatella furcifera, Hemiptera: Delphacidae) and tentatively named Sogatella furcifera hepe-like virus (SfHeV). Its genome is a linear, single-stranded monopartite RNA, 7,312 nucleotides (nt) long with a 66-nt 5&#39; UTR, 54-nt 3&#39; UTR, and 28-nt polyA, showing typical genomic features of viruses in the family Hepeviridae, but highly divergent from known members in the family, with amino acid sequence identities of only 18.9-23% (ORF1), 13.1-18.8% (ORF2) and 1.9-11% (ORF3). Phylogenetic analysis revealed that SfHeV was closer to cutthroat trout virus (CTV), but did not cluster with any members of the family. SfHeV is the first hepe-like virus identified in a hemipteran insect and was detected in all developmental stages suggesting the presence of some level of vertical transmission. On the basis of these data, we propose that SfHeV represents a novel clade in the family Hepeviridae and tentatively name the genus Insecthepevirus.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29605729">
                    <p>在白背飞虱（Sogatella furcifera，半翅目：Delphacidae）中鉴定出新型病毒，并初步命名为Sogatella furcifera类疱疹病毒（SfHeV）。它的基因组是一条线性单链单组RNA，长度为7,312核苷酸（nt），含66nt nt 5&#39;UTR，54 nt 3&#39;UTR和28 nt polyA，显示了病毒在Hepeviridae家族中的典型基因组特征，但与家族中已知成员高度不同，氨基酸序列同一性仅为18.9-23％（ORF1），13.1-18.8％（ORF2）和1.9-11％（ORF3）。系统发育分析显示SfHeV更接近cutthroat鳟鱼病毒（CTV），但没有与任何家庭成员聚类。 SfHeV是第一个在半翅目昆虫中鉴定出的类疱疹病毒，在所有发育阶段都检测到，表明存在一定程度的垂直传播。在这些数据的基础上，我们提出SfHeV代表了一个在Hepeviridae家族中的新进化枝，并暂定名为昆虫类病毒。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Virus Res.">
                Journal: Virus Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29605729" target="_blank">
                PMID: 29605729
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May-02
            </a>

            <a class="btn btn-sm btn-danger" title="Wu N, Zhang P, Liu W, Wang X">
                Authors: Wu N, Zhang P, Li...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29605729">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>176. Long Terminal Repeat CRISPR-CAR-Coupled &quot;Universal&quot; T Cells Mediate Potent Anti-leukemic Effects.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29605708" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29605708" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29605708">
                    <p>Gene editing can be used to overcome allo-recognition, which otherwise limits allogeneic T cell therapies. Initial proof-of-concept applications have included generation of such &quot;universal&quot; T cells expressing chimeric antigen receptors (CARs) against CD19 target antigens combined with transient expression of DNA-targeting nucleases to disrupt the T cell receptor alpha constant chain (TRAC). Although relatively efficient, transgene expression and editing effects were unlinked, yields variable, and resulting T cell populations heterogeneous, complicating dosing strategies. We describe a self-inactivating lentiviral &quot;terminal&quot; vector platform coupling CAR expression with CRISPR/Cas9 effects through incorporation of an sgRNA element into the ΔU3 3&#39; long terminal repeat (LTR). Following reverse transcription and duplication of the hybrid ΔU3-sgRNA, delivery of Cas9 mRNA resulted in targeted TRAC locus cleavage and allowed the enrichment of highly homogeneous (&gt;96%) CAR+ (&gt;99%) TCR- populations by automated magnetic separation. Molecular analyses, including NGS, WGS, and Digenome-seq, verified on-target specificity with no evidence of predicted off-target events. Robust anti-leukemic effects were demonstrated in humanized immunodeficient mice and were sustained longer than by conventional CAR+TCR+ T cells. Terminal-TRAC (TT) CAR T cells offer the possibility of a pre-manufactured, non-HLA-matched CAR cell therapy and will be evaluated in phase 1 trials against B cell malignancies shortly.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29605708">
                    <p>基因编辑可用于克服异体识别，否则会限制同种异体T细胞治疗。初步的概念验证应用包括产生表达针对CD19靶抗原的嵌合抗原受体（CAR）的此类“通用”T细胞以及破坏T细胞受体α恒定链（TRAC）的DNA-靶向核酸酶的瞬时表达。尽管相对有效，但转基因表达和编辑效应不连锁，产生可变性，并且导致T细胞群体异质性，使给药策略复杂化。我们描述了通过将sgRNA元件并入ΔU33&#39;长末端重复序列（LTR）中，将CAR表达与CRISPR / Cas9效应偶联的自灭活慢病毒“末端”载体平台。在杂交ΔU3-sgRNA的逆转录和复制之后，递送Cas9 mRNA导致靶向的TRAC基因座切割并且通过自动磁分离允许高度均一（&gt; 96％）CAR +（&gt; 99％）TCR-群体的富集。包括NGS，WGS和Digenome-seq在内的分子分析验证了靶特异性，没有预测脱靶事件的证据。在人源化免疫缺陷小鼠中证明了强大的抗白血病作用，并且比传统CAR + TCR + T细胞持续更长时间。终端-TRAC（TT）CAR T细胞提供了预先制造的非HLA匹配的CAR细胞治疗的可能性，并将在短期内针对B细胞恶性肿瘤进行的1期临床试验中进行评估。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Mol. Ther.">
                Journal: Mol. Ther. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29605708" target="_blank">
                PMID: 29605708
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May-02
            </a>

            <a class="btn btn-sm btn-danger" title="Georgiadis C, Preece R, Nickolay L, Etuk A, Petrova A, Ladon D, Danyi A, Humphryes-Kirilov N, Ajetunmobi A, Kim D, Kim JS, Qasim W">
                Authors: Georgiadis C, Pre...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29605708">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>177. Next generation sequencing for preimplantation genetic screening improves pregnancy outcomes compared with array comparative genomic hybridization in single thawed euploid embryo transfer cycles.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29605407" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29605407" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29605407">
                    <p>To evaluate whether the use of next generation sequencing (NGS) for preimplantation genetic screening (PGS) in single thawed euploid embryo transfer (STEET) cycles improves pregnancy outcomes compared with array comparative genomic hybridization (aCGH).</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29605407">
                    <p>为了评估与阵列比较基因组杂交（aCGH）相比，在单一解冻整倍体胚胎移植（STEET）周期中使用下一代测序（NGS）进行植入前基因筛选（PGS）是否改善妊娠结局。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Fertil. Steril.">
                Journal: Fertil. Steril. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29605407" target="_blank">
                PMID: 29605407
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Friedenthal J, Maxwell SM, Munné S, Kramer Y, McCulloh DH, McCaffrey C, Grifo JA">
                Authors: Friedenthal J, Ma...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29605407">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>178. Emberger syndrome: A rare association with hearing loss.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29605372" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29605372" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29605372">
                    <p>Emberger Syndrome (ES) is a rare genetic disorder characterized by lymphedema and myelodysplasia. It is also associated with hearing loss. The genetic mutations associated with ES are not part of the comprehensive 80 gene next generation sequencing (NGS) panel. As a result, the otolaryngologist should maintain an index of suspicion for ES in any child with SNHL who presents repeatedly with recurrent infections, lymphedema and/or cutaneous warts. This paper describes the clinical evolution and management of two children who were followed up for hearing loss and eventually were diagnosed with ES.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29605372">
                    <p>Emberger综合征（ES）是以淋巴水肿和骨髓发育不良为特征的罕见遗传疾病。它也与听力损失有关。与ES相关的基因突变不是全面的80基因下一代测序（NGS）小组的一部分。因此，耳鼻喉科医师应该对任何反复出现感染，淋巴水肿和/或皮肤疣的SNHL儿童保持一个怀疑ES的指标。本文介绍了随访听力丧失并最终被诊断为ES的两名儿童的临床演变和管理。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int. J. Pediatr. Otorhinolaryngol.">
                Journal: Int. J. Pediatr. Otorhinolaryngol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29605372" target="_blank">
                PMID: 29605372
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Zawawi F, Sokolov M, Mawby T, Gordon KA, Papsin BC, Cushing SL">
                Authors: Zawawi F, Sokolov...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29605372">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>179. A novel missense mutation in GIPC3 causes sensorineural hearing loss in an Iranian family revealed by targeted next-generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29605370" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29605370" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29605370">
                    <p>Recent studies have confirmed the utility of targeted next-generation sequencing (NGS), providing a remarkable opportunity to find variants in known disease genes, especially in genetically heterogeneous disorders such as hearing loss (HL).</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29605370">
                    <p>最近的研究证实了靶向下一代测序（NGS）的实用性，为发现已知疾病基因的变异体，特别是遗传异质性疾病如听力丧失（HL）提供了显着的机会。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int. J. Pediatr. Otorhinolaryngol.">
                Journal: Int. J. Pediatr. Otorhinolaryngol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29605370" target="_blank">
                PMID: 29605370
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Asgharzade S, Tabatabaiefar MA, Mohammadi-Asl J, Chaleshtori MH">
                Authors: Asgharzade S, Tab...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29605370">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>180. Utility of whole exome sequencing in the diagnosis of Usher syndrome: Report of novel compound heterozygous MYO7A mutations.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29605349" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29605349" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29605349">
                    <p>Next generation sequencing (NGS), such as targeted panel sequencing, whole-exome sequencing and whole-genome sequencing has led to an exponential increase of elucidated genetic causes in both rare diseases, and common but heterogeneous disorders. NGS is applied in both research and clinical settings, and the clinical exome sequencing (CES), which provides not only the sequence variation data but also clinical interpretation, aids in reaching a final conclusion with regards to a genetic diagnosis. Usher syndrome is a group of disorders, characterized by bilateral sensorineural hearing loss, with or without vestibular dysfunction and retinitis pigmentosa. The index patient, a 2-year-old child was initially diagnosed with nonsyndromic hearing impairment. Homozygosity mapping followed by CES was utilized as a diagnostic tool to identify the genetic basis of his hearing loss. A paternally inherited novel insertion, c.198_199insA (p.Val67Serfs*73) and a maternally inherited novel deletion, c.1219_1226del (p.Phe407Aspfs*33) in gene MYO7A were found in compound heterozygous state in the index patient. The result expands the mutational spectrum of MYO7A. In addition it helped in early diagnosis of the syndrome, for planning and adjustments for the patient, and as well as for future family planning. This study highlights the clinical effectiveness of CES for Usher syndrome diagnosis in a child presented with congenital hearing loss.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29605349">
                    <p>新一代测序（NGS），如有针对性的小组测序，全外显子组测序和全基因组测序已导致两种罕见疾病中明确的遗传原因以及常见但异质性疾病的指数增加。 NGS适用于研究和临床环境，临床外显子组测序（CES）不仅提供序列变异数据，而且提供临床解释，有助于就基因诊断达成最终结论。 Usher综合征是一组以两侧感觉神经性听力丧失，伴或不伴前庭功能障碍和色素性视网膜炎为特征的疾病。索引患者，一名2岁儿童最初被诊断为非综合征性听力障碍。使用CES之后的纯合映射作为诊断工具来鉴定他的听力丧失的遗传基础。在基因MYO7A中发现一个父本遗传的新插入片段c.198_199insA（p.Val67Serfs * 73）和一个母本遗传的新的缺失片段c.1219_1226del（p.Phe407Aspfs * 33），该片段在复合杂合子状态中被发现。结果扩展了MYO7A的突变谱。此外，它有助于早期诊断综合征，为患者进行规划和调整，以及为未来的计划生育做出贡献。本研究强调了CES对先天性听力丧失小儿Usher综合征诊断的临床有效性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int. J. Pediatr. Otorhinolaryngol.">
                Journal: Int. J. Pediatr. Otorhinolaryngol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29605349" target="_blank">
                PMID: 29605349
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Ramzan K, Al-Owain M, Huma R, Al-Hazzaa SAF, Al-Ageel S, Imtiaz F, Al-Sayed M">
                Authors: Ramzan K, Al-Owai...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29605349">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>181. Radiotherapy-Sensitized Tumor Photothermal Ablation Using γ-Polyglutamic Acid Nanogels Loaded with Polypyrrole.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29601720" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29601720" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29601720">
                    <p>Development of versatile nanoscale platforms for cancer diagnosis and therapy is of great importance for applications in translational medicine. In this work, we present the use of γ-polyglutamic acid (γ-PGA) nanogels (NGs) to load polypyrrole (PPy) for thermal/photoacoustic (PA) imaging and radiotherapy (RT)-sensitized tumor photothermal therapy (PTT). First, a double emulsion approach was used to prepare the cystamine dihydrochloride (Cys)-cross-linked γ-PGA NGs. Next, the cross-linked NGs served as a reactor to be filled with pyrrole monomers that were subjected to in situ oxidation polymerization in the existence of Fe(III) ions. The formed uniform PPy-loaded NGs having an average diameter of 38.9 ± 8.6 nm exhibited good water-dispersibility and colloid stability. The prominent near-infrared (NIR) absorbance feature due to the loaded PPy endowed the NGs with contrast enhancement in PA imaging. The hybrid NGs possessed excellent photothermal conversion efficiency (64.7%) and stability against laser irradiation, and could be adopted for PA imaging and PTT of cancerous cells and tumor xenografts. Importantly, we also explored the cooperative PTT and X-ray radiation-mediated RT for enhanced tumor therapy. We show that PTT of tumors can be more significantly sensitized by RT using the sequence of laser irradiation followed by X-ray radiation as compared to using the reverse sequence. Our study suggests a promising theranostic platform of hybrid NGs that may be potentially utilized for PA imaging and combination therapy of different types of tumors.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29601720">
                    <p>用于癌症诊断和治疗的多功能纳米级平台的开发对于转化医学中的应用非常重要。在这项工作中，我们提出使用γ-聚谷氨酸（γ-PGA）纳米凝胶（NGs）加载聚吡咯（PPy）进行热/光声（PA）成像和放疗（RT） - 敏化肿瘤光热疗法（PTT）。首先，使用双重乳液方法来制备胱胺二盐酸盐（Cys） - 交联的γ-PGA NGs。接下来，交联的NG用作填充吡咯单体的反应器，所述吡咯单体在存在Fe（III）离子的情况下进行原位氧化聚合。平均直径为38.9±8.6nm的均匀形成PPy的NGs表现出良好的水分散性和胶体稳定性。由于加载PPy，突出的近红外（NIR）吸收特征赋予NG在PA成像中对比度增强。杂合NGs具有优良的光热转换效率（64.7％）和对激光照射的稳定性，可用于癌细胞和肿瘤异种移植物的PA成像和PTT。重要的是，我们还探索了用于增强肿瘤治疗的合作PTT和X射线辐射介导的RT。我们表明，与使用相反序列相比，使用激光照射然后是X射线照射的序列，可以通过RT对肿瘤的PTT进行更显着的敏化。我们的研究表明一种有希望的混合NGs的分析平台，其可能用于PA成像和不同类型的肿瘤的组合治疗。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Biomacromolecules">
                Journal: Biomacromolecules [IF=5.246]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29601720" target="_blank">
                PMID: 29601720
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-06
            </a>

            <a class="btn btn-sm btn-danger" title="Zhou Y, Hu Y, Sun W, Lu S, Cai C, Peng C, Yu J, Popovtzer R, Shen M, Shi X">
                Authors: Zhou Y, Hu Y, Sun...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29601720">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>182. USP8 Mutations in Pituitary Cushing Adenomas-Targeted Analysis by Next-Generation Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29600293" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29600293" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29600293">
                    <p>Gain-of-function somatic mutations in the ubiquitin specific protease 8 (USP8) gene have recently been reported as a cause of pituitary adenomas in Cushing disease. Molecular diagnostic testing of tumor tissue may aid in the diagnosis of specimens obtained through therapeutic transsphenoidal surgery; however, for small tumors, availability of fresh tissue is limited, and contamination with normal tissue is frequent. We performed molecular testing of DNA isolated from single formalin-fixed and paraffin-embedded (FFPE) tissue sections of 42 pituitary adenomas from patients with Cushing disease (27 female patients and 15 male patients; mean age at surgery, 42.5 years; mean tumor size, 12.2 mm). By Sanger sequencing, we identified previously reported USP8 missense mutations in six tumors. Targeted next-generation sequencing (NGS) revealed known or previously undescribed missense mutations in three additional tumors (two with two different mutations each), with mutant allele frequencies as low as 3%. Of the nine tumors with USP8 mutations (mutation frequency, 21.4%), seven were from female patients (mutation frequency, 25.9%), and two were from male patients (mutation frequency, 13.3%). Mutant tumors were on average 11.4 mm in size, and patients with mutations were on average 43.9 years of age. The overall USP8 mutation frequency in our cohort was lower than in previously described cohorts, and we did not observe USP8 deletions that were frequent in other cohorts. We demonstrate that testing for USP8 variants can be performed from small amounts of FFPE tissue. NGS showed higher sensitivity for USP8 mutation detection than did Sanger sequencing. Assessment for USP8 mutations may complement histopathological diagnosis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29600293">
                    <p>最近已报道泛素特异性蛋白酶8（USP8）基因中的功能增益体细胞突变是库欣病中垂体腺瘤的原因。肿瘤组织的分子诊断检测可能有助于通过治疗性蝶窦手术获得的标本的诊断;然而，对于小肿瘤，新鲜组织的可用性是有限的，并且正常组织的污染频繁。我们对来自库欣病患者的42个垂体腺瘤的单个福尔马林固定和石蜡包埋（FFPE）组织切片分离的DNA进行了分子检测（27名女性患者和15名男性患者;平均手术年龄42.5岁;平均肿瘤大小，12.2毫米）。通过Sanger测序，我们确定了先前报道的6个肿瘤中的USP8错义突变。靶向的新一代测序（NGS）揭示了三个额外肿瘤中的已知或先前未描述的错义突变（两个各有两个不同的突变），突变等位基因频率低至3％。 USP8基因突变9例（突变频率21.4％）中，女性患者7例（突变频率25.9％），男性患者2例（突变频率13.3％）。突变肿瘤的平均大小为11.4mm，突变患者的平均年龄为43.9岁。 USP8突变频率在我们的队列中低于先前描述的队列，并且我们没有观察到在其他队列中频繁出现的USP8缺失。我们证明可以从少量的FFPE组织进行USP8变体的测试。与Sanger测序相比，NGS对USP8突变检测的敏感性更高。对USP8突变的评估可以补充组织病理学诊断。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Endocr Soc">
                Journal: J Endocr Soc [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29600293" target="_blank">
                PMID: 29600293
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-01
            </a>

            <a class="btn btn-sm btn-danger" title="Ballmann C, Thiel A, Korah HE, Reis AC, Saeger W, Stepanow S, Köhrer K, Reifenberger G, Knobbe-Thomsen CB, Knappe UJ, Scholl UI">
                Authors: Ballmann C, Thiel...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5838826" target="_blank">
                PMC5838826
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29600293">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>183. Molecular dissection of engraftment in a xenograft model of myelodysplastic syndromes.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29599920" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29599920" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29599920">
                    <p>Myelodysplastic syndromes (MDS) are oligoclonal disorders of the hematopoietic stem cells (HSC). Recurrent gene mutations are involved in the MDS physiopathology along with the medullar microenvironment. To better study the heterogeneity of MDS, it is necessary to create patient derived xenograft (PDX). We have reproduced a PDX model by xenografting HSC (CD34+) and mesenchymal stromal cells (MSC) in NOD/SCID/IL2rγ-/- mice with primary samples from one RAEB2, two RAEB1 and one RARS patients harboring karyotype abnormalities and gene mutations. The average human chimerisms ranged from 59.7% to 0.0175% for the 4 patients. Secondary grafts (G2) were only performed for mice derived from the RAEB2 patient and the average human chimerism was 53.33%. G1 mice 1 and 2, and their derived G2 mice showed less than 20% of medullar blasts whereas mouse 3 and the resulting G2 mice transformed to AML. Clonal architecture was dissected in the different hematopoietic progenitors (HP) harvested from G1 and G2 mice. By direct Sanger sequencing, we found the 4 initial mutations in each HP subpopulation and those mutations had the same variant allele frequency in the CD34+ CD38- HSC from G1 and G2 mice by next generation sequencing (NGS). Targeted NGS analysis done in HSC of mouse 3 did not show any additional driver gene mutations explaining the transformation to AML. To conclude, we have generated a PDX mouse model that perfectly reproduces the MDS founder clone which is stable over time, allowing us to consider this system as a powerful tool to test therapeutic approaches.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29599920">
                    <p>骨髓增生异常综合征（MDS）是造血干细胞（HSC）的寡克隆疾病。复发性基因突变与髓质微环境一起参与MDS病理生理学。为了更好地研究MDS的异质性，有必要创建患者衍生的异种移植物（PDX）。从一个RAEB2，二RAEB1和一个RARS患者窝藏核型异常和基因突变小鼠初级样品 -  /  - ，我们通过在NOD / SCID /IL2rγ异种移植HSC（CD34 +）和间充质基质细胞（MSC）再现的PDX模型。 4名患者的平均人嵌合体范围从59.7％至0.0175％。次要移植物（G2）仅对来自RAEB2患者的小鼠进行，并且平均人嵌合体为53.33％。 G1小鼠1和2及其衍生的G2小鼠显示小于20％的髓质原始细胞，而小鼠3和所得的G2小鼠转化为AML。在从G1和G2小鼠收获的不同造血祖细胞（HP）中解剖克隆结构。通过直接Sanger测序，我们发现每个HP亚群中有4个起始突变，并且这些突变通过下一代测序（NGS）在来自G1和G2小鼠的CD34 + CD38-HSC中具有相同的变异等位基因频率。在小鼠3的HSC中进行的靶向NGS分析没有显示解释向AML转化的任何额外驱动基因突变。总之，我们已经生成了一个PDX鼠标模型，可以完美再现MDS创始人克隆，这种克隆随着时间的推移是稳定的，这使我们可以将此系统视为测试治疗方法的强大工具。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29599920" target="_blank">
                PMID: 29599920
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-13
            </a>

            <a class="btn btn-sm btn-danger" title="Meunier M, Dussiau C, Mauz N, Alary AS, Lefebvre C, Szymanski G, Pezet M, Blanquet F, Kosmider O, Park S">
                Authors: Meunier M, Dussia...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5871091" target="_blank">
                PMC5871091
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29599920">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>184. Genome-wide analysis of replication timing by next-generation sequencing with E/L Repli-seq.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29599440" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29599440" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29599440">
                    <p>This protocol is an extension to: Nat. Protoc. 6, 870-895 (2014); doi:10.1038/nprot.2011.328; published online 02 June 2011Cycling cells duplicate their DNA content during S phase, following a defined program called replication timing (RT). Early- and late-replicating regions differ in terms of mutation rates, transcriptional activity, chromatin marks and subnuclear position. Moreover, RT is regulated during development and is altered in diseases. Here, we describe E/L Repli-seq, an extension of our Repli-chip protocol. E/L Repli-seq is a rapid, robust and relatively inexpensive protocol for analyzing RT by next-generation sequencing (NGS), allowing genome-wide assessment of how cellular processes are linked to RT. Briefly, cells are pulse-labeled with BrdU, and early and late S-phase fractions are sorted by flow cytometry. Labeled nascent DNA is immunoprecipitated from both fractions and sequenced. Data processing leads to a single bedGraph file containing the ratio of nascent DNA from early versus late S-phase fractions. The results are comparable to those of Repli-chip, with the additional benefits of genome-wide sequence information and an increased dynamic range. We also provide computational pipelines for downstream analyses, for parsing phased genomes using single-nucleotide polymorphisms (SNPs) to analyze RT allelic asynchrony, and for direct comparison to Repli-chip data. This protocol can be performed in up to 3 d before sequencing, and requires basic cellular and molecular biology skills, as well as a basic understanding of Unix and R.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29599440">
                    <p>该协议是对以下内容的扩展：Nat。 Protoc。 6，870-895（2014）; DOI：10.1038 / nprot.2011.328;在线发表于2011年6月02日在S期，循环细胞复制其DNA含量，遵循一个称为复制时间（RT）的定义程序。早期和晚期复制区域在突变率，转录活性，染色质标记和亚核位置方面不同。此外，RT在发育过程中受到调节，并在疾病中发生改变。在这里，我们描述了E / L Repli-seq，它是我们Repli芯片协议的扩展。 E / L Repli-seq是一种快速，稳健且相对廉价的用于通过下一代测序（NGS）分析RT的协议，允许全基因组评估细胞过程如何与RT相关联。简而言之，用BrdU脉冲标记细胞，并通过流式细胞术分选早期和晚期S期部分。从两个部分免疫沉淀标记的新生DNA并测序。数据处理导致一个bedGraph文件包含早期和晚期S期组分的新生DNA比例。结果与Repli-chip的结果相当，具有全基因组序列信息和增加的动态范围的额外好处。我们还提供用于下游分析的计算流水线，用于使用单核苷酸多态性（SNPs）分析定相基因组以分析RT等位基因异步性，并且直接与Repli-chip数据进行比较。这个协议可以在测序前3天内完成，需要基本的细胞和分子生物学技能，以及对Unix和R的基本了解。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Nat Protoc">
                Journal: Nat Protoc [IF=10.032]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29599440" target="_blank">
                PMID: 29599440
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Marchal C, Sasaki T, Vera D, Wilson K, Sima J, Rivera-Mulia JC, Trevilla-García C, Nogues C, Nafie E, Gilbert DM">
                Authors: Marchal C, Sasaki...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29599440">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>185. Clinical utility of cell-free DNA for the detection of ALK fusions and genomic mechanisms of ALK inhibitor resistance in non-small cell lung cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29599410" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29599410" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29599410">
                    <p>Patients with advanced non-small cell lung cancer (NSCLC) whose tumors harbor anaplastic lymphoma kinase ALK gene fusions benefit from treatment with ALK inhibitors (ALKi). Analysis of cell-free circulating tumor DNA (cfDNA) may provide a non-invasive way to identify ALK fusions and actionable resistance mechanisms without biopsy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29599410">
                    <p>患有晚期非小细胞肺癌（NSCLC）的患有间变性淋巴瘤激酶ALK基因融合的肿瘤受益于ALK抑制剂（ALKi）的治疗。无细胞循环肿瘤DNA（cfDNA）分析可能提供了一种无创方法来鉴定ALK融合和无需活检的可行性耐药机制。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Cancer Res.">
                Journal: Clin. Cancer Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29599410" target="_blank">
                PMID: 29599410
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-29
            </a>

            <a class="btn btn-sm btn-danger" title="McCoach CE, Blakely CM, Banks KC, Levy B, Chue BM, Raymond VM, Le A, Lee CE, Diaz J, Waqar SN, Purcell WT, Aisner DL, Davies KD, Lanman RB, Shaw AT, Doebele RC">
                Authors: McCoach CE, Blake...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29599410">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>186. Machine-learned analysis of the association of next-generation sequencing based human TRPV1 and TRPA1 genotypes with the sensitivity to heat stimuli and topically applied capsaicin.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29596157" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29596157" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29596157">
                    <p>Heat pain and its modulation by capsaicin varies among subjects in experimental and clinical settings. A plausible cause is a genetic component, of which TRPV1 ion channels, by their response to both heat and capsaicin, are primary candidates. However, TRPA1 channels can heterodimerize with TRPV1 channels and carry genetic variants reported to modulate heat pain sensitivity. To address the role of these candidate genes in capsaicin induced hypersensitization to heat, pain thresholds acquired before and after topical application of capsaicin and TRPA1/TRPV1 exomic sequences derived by next-generation sequencing were assessed in n = 75 healthy volunteers and the genetic information comprised 278 loci. Gaussian mixture modeling indicated two phenotype groups with high or low capsaicin-induced hypersensitization to heat. Unsupervised machine learning implemented as swarm-based clustering hinted at differences in the genetic pattern between these phenotype groups. Several methods of supervised machine learning implemented as random forests, adaptive boosting, k-nearest neighbors, naïve Bayes, support vector machines, and for comparison, binary logistic regression predicted the phenotype group association consistently better when based on the observed genotypes than when using a random permutation of the exomic sequences. Of note, TRPA1 variants were more important for correct phenotype group association than TRPV1 variants. This indicates a role of the TRPA1 and TRPV1 NGS based genetic pattern in the modulation of the individual response to heat related pain phenotypes. When considering earlier evidence that topical capsaicin can induce neuropathy-like quantitative sensory testing patterns in healthy subjects, implications for future analgesic treatments with TRP inhibitors arise.This is an open-access article distributed under the terms of the Creative Commons Attribution-Non Commercial-No Derivatives License 4.0 (CCBY-NC-ND) , where it is permissible to download and share the work provided it is properly cited. The work cannot be changed in any way or used commercially without permission from the journal.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29596157">
                    <p>在实验和临床环境中，辣椒素引起的热痛和其调节作用各不相同。一个可能的原因是遗传因素，其中TRPV1离子通过它们对热和辣椒素的反应而成为主要候选物。然而，TRPA1通道可以与TRPV1通道异源二聚化并携带报道的调节热痛敏感性的遗传变体。为了解决这些候选基因在辣椒素诱导的对热敏感性中的作用，在n = 75名健康志愿者中评估了局部应用辣椒素和TRPA1 / TRPV1外显子序列之前和之后获得的疼痛阈值，所述辣椒素和TRPA1 / TRPV1外显子序列通过下一代测序得到， 278个基因座。高斯混合模型表明两个表型组高辣椒素或低辣椒诱导hypersensitization加热。作为基于群集的聚类实施的无监督机器学习暗示了这些表型群体之间遗传模式的差异。监督机器学习的几种方法被实施为随机森林，自适应增强，k-最近邻居，朴素贝叶斯，支持向量机，并且为了比较，二元逻辑回归预测当基于观察到的基因型时表型群组关联一致地好于当使用外部序列的随机置换。值得注意的是，与TRPV1变体相比，TRPA1变体对于正确的表型组关联更重要。这表明基于TRPA1和TRPV1 NGS的遗传模式在调节对热相关疼痛表型的个体反应中的作用。当考虑早期的证据表明辣椒素可以诱导健康受试者的类神经病样定量感觉测试模式时，对未来使用TRP抑制剂的镇痛治疗产生影响。这是一篇开放存取的文章，根据Creative Commons Attribution-Non Commercial- No Derivatives License 4.0（CCBY-NC-ND），允许下载和共享作品，只要它被正确引用。未经许可，本书不得以任何方式改变或在商业上使用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Pain">
                Journal: Pain [IF=5.445]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29596157" target="_blank">
                PMID: 29596157
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-27
            </a>

            <a class="btn btn-sm btn-danger" title="Kringel D, Geisslinger G, Resch E, Oertel BG, Thrun MC, Heinemann S, Lötsch J">
                Authors: Kringel D, Geissl...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29596157">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>187. Next-generation sequencing provides an added value in determining drug resistance and viral tropism in Cameroonian HIV-1 vertically infected children.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29595649" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29595649" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29595649">
                    <p>With limited and low-genetic barrier drugs used for the prevention of mother-to-child transmission (PMTCT) of HIV in sub-Saharan Africa, vertically transmitted HIV-1 drug-resistance (HIVDR) is concerning and might prompt optimal pediatric strategies.The aim of this study was to ascertain HIVDR and viral-tropism in majority and minority populations among Cameroonian vertically infected children.A comparative analysis among 18 HIV-infected children (7 from PMTCT-exposed mothers and 11 from mothers without PMTCT-exposure) was performed. HIVDR and HIV-1 co-receptor usage was evaluated by analyzing sequences obtained by both Sanger sequencing and ultra-deep 454-pyrosequencing (UDPS), set at 1% threshold.Overall, median (interquartile range) age, viremia, and CD4 count were 6 (4-10) years, 5.5 (4.9-6.0) log10 copies/mL, and 526 (282-645) cells/mm, respectively. All children had wild-type viruses through both Sanger sequencing and UDPS, except for 1 PMTCT-exposed infant harboring minority K103N (8.31%), born to a mother exposed to AZT+3TC+NVP. X4-tropic viruses were found in 5 of 15 (33.3%) children (including 2 cases detected only by UDPS). Rate of X4-tropic viruses was 0% (0/6) below 5 years (also as minority species), and became relatively high above 5 years (55.6% [5/9], P = .040. X4-tropic viruses were higher with CD4 ≤15% (4/9 [44.4%]) versus CD4 &gt;15% (1/6 [16.7%], P = .580); similarly for CD4 ≤200 (3/4 [75%]) versus CD4 &gt;200 (2/11 [18.2%] cells/mm, P = .077.NGS has the ability of excluding NRTI- and NNRTI-mutations as minority species in all but 1 children, thus supporting the safe use of these drug-classes in those without such mutations, henceforth sparing ritonavir-boosted protease inhibitors or integrase inhibitors for the few remaining cases. In children under five years, X4-tropic variants would be rare, suggesting vertical-transmission with CCR5-tropic viruses and possible maraviroc usage at younger ages.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29595649">
                    <p>用撒哈拉以南非洲地区预防艾滋病毒母婴传播（PMTCT）的有限和低基因屏障药物，垂直传播的HIV-1抗药性（HIVDR）与此有关，并可能促成最佳的儿科策略。本研究的目的是确定喀麦隆垂直感染儿童中大多数人和少数民族人群的HIVDR和病毒嗜性。对18名艾滋病毒感染儿童（其中7名来自母婴传播暴露的母亲和11名来自无母婴传播暴露的母亲）进行了比较分析，结果显示，执行。通过分析由Sanger测序和超深454焦磷酸测序（UDPS）获得的序列来评估HIVDR和HIV-1共受体使用，设定为1％阈值。总体而言，中位数（四分位数间距）年龄，病毒血症和CD4计数分别为6（4-10），5.5（4.9-6.0）log 10拷贝/ mL和526（282-645）个细胞/ mm。通过Sanger测序和UDPS测试，所有儿童都有野生型病毒，除了1名暴露于AZT + 3TC + NVP的母亲所生的带有少数K103N（8.31％）的PMTCT暴露婴儿外。 15名儿童中有5名（33.3％）发现X4热带病毒（其中2人仅通过UDPS检测）。 X4热带病毒率在5年以下（也是少数种）为0％（0/6），5年以上相对较高（55.6％[5/9]，P = 0.040; X4热带病毒为与CD4&gt; 15％（1/6 [16.7％]，P = 0.580）相比，CD4≤15％（4/9 [44.4％]）更高;与CD4≤200（3/4 [75％]）相比， CD4&gt; 200（2/11 [18.2％]细胞/ mm，P = 0.077.NGS具有排除NRTI和NNRTI突变作为少数物种在所有儿童中的作用，因此支持这些药物 - 在没有这种突变的那些患者中，这些患者从未使用过利托那韦加强蛋白酶抑制剂或整合酶抑制剂，对于少数患者，5岁以下的儿童中，X4嗜性变异体罕见，提示CCR5热带病毒的垂直传播和可能的马拉韦罗夫病毒使用在较年轻的时候。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Medicine (Baltimore)">
                Journal: Medicine (Baltimore) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29595649" target="_blank">
                PMID: 29595649
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar
            </a>

            <a class="btn btn-sm btn-danger" title="Fokam J, Bellocchi MC, Armenia D, Nanfack AJ, Carioti L, Continenza F, Takou D, Temgoua ES, Tangimpundu C, Torimiro JN, Koki PN, Fokunang CN, Cappelli G, Ndjolo A, Colizzi V, Ceccherini-Silberstein F, Perno CF, Santoro MM">
                Authors: Fokam J, Bellocch...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5895385" target="_blank">
                PMC5895385
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29595649">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>188. Next-Generation Sequencing of Hepatitis C Virus (HCV) Mixed-Genotype Infections in Anti-HCV-Negative Blood Donors.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29594753" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29594753" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29594753">
                    <p>The infection with more than one hepatitis C virus (HCV) genotype especially in subjects with a high risk of multiple HCV exposures has been demonstrated. The role of HCV mixed-genotype infection in viral persistence and treatment effect is not fully understood. The prevalence of such infection varies greatly depending on the technique used for genotype determination and studied population. Next-generation sequencing (NGS) which is suitable for extensive analysis of complex viral populations is a method of choice for studying mixed infections. The aim of the present study was to determine the prevalence of mixed-genotype HCV infections in the Polish seronegative, HCV-RNA-positive blood donors (n = 76). Two-step PCR was used for amplification of 5&#39;-UTR of HCV. Using pyrosequencing altogether, 381,063 reads were obtained. The raw reads were trimmed and subjected to similarity analysis against the entire unfiltered NCBI nt database. Results obtained from NGS were compared with the standard genotyping. One (1.3%) mixed-genotype [3a, 2989 reads (94.8%); 1b, 164 reads (5.2%)] infection was found in a sample diagnosed as genotype 3a only by routine testing. Two samples were identified with different genotypes, compared to routine testing. In conclusion, NGS is a sensitive method for HCV genotyping. The prevalence of mixed-genotype HCV infections in blood donors is low.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29594753">
                    <p>已经证实了特别是在具有多次HCV暴露风险高的受试者中多于一种丙型肝炎病毒（HCV）基因型的感染。 HCV混合基因型感染在病毒持续性和治疗效果中的作用尚未完全了解。这种感染的发生率在很大程度上取决于用于基因型测定和研究人群的技术。适用于复杂病毒种群广泛分析的新一代测序（NGS）是研究混合感染的首选方法。本研究的目的是确定波兰血清阴性，HCV-RNA阳性献血者（n = 76）中混合基因型HCV感染的发病率。两步PCR用于扩增HCV的5&#39;-UTR。全部使用焦磷酸测序，获得381,063个读数。将原始读数修剪并对整个未过滤的NCBI nt数据库进行相似性分析。将从NGS获得的结果与标准基因分型进行比较。一种（1.3％）混合基因型[3a，2989（94.8％）;仅在常规检测中诊断为基因型3a的样品中发现164个读数（5.2％）]感染。与常规测试相比，用不同的基因型鉴定了两个样品。总之，NGS是HCV基因分型的一种敏感方法。献血者中混合基因型HCV感染的发病率很低。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Adv. Exp. Med. Biol.">
                Journal: Adv. Exp. Med. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29594753" target="_blank">
                PMID: 29594753
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-29
            </a>

            <a class="btn btn-sm btn-danger" title="Janiak M, Caraballo Cortés K, Perlejewski K, Kubicka-Russel D, Grabarczyk P, Demkow U, Radkowski M">
                Authors: Janiak M, Carabal...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29594753">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>189. Incidentalome in Neurogenetics: Pathogenic Variant of NSD1 in a Patient With Spinocerebellar Ataxia (SCA).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29593781" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29593781" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29593781">
                    <p>Background: Genetic studies of late-onset sporadic ataxias (&gt;40 years of age) are not routinely indicated. For unresolved cases, next-generation sequencing (NGS) tools, such as whole-exome sequencing (WES), are available for a definitive diagnosis. Case presentation: Our patient is a woman with a usual facial phenotype and anthropometry, who developed ataxia at 45 years of age, with no relevant family history and an initial clinical approach that ruled out common aetiologies. WES was performed when the patient was 54 years old. The results identified the heterozygous pathogenic variant c.248delA (p.N83MfsX4) in the nuclear receptor-binding SET domain protein 1 (NSD1; MIM 606681) gene (related to Sotos syndrome), which was not associated with ataxia and is not related to the patient&#39;s phenotype. Sanger sequencing of NSD1 in two different laboratories confirmed the variant. Conclusions: NGS findings generally offer valuable information that can be used for clinical decision-making. However, an incidental finding that leads to defining new clinical and bioethical actions is also possible. Consequently, the biological importance of this type of genetic &quot;incidentalome&quot; must be determined.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29593781">
                    <p>背景：迟发型散发性共济失调（&gt; 40岁）的遗传学研究并未常规指明。对于未解决的病例，新一代测序（NGS）工具（如全外显子组测序（WES））可用于确定诊断。病例介绍：我们的病人是一名具有常见面部表型和人体测量学的女性，他们在45岁时出现共济失调，没有相关的家族史和排除常见病因的初始临床方法。当患者54岁时进行WES。结果鉴定了核受体结合的SET结构域蛋白1（NSD1; MIM606681）基因（与Sotos综合征有关）中的杂合致病变体c.248delA（p.N83MfsX4），其与共济失调不相关并且与病人的表型。两个不同实验室的NSD1的Sanger测序证实了该变体。结论：NGS发现通常提供可用于临床决策的有价值的信息。然而，导致界定新的临床和生命伦理行为的偶然发现也是可能的。因此，必须确定这种遗传“附带体”的生物学重要性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Genet">
                Journal: Front Genet [IF=3.789]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29593781" target="_blank">
                PMID: 29593781
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Velasco H, Ramírez-Montaño D">
                Authors: Velasco H, Ramíre...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861145" target="_blank">
                PMC5861145
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29593781">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>190. High-Throughput Sequencing Strategy for Microsatellite Genotyping Using Neotropical Fish as a Model.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29593777" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29593777" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29593777">
                    <p>Genetic diversity and population studies are essential for conservation and wildlife management programs. However, monitoring requires the analysis of multiple loci from many samples. These processes can be laborious and expensive. The choice of microsatellites and PCR calibration for genotyping are particularly daunting. Here we optimized a low-cost genotyping method using multiple microsatellite loci for simultaneous genotyping of up to 384 samples using next-generation sequencing (NGS). We designed primers with adapters to the combinatorial barcoding amplicon library and sequenced samples by MiSeq. Next, we adapted a bioinformatics pipeline for genotyping microsatellites based on read-length and sequence content. Using primer pairs for eight microsatellite loci from the fish Prochilodus costatus, we amplified, sequenced, and analyzed the DNA of 96, 288, or 384 individuals for allele detection. The most cost-effective methodology was a pseudo-multiplex reaction using a low-throughput kit of 1 M reads (Nano) for 384 DNA samples. We observed an average of 325 reads per individual per locus when genotyping eight loci. Assuming a minimum requirement of 10 reads per loci, two to four times more loci could be tested in each run, depending on the quality of the PCR reaction of each locus. In conclusion, we present a novel method for microsatellite genotyping using Illumina combinatorial barcoding that dispenses exhaustive PCR calibrations, since non-specific amplicons can be eliminated by bioinformatics analyses. This methodology rapidly provides genotyping data and is therefore a promising development for large-scale conservation-genetics studies.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29593777">
                    <p>遗传多样性和人口研究对保护和野生动物管理计划至关重要。但是，监测需要分析多个样本的多个位点。这些过程可能费力并且昂贵。用于基因分型的微卫星和PCR校准的选择尤其艰巨​​。在这里，我们利用多重微卫星基因座优化了低成本基因分型方法，使用新一代测序（NGS）对多达384个样品进行同时基因分型。我们设计了带有适配器的引物，用于组合条形扩增子文库和MiSeq的测序样品。接下来，我们根据阅读长度和序列内容改编了用于基因分型微卫星的生物信息学流水线。我们使用引物对来自鱼Prochilodus costatus的8个微卫星基因座，我们扩增，测序并分析了96,288或384个个体的DNA用于等位基因检测。最具成本效益的方法是使用1 M读数（Nano）的低通量试剂盒用于384个DNA样品的假多重反应。当对8个基因座进行基因分型时，我们观察到每个基因座平均每个基因座有325个读段。假设每个基因座最少需要10次读数，每次运行可以测试两到四倍多的基因座，这取决于每个基因座的PCR反应的质量。总之，我们提出了一种使用Illumina组合条形码进行微卫星基因分型的新方法，该方法省去了彻底的PCR校准，因为通过生物信息学分析可以消除非特异性扩增子。这种方法快速提供基因分型数据，因此是大规模保存遗传学研究的有希望的发展。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Genet">
                Journal: Front Genet [IF=3.789]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29593777" target="_blank">
                PMID: 29593777
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Pimentel JSM, Carmo AO, Rosse IC, Martins APV, Ludwig S, Facchin S, Pereira AH, Brandão-Dias PFP, Abreu NL, Kalapothakis E">
                Authors: Pimentel JSM, Car...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5855144" target="_blank">
                PMC5855144
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29593777">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>191. Identification and DNA annotation of a plasmid isolated from Chromobacterium violaceum.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29593241" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29593241" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29593241">
                    <p>Chromobacterium violaceum is a ß-proteobacterium found widely worldwide with important biotechnological properties and is associated to lethal sepsis in immune-depressed individuals. In this work, we report the discover, complete sequence and annotation of a plasmid detected in C. violaceum that has been unnoticed until now. We used DNA single-molecule analysis to confirm that the episome found was a circular molecule and then proceeded with NGS sequencing. After DNA annotation, we found that this extra-chromosomal DNA is probably a defective bacteriophage of approximately 44 kilobases, with 39 ORFs comprising, mostly hypothetical proteins. We also found DNA sequences that ensure proper plasmid replication and partitioning as well as a toxin addiction system. This report sheds light on the biology of this important species, helping us to understand the mechanisms by which C. violaceum endures to several harsh conditions. This discovery could also be a first step in the development of a DNA manipulation tool in this bacterium.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29593241">
                    <p>Chromobacterium v​​iolaceum是广泛存在于世界范围内具有重要生物技术特性的β-变形杆菌，与免疫抑制个体中的致死性败血症有关。在这项工作中，我们报告了迄今未被注意的C. violaceum中检测到的质粒的发现，完整序列和注释。我们使用DNA单分子分析来确认发现的附加物是环状分子，然后进行NGS测序。经过DNA注释，我们发现这种染色体外DNA可能是一种约44千碱基的有缺陷的噬菌体，其中39个ORF包含大部分假想蛋白。我们还发现了确保合适的质粒复制和分配以及毒素成瘾系统的DNA序列。本报告揭示了这一重要物种的生物学特征，帮助我们了解了紫色花橙忍受了几种苛刻条件的机制。这一发现也可能是在这种细菌中开发DNA操作工具的第一步。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29593241" target="_blank">
                PMID: 29593241
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-28
            </a>

            <a class="btn btn-sm btn-danger" title="Lima DC, Nyberg LK, Westerlund F, Batistuzzo de Medeiros SR">
                Authors: Lima DC, Nyberg L...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5871888" target="_blank">
                PMC5871888
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29593241">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>192. C3 glomerulopathy in cystic fibrosis: a case report.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29592796" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29592796" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29592796">
                    <p>C3 glomerulonephritis is a rare glomerulopathy characterized at renal biopsy by C3 deposition, alone or with scanty immunoglobulins, as well as by an electron-dense material in mesangium, subendothelial and subepithelial space. An abnormal systemic activation of the alternative pathway of the complement cascade is responsible for the development of the disease if triggered by several possible environmental conditions. We report the first case in literature of a patient affected by cystic fibrosis and C3GN.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29592796">
                    <p>C3肾小球肾炎是一种罕见的肾小球病，其特征在于通过C3沉积，单独或少量免疫球蛋白以及系膜，内皮下和上皮下空间中的电子致密物质进行肾活组织检查。如果由几种可能的环境条件触发，则补体级联的旁路途径的异常全身激活负责疾病的发展。我们报告了第一例患有囊性纤维化和C3GN患者的文献。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Nephrol">
                Journal: BMC Nephrol [IF=2.289]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29592796" target="_blank">
                PMID: 29592796
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-28
            </a>

            <a class="btn btn-sm btn-danger" title="Santoro D, Siligato R, Vadalà C, Lucanto M, Cristadoro S, Conti G, Buemi M, Costa S, Sabadini E, Magazzù G">
                Authors: Santoro D, Siliga...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5875003" target="_blank">
                PMC5875003
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29592796">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>193. Identification of Three Novel Frameshift Mutations in the PKD1 Gene in Iranian Families with Autosomal Dominant Polycystic Kidney Disease Using Efficient Targeted Next-Generation Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29590654" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29590654" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29590654">
                    <p>Autosomal dominant polycystic kidney disease (ADPKD) is one of the most common inherited cystic kidney diseases caused by mutations in two large multi-exon genes, PKD1 and PKD2. High allelic heterogeneity and duplication of PKD1 exons 1-32 as six pseudo genes on chromosome 16 complicate molecular analysis of this disease.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29590654">
                    <p>常染色体显性多囊肾病（ADPKD）是由两个大型多外显子基因PKD1和PKD2突变引起的最常见的遗传性囊性肾病之一。 PKD1外显子1-32的高度等位基因异质性和重复作为16号染色体上的六个假基因使得该疾病的分子分析复杂化。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Kidney Blood Press. Res.">
                Journal: Kidney Blood Press. Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29590654" target="_blank">
                PMID: 29590654
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-22
            </a>

            <a class="btn btn-sm btn-danger" title="Ranjzad F, Aghdami N, Tara A, Mohseni M, Moghadasali R, Basiri A">
                Authors: Ranjzad F, Aghdam...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29590654">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>194. Gene panel analysis for nonsyndromic cryptogenic neonatal/infantile epileptic encephalopathy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29588952" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29588952" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29588952">
                    <p>Epileptic encephalopathy (EE) is a heterogeneous condition associated with deteriorations of cognitive, sensory and/or motor functions as a consequence of epileptic activity. The phenomenon is the most common and severe in infancy and early childhood. Genetic-based diagnosis in EE patients is challenging owing to genetic and phenotypic heterogeneity of numerous monogenic disorders and the fact that thousands of genes are involved in neurodevelopment. Therefore, high-throughput next-generation sequencing (NGS) was used to investigate the genetic causes of non-syndromic cryptogenic neonatal/infantile EE (NIEE).</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29588952">
                    <p>癫痫性脑病（EE）是由于癫痫活动而与认知，感觉和/或运动功能恶化相关的异质性病症。这种现象在婴儿期和幼儿期最为常见和严重。由于许多单基因疾病的遗传和表型异质性以及数以千计的基因参与神经发育的事实，所以EE患者基于遗传的诊断具有挑战性。因此，使用高通量下一代测序（NGS）来研究非综合征型隐源性新生儿/婴儿EE（NIEE）的遗传原因。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Epilepsia Open">
                Journal: Epilepsia Open [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29588952" target="_blank">
                PMID: 29588952
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2017-Jun
            </a>

            <a class="btn btn-sm btn-danger" title="Fung CW, Kwong AK, Wong VC">
                Authors: Fung CW, Kwong AK...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5719849" target="_blank">
                PMC5719849
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29588952">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>195. HLA and proteasome expression body map.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29587858" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29587858" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29587858">
                    <p>The presentation of HLA peptide complexes to T cells is a highly regulated and tissue specific process involving multiple transcriptionally controlled cellular components. The extensive polymorphism of HLA genes and the complex composition of the proteasome make it difficult to map their expression profiles across tissues.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29587858">
                    <p>将HLA肽复合物呈递给T细胞是涉及多种转录控制的细胞组分的高度调节和组织特异性过程。 HLA基因的广泛多态性和蛋白酶体的复杂组成使其难以跨组织表达它们的表达谱。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Med Genomics">
                Journal: BMC Med Genomics [IF=2.848]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29587858" target="_blank">
                PMID: 29587858
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-27
            </a>

            <a class="btn btn-sm btn-danger" title="Boegel S, Löwer M, Bukur T, Sorn P, Castle JC, Sahin U">
                Authors: Boegel S, Löwer M...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5872580" target="_blank">
                PMC5872580
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29587858">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>196. 5&#39;/ 3&#39; imbalance strategy to detect ALK fusion genes in circulating tumor RNA from patients with non-small cell lung cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29587818" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29587818" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29587818">
                    <p>Detecting an ALK fusion gene in patients with non-small cell lung cancer (NSCLC) could provide evidence to guide individualized therapy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29587818">
                    <p>在非小细胞肺癌（NSCLC）患者中检测ALK融合基因可以提供指导个体化治疗的证据。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Exp. Clin. Cancer Res.">
                Journal: J. Exp. Clin. Cancer Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29587818" target="_blank">
                PMID: 29587818
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-27
            </a>

            <a class="btn btn-sm btn-danger" title="Tong Y, Zhao Z, Liu B, Bao A, Zheng H, Gu J, McGrath M, Xia Y, Tan B, Song C, Li Y">
                Authors: Tong Y, Zhao Z, L...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5870746" target="_blank">
                PMC5870746
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29587818">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>197. NGS Analysis of Human Embryo Culture Media Reveals miRNAs of Extra Embryonic Origin.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29587610" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29587610" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29587610">
                    <p>Our objective in this work was to isolate, identify, and compare micro-RNAs (miRNAs) found in spent culture media of euploid and aneuploid in vitro fertilization (IVF) embryos. Seventy-two embryos from 62 patients were collected, and their spent media were retained. A total of 108 spent conditioned media samples were analyzed (n = 36 day 3 euploid embryos, n = 36 day 3 aneuploid embryos, and n = 36 matched control media). Fifty hed-control media embryos were analyzed using next-generation sequencing (NGS) technology. We detected 53 known human miRNAs present in the spent conditioned media of euploid and aneuploid IVF embryos. miR-181b-5p and miR-191-5p were found the most represented. We validated our results by quantitative polymerase chain reaction (qPCR), but no significant results were obtained between the groups. In conclusion, we obtained the list of miRNAs present in the spent conditioned media from euploid and aneuploid IVF embryos, but our data suggest that these miRNAs could have a nonembryonic origin.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29587610">
                    <p>我们的目标是分离，鉴定和比较在整倍体和非整倍体体外受精（IVF）胚胎的废培养基中发现的微RNA（miRNA）。收集62名患者的72个胚胎，并保留其用过的培养基。分析总共108个用过的条件培养基样品（n = 36天3倍整倍体胚胎，n = 36天3个非整倍体胚胎，并且n = 36个匹配的对照培养基）。使用新一代测序（NGS）技术分析50个对照介质胚胎。我们在整倍体和非整倍体IVF胚胎的废调理培养基中检测到53种已知的人类miRNA。发现miR-181b-5p和miR-191-5p的代表最多。我们通过定量聚合酶链式反应（qPCR）验证了我们的结果，但是没有获得显着的结果。总之，我们从整倍体和非整倍体IVF胚胎中获得了存在于废调理培养基中的miRNA列表，但是我们的数据表明这些miRNA可能具有非胚胎来源。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Reprod Sci">
                Journal: Reprod Sci [IF=2.443]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29587610" target="_blank">
                PMID: 29587610
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jan-01
            </a>

            <a class="btn btn-sm btn-danger" title="Sánchez-Ribas I, Diaz-Gimeno P, Quiñonero A, Ojeda M, Larreategui Z, Ballesteros A, Domínguez F">
                Authors: Sánchez-Ribas I, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29587610">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>198. MDM2-p53 Interactions in Human Hepatocellular Carcinoma: What Is the Role of Nutlins and New Therapeutic Options?</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29584707" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29584707" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29584707">
                    <p>Human hepatocellular carcinoma (HCC) is the fifth most common cancer and is associated with poor prognosis worldwide. The molecular mechanisms underlying the pathogenesis of HCC have been an area of continuing interest, and recent studies using next generation sequencing (NGS) have revealed much regarding previously unsettled issues. Molecular studies using HCC samples have been mainly targeted with the aim to identify the fundamental mechanisms contributing to HCC and identify more effective treatments. In response to cellular stresses (e.g., DNA damage or oncogenes), activated p53 elicits appropriate responses that aim at DNA repair, genetic stability, cell cycle arrest, and the deletion of DNA-damaged cells. On the other hand, the murine double minute 2 (MDM2) oncogene protein is an important cellular antagonist of p53. MDM2 negatively regulates p53 activity through the induction of p53 protein degradation. However, current research has shown that the mechanisms underlying MDM2-p53 interactions are more complex than previously thought. Microarray data have added new insight into the transcription changes in HCC. Recently, Nutlin-3 has shown potency against p53-MDM2 binding and the enhancement of p53 stabilization as well as an increment of p53 cellular accumulation with potential therapeutic effects. This review outlines the molecular mechanisms involved in the p53-MDM2 pathways, the biological factors influencing these pathways, and their roles in the pathogenesis of HCC. It also discusses the action of Nutlin-3 treatment in inducing growth arrest in HCC and elaborates on future directions in research in this area. More research on the biology of p53-MDM2 interactions may offer a better understanding of these mechanisms and discover new biomarkers, sensitive prognostic indicators as well as new therapeutic interventions in HCC.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29584707">
                    <p>人类肝细胞癌（HCC）是第五常见的癌症，并且与全球预后不良有关。 HCC发病机制的分子机制一直是人们持续关注的领域，近期使用下一代测序（NGS）的研究已经揭示了许多关于以前未解决的问题。使用HCC样品的分子研究主要是为了确定导致HCC的基本机制并确定更有效的治疗。响应于细胞应激（例如DNA损伤或致癌基因），激活的p53引发以DNA修复，遗传稳定性，细胞周期停滞和DNA受损细胞缺失为目标的适当应答。另一方面，鼠双分钟2（MDM2）癌基因蛋白是p53的重要细胞拮抗剂。 MDM2通过诱导p53蛋白降解负调节p53活性。然而，目前的研究表明MDM2-p53相互作用的机制比以前想象的更复杂。微阵列数据增加了对HCC转录变化的新见解。最近，Nutlin-3已显示出对p53-MDM2结合的效力和p53稳定性的增强以及具有潜在治疗效果的p53细胞积累的增加。本综述概述了p53-MDM2通路中涉及的分子机制，影响这些通路的生物学因素及其在HCC发病机制中的作用。它还讨论了Nutlin-3治疗诱导HCC生长停滞的作用，并阐述了该领域未来的研究方向。对p53-MDM2相互作用的生物学的更多研究可能提供对这些机制的更好理解，并发现新的生物标志物，敏感的预后指标以及HCC中的新的治疗干预措施。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Clin Med">
                Journal: J Clin Med [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29584707" target="_blank">
                PMID: 29584707
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-27
            </a>

            <a class="btn btn-sm btn-danger" title="Azer SA">
                Authors: Azer SA
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5920438" target="_blank">
                PMC5920438
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29584707">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>199. Two novel totiviruses in the white-backed planthopper, Sogatella furcifera.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29580322" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29580322" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29580322">
                    <p>There is little information about commensal viruses in the white-backed planthopper, Sogatella furcifera, although it is an important agricultural insect. Here, two novel double-stranded RNA viruses related to the viruses in the family Totiviridae were identified using next-generation sequencing and tentatively named Sogatella furcifera totivirus 1 and 2 (SfTV1 and SfTV2). Their complete genomes consist of 6310 and 6303 nt, respectively, showing typical genomic features with viruses in the family Totiviridae. Identity, phylogenetic and conserved sequence analyses showed that SfTV1, SfTV2 and three other insect viruses may form a proposed novel genus of the family Totiviridae. Vertical transmission of the two viruses was highly efficient, and they were detected in all insect tissues and developmental stages, with the highest titres in the adult and in the haemolymph and reproductive organs. To our knowledge, this is the first report of viruses in the family Totiviridae found in a hemipteran insect.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29580322">
                    <p>关于白背飞虱Sogatella furcifera中共生病毒的信息很少，尽管它是一种重要的农业昆虫。在这里，使用新一代测序并暂时命名为Sogatella furcifera totivirus 1和2（SfTV1和SfTV2）鉴定了与Totiviridae科病毒相关的两种新型双链RNA病毒。他们的完整基因组分别由6310和6303核苷酸组成，显示了典型的基因组特征以及病毒在Totiviridae中的病毒。同源性，系统发育性和保守性序列分析表明，SfTV1，SfTV2和其他三种昆虫病毒可能形成一个新的Totiviridae科属。这两种病毒的垂直传播非常有效，并且在所有昆虫组织和发育阶段都能检测到，它们在成人以及血淋巴和生殖器官中的滴度最高。据我们所知，这是第一次发现在半翅目昆虫中发现的Totiviridae病毒的报道。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Gen. Virol.">
                Journal: J. Gen. Virol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29580322" target="_blank">
                PMID: 29580322
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Zhang P, Liu W, Cao M, Massart S, Wang X">
                Authors: Zhang P, Liu W, C...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29580322">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>200. Liquid biopsy of bile for the molecular diagnosis of gallbladder cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29580156" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29580156" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29580156">
                    <p>Tissue sampling of gallbladder cancer (GBCa) is challenging because of the anatomy of the gallbladder. The aim of this study is to investigate the possibility of diagnosing GBCa patients by performing a liquid biopsy of bile in addition to current diagnostic methods.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29580156">
                    <p>由于胆囊的解剖结构，胆囊癌组织取样（GBCa）具有挑战性。这项研究的目的是调查除目前的诊断方法之外，通过进行胆汁液体活检来诊断GBCa患者的可能性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Biol. Ther.">
                Journal: Cancer Biol. Ther. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29580156" target="_blank">
                PMID: 29580156
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-26
            </a>

            <a class="btn btn-sm btn-danger" title="Kinugasa H, Nouso K, Ako S, Dohi C, Matsushita H, Matsumoto K, Kato H, Okada H">
                Authors: Kinugasa H, Nouso...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29580156">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>201. Transition-Metal-Free Decarboxylative Propargylic Substitution/Cyclization with either Azolium Enolates or Acyl Anions.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29577534" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29577534" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29577534">
                    <p>Presented herein is an unprecedented transition-metal-free propargylic substitution reaction with either azolium enolates or acyl anions, which are generated from aldehydes under N-heterocyclic carbene catalysis. This new catalytic activation operates on readily available cyclic propargylic carbamates through decarboxylation, and generates reactive allene intermediates that can undergo divergent cyclization pathways to deliver skeletally diverse polycyclic compounds with high levels of efficiency and excellent enantioselectivities.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29577534">
                    <p>本文呈现的是在N-杂环卡宾催化下由醛衍生的烯醇鎓烯醇盐或酰基阴离子的前所未有的无过渡金属炔丙基取代反应。这种新的催化活化通过脱羧作用于容易得到的环状炔丙基氨基甲酸酯上，并且产生可以经历不同环化途径的活性丙二烯中间体，以提供具有高水平效率和优异的对映选择性的骨架不同的多环化合物。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Angew. Chem. Int. Ed. Engl.">
                Journal: Angew. Chem. Int. Ed. Engl. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29577534" target="_blank">
                PMID: 29577534
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-25
            </a>

            <a class="btn btn-sm btn-danger" title="Lu S, Ong JY, Poh SB, Tsang T, Zhao Y">
                Authors: Lu S, Ong JY, Poh...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29577534">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>202. An optimized FAIRE procedure for low cell numbers in yeast.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29577419" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29577419" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29577419">
                    <p>We report an optimised low-input FAIRE-seq (Formaldehyde-Assisted Isolation of Regulatory Elements-sequencing) procedure to assay chromatin accessibility from limited amounts of yeast cells. We demonstrate that the method performs well on as little as 4 mg cells scraped directly from a few colonies. Sensitivity, specificity and reproducibility of the scaled-down method are comparable to that of regular, higher input amounts, and allows the use of 100-fold fewer cells than existing procedures. The method enables epigenetic analysis of chromatin structure without the need for cell multiplication of exponentially growing cells in liquid culture, thus opening the possibility to study colony cell subpopulations, or those that can be isolated directly from environmental samples.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29577419">
                    <p>我们报告了优化的低输入FAIRE-seq（调节元件测序的甲醛辅助分离）程序以测定来自有限量酵母细胞的染色质可接近性。我们证明该方法在少量直接从几个菌落中刮取的4毫克细胞上表现良好。按比例缩小的方法的灵敏度，特异性和再现性与常规的较高输入量相当，并且允许使用比现有程序少100倍的细胞。该方法能够进行表观遗传学分析染色质结构，而不需要液体培养中指数生长细胞的细胞增殖，因此开辟了研究菌落细胞亚群的可能性，或者可以直接从环境样品中分离的那些。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Yeast">
                Journal: Yeast [IF=1.990]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29577419" target="_blank">
                PMID: 29577419
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-25
            </a>

            <a class="btn btn-sm btn-danger" title="Segorbe D, Wilkinson D, Mizeranschi A, Hughes T, Aaløkken R, Vachova L, Palkova Z, Gilfillan GD">
                Authors: Segorbe D, Wilkin...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29577419">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>203. Enhancing the accuracy of next-generation sequencing for detecting rare and subclonal mutations.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29576615" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29576615" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29576615">
                    <p>Mutations, the fuel of evolution, are first manifested as rare DNA changes within a population of cells. Although next-generation sequencing (NGS) technologies have revolutionized the study of genomic variation between species and individual organisms, most have limited ability to accurately detect and quantify rare variants among the different genome copies in heterogeneous mixtures of cells or molecules. We describe the technical challenges in characterizing subclonal variants using conventional NGS protocols and the recent development of error correction strategies, both computational and experimental, including consensus sequencing of single DNA molecules. We also highlight major applications for low-frequency mutation detection in science and medicine, describe emerging methodologies and provide our vision for the future of DNA sequencing.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29576615">
                    <p>突变，进化的燃料，首先表现为细胞群体内罕见的DNA变化。尽管下一代测序（NGS）技术已经彻底改变了物种和个体生物体之间基因组变异的研究，但大多数能力有限，无法准确检测和量化不同基因组拷贝中异种细胞或分子混合物中的罕见变异。我们描述了使用常规NGS方案表征亚克隆变体的技术挑战以及计算和实验的错误纠正策略的最新发展，包括单个DNA分子的共有序列。我们还重点介绍了科学和医学中低频突变检测的主要应用，描述新兴方法并提供了DNA测序未来的展望。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Nat. Rev. Genet.">
                Journal: Nat. Rev. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29576615" target="_blank">
                PMID: 29576615
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Salk JJ, Schmitt MW, Loeb LA">
                Authors: Salk JJ, Schmitt ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29576615">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>204. Molecular Screening of Small Biopsy Samples Using Next-Generation Sequencing in Korean Patients with Advanced Non-small Cell Lung Cancer: Korean Lung Cancer Consortium (KLCC-13-01).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29575851" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29575851" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29575851">
                    <p>Non-small cell lung cancer (NSCLC) is a common type of cancer with typically poor prognosis. As individual cancers exhibit unique mutation patterns, identifying and characterizing gene mutations in NSCLC might help predict patient outcomes and guide treatment. The aim of this study was to evaluate the clinical adequacy of molecular testing using next-generation sequencing (NGS) for small biopsy samples and characterize the mutational landscape of Korean patients with advanced NSCLC.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29575851">
                    <p>非小细胞肺癌（NSCLC）是一种常见的预后不良的癌症类型。由于个体癌症表现出独特的突变模式，鉴别和表征NSCLC中的基因突变可能有助于预测患者结果并指导治疗。本研究的目的是评估使用新一代测序（NGS）进行小活检标本的分子检测的临床适宜性，并描述韩国晚期NSCLC患者的突变情况。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Pathol Transl Med">
                Journal: J Pathol Transl Med [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29575851" target="_blank">
                PMID: 29575851
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-26
            </a>

            <a class="btn btn-sm btn-danger" title="Ku BM, Heo MH, Kim JH, Cho BC, Cho EK, Min YJ, Lee KH, Sun JM, Lee SH, Ahn JS, Park K, Kim TJ, Lee HY, Kim H, Lee KJ, Ahn MJ">
                Authors: Ku BM, Heo MH, Ki...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29575851">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>205. Simultaneous detection of ABL1 mutation and IKZF1 deletion in Philadelphia chromosome-positive acute lymphoblastic leukemia using a customized target enrichment system panel.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29575541" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29575541" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29575541">
                    <p>Recent clinical outcomes of pediatric Philadelphia chromosome-positive acute lymphoblastic leukemia (Ph+ALL) vastly improved owing to tyrosine kinase inhibitor (TKI). However, the genetic status would be different in each case with ABL1 gene mutation or copy number variants (CNVs) such as IKZF1 deletion. In particular, the TKI resistant clone with ABL1 kinase mutation remains problematic. The comprehensive assessment of genetic status including mutation, insertion and deletion (indel) and CNVs is necessary.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29575541">
                    <p>由于酪氨酸激酶抑制剂（TKI），儿科费城染色体阳性急性淋巴细胞白血病（Ph + ALL）的近期临床结果大大改善。然而，在ABL1基因突变或拷贝数变异（CNV）如IKZF1缺失的情况下，遗传状态将不同。特别是，具有ABL1激酶突变的TKI抗性克隆仍然存在问题。遗传状态的综合评估包括突变，插入和删除（indel）和CNV是必要的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Lab Hematol">
                Journal: Int J Lab Hematol [IF=2.030]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29575541" target="_blank">
                PMID: 29575541
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-25
            </a>

            <a class="btn btn-sm btn-danger" title="Aoe M, Ishida H, Matsubara T, Karakawa S, Kawaguchi H, Fujiwara K, Kanamitsu K, Washio K, Okada K, Shibakura M, Shimada A">
                Authors: Aoe M, Ishida H, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29575541">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>206. Repair of DNA damage caused by cytosine deamination in mitochondrial DNA of forensic case samples.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29573606" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29573606" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29573606">
                    <p>DNA sequence damage from cytosine deamination is well documented in degraded samples, such as those from ancient and forensic contexts. This study examined the effect of a DNA repair treatment on mitochondrial DNA (mtDNA) from aged and degraded skeletal samples. DNA extracts from 21 non-probative, degraded skeletal samples (aged 50-70 years) were utilized for the analysis. A portion of each sample extract was subjected to DNA repair using a commercial repair kit, the New England BioLabs&#39; NEBNext FFPE DNA Repair Kit (Ipswich, MA). MtDNA was enriched using PCR and targeted capture in a side-by-side experiment of untreated and repaired DNA. Sequencing was performed using both traditional (Sanger-type; STS) and next-generation sequencing (NGS) methods Although cytosine deamination was evident in the mtDNA sequence data, the observed level of damaged bases varied by sequencing method as well as by enrichment type. The STS PCR amplicon data did not show evidence of cytosine deamination that could be distinguished from background signal in either the untreated or repaired sample set. However, the same PCR amplicons showed 850 C → T/G → A substitutions consistent with cytosine deamination with variant frequencies (VFs) of up to 25% when sequenced using NGS methods The occurrence of base misincorporation due to cytosine deamination was reduced by 98% (to 10) in the NGS amplicon data after repair. The NGS capture data indicated low levels (1-2%) of cytosine deamination in mtDNA fragments that was effectively mitigated by DNA repair. The observed difference in the level of cytosine deamination between the PCR and capture enrichment methods can be attributed to the greater propensity for stochastic effects from the PCR enrichment technique employed (e.g., low template input, increased PCR cycles). Altogether these results indicate that DNA repair may be required when sequencing PCR-amplified DNA from degraded forensic case samples with NGS methods.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29573606">
                    <p>来自胞嘧啶脱氨基的DNA序列损伤在降解样品中有很好的记载，例如来自古代和法医环境的样品。本研究检测了DNA修复治疗对来自老化和退化骨骼样品的线粒体DNA（mtDNA）的影响。使用来自21个未证实的降解骨骼样品（50-70岁）的DNA提取物进行分析。使用商业修复试剂盒New England BioLabs的NEBNext FFPE DNA修复试剂盒（Ipswich，MA）对每份样品提取物的一部分进行DNA修复。使用PCR富集MtDNA并在未经处理和修复的DNA的并行实验中靶向捕获。使用传统（Sanger型; STS）和下一代测序（NGS）方法进行测序尽管胞嘧啶脱氨基在mtDNA序列数据中很明显，但观察到的受损碱基水平因测序方法和富集类型而异。 STS PCR扩增子数据没有显示在未处理或修复的样品组中可以与背景信号区分的胞嘧啶脱氨基的证据。然而，使用NGS方法进行测序时，相同的PCR扩增子显示850 C→T / G→A取代与胞嘧啶脱氨基一致，变异频率（VF）高达25％。由于胞嘧啶脱氨基引起碱基错误掺入的发生减少了98％ （至10）修复后的NGS扩增子数据。 NGS捕获数据表明，mtDNA片段中胞嘧啶脱氨基的水平较低（1-2％），DNA修复有效缓解了这种情况。观察到的PCR和捕获富集方法之间胞嘧啶脱氨基水平的差异可归因于来自所采用的PCR富集技术的随机效应的更大倾向（例如，低模板输入，增加的PCR循环）。总之，这些结果表明，当使用NGS方法对来自降解法医病例样品的PCR扩增DNA进行测序时，可能需要DNA修复。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Forensic Sci Int Genet">
                Journal: Forensic Sci Int Genet [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29573606" target="_blank">
                PMID: 29573606
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Gorden EM, Sturk-Andreaggi K, Marshall C">
                Authors: Gorden EM, Sturk-...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29573606">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>207. Liquid biopsy by NGS: differential presence of exons (DPE) in cell-free DNA reveals different patterns in metastatic and nonmetastatic colorectal cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29573240" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29573240" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29573240">
                    <p>Next-generation sequencing (NGS) has been proposed as a suitable tool for liquid biopsy in colorectal cancer (CRC), although most studies to date have focused almost exclusively on sequencing of panels of potential clinically actionable genes. We evaluated the clinical value of whole-exome sequencing (WES) of cell-free DNA (cfDNA) circulating in plasma, with the goal of identifying differential clinical profiles in patients with CRC. To this end, we applied an original concept, &quot;differential presence of exons&quot; (DPE). We determined differences in levels of 379 exons in plasma cfDNA and used DPE analysis to cluster and classify patients with disseminated and localized disease. The resultant bioinformatics analysis pipeline allowed us to design a predictive DPE algorithm in a small subset of patients that could not be initially classified based on the selection criteria. This DPE suggests that these nucleic acids could be actively released by both tumor and nontumor cells as a means of intercellular communication and might thus play a role in the process of malignant transformation. DPE is a new technique for the study of plasma cfDNA by WES that might have predictive and prognostic value in patients with CRC.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29573240">
                    <p>新一代测序（NGS）已被提出作为结肠直肠癌（CRC）中液体活检的合适工具，尽管迄今为止大多数研究几乎都专注于潜在临床可行基因的测序。我们评估了在血浆中循环的无细胞DNA（cfDNA）的全外显子组测序（WES）的临床价值，目的是鉴定CRC患者中的不同临床表现。为此，我们应用了一个原创的概念，“外显子的差异化存在”（DPE）。我们测定了血浆cfDNA中379个外显子的水平差异，并使用DPE分析来聚类和分类播散性和局限性疾病患者。由此产生的生物信息学分析流程使我们能够在一小部分患者中设计预测性DPE算法，这些患者无法根据选择标准进行初始分类。该DPE表明这些核酸可以作为细胞间通讯的手段而被肿瘤细胞和非肿瘤细胞主动释放，因此可能在恶变过程中起作用。 DPE是WES研究血浆cfDNA的新技术，可能对CRC患者具有预测价值和预后价值。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Med">
                Journal: Cancer Med [IF=3.362]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29573240" target="_blank">
                PMID: 29573240
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Olmedillas-López S, García-Olmo DC, García-Arranz M, Peiró-Pastor R, Aguado B, García-Olmo D">
                Authors: Olmedillas-López ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29573240">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>208. Single, short in-del, and copy number variations detection in monogenic dyslipidemia using a next-generation sequencing strategy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29572815" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29572815" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29572815">
                    <p>Optimal molecular diagnosis of primary dyslipidemia is challenging to confirm the diagnosis, test and identify at risk relatives. The aim of this study was to test the application of a single targeted next-generation sequencing (NGS) panel for hypercholesterolemia, hypocholesterolemia, and hypertriglyceridemia molecular diagnosis. NGS workflow based on a custom AmpliSeq panel was designed for sequencing the most prevalent dyslipidemia-causing genes (ANGPTL3, APOA5, APOC2, APOB, GPIHBP1, LDLR, LMF1, LPL, PCSK9) on the Ion PGM Sequencer. One hundred and forty patients without molecular diagnosis were studied. In silico analyses were performed using the NextGENe software and homemade tools for detection of copy number variations (CNV). All mutations were confirmed using appropriate tools. Eighty seven variations and 4 CNV were identified, allowing a molecular diagnosis for 40/116 hypercholesterolemic patients, 5/13 hypocholesterolemic patients, and 2/11, hypertriglyceridemic patients respectively. This workflow allowed the detection of CNV contrary to our previous strategy. Some variations were found in previously unexplored regions providing an added value for genotype-phenotype correlation and familial screening. In conclusion, this new NGS process is an effective mutation detection method and allows better understanding of phenotype. Consequently this assay meets the medical need for individualized diagnosis of dyslipidemia.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29572815">
                    <p>原发性血脂异常的最佳分子诊断对确认危险亲属的诊断，检测和鉴定具有挑战性。本研究的目的是测试单一靶向下一代测序（NGS）小组在高胆固醇血症，低胆固醇血症和高甘油三酯血症分子诊断中的应用。基于定制AmpliSeq小组的NGS工作流程设计用于对Ion PGM测序仪上最常见的血脂异常引起的基因（ANGPTL3，APOA5，APOC2，APOB，GPIHBP1，LDLR，LMF1，LPL，PCSK9）进行测序。研究了140位无分子诊断的患者。在计算机分析中使用NextGENe软件和自制工具检测拷贝数变异（CNV）。所有突变均使用合适的工具进行确认鉴定出87种变异和4种CNV，分别诊断40/116例高胆固醇血症患者，5/13例低胆固醇血症患者和2/11例高甘油三酯血症患者。这个工作流程允许CNV的检测违背我们以前的策略。在以前未开发的区域发现了一些变异，为基因型 - 表型相关性和家族筛查提供了附加价值。总之，这种新的NGS过程是一种有效的突变检测方法，可以更好地理解表型。因此该测定法满足了对血脂异常个体化诊断的医学需求。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Genet.">
                Journal: Clin. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29572815" target="_blank">
                PMID: 29572815
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Marmontel O, Charrière S, Simonet T, Bonnet V, Dumont S, Mahl M, Jacobs C, Nony S, Chabane K, Bozon D, Janin A, Peretti N, Lachaux A, Bardel C, Millat G, Moulin P, Marçais C, Di Filippo M">
                Authors: Marmontel O, Char...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29572815">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>209. Methodology challenges in studying human gut microbiota - effects of collection, storage, DNA extraction and next generation sequencing technologies.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29572539" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29572539" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29572539">
                    <p>The information on microbiota composition in the human gastrointestinal tract predominantly originates from the analyses of human faeces by application of next generation sequencing (NGS). However, the detected composition of the faecal bacterial community can be affected by various factors including experimental design and procedures. This study evaluated the performance of different protocols for collection and storage of faecal samples (native and OMNIgene.GUT system) and bacterial DNA extraction (MP Biomedicals, QIAGEN and MO BIO kits), using two NGS platforms for 16S rRNA gene sequencing (Ilumina MiSeq and Ion Torrent PGM). OMNIgene.GUT proved as a reliable and convenient system for collection and storage of faecal samples although favouring Sutterella genus. MP provided superior DNA yield and quality, MO BIO depleted Gram positive organisms while using QIAGEN with OMNIgene.GUT resulted in greatest variability compared to other two kits. MiSeq and IT platforms in their supplier recommended setups provided comparable reproducibility of donor faecal microbiota. The differences included higher diversity observed with MiSeq and increased capacity of MiSeq to detect Akkermansia muciniphila, [Odoribacteraceae], Erysipelotrichaceae and Ruminococcaceae (primarily Faecalibacterium prausnitzii). The results of our study could assist the investigators using NGS technologies to make informed decisions on appropriate tools for their experimental pipelines.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29572539">
                    <p>关于人类胃肠道微生物群成分的信息主要来源于应用下一代测序（NGS）分析人类粪便。然而，检测到的粪便细菌群落组成可能受各种因素影响，包括实验设计和程序。本研究使用两个用于16S rRNA基因测序的NGS平台（Ilumina MiSeq）评估了用于收集和储存粪便样品（天然和OMNIgene.GUT系统）和细菌DNA提取（MP Biomedicals，QIAGEN和MO BIO试剂盒）的不同方案的性能和Ion Torrent PGM）。 OMNIgene.GUT被证明是一个可靠和便利的系统，用于收集和储存粪便样品，虽然有利于Sutterella属。 MP提供了优异的DNA产量和质量，MO BIO消耗革兰氏阳性生物体，而使用QIAGEN和OMNIgene.GUT产生与其他两种试剂盒相比最大的变异性。 MiSeq和IT平台在其供应商推荐的设置中提供了可比的重现性供体粪便微生物群。这些差异包括MiSeq观察到的更高的多样性和MiSeq检测Akkermansia muciniphila，[Odoribacteraceae]，Erysipelotrichaceae和Ruminococcaceae（主要是Faecalibacterium prausnitzii）的能力增加。我们研究的结果可以帮助调查人员使用NGS技术为他们的实验管道合适的工具做出明智的决定。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29572539" target="_blank">
                PMID: 29572539
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Panek M, Čipčić Paljetak H, Barešić A, Perić M, Matijašić M, Lojkić I, Vranešić Bender D, Krznarić Ž, Verbanac D">
                Authors: Panek M, Čipčić P...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5865204" target="_blank">
                PMC5865204
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29572539">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>210. A new hybrid approach for MHC genotyping: high-throughput NGS and long read MinION nanopore sequencing, with application to the non-model vertebrate Alpine chamois (Rupicapra rupicapra).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29572469" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29572469" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29572469">
                    <p>The major histocompatibility complex (MHC) acts as an interface between the immune system and infectious diseases. Accurate characterization and genotyping of the extremely variable MHC loci are challenging especially without a reference sequence. We designed a combination of long-range PCR, Illumina short-reads, and Oxford Nanopore MinION long-reads approaches to capture the genetic variation of the MHC II DRB locus in an Italian population of the Alpine chamois (Rupicapra rupicapra). We utilized long-range PCR to generate a 9 Kb fragment of the DRB locus. Amplicons from six different individuals were fragmented, tagged, and simultaneously sequenced with Illumina MiSeq. One of these amplicons was sequenced with the MinION device, which produced long reads covering the entire amplified fragment. A pipeline that combines short and long reads resolved several short tandem repeats and homopolymers and produced a de novo reference, which was then used to map and genotype the short reads from all individuals. The assembled DRB locus showed a high level of polymorphism and the presence of a recombination breakpoint. Our results suggest that an amplicon-based NGS approach coupled with single-molecule MinION nanopore sequencing can efficiently achieve both the assembly and the genotyping of complex genomic regions in multiple individuals in the absence of a reference sequence.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29572469">
                    <p>主要组织相容性复合体（MHC）充当免疫系统和传染病之间的界面。特别是在没有参考序列的情况下，对极其多变的MHC基因座进行准确的表征和基因分型具有挑战性。我们设计了长距离PCR，Illumina短读数和Oxford Nanopore MinION长读方法，以捕获意大利阿尔卑斯山羚羊（Rupicapra rupicapra）群体中MHC II DRB基因座的遗传变异。我们利用长程PCR产生DRB基因座的一个9Kb片段。来自6个不同个体的扩增子被片段化，标记，并同时用Illumina MiSeq进行测序。其中一个扩增子用MinION设备进行测序，产生覆盖整个扩增片段的长片段。一个结合短和长阅读的管道解决了几个短串联重复序列和均聚物，并产生了一个从头参考，然后用它来对来自所有个体的短读数进行映射和基因分型。组装的DRB基因座显示出高水平的多态性并存在重组断点。我们的研究结果表明，基于扩增子的NGS方法与单分子MinION纳米孔测序相结合，可以有效地在缺少参考序列的情况下实现多个个体中复杂基因组区域的基因组分型和基因分型。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Heredity (Edinb)">
                Journal: Heredity (Edinb) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29572469" target="_blank">
                PMID: 29572469
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-24
            </a>

            <a class="btn btn-sm btn-danger" title="Fuselli S, Baptista RP, Panziera A, Magi A, Guglielmi S, Tonin R, Benazzo A, Bauzer LG, Mazzoni CJ, Bertorelle G">
                Authors: Fuselli S, Baptis...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29572469">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>211. Assessing airborne fungal communities by high-throughput sequencing using passive traps.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29572213" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29572213" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29572213">
                    <p>Techniques based on the high-throughput sequencing (HTS) of environmental DNA have provided a new way of studying fungal diversity. However, these techniques suffer from a number of methodological biases which may appear at any of the steps involved in a metabarcoding study. Air is one of the most important environments where fungi can be found because it is the primary medium of dispersal for many species. Looking ahead to future developments, it was decided to test 20 protocols including different passive spore traps, spore recovery procedures, DNA extraction kits and barcode loci. HTS was performed with the Illumina MiSeq platform targeting two sub loci of the fungal internal transcribed spacer. Multivariate analysis and generalized linear models showed that the type of passive spore trap, the spore recovery procedure and the barcode all impact the description of fungal communities in terms of richness and diversity when assessed by HTS metabarcoding. In contrast, DNA extraction kits did not significantly impact these results. Although passive traps may be used to describe airborne fungal communities, a study using specific real-time PCR and a mock community showed that these kinds of trap are affected by environmental conditions that may induce losses of biological material, impacting diversity and community composition results.Importance The advent of high throughput sequencing (HTS) methods such as those offered by next-generation sequencing (NGS) techniques has opened a new era in the study of fungal diversity in different environmental substrates. In this study we show that the assessment of the diversity of airborne fungal communities can reliably be achieved by the use of simple and robust passive spore traps. However, comparison of sample processing protocols showed that several methodological biases may impact results of fungal diversity when assessed by metabarcoding. Our data suggest that identifying these biases is of paramount importance to enable a correct identification and relative quantification of community members.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29572213">
                    <p>基于环境DNA的高通量测序（HTS）的技术为研究真菌多样性提供了新的途径。然而，这些技术遭受了一些方法上的偏见，这些偏见可能出现在metabarcoding研究的任何步骤中。空气是真菌可以被发现的最重要的环境之一，因为它是许多物种分散的主要媒介。展望未来的发展，决定测试20种方案，包括不同的被动孢子陷阱，孢子恢复程序，DNA提取试剂盒和条形码基因座。用Illumina MiSeq平台针对真菌内转录间隔区的两个亚位点进行HTS。多变量分析和广义线性模型表明，被HTS metabarcoding评估时，被动孢子捕捉器的类型，孢子恢复程序和条形码在丰富度和多样性方面均影响真菌群落的描述。相反，DNA提取试剂盒并没有显着影响这些结果。虽然被动圈闭可能被用来描述空气真菌群落，但是一项使用特定实时PCR和模拟群落的研究表明，这些圈套受环境条件的影响，可能会导致生物材料的损失，从而影响多样性和群落组成结果。重要性高通量测序（HTS）方法（如新一代测序（NGS）技术提供的方法）的出现为研究不同环境底物的真菌多样性打开了一个新时代。在这项研究中，我们表明，通过使用简单而强大的被动孢子陷阱，可以可靠地实现对空气真菌群落多样性的评估。然而，样品处理方案的比较表明，在用元条形码评估时，若干方法学偏差可能影响真菌多样性的结果。我们的数据表明，识别这些偏见对于社区成员的正确识别和相对量化至关重要。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Appl. Environ. Microbiol.">
                Journal: Appl. Environ. Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29572213" target="_blank">
                PMID: 29572213
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Aguayo J, Fourrier-Jeandel C, Husson C, Ioos R">
                Authors: Aguayo J, Fourrie...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29572213">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>212. Clinicopathological characteristics of POLE mutation in patients with non-small-cell lung cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29572003" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29572003" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29572003">
                    <p>Mutations in polymerase ε (POLE), a DNA polymerase involved in DNA replication and repair, have been investigated in endometrial cancers and response to programmed cell death protein 1 (PD-1)/PD-1 ligand (PD-L1) immunotherapy. However, the frequency of POLE gene mutation in patients with non-small cell lung cancer (NSCLC) has not been reported.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29572003">
                    <p>已经在子宫内膜癌和对程序性细胞死亡蛋白1（PD-1）/ PD-1配体（PD-L1）免疫疗法的反应中研究了聚合酶ε（POLE）（一种参与DNA复制和修复的DNA聚合酶）中的突变。然而，非小细胞肺癌（NSCLC）患者POLE基因突变的频率尚未见报道。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Lung Cancer">
                Journal: Lung Cancer [IF=4.294]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29572003" target="_blank">
                PMID: 29572003
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Song Z, Cheng G, Xu C, Wang W, Shao Y, Zhang Y">
                Authors: Song Z, Cheng G, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29572003">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>213. Identification of a novel KIF13A-RET fusion in lung adenocarcinoma by next-generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29571998" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29571998" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29571998">
                    <p>RET fusions have been reported in 1-2% of lung adenocarcinomas, and represent an actionable target. Patients whose tumors possess RET fusion are associated with clinical benefit from the treatment with multi-kinase inhibitors such as cabozantinib and vandetanib. Further molecular screening for RET fusions is warranted. Novel KIF13A-RET fusion containing an intact RET kinase domain involving exons 1-18 of KIF13A and exons 12-20 of RET was identified in a lung cancer specimen from an 74-year-old Asian never smoker by next-generation sequencing (NGS) during clinical care. The patient was negative for EGFR, ALK, ROS1 and other putative driver alterations. Fusion analysis is consistent with other described RET fusions and is predicted to result in aberrant constitutive activation caused by dimerization and sensitivity to RET-directed therapies. We describe a novel RET-fusion with molecular characteristics consistent with RET-driven non-small cell lung cancer. Our case expands the spectrum of RET fusion partners and supports broad molecular profiling in non-small cell lung cancer optimizing patient therapeutic options. The new RET fusion has immediate clinical implications for cancer patients.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29571998">
                    <p>据报道，1-2％的肺腺癌中有RET融合，并且是可行的靶点。其肿瘤具有RET融合的患者与用多激酶抑制剂如卡洛替尼和凡德他尼治疗的临床益处相关。有必要进一步分子筛选RET融合。通过下一代测序（NGS）从74岁的亚洲人永不吸烟的肺癌样本中鉴定出含有完整的RET激酶结构域的新型KIF13A-RET融合蛋白，该结构涉及KIF13A的外显子1-18和RET的外显子12-20，在临床护理期间。患者对EGFR，ALK，ROS1和其他假定的驱动改变是阴性的。融合分析与其他描述的RET融合是一致的，并且预测导致由RET聚合和对RET定向治疗的敏感性引起的异常组成型活化。我们描述了一种新型RET融合蛋白，其分子特征与RET驱动的非小细胞肺癌一致。我们的案例扩展了RET融合伴侣的范围，并支持在非小细胞肺癌中优化患者治疗选择的广泛分子谱分析。新的RET融合对癌症患者具有直接的临床意义。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Lung Cancer">
                Journal: Lung Cancer [IF=4.294]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29571998" target="_blank">
                PMID: 29571998
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Zhang X, Li Y, Liu C, Wang W, Li M, Lv D, Sun G, Chen H, Dong X, Miao Z, Yao M, Wang K, Tian H">
                Authors: Zhang X, Li Y, Li...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29571998">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>214. Response to crizotinib in advanced ALK-rearranged non-small cell lung cancers with different ALK-fusion variants.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29571990" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29571990" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29571990">
                    <p>Anaplastic lymphoma kinase (ALK) rearrangements are present in approximately 5% of non-small-cell lung cancers (NSCLCs). NSCLCs with ALK-rearrangement can be effectively treated with crizotinib. However, magnitude and duration of responses are found to be heterogeneous. This study explored the clinical efficacy of crizotinib in different ALK variants.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29571990">
                    <p>约5％的非小细胞肺癌（NSCLC）中存在间变性淋巴瘤激酶（ALK）重排。用克唑替尼可有效治疗ALK重排的NSCLCs。然而，反应的程度和持续时间是多种多样的。本研究探讨了克唑替尼在不同ALK变体中的临床疗效。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Lung Cancer">
                Journal: Lung Cancer [IF=4.294]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29571990" target="_blank">
                PMID: 29571990
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Li Y, Zhang T, Zhang J, Li W, Yuan P, Xing P, Zhang Z, Chuai S, Li J, Ying J">
                Authors: Li Y, Zhang T, Zh...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29571990">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>215. Acquired EGFR L718V mutation mediates resistance to osimertinib in non-small cell lung cancer but retains sensitivity to afatinib.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29571986" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29571986" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29571986">
                    <p>Epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors (TKIs) are promising targeted therapies for EGFR-mutated non-small-cell lung cancer (NSCLC) patients. However, acquired resistance inevitably develops. Comprehensive and dynamic companion genomic diagnosis can gain insights into underlying resistance mechanisms, thereby help oncologists and patients to make informed decision on the potential benefit of the treatment.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29571986">
                    <p>表皮生长因子受体（EGFR）酪氨酸激酶抑制剂（TKIs）是EGFR突变的非小细胞肺癌（NSCLC）患者的有希望的靶向治疗。然而，获得性抵抗不可避免地会发展。全面和动态的伴侣基因组诊断可以深入了解潜在的耐药机制，从而帮助肿瘤学家和患者对治疗的潜在益处作出明智的决定。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Lung Cancer">
                Journal: Lung Cancer [IF=4.294]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29571986" target="_blank">
                PMID: 29571986
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Liu Y, Li Y, Ou Q, Wu X, Wang X, Shao YW, Ying J">
                Authors: Liu Y, Li Y, Ou Q...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29571986">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>216. Comprehensive analysis of cancers of unknown primary for the biomarkers of response to immune checkpoint blockade therapy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29571084" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29571084" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29571084">
                    <p>Cancer of unknown primary (CUP) accounts for approximately 3% of all malignancies. Avoiding immune destruction is a major cancer characteristic and therapies aimed at immune checkpoint blockade are in use for several specific cancer types. A comprehensive survey of predictive biomarkers to immune checkpoint blockade in CUP were explored in this study.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29571084">
                    <p>未知原发癌（CUP）约占所有恶性肿瘤的3％。避免免疫破坏是一种主要的癌症特征，针对免疫检查点阻断的疗法正在用于几种特定的癌症类型。本研究探讨了预防性生物标志物对银屑病免疫检查点阻滞的全面调查。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Cancer">
                Journal: Eur. J. Cancer [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29571084" target="_blank">
                PMID: 29571084
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Gatalica Z, Xiu J, Swensen J, Vranic S">
                Authors: Gatalica Z, Xiu J...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29571084">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>217. Novel putative drivers revealed by targeted exome sequencing of advanced solid tumors.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29570743" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29570743" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29570743">
                    <p>Next generation sequencing (NGS) is becoming increasingly integrated into oncological practice and clinical research. NGS methods have also provided evidence for clonal evolution of cancers during disease progression and treatment. The number of variants associated with response to specific therapeutic agents keeps increasing. However, the identification of novel driver mutations as opposed to passenger (phenotypically silent or clinically irrelevant) mutations remains a major challenge. We conducted targeted exome sequencing of advanced solid tumors from 44 pre-treated patients with solid tumors including breast, colorectal and lung carcinomas, neuroendocrine tumors, sarcomas and others. We catalogued established driver mutations and putative new drivers as predicted by two distinct algorithms. The established drivers we detected were consistent with published observations. However, we also detected a significant number of mutations with driver potential never described before in each tumor type we studied. These putative drivers belong to key cell fate regulatory networks, including potentially druggable pathways. Should our observations be confirmed, they would support the hypothesis that new driver mutations are selected by treatment in clinically aggressive tumors, and indicate a need for longitudinal genomic testing of solid tumors to inform second line cancer treatment.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29570743">
                    <p>新一代测序（NGS）正日益融入肿瘤学实践和临床研究。 NGS方法也为疾病进展和治疗过程中癌症的克隆演变提供了证据。与对特定治疗剂的反应相关的变体的数量不断增加。然而，与乘客（表型沉默或临床无关的）突变相对的新型驱动突变的鉴定仍然是主要挑战。我们对来自44例接受过实体瘤治疗的患者（包括乳腺癌，结肠直肠癌和肺癌，神经内分泌肿瘤，肉瘤等）的高级实体瘤进行了靶向外显子组测序。按照两种不同的算法预测，我们编制了既定的驱动程序突变和假定的新驱动程序。我们检测到的既定驱动程序与发表的观察结果一致然而，我们还发现了在我们研究的每种肿瘤类型中前面没有描述过的具有驱动潜能的大量突变。这些推定的驱动因素属于关键的细胞命运监管网络，包括潜在的可药用途径。如果我们的观察结果得到证实，他们将支持这样的假设，即通过临床侵袭性肿瘤的治疗选择新的驱动突变，并且表明需要对实体瘤进行纵向基因组测试以通知二线癌症治疗。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29570743" target="_blank">
                PMID: 29570743
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Pannuti A, Filipovic A, Hicks C, Lefkowitz E, Ptacek T, Stebbing J, Miele L">
                Authors: Pannuti A, Filipo...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5865730" target="_blank">
                PMC5865730
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29570743">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>218. A Whole Germline BRCA2 Gene Deletion: How to Learn from CNV In Silico Analysis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29570666" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29570666" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29570666">
                    <p>BRCA1/2 screening in Hereditary Breast and Ovarian Syndrome (HBOC) is an essential step for effective patients&#39; management. Next-Generation Sequencing (NGS) can rapidly provide high throughput and reliable information about the qualitative and quantitative status of tumor-associated genes. Straightforwardly, bioinformatics methods play a key role in molecular diagnostics pipelines. BRCA1/2 genes were evaluated with our NGS workflow, coupled with Multiplex Amplicon Quantification (MAQ) and Multiplex Ligation-dependent Probe Amplification (MLPA) assays. Variant calling was performed on Amplicon Suite, while Copy Number Variant (CNV) prediction by in house and commercial CNV tools, before confirmatory MAQ/MLPA testing. The germline profile of BRCA genes revealed a unique HBOC pattern. Although variant calling analysis pinpointed heterozygote and homozygote polymorphisms on BRCA1 and BRCA2, respectively, the CNV predicted by our script suggested two conflicting interpretations: BRCA1 duplication and/or BRCA2 deletion. Our commercial software reported a BRCA1 duplication, in contrast with variant calling results. Finally, the MAQ/MLPA assays assessed a whole BRCA2 copy loss. In silico CNV analysis is a time and cost-saving procedure to powerfully identify possible Large Rearrangements using robust and efficient NGS pipelines. Our layout shows as bioinformatics algorithms alone cannot completely and correctly identify whole BRCA1/2 deletions/duplications. In particular, the complete deletion of an entire gene, like in our case, cannot be solved without alternative strategies as MLPA/MAQ. These findings support the crucial role of bioinformatics in deciphering pitfalls within NGS data analysis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29570666">
                    <p>BRCA1 / 2筛查遗传性乳腺癌和卵巢综合征（HBOC）是有效治疗患者的必要步骤。新一代测序（NGS）可以快速提供关于肿瘤相关基因的定性和定量状态的高通量和可靠信息。简而言之，生物信息学方法在分子诊断流水线中发挥关键作用。用我们的NGS工作流程评估BRCA1 / 2基因，并结合多重扩增子定量（MAQ）和多重连接依赖性探针扩增（MLPA）分析。在确认MAQ / MLPA测试之前，在Amplicon Suite上进行变体调用，同时使用内部和商业CNV工具进行拷贝数变异（CNV）预测。 BRCA基因的种系特征揭示了独特的HBOC模式。虽然变异调用分析针对BRCA1和BRCA2上的杂合子和纯合子多态性，我们的脚本预测的CNV提示两个相互矛盾的解释：BRCA1重复和/或BRCA2缺失。我们的商业软件报告了BRCA1重复，与变体调用结果相比较。最后，MAQ / MLPA分析评估了整个BRCA2拷贝丢失。计算机模拟CNV分析是一个节省时间和成本的过程，可以使用强大而高效的NGS管道强有力地识别可能的大型重排。我们的布局显示单独的生物信息学算法不能完全和正确地识别整个BRCA1 / 2缺失/重复。特别是，如果没有MLPA / MAQ等替代策略，就不能解决整个基因的完全缺失，就像我们的情况一样。这些发现支持生物信息学在解析NGS数据分析中的缺陷方面的重要作用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Mol Sci">
                Journal: Int J Mol Sci [IF=3.226]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29570666" target="_blank">
                PMID: 29570666
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Scaglione GL, Concolino P, De Bonis M, De Paolis E, Minucci A, Ferrandina G, Scambia G, Capoluongo E">
                Authors: Scaglione GL, Con...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29570666">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>219. Rhizosphere Microbiome Modulators: Contributions of Nitrogen Fixing Bacteria towards Sustainable Agriculture.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29570619" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29570619" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29570619">
                    <p>Rhizosphere microbiome which has been shown to enhance plant growth and yield are modulated or influenced by a few environmental factors such as soil type, plant cultivar, climate change and anthropogenic activities. In particular, anthropogenic activity, such as the use of nitrogen-based chemical fertilizers, is associated with environmental destruction and this calls for a more ecofriendly strategy to increase nitrogen levels in agricultural land. This feat is attainable by harnessing nitrogen-fixing endophytic and free-living rhizobacteria. Rhizobium, Pseudomonas, Azospirillum and Bacillus, have been found to have positive impacts on crops by enhancing both above and belowground biomass and could therefore play positive roles in achieving sustainable agriculture outcomes. Thus, it is necessary to study this rhizosphere microbiome with more sophisticated culture-independent techniques such as next generation sequencing (NGS) with the prospect of discovering novel bacteria with plant growth promoting traits. This review is therefore aimed at discussing factors that can modulate rhizosphere microbiome with focus on the contributions of nitrogen fixing bacteria towards sustainable agricultural development and the techniques that can be used for their study.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29570619">
                    <p>已显示增强植物生长和产量的根际微生物组受到少数环境因素如土壤类型，植物栽培品种，气候变化和人类活动的调节或影响。特别是人为活动，如使用氮基化肥，与环境破坏有关，这就要求采取更加环保的策略来增加农地的氮含量。这项壮举是通过利用固氮内生菌和自由活的根瘤菌来实现的。根瘤菌，假单胞菌属，固氮螺菌属和芽孢杆菌属已被发现通过提高地上和地下生物量对作物产生积极影响，因此可以在实现可持续农业成果方面发挥积极作用。因此，有必要用更复杂的不依赖于培养的技术（如下一代测序（NGS））研究这种根际微生物群，以期发现具有植物生长促进特性的新细菌。因此，这篇综述旨在讨论调节根际微生物组的因素，重点讨论固氮细菌对可持续农业发展的贡献以及可用于他们研究的技术。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Environ Res Public Health">
                Journal: Int J Environ Res Public Health [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29570619" target="_blank">
                PMID: 29570619
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Igiehon NO, Babalola OO">
                Authors: Igiehon NO, Babal...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29570619">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>220. Detection of Significant Copy Number Variations From Multiple Samples in Next-Generation Sequencing Data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29570071" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29570071" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29570071">
                    <p>Analyzing copy number variations (CNVs) from next-generation sequencing (NGS) data has become a common approach to detect disease susceptibility genes. The main challenge is how to utilize the NGS data with limited coverage depth to detect significant CNVs. Here, we introduce a new statistical method, the derivative of correlation coefficient (DCC), to detect significant CNVs that recurrently occur in multiple samples using read depth signals. We use a sliding window to calculate a correlation coefficient for each genome bin, and compute corresponding derivatives by fitting curves to the correlation coefficient. Then, the detection of significant CNVs was transformed into a problem of detecting significant derivatives reflecting genome breakpoints that can be solved using statistical hypothesis testing. We tested and compared the performance of DCC against several peer methods using a large number of simulation data sets, and validated DCC using several real sequencing data sets derived from the European Genome-Phenome archive, DNA Data Bank of Japan, and the 1000 Genomes Project. Experimental results suggest that DCC is an effective approach for identifying CNVs, outperforming peer methods in the terms of detection power and accuracy. DCC can be used to detect significant or recurrent CNVs in various NGS data sets, thus providing useful information to study genomic mutations and find disease susceptibility genes.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29570071">
                    <p>分析来自新一代测序（NGS）数据的拷贝数变异（CNV）已成为检测疾病易感基因的常用方法。主要挑战是如何利用覆盖深度有限的NGS数据来检测重要的CNV。在这里，我们引入一种新的统计方法，即相关系数（DCC）的导数，以检测使用读取深度信号在多个样本中反复发生的显着CNV。我们使用滑动窗口计算每个基因组bin的相关系数，并通过拟合相关系数曲线来计算相应的导数。然后，将显着CNV的检测转化成检测显着衍生物的问题，所述衍生物反映可使用统计假设检验来解决的基因组断点。我们使用大量模拟数据集对DCC的性能进行了测试并与几种同类方法进行了比较，并使用来自欧洲Genome-Phenome存档，日本DNA数据库和1000基因组计划的几个真实测序数据集验证了DCC 。实验结果表明，DCC是识别CNV的有效方法，其在检测能力和准确度方面优于对等方法。 DCC可用于检测各种NGS数据集中的显着或复发CNV，从而为研究基因组突变和发现疾病易感基因提供有用的信息。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=IEEE Trans Nanobioscience">
                Journal: IEEE Trans Nanobioscience [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29570071" target="_blank">
                PMID: 29570071
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar
            </a>

            <a class="btn btn-sm btn-danger" title="Yuan X, Zhang J, Yang L, Bai J, Fan P">
                Authors: Yuan X, Zhang J, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29570071">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>221. Quantitative T-cell repertoire analysis of peripheral blood mononuclear cells from lung cancer patients following long-term cancer peptide vaccination.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29568993" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29568993" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29568993">
                    <p>Therapeutic cancer peptide vaccination is an immunotherapy designed to elicit cytotoxic T-lymphocyte (CTL) responses in patients. A number of therapeutic vaccination trials have been performed, nevertheless there are only a few reports that have analyzed the T-cell receptors (TCRs) expressed on tumor antigen-specific CTLs. Here, we use next-generation sequencing (NGS) to analyze TCRs of vaccine-induced CTL clones and the TCR repertoire of bulk T cells in peripheral blood mononuclear cells (PBMCs) from two lung cancer patients over the course of long-term vaccine therapy. In both patients, vaccination with two epitope peptides derived from cancer/testis antigens (upregulated lung cancer 10 (URLC10) and cell division associated 1 (CDCA1)) induced specific CTLs expressing various TCRs. All URLC10-specific CTL clones tested showed Ca2+ influx, IFN-γ production, and cytotoxicity when co-cultured with URLC10-pulsed tumor cells. Moreover, in CTL clones that were not stained with the URLC10/MHC-multimer, the CD3 ζ chain was not phosphorylated. NGS of the TCR repertoire of bulk PBMCs demonstrated that the frequency of vaccine peptide-specific CTL clones was near the minimum detectable threshold level. These results demonstrate that vaccination induces antigen-specific CTLs expressing various TCRs at different time points in cancer patients, and that some CTL clones are maintained in PBMCs during long-term treatment, including some with TCRs that do not bind peptide/MHC-multimer.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29568993">
                    <p>治疗性癌症肽疫苗接种是旨在引发患者中的细胞毒性T淋巴细胞（CTL）应答的免疫疗法。已经进行了许多治疗性疫苗接种试验，但是仅有少数报道分析了在肿瘤抗原特异性CTL上表达的T细胞受体（TCR）。在这里，我们使用新一代测序（NGS）来分析疫苗诱导的CTL克隆的TCR和在两种肺癌患者的长期疫苗疗法过程中的外周血单核细胞（PBMC）中的大量T细胞的TCR库。在两例患者中，用来源于癌症/睾丸抗原的两种表位肽（上调的肺癌10（URLC10）和细胞分裂相关的1（CDCA1））的疫苗接种诱导表达各种TCR的特异性CTL。与URLC10脉冲的肿瘤细胞共培养时，所有测试的URLC10特异性CTL克隆显示出Ca2 +内流，IFN-γ产生和细胞毒性。此外，在未用URLC10 / MHC多聚体染色的CTL克隆中，CD3ζ链不被磷酸化。大量PBMC的TCR库的NGS证明疫苗肽特异性CTL克隆的频率接近最小可检测阈值水平。这些结果表明接种疫苗诱导在癌症患者的不同时间点表达各种TCR的抗原特异性CTL，并且一些CTL克隆在长期治疗期间保持在PBMC中，包括一些TCR不结合肽/ MHC多聚体。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Immunol. Immunother.">
                Journal: Cancer Immunol. Immunother. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29568993" target="_blank">
                PMID: 29568993
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-22
            </a>

            <a class="btn btn-sm btn-danger" title="Takeda K, Kitaura K, Suzuki R, Owada Y, Muto S, Okabe N, Hasegawa T, Osugi J, Hoshino M, Tsunoda T, Okumura K, Suzuki H">
                Authors: Takeda K, Kitaura...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29568993">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>222. Ten steps to get started in Genome Assembly and Annotation.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29568489" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29568489" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29568489">
                    <p>As a part of the ELIXIR-EXCELERATE efforts in capacity building, we present here 10 steps to facilitate researchers getting started in genome assembly and genome annotation. The guidelines given are broadly applicable, intended to be stable over time, and cover all aspects from start to finish of a general assembly and annotation project. Intrinsic properties of genomes are discussed, as is the importance of using high quality DNA. Different sequencing technologies and generally applicable workflows for genome assembly are also detailed. We cover structural and functional annotation and encourage readers to also annotate transposable elements, something that is often omitted from annotation workflows. The importance of data management is stressed, and we give advice on where to submit data and how to make your results Findable, Accessible, Interoperable, and Reusable (FAIR).</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29568489">
                    <p>作为ELIXIR-EXCELERATE在能力建设方面努力的一部分，我们在这里介绍10个步骤，以促进研究人员开始基因组组装和基因组注释。所给出的指导方针是广泛适用的，旨在保持一段时间的稳定性，涵盖从大会和注释项目开始到结束的所有方面。讨论基因组的内在特性，以及使用高质量DNA的重要性。还详细介绍了不同的测序技术和通用的基因组组装工作流程。我们涵盖结构和功能注释，并鼓励读者也对转座元素进行注释，这些注释工作流程通常会忽略这些元素。强调数据管理的重要性，并就如何提交数据以及如何使结果可查找，可访问，可互操作和可重复使用（FAIR）提供建议。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=F1000Res">
                Journal: F1000Res [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29568489" target="_blank">
                PMID: 29568489
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Dominguez Del Angel V, Hjerde E, Sterck L, Capella-Gutierrez S, Notredame C, Vinnere Pettersson O, Amselem J, Bouri L, Bocs S, Klopp C, Gibrat JF, Vlasova A, Leskosek BL, Soler L, Binzer-Panchal M, Lantz H">
                Authors: Dominguez Del Ang...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850084" target="_blank">
                PMC5850084
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29568489">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>223. Performance evaluation method for read mapping tool in clinical panel sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29568413" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29568413" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29568413">
                    <p>In addition to the rapid advancement in Next-Generation Sequencing (NGS) technology, clinical panel sequencing is being used increasingly in clinical studies and tests. However, tools that are used in NGS data analysis have not been comparatively evaluated in performance for panel sequencing. This study aimed to evaluate the tools used in the alignment process, the first procedure in bioinformatics analysis, by comparing tools that have been widely used with ones that have been introduced recently. With the accumulated panel sequencing data, detected variant lists were cataloged and inserted into simulated reads produced from the reference genome (h19). The amount of unmapped reads and misaligned reads, mapping quality distribution, and runtime were measured as standards for comparison. As the most widely used tools, Bowtie2 and BWA-MEM each showed explicit performance with AUC of 0.9984 and 0.9970 respectively. Kart, maintaining superior runtime and less number of misaligned read, also similarly possessed high level of AUC (0.9723). Such selection and optimization method of tools appropriate for panel sequencing can be utilized for fields requiring error minimization, such as clinical application and liquid biopsy studies.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29568413">
                    <p>除了新一代测序（NGS）技术的快速发展之外，临床小组测序正在越来越多地用于临床研究和测试。然而，用于NGS数据分析的工具在面板测序的性能方面还没有得到相对的评估。本研究旨在通过比较已广泛使用的工具和最近推出的工具，评估对齐过程中使用的工具，这是生物信息学分析的第一步。利用累积的面板测序数据，检测到的变体列表被编目并插入到参考基因组（h19）产生的模拟读数中。测量未映射读取和未对齐读取的数量，映射质量分布和运行时间作为比较标准。作为使用最广泛的工具，Bowtie2和BWA-MEM各自显示出明显的性能，AUC分别为0.9984和0.9970。卡丁车保持卓越的运行时间和较少的错位读取数量，同样具有较高的AUC水平（0.9723）。适用于面板测序的工具的这种选择和优化方法可用于需要误差最小化的领域，例如临床应用和液体活检研究。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genes Genomics">
                Journal: Genes Genomics [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29568413" target="_blank">
                PMID: 29568413
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Lee H, Lee KW, Lee T, Park D, Chung J, Lee C, Park WY, Son DS">
                Authors: Lee H, Lee KW, Le...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5846869" target="_blank">
                PMC5846869
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29568413">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>224. The single-nucleotide polymorphisms in CHD5 affect the prognosis of patients with hepatocellular carcinoma.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29568352" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29568352" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29568352">
                    <p>Previous studies showed that the low expressions of chromodomain-helicase-DNA-binding protein 5 (CHD5) were intensively associated with deteriorative biologic and clinical characteristics as well as outcomes in many tumors. The aim of this study is to determine whether CHD5 single nucleotide polymorphisms (SNPs) contribute to the prognosis of hepatocellular carcima (HCC). The SNPs were selected according to their linkage disequilibrium (LD) in the targeted next-generation sequencing (NGS) and then genotyped with TaqMan probers. We revealed a rare haplotype AG in CHD5 (SNPs: rs12564469-rs9434711) was markedly associated with HCC prognosis. The univariate and multivariate regression analyses revealed the patients with worse overall survival time were those with tumor metastasis and haplotype AG, as well as cirrhosis, poor differentiation and IV-TNM stage. Based on the available public databases, we discovered the significant association between haplotype AG and CHD5 mRNA expressions only existed in Chinese. These data proposed that the potentially genetic haplotype might functionally contribute to HCC prognosis and CHD5 mRNA expressions.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29568352">
                    <p>以前的研究表明chromodomain-helicase-DNA-binding protein 5（CHD5）的低表达与许多肿瘤的恶化生物和临床特征以及结果密切相关。本研究的目的是确定CHD5单核苷酸多态性（SNPs）是否有助于肝细胞癌（HCC）的预后。根据目标下一代测序（NGS）中的连锁不平衡（LD）选择SNP，然后用TaqMan探针进行基因分型。我们揭示了CHD5中罕见的单体型AG（SNPs：rs12564469-rs9434711）与HCC预后显着相关。单因素和多因素回归分析显示总生存时间较短的患者为肿瘤转移和单体型AG，以及肝硬化，分化差和IV-TNM分期。基于现有的公共数据库，我们发现单倍型AG与CHD5 mRNA表达之间的显着相关性仅存在于汉语中。这些数据提出，潜在的遗传单倍型可能在功能上有助于HCC预后和CHD5 mRNA表达。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29568352" target="_blank">
                PMID: 29568352
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-02
            </a>

            <a class="btn btn-sm btn-danger" title="Zhu X, Kong Q, Xie L, Chen Z, Li H, Zhu Z, Huang Y, Lan F, Luo H, Zhan J, Ding H, Lei J, Xiao Q, Fu W, Fan W, Zhang J, Luo H">
                Authors: Zhu X, Kong Q, Xi...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5862573" target="_blank">
                PMC5862573
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29568352">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>225. A complete, homozygous CRX deletion causing nullizygosity is a new genetic mechanism for Leber congenital amaurosis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29568065" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29568065" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29568065">
                    <p>CRX is a transcription factor required for activating the expression of many photoreceptor-neuron genes. CRX may be mutated in three forms of human blindness; Leber congenital amaurosis (LCA), cone-rod degeneration (CRD) and retinitis pigmentosa (RP). The pathogenic mechanism in most cases is likely dominant negative, with gain of function. We report a novel, complete homozygous CRX deletion in LCA. We identified a Lebanese family with 3 affected LCA cases. The proband was sequenced by NGS. Quantitative PCR, array comparative genomic hybridization, and long range PCR were performed. Full eye examinations, OCT and photography were performed. We identified a homozygous 56,000 bp deletion of CRX, which co-segregates and is heterozygous in four parents, who report normal vision. The blind children with LCA manifest severe retinal degeneration, a phenotype typical for CRX and LCA. We hypothesized that a single copy of CRX (haplo-insufficiency) in the causes mild abnormal foveal development, but not LCA. Two parents had significant inner and outer foveal and photoreceptor abnormalities. This is the first reported case of a homozygous, complete CRX deletion. Nullizygosity of CRX thus causes LCA while haplo-insufficiency of CRX causes abnormal foveal development, but not LCA. Our data suggest a new disease mechanism for CRX.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29568065">
                    <p>CRX是激活许多光感受器 - 神经元基因表达所需的转录因子。 CRX可能以三种人类失明形式发生突变;莱伯先天性黑蒙（LCA），锥形杆变性（CRD）和视网膜色素变性（RP）。大多数病例的致病机制可能占主导地位，功能获益。我们报告了LCA中新的完全纯合CRX缺失。我们确定了一个黎巴嫩家庭，其中有3个受影响的LCA病例。先证者被NGS测序。进行定量PCR，阵列比较基因组杂交和长距离PCR。进行全面的眼科检查，OCT和摄影。我们确定了CRX的纯合56,000bp缺失，其在共报道并且在报告正常视力的四位父母中是杂合的。患有LCA的盲童表现出严重的视网膜变性，这是CRX和LCA典型的表型。我们假设CRX（单倍体不全）的单个副本导致轻度异常中心凹发育，但不是LCA。两名父母有显着的内部和外部中心凹和感光异常。这是首例报道的纯合CRX完全缺失病例。 CRX的无效合成因此导致LCA，而CRX的单倍体不足导致异常中心凹发育，但不是LCA。我们的数据表明CRX有一种新的疾病机制。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29568065" target="_blank">
                PMID: 29568065
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-22
            </a>

            <a class="btn btn-sm btn-danger" title="Ibrahim MT, Alarcon-Martinez T, Lopez I, Fajardo N, Chiang J, Koenekoop RK">
                Authors: Ibrahim MT, Alarc...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5864841" target="_blank">
                PMC5864841
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29568065">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>226. False positive plasma genotyping due to clonal hematopoiesis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29567812" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29567812" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29567812">
                    <p>Plasma cell-free DNA (cfDNA) genotyping is increasingly used in cancer care, but assay accuracy is debated. Because most cfDNA is derived from peripheral blood cells (PBC), we hypothesized that nonmalignant mutations harbored by hematopoietic cells (clonal hematopoiesis, CH) could be a cause of false positive plasma genotyping.  Experimental Design: We identified patients with advanced NSCLC with KRAS, JAK2, or TP53 mutations identified in cfDNA. With consent, PBC DNA was tested using droplet digital PCR (ddPCR) or next-generation sequencing (NGS) to test for CH-derived mutations.  Results: We first studied plasma ddPCR results from 58 EGFR-mutant NSCLC patients. Two had KRAS G12X detected in cfDNA and both were present in PBC, including one where the KRAS mutation was detected serially for 20 months. We then studied 143 plasma NGS results from 122 NSCLC patients, and identified 5 JAK2 V617F mutations derived from PBC. Additionally, 108 TP53 mutations were detected in cfDNA; for 33 of the TP53 mutations, PBC and tumor NGS were available for comparison, and 5 were present in PBC but absent in tumor, consistent with CH.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29567812">
                    <p>无细胞浆细胞DNA（cfDNA）基因分型越来越多地用于癌症治疗，但检测的准确性还存在争议。由于大多数cfDNA来自外周血细胞（PBC），我们假设造血细胞（克隆造血细胞，CH）含有的非恶性突变可能是假阳性血浆基因分型的原因。实验设计：我们鉴定了患有晚期NSCLC的患者，其具有在cfDNA中鉴定的KRAS，JAK2或TP53突变。经同意，使用液滴数字PCR（ddPCR）或下一代测序（NGS）测试PBC DNA以测试CH-衍生的突变。结果：我们首先研究了来自58个EGFR突变NSCLC患者的血浆ddPCR结果。两个在c​​fDNA中检测到KRAS G12X，两者均存在于PBC中，包括其中连续20个月检测到KRAS突变的一个。然后，我们研究了来自122名NSCLC患者的143种血浆NGS结果，并​​确定了5种来源于PBC的JAK2 V617F突变。另外，在cfDNA中检测到108个TP53突变;对于TP53突变中的33个，PBC和肿瘤NGS可用于比较，并且5个存在于PBC中但在肿瘤中不存在，与CH一致。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Cancer Res.">
                Journal: Clin. Cancer Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29567812" target="_blank">
                PMID: 29567812
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-22
            </a>

            <a class="btn btn-sm btn-danger" title="Hu Y, Ulrich B, Supplee J, Kuang Y, Lizotte PH, Feeney N, Guibert N, Awad MM, Wong KK, Janne PA, Paweletz CP, Oxnard GR">
                Authors: Hu Y, Ulrich B, S...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29567812">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>227. The characteristics of ctDNA reveal the high complexity in matching the corresponding tumor tissues.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29566644" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29566644" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29566644">
                    <p>Next-generation sequencing (NGS) is an efficient and sensitive method to detect mutations from ctDNA. Many features and clinical conditions could significantly affect the concordance between ctDNA and corresponding tumor tissues. Our goal was to systematically investigate the critical factors contributing to different concordance between ctDNA and corresponding tumor tissues.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29566644">
                    <p>新一代测序（NGS）是检测ctDNA突变的有效方法。许多特征和临床条件可以显着影响ctDNA和相应肿瘤组织之间的一致性。我们的目标是系统调查ctDNA与相应肿瘤组织不同一致性的关键因素。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Cancer">
                Journal: BMC Cancer [IF=3.288]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29566644" target="_blank">
                PMID: 29566644
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-23
            </a>

            <a class="btn btn-sm btn-danger" title="Yang N, Li Y, Liu Z, Qin H, Du D, Cao X, Cao X, Li J, Li D, Jiang B, Duan L, Yang H, Zhang Z, Lin H, Li J, Yang Z, Xiong L, Shen H, Lin L, Li F">
                Authors: Yang N, Li Y, Liu...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5865353" target="_blank">
                PMC5865353
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29566644">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>228. Next-generation sequencing analysis of a cluster of hepatitis C virus infections in a haematology and oncology center.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29566084" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29566084" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29566084">
                    <p>Molecular characterization of early hepatitis C virus (HCV) infection remains rare. Ten out of 78 patients of a hematology/oncology center were found to be HCV RNA positive two to four months after hospitalization. Only two of the ten patients were anti-HCV positive. HCV hypervariable region 1 (HVR1) was amplified in seven patients (including one anti-HCV positive) and analyzed by next generation sequencing (NGS). Genetic variants were reconstructed by Shorah and an empirically established 0.5% variant frequency cut-off was implemented. These sequences were compared by phylogenetic and diversity analyses. Ten unrelated blood donors with newly acquired HCV infection detected at the time of donation (HCV RNA positive and anti-HCV negative) served as controls. One to seven HVR1 variants were found in each patient. Sequences intermixed phylogenetically with no evidence of clustering in individual patients. These sequences were more similar to each other (similarity 95.4% to 100.0%) than to those of controls (similarity 64.8% to 82.6%). An identical predominant variant was present in four patients, whereas other closely related variants dominated in the remaining three patients. In five patients the HCV population was limited to a single variant or one predominant variant and minor variants of less than 10% frequency. In conclusion, NGS analysis of a cluster of HCV infections acquired in the hospital setting revealed the presence of low diversity, very closely related variants in all patients, suggesting an early-stage infection with the same virus. NGS combined with phylogenetic analysis and classical epidemiological analysis could help in tracking of HCV outbreaks.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29566084">
                    <p>早期丙型肝炎病毒（HCV）感染的分子特征仍然很少见。在血液学/肿瘤中心的78例患者中，有10例在住院后两到四个月内发现HCV RNA阳性。十名患者中只有两名患者抗-HCV阳性。在7位患者（包括一个抗-HCV阳性）中扩增HCV高变区1（HVR1）并通过下一代测序（NGS）进行分析。遗传变异由Shorah重建，并实施了经验建立的0.5％变异频率截止值。这些序列通过系统发育和多样性分析进行比较。在捐献时检测到10个与HCV新感染感染无关的献血者（HCV RNA阳性和抗HCV阴性）作为对照。在每个患者中发现1至7个HVR1变体。序列混杂在系统发育中，没有证据表明个体患者聚集。这些序列彼此更相似（相似性95.4％至100.0％）比对照者（相似性64.8％至82.6％）更相似。 4名患者中存在相同的主要变体，而其余3名患者中以其他密切相关的变体为主。在五名患者中，HCV群体限于单个变体或一个主要变体和少于10％频率的次要变体。总之，在医院环境中获得的一组HCV感染的NGS分析显示所有患者中存在低多样性，非常密切相关的变体，表明与相同病毒的早期感染。 NGS与系统发育分析和经典流行病学分析相结合有助于追踪HCV暴发。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=PLoS ONE">
                Journal: PLoS ONE [IF=2.806]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29566084" target="_blank">
                PMID: 29566084
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Caraballo Cortes K, Rosińska M, Janiak M, Stępień M, Zagordi O, Perlejewski K, Osuch S, Pawełczyk A, Bukowska-Ośko I, Płoski R, Grabarczyk P, Laskus T, Radkowski M">
                Authors: Caraballo Cortes ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5864040" target="_blank">
                PMC5864040
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29566084">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>229. JAK2 V617F mutation in plasma cell-free DNA preceding clinically overt myelofibrosis: Implications for early diagnosis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29565699" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29565699" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29565699">
                    <p>A 52 year-old man with Erdheim-Chester Disease (ECD) (a non-Lanherhans polyostotic sclerosing histiocytosis) had next-generation sequencing (NGS) performed as part of his diagnostic workup. In addition to the tissue BRAF V600E mutation that is found in over 50% of ECD cases, he was also found to have a JAK2 V617F alteration in cell-free circulating tumor DNA (ctDNA) (liquid biopsy). The latter was thought to be an &quot;incidental&quot; finding, perhaps due to clonal hematopoiesis (though this usually occurs in older individuals), as his blood counts were normal and he had no splenomegaly. Approximately 13 months after the ctDNA test showing JAK2 V617F, he developed anemia, thrombocytopenia, and splenomegaly. Marrow biopsy then showed megakaryocytic atypia and markedly increased marrow fibrosis, consistent with WHO grade 2 of 3 myelofibrosis. Therefore, the patient was determined to have ECD with a typical BRAF V600E mutation, as well as primary myelofibrosis, with the latter diagnosis manifesting clinically over one year after the JAK2 V617F was first detected in ctDNA. He recently was started on the JAK2 inhibitor ruxolitinib. This case demonstrates that genomic alterations detected by liquid biopsy for evaluation of specific malignancies already present may serve as an early harbinger of hematological disease.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29565699">
                    <p>Erdheim-Chester病（ECD）（一种非Lanherhans多形性硬化性组织细胞增多症）的52岁男性在其诊断检查中进行了下一代测序（NGS）。除了在超过50％的ECD病例中发现的组织BRAF V600E突变之外，他还发现在无细胞循环肿瘤DNA（ctDNA）（液体活组织检查）中具有JAK2 V617F改变。后者被认为是“偶然的”发现，可能是由于克隆性造血作用（尽管这通常发生在年长的个体中），因为他的血细胞计数正常并且没有脾肿大。在ctDNA检测显示JAK2 V617F约13个月后，他发生贫血，血小板减少和脾肿大。骨髓活检显示巨核细胞异型性明显增加，骨髓纤维化增多，符合WHO 2级骨髓纤维化3级。因此，患者被确定患有典型的BRAF V600E突变的ECD以及原发性骨髓纤维化，后者在ctDNA中首次检测到JAK2 V617F后临床表现临床超过一年。他最近开始使用JAK2抑制剂ruxolitinib。本案例表明，液体活检检测到的基因组改变可用于评估已存在的特定恶性肿瘤，可能是血液疾病的早期预兆。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Biol. Ther.">
                Journal: Cancer Biol. Ther. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29565699" target="_blank">
                PMID: 29565699
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-22
            </a>

            <a class="btn btn-sm btn-danger" title="Choi MY, Kato S, Wang HY, Lin JH, Lanman RB, Kurzrock R">
                Authors: Choi MY, Kato S, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29565699">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>230. Heterogeneity of Circulating Tumor Cells in Neoadjuvant Chemotherapy of Breast Cancer.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29565320" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29565320" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29565320">
                    <p>The biological properties of circulating tumor cells (CTCs), and their dynamics during neoadjuvant chemotherapy are important, both for disease progression prediction and therapeutic target determination, with the aim of preventing disease progression. The aim of our study was to estimate of different CTC subsets in breast cancer during the NACT (neoadjuvant chemotherapy). The prospective study includes 27 patients with invasive breast cancer, T2-4N0-3M0, aged 32 to 60 years. Venous heparinized blood samples, taken before and after biopsy, after each courses of chemotherapy (on days 3-7), and before surgical intervention, served as the material for this study. Different subsets of circulating tumor cells were determined on the basis of the expression of EpCAM, CD45, CD44, CD24, and N-Cadherin using flow cytometry. As the result of this study, it has been observed that significant changes in the quantity of the different subsets of circulating tumor cells in patients&#39; blood were observed after carrying out the 3rd course of NACT. NACT causes significant changes in the quantity of six CTC subsets, with various combinations of stemness and epithelial-mesenchymal transition (EMT) properties.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29565320">
                    <p>循环肿瘤细胞（CTC）的生物学特性及其在新辅助化疗过程中的动态变化对疾病进展预测和治疗靶点确定都很重要，目的在于预防疾病进展。我们研究的目的是评估NACT（新辅助化疗）期间乳腺癌中不同的CTC亚型。前瞻性研究包括27例浸润性乳腺癌患者T2-4N0-3M0，年龄32-60岁。活检前后，每次化疗后（第3-7天）以及手术干预前采集静脉肝素化血液样本作为本研究的材料。使用流式细胞术根据EpCAM，CD45，CD44，CD24和N-钙粘蛋白的表达来确定循环肿瘤细胞的不同子集。作为这项研究的结果，已经观察到在进行NACT的第三个疗程后，观察到患者血液中循环肿瘤细胞的不同亚组的数量的显着变化。 NACT引起六种CTC亚群数量的显着变化，具有干性和上皮 - 间充质转变（EMT）性质的各种组合。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Molecules">
                Journal: Molecules [IF=2.861]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29565320" target="_blank">
                PMID: 29565320
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-22
            </a>

            <a class="btn btn-sm btn-danger" title="Kaigorodova EV, Savelieva OE, Tashireva LA, Tarabanovskaya NA, Simolina EI, Denisov EV, Slonimskaya EM, Choynzonov EL, Perelmuter VM">
                Authors: Kaigorodova EV, S...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29565320">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>231. Looking for Broken TAD Boundaries and Changes on DNA Interactions: Clinical Guide to 3D Chromatin Change Analysis in Complex Chromosomal Rearrangements and Chromothripsis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29564835" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29564835" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29564835">
                    <p>Apparition of next-generation sequencing (NGS) was a breakthrough on knowledge of genome structure. Bioinformatic tools are a key point to analyze this huge amount of data from NGS and characterize the three-dimensional organization of chromosomes. This chapter describes usage of different browsers to explore publicly available online data and to search for possible 3D chromatin changes involved during complex chromosomal rearrangements as chromothripsis. Their pathogenic impact on clinical phenotype and gene misexpression can also be evaluated with annotated databases.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29564835">
                    <p>下一代测序（NGS）的发现是基因组结构知识的一个突破。生物信息学工具是分析来自NGS的大量数据并表征染色体的三维组织的关键。本章介绍不同浏览器的用法，以探索公开可用的在线数据，并搜索染色体重排期间可能发生的3D染色质变化。它们对临床表型和基因表达的致病影响也可以用注释数据库进行评估。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Methods Mol. Biol.">
                Journal: Methods Mol. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29564835" target="_blank">
                PMID: 29564835
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Yauy K, Gatinois V, Guignard T, Sati S, Puechberty J, Gaillard JB, Schneider A, Pellestor F">
                Authors: Yauy K, Gatinois ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29564835">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>232. RNA-Seq Analysis to Detect Abnormal Fusion Transcripts Linked to Chromothripsis.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29564822" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29564822" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29564822">
                    <p>RNA-Seq approach enables the detection and characterization of fusion or chimeric transcript associated to complex genome rearrangement. Until now, these events are classically identified at DNA level.Here we describe a complete procedure including a novel way of analyzing reads that combines genomic locations and local coverage to directly infer chimeric junctions with a high sensitivity and specificity, allowing identification of different classes of chimeric RNA events. We also recommend the best practices for the bioinformatics analysis and describe the experimental process for RNA validation using real-time PCR and sequencing.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29564822">
                    <p>RNA-Seq方法能够检测和表征与复杂基因组重排相关的融合或嵌合转录物。到目前为止，这些事件是在DNA水平上经典鉴定的。在这里我们描述了一个完整的程序，包括一种分析读数的新方法，它将基因组位置和局部覆盖率结合起来，以高灵敏度和特异性直接推断嵌合结，从而可以鉴定嵌合RNA事件。我们还推荐生物信息学分析的最佳实践，并描述使用实时PCR和测序进行RNA验证的实验过程。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Methods Mol. Biol.">
                Journal: Methods Mol. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29564822" target="_blank">
                PMID: 29564822
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Bougé AL, Rufflé F, Riquier S, Guibert B, Audoux J, Commes T">
                Authors: Bougé AL, Rufflé ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29564822">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>233. Gut Microbial Dysbiosis in Indian Children with Autism Spectrum Disorders.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29564487" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29564487" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29564487">
                    <p>Autism spectrum disorder (ASD) is a term associated with a group of neurodevelopmental disorders. The etiology of ASD is not yet completely understood; however, a disorder in the gut-brain axis is emerging as a prominent factor leading to autism. To identify the taxonomic composition and markers associated with ASD, we compared the fecal microbiota of 30 ASD children diagnosed using Childhood Autism Rating Scale (CARS) score, DSM-5 approved AIIMS-modified INCLEN Diagnostic Tool for Autism Spectrum Disorder (INDT-ASD), and Indian Scale for Assessment of Autism (ISAA) tool, with family-matched 24 healthy children from Indian population using next-generation sequencing (NGS) of 16S rRNA gene amplicon. Our study showed prominent dysbiosis in the gut microbiome of ASD children, with higher relative abundances of families Lactobacillaceae, Bifidobacteraceae, and Veillonellaceae, whereas the gut microbiome of healthy children was dominated by the family Prevotellaceae. Comparative meta-analysis with a publicly available dataset from the US population consisting of 20 ASD and 20 healthy control samples from children of similar age, revealed a significantly high abundance of genus Lactobacillus in ASD children from both the populations. The results reveal the microbial dysbiosis and an association of selected Lactobacillus species with the gut microbiome of ASD children.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29564487">
                    <p>自闭症谱系障碍（ASD）是与一组神经发育障碍相关的术语。 ASD的病因尚未完全了解;然而，肠 - 脑轴紊乱正在成为导致自闭症的突出因素。为了确定与ASD相关的分类学组成和标记，我们比较了使用儿童孤独症评定量表（CARS）评分，DSM-5批准的AIIMS修改的INCLEN自闭症谱系障碍诊断工具（INDT-ASD）的30名ASD儿童的粪便微生物群， ，以及印度自闭症评估量表（ISAA）工具，使用16S rRNA基因扩增子的新一代测序（NGS）与来自印度人群的24名健康家庭成员进行比较。我们的研究显示，ASD儿童的肠道微生物组中存在显着的生态失调，家族中乳酸杆菌科，双歧杆菌科和Veillonellaceae的相对丰度较高，而健康儿童的肠道微生物群则以Prevotellaceae科为主。对来自美国人群的20个ASD和20个来自相似年龄的儿童的健康对照样品的公开可用数据集进行的比较荟萃分析揭示了来自两个群体的ASD儿童中乳酸杆菌属的丰度非常高。结果揭示了微生物生态失调以及选定的乳酸杆菌属物种与ASD儿童的肠道微生物群的关联。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Microb. Ecol.">
                Journal: Microb. Ecol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29564487" target="_blank">
                PMID: 29564487
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-21
            </a>

            <a class="btn btn-sm btn-danger" title="Pulikkan J, Maji A, Dhakan DB, Saxena R, Mohan B, Anto MM, Agarwal N, Grace T, Sharma VK">
                Authors: Pulikkan J, Maji ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29564487">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>234. Investigation of targetable predictive and prognostic markers in gallbladder carcinoma.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29564177" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29564177" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29564177">
                    <p>Gallbladder carcinoma, uncommon in most of the world, is frequent in South East Asia. It presents in advanced stages with limited therapeutic options. We investigated targetable predictive and prognostic Markers in 268 cases including 233 primary site lesions and 35 metastatic gallbladder carcinoma.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29564177">
                    <p>在世界大部分地区罕见的胆囊癌在东南亚频繁出现。它在晚期呈现有限的治疗选择。我们在268例病例中研究了可靶向的预测和预后标记物，包括233个原发部位病变和35个转移性胆囊癌。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Gastrointest Oncol">
                Journal: J Gastrointest Oncol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29564177" target="_blank">
                PMID: 29564177
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb
            </a>

            <a class="btn btn-sm btn-danger" title="Neyaz A, Husain N, Gupta S, Kumari S, Arora A, Awasthi NP, Malhotra KP, Misra S">
                Authors: Neyaz A, Husain N...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5848045" target="_blank">
                PMC5848045
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29564177">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>235. Classification and Biomarker Genes Selection for Cancer Gene Expression Data Using Random Forest.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29563929" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29563929" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29563929">
                    <p>Microarray and next generation sequencing (NGS) data are the important sources to find helpful molecular patterns. Also, the great number of gene expression data increases the challenge of how to identify the biomarkers associated with cancer. The random forest (RF) is used to effectively analyze the problems of large-p and small-n. Therefore, RF can be used to select and rank the genes for the diagnosis and effective treatment of cancer.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29563929">
                    <p>微阵列和下一代测序（NGS）数据是寻找有用分子模式的重要来源。此外，大量基因表达数据增加了如何鉴定与癌症相关的生物标志物的挑战。随机森林（RF）用于有效分析大-p和小-n的问题。因此，RF可用于选择和排列用于癌症诊断和有效治疗的基因。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Iran J Pathol">
                Journal: Iran J Pathol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29563929" target="_blank">
                PMID: 29563929
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2017-Fall
            </a>

            <a class="btn btn-sm btn-danger" title="Ram M, Najafi A, Shakeri MT">
                Authors: Ram M, Najafi A, ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5844678" target="_blank">
                PMC5844678
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29563929">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>236. Accelerating public sector rice breeding with high-density KASP markers derived from whole genome sequencing of indica rice.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29563850" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29563850" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29563850">
                    <p>Few public sector rice breeders have the capacity to use NGS-derived markers in their breeding programmes despite rapidly expanding repositories of rice genome sequence data. They rely on &gt; 18,000 mapped microsatellites (SSRs) for marker-assisted selection (MAS) using gel analysis. Lack of knowledge about target SNP and InDel variant loci has hampered the uptake by many breeders of Kompetitive allele-specific PCR (KASP), a proprietary technology of LGC genomics that can distinguish alleles at variant loci. KASP is a cost-effective single-step genotyping technology, cheaper than SSRs and more flexible than genotyping by sequencing (GBS) or array-based genotyping when used in selection programmes. Before this study, there were 2015 rice KASP marker loci in the public domain, mainly identified by array-based screening, leaving large proportions of the rice genome with no KASP coverage. Here we have addressed the urgent need for a wide choice of appropriate rice KASP assays and demonstrated that NGS can detect many more KASP to give full genome coverage. Through re-sequencing of nine indica rice breeding lines or released varieties, this study has identified 2.5 million variant sites. Stringent filtering of variants generated 1.3 million potential KASP assay designs, including 92,500 potential functional markers. This strategy delivers a 650-fold increase in potential selectable KASP markers at a density of 3.1 per 1 kb in the indica crosses analysed and 377,178 polymorphic KASP design sites on average per cross. This knowledge is available to breeders and has been utilised to improve the efficiency of public sector breeding in Nepal, enabling identification of polymorphic KASP at any region or quantitative trait loci in relevant crosses. Validation of 39 new KASP was carried out by genotyping progeny from a range of crosses to show that they detected segregating alleles. The new KASP have replaced SSRs to aid trait selection during marker-assisted backcrossing in these crosses, where target traits include rice blast and BLB resistance loci. Furthermore, we provide the software for plant breeders to generate KASP designs from their own datasets.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29563850">
                    <p>尽管水稻基因组序列数据库迅速扩大，但很少有公共部门的育种者有能力在其育种计划中使用源自NGS的标记。他们利用凝胶分析依靠&gt; 18,000个微卫星标记（SSR）进行标记辅助选择（MAS）。缺乏关于目标SNP和InDel变异基因座的知识妨碍了许多育种者对Kompetitive等位基因特异性PCR（KASP）的吸收，KASP是LGC基因组学的专有技术，可以区分变异基因座上的等位基因。 KASP是一种具有成本效益的单步基因分型技术，比SSR便宜，比用于选择程序的基因分型（GBS）或基于阵列的基因分型更灵活。在本研究之前，2015年公共领域有大米KASP标记位点，主要通过基于阵列的筛选鉴定，留下大部分水稻基因组，没有KASP覆盖。在这里，我们解决了迫切需要广泛选择适当的大米KASP检测方法，并证明NGS可以检测更多的KASP以提供全基因组覆盖。通过重新测序9个籼稻育种品系或释放品种，本研究确定了250万个变异位点。对变体进行严格过滤可产生130万个可能的KASP测定设计，其中包括92,500个潜在的功能标记。该策略在分析的籼稻杂交种中以每1kb 3.1的密度和平均每个杂交种377,178个多态KASP设计位点的密度增加了650倍的可选择的KASP标记物。这些知识可为育种者提供，并已用于提高尼泊尔公共部门育种的效率，从而能够鉴定任何区域的多态KASP或相关杂交的数量性状基因座。通过对来自一系列杂交的后代进行基因分型来验证39新的KASP以显示它们检测到分离等位基因。新的KASP取代了SSRs以帮助在这些杂交中进行标记辅助回交期间的特征选择，其中目标性状包括稻瘟病和BLB抗性基因座。此外，我们还为植物育种者提供软件，用它们自己的数据集生成KASP设计。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Mol. Breed.">
                Journal: Mol. Breed. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29563850" target="_blank">
                PMID: 29563850
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Steele KA, Quinton-Tulloch MJ, Amgai RB, Dhakal R, Khatiwada SP, Vyas D, Heine M, Witcombe JR">
                Authors: Steele KA, Quinto...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5842261" target="_blank">
                PMC5842261
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29563850">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>237. Next-Generation Sequencing for Characterizing Drug Resistant Mycobacterium tuberculosis Genes from Clinical Isolates in the Ukraine.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29563202" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29563202" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29563202">
                    <p>The Ukraine ranks among the top twenty countries with the highest number of multi-drug resistant (MDR), and extensively drug resistant (XDR) Mycobacterium tuberculosis (MTB) cases in the world. However, little is known of the genetic diversity, i.e., resistance signatures in clinical isolates from this region. We analyzed seven of most prevalent MDR/XDR antibiotic resistance conferring genes from clinical isolates (N = 75) collected from geographically diverse Ukrainian oblasts and its southern Crimean peninsula. Genomic analysis revealed 6 (8 %) were sensitive, 3 (4 %) were resistant to at least one antibiotic, 40 (53 %) were MDR, and 26 (35 %) were XDR. The majority of isolates (81 %) were of the Beijing-like lineage. This is the first study to use NGS of clinical isolates from the Ukraine to characterize mutations in genes conferring MTB drug resistance. Several isolates harbored drug resistance signatures that have not been observed in other high burden TB countries. Most notably, the absence of inhA gene promoter mutations, a diversity of mutation types in the rpoB determining region, and detection of heteroresistance provide a broader understanding of MDR/XDR from this area of the world.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29563202">
                    <p>乌克兰位居全球前20位耐多药（MDR）和广泛耐药（XDR）结核分枝杆菌（MTB）病例数最高的国家之列。然而，关于遗传多样性的知之甚少，即来自该地区的临床分离株的抗性标记。我们分析了来自地理上多样的乌克兰州及其南部克里米亚半岛的临床分离株（N = 75）中7种最流行的MDR / XDR抗生素抗性赋予基因。基因组分析显示6例（8％）敏感，3例（4％）对至少一种抗生素耐药，40例（53％）为耐多药，26例（35％）为XDR。大部分分离株（81％）属于北京人的血统。这是第一个使用来自乌克兰的临床分离株的NGS来鉴定赋予MTB耐药性的基因突变的研究。几个分离株具有在其他高负担结核病国家尚未观察到的耐药特征。最值得注意的是，inhA基因启动子突变的缺失，rpoB决定区的多样性突变类型以及异源性的检测提供了来自世界这个地区的MDR / XDR的更广泛的理解。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Clin. Microbiol.">
                Journal: J. Clin. Microbiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29563202" target="_blank">
                PMID: 29563202
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-21
            </a>

            <a class="btn btn-sm btn-danger" title="Daum LT, Konstantynovska OS, Solodiankin OS, Liashenko OO, Poteiko PI, Bolotin VI, Hrek II, Rohozhyn AV, Rodriguez JD, Fischer GW, Chambers JP, Gerilovych AP">
                Authors: Daum LT, Konstant...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29563202">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>238. Reproducibility and repeatability of six high-throughput 16S rDNA sequencing protocols for microbiota profiling.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29563060" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29563060" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29563060">
                    <p>Culture-independent molecular techniques and advances in next generation sequencing (NGS) technologies make large-scale epidemiological studies on microbiota feasible. A challenge using NGS is to obtain high reproducibility and repeatability, which is mostly attained through robust amplification. We aimed to assess the reproducibility of saliva microbiota by comparing triplicate samples. The microbiota was produced with simplified in-house 16S amplicon assays taking advantage of large number of barcodes. The assays included primers with Truseq (TS-tailed) or Nextera (NX-tailed) adapters and either with dual index or dual index plus a 6-nt internal index. All amplification protocols produced consistent microbial profiles for the same samples. Although, in our study, reproducibility was highest for the TS-tailed method. Five replicates of a single sample, prepared with the TS-tailed 1-step protocol without internal index sequenced on the HiSeq platform provided high alpha-diversity and low standard deviation (mean Shannon and Inverse Simpson diversity was 3.19 ± 0.097 and 13.56 ± 1.634 respectively). Large-scale profiling of microbiota can consistently be produced by all 16S amplicon assays. The TS-tailed-1S dual index protocol is preferred since it provides repeatable profiles on the HiSeq platform and are less labour intensive.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29563060">
                    <p>独立于文化的分子技术和下一代测序（NGS）技术的进步使得大规模的微生物群流行病学研究成为可能。使用NGS的挑战是获得高重现性和可重复性，这主要是通过稳健放大来实现的。我们旨在通过比较一式三份样品来评估唾液微生物群的再现性。利用大量条形码利用简化的内部16S扩增子分析生成微生物群。测定包括带有Truseq（TS尾）或Nextera（NX尾）衔接头的引物，以及具有双索引或双索引加上6-nt内部索引的引物。所有的扩增方案对同一样品产生一致的微生物谱。尽管在我们的研究中，TS尾加法的再现性最高。用HiSeq平台上没有内部索引序列的TS尾加1步方案制备的单个样品的5个重复样品提供了高α多样性和低标准差（平均Shannon和逆Simpson多样性分别为3.19±0.097和13.56±1.634 ）。所有16S扩增子检测均可持续产生大规模的微生物群谱分析。 TS-tailed-1S双索引协议是首选，因为它在HiSeq平台上提供可重复的配置文件，并且劳动力密集度较低。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Microbiol. Methods">
                Journal: J. Microbiol. Methods [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29563060" target="_blank">
                PMID: 29563060
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Raju SC, Lagström S, Ellonen P, de Vos WM, Eriksson JG, Weiderpass E, Rounge TB">
                Authors: Raju SC, Lagström...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29563060">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>239. Effective osimertinib treatment in a patient with discordant T790 M mutation detection between liquid biopsy and tissue biopsy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29562902" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29562902" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29562902">
                    <p>We report the successful treatment of the patient with osimertinib 80 mg/day following disease progression and a discordance in the detection of a mechanism of resistance epithelial growth factor receptor (EGFR) T790 M between liquid biopsy and tissue biopsy methods.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29562902">
                    <p>我们报告了患者在疾病进展后使用奥西莫替尼80mg /天的成功治疗，以及检测液体活组织检查方法和组织活检方法之间的抗性上皮生长因子受体（EGFR）T790M机制的不一致性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Cancer">
                Journal: BMC Cancer [IF=3.288]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29562902" target="_blank">
                PMID: 29562902
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-21
            </a>

            <a class="btn btn-sm btn-danger" title="Mambetsariev I, Vora L, Yu KW, Salgia R">
                Authors: Mambetsariev I, V...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5863467" target="_blank">
                PMC5863467
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29562902">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>240. UDiTaS™, a genome editing detection method for indels and genome rearrangements.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29562890" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29562890" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29562890">
                    <p>Understanding the diversity of repair outcomes after introducing a genomic cut is essential for realizing the therapeutic potential of genomic editing technologies. Targeted PCR amplification combined with Next Generation Sequencing (NGS) or enzymatic digestion, while broadly used in the genome editing field, has critical limitations for detecting and quantifying structural variants such as large deletions (greater than approximately 100 base pairs), inversions, and translocations.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29562890">
                    <p>在引入基因组切割后了解修复结果的多样性对于实现基因组编辑技术的治疗潜力至关重要。目标PCR扩增与下一代测序（NGS）或酶促消化相结合，虽然广泛用于基因组编辑领域，但对于检测和定量结构变体（例如大的缺失（大于约100个碱基对），倒位和易位。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=BMC Genomics">
                Journal: BMC Genomics [IF=3.729]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29562890" target="_blank">
                PMID: 29562890
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-03-21
            </a>

            <a class="btn btn-sm btn-danger" title="Giannoukos G, Ciulla DM, Marco E, Abdulkerim HS, Barrera LA, Bothmer A, Dhanapal V, Gloskowski SW, Jayaram H, Maeder ML, Skor MN, Wang T, Myer VE, Wilson CJ">
                Authors: Giannoukos G, Ciu...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5861650" target="_blank">
                PMC5861650
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29562890">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>241. [Spectrum of somatic mutations and their prognostic significance in adult patients with B cell acute lymphoblastic leukemia].</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29562441" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29562441" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29562441">
                    <p>Objective: To investigate the spectrum of gene mutations in adult patients with B-acute lymphoblastic leukemia (B-ALL), and to analyze the influences of different gene mutations on prognosis. Methods: DNA samples from 113 adult B-ALL patients who administered from June 2009 to September 2015 were collected. Target-specific next generation sequencing (NGS) approach was used to analyze the mutations of 112 genes (focused on the specific mutational hotspots) and all putative mutations were compared against multiple databases to calculate the frequency spectrum. The impact of gene mutation on the patients&#39; overall survival (OS) and recurrence free survival (RFS) was analyzed by the putative mutations through Kaplan-Meier, and Cox regression methods. Results: Of the 113 patients, 103 (92.0%) harbored at least one mutation and 29 (25.6%) harbored more than 3 genes mutation. The five most frequently mutated genes in B-ALL are SF1, FAT1, MPL, PTPN11 and NRAS. Gene mutations are different between Ph+ B-ALL and Ph- B-ALL patients. Ph- B-ALL patients with JAK-STAT signal pathway related gene mutation, such as JAK1/JAK2 mutation showed a poor prognosis compared to the patients without mutation (OS: P=0.011, 0.001; RFS: P=0.014,&lt;0.001). Patients with PTPN11 mutation showed better survival than those without mutation, but the difference was not statistically significant (P value &gt; 0.05). Besides, in Ph+ B-ALL patients whose epigenetic modifications related signaling pathway genes were affected, they had a worse prognosis (OS: P=0.038; RFS: P=0.047). Conclusion: Gene mutations are common in adult ALL patients, a variety of signaling pathways are involved. The frequency and spectrum are varied in different types of B-ALL. JAK family gene mutation usually indicates poor prognosis. The co-occurrence of somatic mutations in adult B-ALL patients indicate the genetic complex and instability of adult B-ALL patients.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29562441">
                    <p>目的：探讨成人急性B淋巴细胞白血病（B-ALL）患者基因突变谱，分析不同基因突变对预后的影响。方法：收集2009年6月至2015年9月施行的113名成人B-ALL患者的DNA样本。使用靶特异性下一代测序（NGS）方法分析112个基因的突变（集中在特定的突变热点），并将所有推定的突变与多个数据库进行比较以计算频谱。基因突变对患者总生存（OS）和无复发生存（RFS）的影响通过Kaplan-Meier和Cox回归方法通过推定的突变进行分析。结果：113例患者中，103例（92.0％）存在至少一处突变，29例（25.6％）存在3处以上基因突变。 B-ALL中五个最频繁突变的基因是SF1，FAT1，MPL，PTPN11和NRAS。 Ph + B-ALL和Ph-B-ALL患者之间的基因突变不同。具有JAK-STAT信号通路相关基因突变例如JAK1 / JAK2突变的Ph-B-ALL患者预后较差（OS：P = 0.011,0.001; RFS：P = 0.014，&lt;0.001） 。 PTPN11突变患者的生存率优于无突变者，但差异无统计学意义（P值&gt; 0.05）。此外，Ph + B-ALL患者的表观遗传修饰相关信号通路基因受到影响，其预后较差（OS：P = 0.038; RFS：P = 0.047）。结论：基因突变在成人ALL患者中很常见，涉及多种信号通路。频率和频谱在不同类型的B-ALL中有所不同。 JAK家族基因突变通常表明预后不良。成人B-ALL患者体细胞突变的同时发生表明成人B-ALL患者的遗传复杂性和不稳定性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Zhonghua Xue Ye Xue Za Zhi">
                Journal: Zhonghua Xue Ye Xue Za Zhi [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29562441" target="_blank">
                PMID: 29562441
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb-14
            </a>

            <a class="btn btn-sm btn-danger" title="Feng J, Gong XY, Jia YJ, Liu KQ, Li Y, Dong XB, Fang QY, Ru K, Li QH, Wang HJ, Zhao XL, Jia YN, Song Y, Tian Z, Wang M, Tang KJ, Wang JX, Mi YC">
                Authors: Feng J, Gong XY, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29562441">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>242. Use of the Ion PGM and the GeneReader NGS Systems in Daily Routine Practice for Advanced Lung Adenocarcinoma Patients: A Practical Point of View Reporting a Comparative Study and Assessment of 90 Patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29561830" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29561830" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29561830">
                    <p>Background: With the integration of various targeted therapies into the clinical management of patients with advanced lung adenocarcinoma, next-generation sequencing (NGS) has become the technology of choice and has led to an increase in simultaneously interrogated genes. However, the broader adoption of NGS for routine clinical practice is still hampered by sophisticated workflows, complex bioinformatics analysis and medical interpretation. Therefore, the performance of the novel QIAGEN GeneReader NGS system was compared to an in-house ISO-15189 certified Ion PGM NGS platform. Methods: Clinical samples from 90 patients (60 Retrospectively and 30 Prospectively) with lung adenocarcinoma were sequenced with both systems. Mutations were analyzed and EGFR, KRAS, BRAF, NRAS, ALK, PIK3CA and ERBB2 genes were compared and sampling time and suitability for clinical testing were assessed. Results: Both sequencing systems showed perfect concordance for the overlapping genes. Correlation of allele frequency was r² = 0.93 for the retrospective patients and r² = 0.81 for the prospective patients. Hands-on time and total run time were shorter using the PGM system, while the GeneReader platform provided good traceability and up-to-date interpretation of the results. Conclusion: We demonstrated the suitability of the GeneReader NGS system in routine practice in a clinical pathology laboratory setting.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29561830">
                    <p>背景：随着各种靶向治疗与晚期肺腺癌患者的临床管理相结合，新一代测序（NGS）已成为首选技术，并导致同时询问基因的增加。然而，日常临床实践中更广泛地采用NGS仍然受到复杂的工作流程，复杂的生物信息学分析和医学解释的阻碍。因此，新型QIAGEN GeneReader NGS系统的性能与内部通过ISO-15189认证的Ion PGM NGS平台进行了比较。方法：采用两种系统对90例患者（60例回顾性和30例前瞻性）肺腺癌患者的临床样本进行测序。分析突变，并比较EGFR，KRAS，BRAF，NRAS，ALK，PIK3CA和ERBB2基因，并评估采样时间和临床测试的适用性。结果：两种测序系统都显示出重叠基因的完美一致性。回顾性患者的等位基因频率相关性r2 = 0.93，未来患者相关性r2 = 0.81。使用PGM系统的实验时间和总运行时间更短，而GeneReader平台提供了良好的可追溯性和最新的结果解释。结论：我们证明了GeneReader NGS系统在临床病理学实验室环境中的常规实践中的适用性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancers (Basel)">
                Journal: Cancers (Basel) [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29561830" target="_blank">
                PMID: 29561830
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-21
            </a>

            <a class="btn btn-sm btn-danger" title="Heeke S, Hofman V, Long-Mira E, Lespinet V, Lalvée S, Bordone O, Ribeyre C, Tanga V, Benzaquen J, Leroy S, Cohen C, Mouroux J, Marquette CH, Ilié M, Hofman P">
                Authors: Heeke S, Hofman V...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29561830">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>243. Genetic cardiomyopathies.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29561320" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29561320" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29561320">
                    <p>To describe recent advancements in cardiovascular genetics made possible by leveraging next-generation sequencing (NGS), and to provide a framework for practical applications of genetic testing for hypertrophic (HCM), dilated (DCM), and arrhythmogenic right ventricular cardiomyopathies (ARVC).</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29561320">
                    <p>通过利用新一代测序（NGS）描述近期心血管遗传学进展，并为肥厚型（HCM），扩张型（DCM）和致心律失常性右心室心肌病（ARVC）基因检测的实际应用提供框架。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Curr. Opin. Cardiol.">
                Journal: Curr. Opin. Cardiol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29561320" target="_blank">
                PMID: 29561320
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Wilcox JE, Hershberger RE">
                Authors: Wilcox JE, Hershb...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29561320">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>244. Detection of circulating tumor DNA in patients with osteosarcoma.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29560102" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29560102" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29560102">
                    <p>Identification and quantification of somatic alterations in plasma-derived, circulating tumor DNA (ctDNA) is gaining traction as a non-invasive and cost effective method of disease monitoring in cancer patients, particularly to evaluate response to treatment and monitor for disease recurrence. To our knowledge, genetic analysis of ctDNA in osteosarcoma has not yet been studied. To determine whether somatic alterations can be detected in ctDNA and perhaps applied to patient management in this disease, we collected germline, tumor, and serial plasma samples from pediatric, adolescent, and young adult patients with osteosarcoma and used targeted Next Generation Sequencing (NGS) to identify somatic single nucleotide variants (SNV), insertions and deletions (INDELS), and structural variants (SV) in 7 genes commonly mutated in osteosarcoma. We demonstrate that patient-specific somatic alterations identified through comparison of tumor-germline pairs can be detected and quantified in cell-free DNA of osteosarcoma patients.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29560102">
                    <p>在血浆来源的循环肿瘤DNA（ctDNA）中体细胞改变的鉴定和定量作为癌症患者的非侵入性和成本有效的疾病监测方法，特别是评估对治疗的反应和监测疾病复发，正在得到越来越多的关注。据我们所知，骨肉瘤中ctDNA的遗传分析尚未研究。为了确定是否可以在ctDNA中检测到体细胞改变，并且可能应用于本病的患者管理，我们收集了来自儿科，青少年和年轻成人骨肉瘤患者的生殖系统，肿瘤和系列血浆样本，并使用有针对性的新一代测序（NGS）以鉴定在骨肉瘤中通常发生突变的7个基因中的体细胞单核苷酸变体（SNV），插入和缺失（INDELS）和结构变体（SV）。我们证明，通过比较肿瘤生殖细胞对鉴定的患者特异性体细胞改变可以在骨肉瘤患者的无细胞DNA中检测和定量。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29560102" target="_blank">
                PMID: 29560102
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb-27
            </a>

            <a class="btn btn-sm btn-danger" title="Barris DM, Weiner SB, Dubin RA, Fremed M, Zhang X, Piperdi S, Zhang W, Maqbool S, Gill J, Roth M, Hoang B, Geller D, Gorlick R, Weiser DA">
                Authors: Barris DM, Weiner...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5849166" target="_blank">
                PMC5849166
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29560102">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>245. Anaplastic lymphoma kinase mutation (ALK F1174C) in small cell carcinoma of the prostate and molecular response to alectinib.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29559559" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29559559" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29559559">
                    <p>Small cell carcinoma of the prostate (SCCP) is an aggressive disease that can arise de novo or by transdifferentiation from prostate adenocarcinoma. Alterations in anaplastic lymphoma kinase (ALK) gene are involved in neuroblastoma, lung cancer, and other malignancies but its role in SCCP has not been documented. We describe a patient with refractory de novo SCCP with ALK F1174C activating mutation who obtained clinical benefit from treatment with ALK inhibitor.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29559559">
                    <p>前列腺小细胞癌（SCCP）是一种侵袭性疾病，可从头发或从前列腺腺癌转分化产生。间变性淋巴瘤激酶（ALK）基因的改变参与神经母细胞瘤，肺癌和其他恶性肿瘤，但其在SCCP中的作用尚未见报道。我们描述了一位患有ALK F1174C激活突变的难治性新发SCCP患者，其通过ALK抑制剂治疗获得了临床益处。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Cancer Res.">
                Journal: Clin. Cancer Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29559559" target="_blank">
                PMID: 29559559
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-20
            </a>

            <a class="btn btn-sm btn-danger" title="Carneiro BA, Pamarthy S, Shah AN, Sagar V, Unno K, Han H, Yang XJ, Costa RB, Nagy RJ, Lanman RB, Kuzel TM, Ross JS, Gay L, Elvin JA, Ali SM, Cristofanilli M, Chae YK, Giles FJ, Abdulkadir SA">
                Authors: Carneiro BA, Pama...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29559559">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>246. Overhydrated stomatocytosis associated with a complex RHAG genotype including a novel de novo mutation.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29559519" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29559519" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29559519">
                    <p>Overhydrated stomatocytosis is a rare autosomal dominant disorder known to cause variably severe haemolytic anaemia due to heterozygous mutations in the RHAG gene. We report a 26-year-old man with recurring jaundice, splenohepatomegaly and mild chronic haemolytic anaemia with significant stomatocytosis. Extensive haemolytic work-up including flow cytometry for eosin-5&#39;-maleimide and CD47 expression levels was carried out. Targeted resequencing revealed two probably causative heterozygous mutations in RHAG (Leu336Ser and Ile149Met) and one heterozygous mutation in ANK1 (Glu1046Lys). RHAG involvement was confirmed by decreased RhAG macrocomplex component indicated by the reduced CD47 expression on erythrocytes. In silico analysis concordantly flagged RHAG:Leu336Ser and ANK1:Glu1046Lys as likely deleterious mutation, whereas RHAG:Ile149Met was reported as likely neutral by PROVEAN. Family screening by Sanger sequencing revealed RHAG:Leu336Ser in a mother and ANK1:Glu1046Lys in a father who were both asymptomatic, excluding them as causative dominant events, thus establishing RHAG:Ile149Met, novel de novo mutation as probably causative. This case illustrates the importance of family screening in interpreting next-generation sequencing (NGS) data, as in silico analysis alone can be misleading. Erudite generation of diagnostic possibilities based on a thorough baseline clinical and laboratory work-up remains as important as ever, even as NGS brings about a paradigm shift in the diagnostic work-up of rare haemolytic anaemias.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29559519">
                    <p>水合型口腔细胞增多症是一种罕见的常染色体显性疾病，由于RHAG基因中的杂合突变而引起不同程度的严重溶血性贫血。我们报告一名26岁男性反复发作性黄疸，脾脏肿大和轻度慢性溶血性贫血伴显着口腔细胞增多症。进行广泛的溶血处理，包括用于伊红-5&#39;-马来酰亚胺和CD47表达水平的流式细胞术。靶向重测序揭示了RHAG（Leu336Ser和Ile149Met）中两个可能的致病杂合突变和ANK1（Glu1046Lys）中的一个杂合突变。 RHAG受累通过红细胞上减少的CD47表达指示的减少的RhAG大复合物组分来证实。在计算机分析中，一致标记RHAG：Leu336Ser和ANK1：Glu1046Lys为可能的有害突变，而据报告PROVEAN可能中性RHAG：Ile149Met。通过Sanger测序进行的家庭筛查揭示了母亲中的RHAG：Leu336Ser和父亲中的ANK1：Glu1046Lys，两者都是无症状的，排除它们作为致病性显性事件，由此建立RHAG：Ile149Met，可能是致病的新型新生突变。这个案例说明了家庭筛查在解释下一代测序（NGS）数据中的重要性，因为单纯的计算机分析可能会产生误导。即使NGS在罕见的溶血性贫血的诊断工作中引起了范式转变，基于彻底的基线临床和实验室检查的诊断可能性仍然与以往一样重要。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Clin. Pathol.">
                Journal: J. Clin. Pathol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29559519" target="_blank">
                PMID: 29559519
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-20
            </a>

            <a class="btn btn-sm btn-danger" title="Jamwal M, Aggarwal A, Sachdeva MUS, Sharma P, Malhotra P, Maitra A, Das R">
                Authors: Jamwal M, Aggarwa...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29559519">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>247. Studying microbial functionality within the gut ecosystem by systems biology.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29556373" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29556373" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29556373">
                    <p>Humans are not autonomous entities. We are all living in a complex environment, interacting not only with our peers, but as true holobionts; we are also very much in interaction with our coexisting microbial ecosystems living on and especially within us, in the intestine. Intestinal microorganisms, often collectively referred to as intestinal microbiota, contribute significantly to our daily energy uptake by breaking down complex carbohydrates into simple sugars, which are fermented to short-chain fatty acids and subsequently absorbed by human cells. They also have an impact on our immune system, by suppressing or enhancing the growth of malevolent and beneficial microbes. Our lifestyle can have a large influence on this ecosystem. What and how much we consume can tip the ecological balance in the intestine. A &quot;western diet&quot; containing mainly processed food will have a different effect on our health than a balanced diet fortified with pre- and probiotics. In recent years, new technologies have emerged, which made a more detailed understanding of microbial communities and ecosystems feasible. This includes progress in the sequencing of PCR-amplified phylogenetic marker genes as well as the collective microbial metagenome and metatranscriptome, allowing us to determine with an increasing level of detail, which microbial species are in the microbiota, understand what these microorganisms do and how they respond to changes in lifestyle and diet. These new technologies also include the use of synthetic and in vitro systems, which allow us to study the impact of substrates and addition of specific microbes to microbial communities at a high level of detail, and enable us to gather quantitative data for modelling purposes. Here, we will review the current state of microbiome research, summarizing the computational methodologies in this area and highlighting possible outcomes for personalized nutrition and medicine.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29556373">
                    <p>人类不是自主实体。我们都生活在一个复杂的环境中，不仅与同龄人互动，而且与真正的同伴交往;我们也与我们共存的微生物生态系统相互作用，这些微生物生态系统生活在肠道内，尤其是在我们内部。肠道微生物（通常统称为肠道微生物群）通过将复杂的碳水化合物分解成单糖，将其发酵成短链脂肪酸并随后被人类细胞吸收，对我们的日常能量摄取作出显着贡献。它们也通过抑制或促进恶性微生物和有益微生物的生长而影响我们的免疫系统。我们的生活方式会对这个生态系统产生巨大影响。我们消耗多少和多少可以降低肠道的生态平衡。以加工食品为主的“西方饮食”对于我们的健康有着不同的影响，而不是强化了预先和益生菌的均衡饮食。近年来，出现了新技术，使得微生物群落和生态系统更为详细的了解成为可能。这包括PCR扩增系统发育标记基因测序的进展，以及集体微生物宏基因组和metatranscriptome，使我们能够确定细菌的不断增加的水平，哪些微生物物种在微生物群中，了解这些微生物的作用以及它们是如何应对生活方式和饮食的变化。这些新技术还包括使用合成和体外系统，这些系统使我们能够高度详细地研究底物的影响和微生物群落中特定微生物的添加，并使我们能够为建模目的收集定量数据。在这里，我们将回顾微生物组研究的当前状况，总结这一领域的计算方法并强调个性化营养和医学的可能结果。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genes Nutr">
                Journal: Genes Nutr [IF=2.797]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29556373" target="_blank">
                PMID: 29556373
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Hornung B, Martins Dos Santos VAP, Smidt H, Schaap PJ">
                Authors: Hornung B, Martin...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5840735" target="_blank">
                PMC5840735
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29556373">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>248. High-density genetic map construction and comparative genome analysis in asparagus bean.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29555986" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29555986" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29555986">
                    <p>Genetic maps are a prerequisite for quantitative trait locus (QTL) analysis, marker-assisted selection (MAS), fine gene mapping, and assembly of genome sequences. So far, several asparagus bean linkage maps have been established using various kinds of molecular markers. However, these maps were all constructed by gel- or array-based markers. No maps based on sequencing method have been reported. In this study, an NGS-based strategy, SLAF-seq, was applied to create a high-density genetic map for asparagus bean. Through SLAF library construction and Illumina sequencing of two parents and 100 F2 individuals, a total of 55,437 polymorphic SLAF markers were developed and mined for SNP markers. The map consisted of 5,225 SNP markers in 11 LGs, spanning a total distance of 1,850.81 cM, with an average distance between markers of 0.35 cM. Comparative genome analysis with four other legume species, soybean, common bean, mung bean and adzuki bean showed that asparagus bean is genetically more related to adzuki bean. The results will provide a foundation for future genomic research, such as QTL fine mapping, comparative mapping in pulses, and offer support for assembling asparagus bean genome sequence.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29555986">
                    <p>遗传图谱是数量性状基因座（QTL）分析，标记辅助选择（MAS），精细基因定位和基因组序列组装的先决条件。到目前为止，已经使用各种分子标记建立了几个芦笋豆链接图。然而，这些地图全部是由基于凝胶或阵列的标记构建的。没有报道基于测序方法的地图。在这项研究中，基于NGS的策略SLAF-seq被应用于创建芦笋豆的高密度遗传图谱。通过SLAF文库构建和两个亲本和100个F2个体的Illumina测序，共开发了55,437个多态性SLAF标记并开采了SNP标记。该图由11个LGs中的5,225个SNP标记组成，跨越总距离1,850.81cM，标记之间的平均距离为0.35cM。与其他四种豆科植物，大豆，普通豆，绿豆和小豆的比较基因组分析显示，芦笋豆与小豆的基因关系更为密切。这些结果将为未来的基因组研究提供基础，如QTL精细定位，脉冲比较测绘，并为组装芦笋豆基因组序列提供支持。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29555986" target="_blank">
                PMID: 29555986
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-19
            </a>

            <a class="btn btn-sm btn-danger" title="Huang H, Tan H, Xu D, Tang Y, Niu Y, Lai Y, Tie M, Li H">
                Authors: Huang H, Tan H, X...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5859152" target="_blank">
                PMC5859152
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29555986">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>249. Clinical and genetic characteristics of 251 consecutive patients with macular and cone/cone-rod dystrophy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29555955" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29555955" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29555955">
                    <p>Macular and cone/cone-rod dystrophies (MD/CCRD) demonstrate a broad genetic and phenotypic heterogeneity, with retinal alterations solely or predominantly involving the central retina. Targeted next-generation sequencing (NGS) is an efficient diagnostic tool for identifying mutations in patient with retinitis pigmentosa, which shows similar genetic heterogeneity. To detect the genetic causes of disease in patients with MD/CCRD, we implemented a two-tier procedure consisting of Sanger sequencing and targeted NGS including genes associated with clinically overlapping conditions. Disease-causing mutations were identified in 74% of 251 consecutive MD/CCRD patients (33% of the variants were novel). Mutations in ABCA4, PRPH2 and BEST1 accounted for 57% of disease cases. Further mutations were identified in CDHR1, GUCY2D, PROM1, CRX, GUCA1A, CERKL, MT-TL1, KIF11, RP1L1, MERTK, RDH5, CDH3, C1QTNF5, CRB1, JAG1, DRAM2, POC1B, NPHP1 and RPGR. We provide detailed illustrations of rare phenotypes, including autofluorescence and optical coherence tomography imaging. Targeted NGS also identified six potential novel genotype-phenotype correlations for FAM161A, INPP5E, MERTK, FBLN5, SEMA4A and IMPDH1. Clinical reassessment of genetically unsolved patients revealed subgroups with similar retinal phenotype, indicating a common molecular disease cause in each subgroup.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29555955">
                    <p>黄斑和锥形/锥形杆营养不良（MD / CCRD）表现出广泛的遗传和表型异质性，其中视网膜改变完全或主要涉及中央视网膜。有针对性的下一代测序（NGS）是识别患有视网膜色素变性的患者突变的有效诊断工具，其显示出相似的遗传异质性。为了检测MD / CCRD患者的疾病遗传原因，我们实施了一个由桑格测序和靶向NGS组成的双层方案，包括与临床重叠病症相关的基因。在连续251例MD / CCRD患者中有74％发现了致病突变（33％的变异体是新型的）。 ABCA4，PRPH2和BEST1突变占疾病病例的57％。在CDHR1，GUCY2D，PROM1，CRX，GUCA1A，CERKL，MT-TL1，KIF11，RP1L1，MERTK，RDH5，CDH3，C1QTNF5，CRB1，JAG1，DRAM2，POC1B，NPHP1和RPGR中鉴定出进一步的突变。我们提供了罕见表型的详细说明，包括自发荧光和光学相干断层扫描成像。靶向的NGS还鉴定了FAM161A，INPP5E，MERTK，FBLN5，SEMA4A和IMPDH1的六种潜在的新型基因型 - 表型相关性。临床重新评估遗传性未解决的患者显示具有相似视网膜表型的亚组，表明每个亚组中共同的分子病原因。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Sci Rep">
                Journal: Sci Rep [IF=4.259]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29555955" target="_blank">
                PMID: 29555955
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-19
            </a>

            <a class="btn btn-sm btn-danger" title="Birtel J, Eisenberger T, Gliem M, Müller PL, Herrmann P, Betz C, Zahnleiter D, Neuhaus C, Lenzner S, Holz FG, Mangold E, Bolz HJ, Charbel Issa P">
                Authors: Birtel J, Eisenbe...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5859282" target="_blank">
                PMC5859282
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29555955">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>250. Approach to pancytopenia: Diagnostic algorithm for clinical hematologists.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29555368" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29555368" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29555368">
                    <p>Pancytopenia is a relatively common phenomenon encountered in clinical practice. The evaluation of a patient with pancytopenia requires a comprehensive approach and identifying the underlying cause can be challenging given the wide range of etiologies including drugs, autoimmune conditions, malignancies, infections, hemophagocytosis, and inheritable conditions. Recent advances in molecular hematology which include genomic profiling and next-generation sequencing have helped gain major insights into various hematological conditions and can guide diagnosing specific diseases in a shorter time at lower costs. However the approach to manage patients with pancytopenia in the current era of genomics is not well defined in the literature and is widely variable in practice. Herein, we conducted a systematic review to help devise an algorithm and management approach for pancytopenia, which serves as a general consultative approach.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29555368">
                    <p>全血细胞减少症是在临床实践中遇到的相对普遍的现象。对全血细胞减少症患者的评估需要一种全面的方法，鉴于各种病因包括药物，自身免疫病症，恶性肿瘤，感染，噬血细胞增多症和遗传病，鉴定其根本原因可能具有挑战性。包括基因组谱分析和新一代测序在内的分子血液学研究的最新进展有助于深入了解各种血液学情况，并可以在较短时间内以较低成本指导诊断特定疾病。然而，在目前的基因组学时代，治疗全血细胞减少症的方法在文献中没有很好的定义，并且在实践中广泛地变化。在此，我们进行了系统评价，以帮助制定全血细胞减少症的算法和管理方法，作为一般咨询方法。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Blood Rev.">
                Journal: Blood Rev. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29555368" target="_blank">
                PMID: 29555368
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-05
            </a>

            <a class="btn btn-sm btn-danger" title="Gnanaraj J, Parnes A, Francis CW, Go RS, Takemoto CM, Hashmi SK">
                Authors: Gnanaraj J, Parne...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29555368">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>251. [Multiple lung carcinoma: Primary or intrapulmonary metastasis?]</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29555057" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29555057" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29555057">
                    <p>Multiple lung carcinomas are 5 to 11,5% of lung carcinomas. The distinction between primary lung carcinomas from carcinomas with intrapulmonary metastasis is essential for optimal patient management. The histopathological analysis is very useful but it has to be completed by genotypic assessment using molecular biology (NGS). Molecular biology can also identify genetic alterations with therapeutic implications. We present the case of a patient with a history of surgery for multiple lung carcinomas diagnosed from 2013 to 2017.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29555057">
                    <p>多发性肺癌占肺癌的5〜11.5％。原发性肺癌与肺内转移之间的区别对于最佳的患者管理至关重要。组织病理学分析非常有用，但必须通过使用分子生物学（NGS）进行基因型评估来完成。分子生物学也可以鉴定具有治疗意义的遗传改变。我们介绍2013年至2017年诊断的多发肺癌手术史患者的病例。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ann Pathol">
                Journal: Ann Pathol [IF=0.615]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29555057" target="_blank">
                PMID: 29555057
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-16
            </a>

            <a class="btn btn-sm btn-danger" title="Ravella L, Barritault M, Bringuier PP, Chalabreysse L, Thivolet-Bejui F, Maury JM, Duruisseaux M, Brevet M">
                Authors: Ravella L, Barrit...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29555057">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>252. Microbiome Profiles of Commercial Broilers Through Evisceration and Immersion Chilling During Poultry Slaughter and the Identification of Potential Indicator Microorganisms.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29552001" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29552001" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29552001">
                    <p>Commercial poultry abattoirs were evaluated to determine the efficacy of the multi-hurdle antimicrobial strategy employed to reduce the microbial load present on incoming broilers from the farm. As next generation sequencing (NGS) has been recently employed to characterize the poultry production system, this study utilized 16S High throughput sequencing (HTS) and quantitative plating data to profile the microbiota of chicken carcasses and determine the efficacy of the multi-hurdle antimicrobial system. Aerobic plate count (APC) and Enterobacteriaceae (EB) microbial counts were quantified from whole bird carcass rinsates (WBCR). The remaining rinsates underwent microbiome analysis using 16S rRNA gene fragments on an Illumina MiSeq and were analyzed by Quantitative Insights into Microbial Ecology (QIIME). The key stages of processing were determined to be at rehang, pre-chill, and post-chill as per the Salmonella Reduction Regulation (75 Fed. Reg. 27288-27294). The APC microbial data from rehang, pre-chill, and post-chill were mean log 4.63 CFU/mL, 3.21 CFU/mL, and 0.89 CFU/mL and EB counts were mean log 2.99 CFU/mL, 1.95 CFU/mL, and 0.35 CFU/mL. NGS of WBCR identified 222 Operational Taxonomic Units&#39; (OTU&#39;s) of which only 23 OTU&#39;s or 10% of the population was recovered post-chill. Microbiome data suggested a high relative abundance of Pseudomonas at post-chill. Additionally, Pseudomonas, Enterobacteriaceae, and Weeksellaceae Chryseobacterium have been identified as potential indicator organisms having been isolated from all processing abattoirs and sampling locations. This study provides insight into the microbiota of commercial broilers during poultry processing.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29552001">
                    <p>对商业家禽屠宰场进行评估，以确定用于减少来自农场的进入肉鸡的微生物负荷的多跨栏抗菌策略的功效。由于下一代测序（NGS）最近被用于表征家禽生产系统，本研究利用16S高通量测序（HTS）和定量电镀数据来描绘鸡尸体的微生物群并确定多重跨栏抗微生物系统。从全禽尸体淋巴液（WBCR）定量厌氧平板计数（APC）和肠杆菌科（EB）微生物计数。剩余的漂洗液在Illumina MiSeq上使用16S rRNA基因片段进行微生物组分析，并通过Quantitative Insights分析入微生物生态学（QIIME）进行分析。根据沙门氏菌减量法规（75 Fed。Reg。27288-27294），处理的关键阶段被确定为处理，预冷和后冷处理。 rehang，pre-chill和post-chill的APC微生物数据是平均log 4.63 CFU / mL，3.21 CFU / mL和0.89 CFU / mL，EB计数是平均log 2.99 CFU / mL，1.95 CFU / mL和0.35 CFU / mL。 WBCR的NGS确定了222个业务分类单元（OTU），其中只有23个OTU或10％的人口在寒冷后回收。微生物数据显示冷杉后期假单胞菌的相对丰度较高。此外，假单胞菌，肠杆菌科和水丝虫科已被鉴定为已从所有加工屠宰场和采样位置分离出的潜在指示生物体。这项研究提供了对家禽加工过程中商业肉鸡微生物群的了解。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Microbiol">
                Journal: Front Microbiol [IF=4.076]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29552001" target="_blank">
                PMID: 29552001
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Handley JA, Park SH, Kim SA, Ricke SC">
                Authors: Handley JA, Park ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5841210" target="_blank">
                PMC5841210
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29552001">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>253. Messenger RNA biomarker signatures for forensic body fluid identification revealed by targeted RNA sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29549744" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29549744" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29549744">
                    <p>The recovery of a DNA profile from the perpetrator or victim in criminal investigations can provide valuable &#39;source level&#39; information for investigators. However, a DNA profile does not reveal the circumstances by which biological material was transferred. Some contextual information can be obtained by a determination of the tissue or fluid source of origin of the biological material as it is potentially indicative of some behavioral activity on behalf of the individual that resulted in its transfer from the body. Here, we sought to improve upon established RNA based methods for body fluid identification by developing a targeted multiplexed next generation mRNA sequencing assay comprising a panel of approximately equal sized gene amplicons. The multiplexed biomarker panel includes several highly specific gene targets with the necessary specificity to definitively identify most forensically relevant biological fluids and tissues (blood, semen, saliva, vaginal secretions, menstrual blood and skin). In developing the biomarker panel we evaluated 66 gene targets, with a progressive iteration of testing target combinations that exhibited optimal sensitivity and specificity using a training set of forensically relevant body fluid samples. The current assay comprises 33 targets: 6 blood, 6 semen, 6 saliva, 4 vaginal secretions, 5 menstrual blood and 6 skin markers. We demonstrate the sensitivity and specificity of the assay and the ability to identify body fluids in single source and admixed stains. A 16 sample blind test was carried out by one lab with samples provided by the other participating lab. The blinded lab correctly identified the body fluids present in 15 of the samples with the major component identified in the 16th. Various classification methods are being investigated to permit inference of the body fluid/tissue in dried physiological stains. These include the percentage of reads in a sample that are due to each of the 6 tissues/body fluids tested and inter-sample differential gene expression revealed by agglomerative hierarchical clustering.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29549744">
                    <p>犯罪调查中从罪犯或受害者身上恢复DNA概况可为调查人员提供有价值的“来源级别”信息。然而，DNA档案没有揭示生物材料转移的情况。一些背景信息可以通过确定生物材料的组织或流体来源来获得，因为它可能代表个体导致其从身体转移的一些行为活动。在这里，我们试图通过开发靶向多路复用的下一代mRNA测序测定来改进建立的基于RNA的体液鉴定方法，所述测定包括一组大小基本相同的基因扩增子。多路复用生物标志物组包括几个高度特异性的基因靶，具有确定识别大多数法医相关生物体液和组织（血液，精液，唾液，阴道分泌物，经血和皮肤）的必要特异性。在开发生物标志物专家组时，我们评估了66个基因靶点，并使用训练集的法医相关体液样品对测试靶点组合进行渐进式迭代，结果表现出最佳灵敏度和特异性。目前的测定包括33个目标：6血，6精液，6唾液，4阴道分泌物，5经血和6皮肤标记。我们证明了该检测方法的灵敏度和特异性以及在单一来源和混合污渍中识别体液的能力。由一个实验室对另一参与实验室提供的样品进行16次样品盲测。盲法实验室正确识别出15个样品中存在的体液，其中主要成分在16日鉴定。正在研究各种分类方法以允许干燥生理污渍中的体液/组织的推断。这些包括样本中读数的百分比，这些百分比是由于所测试的6种组织/体液中的每一种造成的，并且样本间差异基因表达通过凝聚等级聚类显示。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Forensic Sci Int Genet">
                Journal: Forensic Sci Int Genet [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29549744" target="_blank">
                PMID: 29549744
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Hanson E, Ingold S, Haas C, Ballantyne J">
                Authors: Hanson E, Ingold ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29549744">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>254. Development and evaluation of a culture-free microbiota profiling platform (MYcrobiota) for clinical diagnostics.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29549470" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29549470" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29549470">
                    <p>Microbiota profiling has the potential to greatly impact on routine clinical diagnostics by detecting DNA derived from live, fastidious, and dead bacterial cells present within clinical samples. Such results could potentially be used to benefit patients by influencing antibiotic prescribing practices or to generate new classical-based diagnostic methods, e.g., culture or PCR. However, technical flaws in 16S rRNA gene next-generation sequencing (NGS) protocols, together with the requirement for access to bioinformatics, currently hinder the introduction of microbiota analysis into clinical diagnostics. Here, we report on the development and evaluation of an &quot;end-to-end&quot; microbiota profiling platform (MYcrobiota), which combines our previously validated micelle PCR/NGS (micPCR/NGS) methodology with an easy-to-use, dedicated bioinformatics pipeline. The newly designed bioinformatics pipeline processes micPCR/NGS data automatically and summarizes the results in interactive, but simple web reports. In order to explore the utility of MYcrobiota in clinical diagnostics, 47 clinical samples (40 &quot;damaged skin&quot; samples and 7 synovial fluids) were investigated using routine bacterial culture as comparator. MYcrobiota confirmed the presence of bacterial DNA in 37/37 culture-positive samples and detected bacterial taxa in 2/10 culture-negative samples. Moreover, 36/38 potentially relevant aerobic bacterial taxa and 3/3 mixtures of anaerobic bacteria were identified using culture and MYcrobiota, with the sensitivity and specificity being 95%. Interestingly, the majority of the 448 bacterial taxa identified using MYcrobiota were not identified using culture, which could potentially have an impact on clinical decision-making. Taken together, the development of MYcrobiota is a promising step towards the introduction of microbiota analysis into clinical diagnostic laboratories.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29549470">
                    <p>通过检测临床样品中存在的活的，苛刻的和死亡的细菌细胞的DNA，微生物群谱分析有可能大大影响常规临床诊断。通过影响抗生素处方实践或产生新的基于经典的诊断方法，例如培养或PCR，这些结果可能有助于患者受益。然而，16S rRNA基因新一代测序（NGS）方案中的技术缺陷以及获取生物信息学的要求目前阻碍了微生物群分析进入临床诊断。在这里，我们报告了“端对端”微生物群谱分析平台（MYcrobiota）的开发和评估，该平台结合了我们以前验证的胶束PCR / NGS（micPCR / NGS）方法和易于使用的专用生物信息学管道。新设计的生物信息学流程自动处理micPCR / NGS数据，并在交互式但简单的网络报告中总结结果。为了探索MYcrobiota在临床诊断中的应用，使用常规细菌培养物作为比较物研究了47个临床样品（40个“损伤的皮肤”样品和7个滑液）。 MYcrobiota证实37/37培养阳性样品中存在细菌DNA，并在2/10培养阴性样品中检测到细菌分类群。此外，36/38潜在相关好氧细菌分类群和3/3混合厌氧菌被确定使用文化和MYcrobiota，与敏感性和特异性为95％。有趣的是，使用MYcrobiota鉴定的448种细菌类群中，大部分未使用培养物进行鉴定，这可能会对临床决策产生影响。综合起来看，MYcrobiota的发展是将微生物群分析引入临床诊断实验室的有希望的一步。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Clin. Microbiol. Infect. Dis.">
                Journal: Eur. J. Clin. Microbiol. Infect. Dis. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29549470" target="_blank">
                PMID: 29549470
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-16
            </a>

            <a class="btn btn-sm btn-danger" title="Boers SA, Hiltemann SD, Stubbs AP, Jansen R, Hays JP">
                Authors: Boers SA, Hiltema...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29549470">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>255. Bacterial next generation sequencing (NGS) made easy.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29548687" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29548687" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29548687">
                    <p>.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29548687">
                    <p>.</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin. Microbiol. Infect.">
                Journal: Clin. Microbiol. Infect. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29548687" target="_blank">
                PMID: 29548687
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Tassios PT, Moran-Gilad J">
                Authors: Tassios PT, Moran...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29548687">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>256. Evaluation of preimplantation genetic testing for chromosomal structural rearrangement by a commonly used next generation sequencing workflow.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29547808" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29547808" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29547808">
                    <p>To evaluate the applicability of a commonly used next generation sequencing workflow in detecting unbalanced meiotic segregation products for reciprocal translocation and inversion carriers.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29547808">
                    <p>评估常用下一代测序工作流程在检测互补易位和倒位载体的不平衡减数分裂产物方面的适用性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Obstet. Gynecol. Reprod. Biol.">
                Journal: Eur. J. Obstet. Gynecol. Reprod. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29547808" target="_blank">
                PMID: 29547808
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Chow JFC, Yeung WSB, Lee VCY, Lau EYL, Ng EHY">
                Authors: Chow JFC, Yeung W...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29547808">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>257. Hydrothermal Synthesis of a New Kind of N-Doped Graphene Gel-like Hybrid As an Enhanced ORR Electrocatalyst.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29547254" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29547254" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29547254">
                    <p>In this work, g-C3N4@GO gel-like hybrid is obtained by assembling intentionally exfoliated g-C3N4 sheets on graphene oxide (GO) sheets under a hydrothermal condition. A specific N-doping process is first designed by heating the g-C3N4@GO interlaced hybrid in vacuum to form nitrogen-doped graphene nanosheets (NGS) with high level of pyridinic-N (56.0%) and edge-rich defect structure. The prepared NGS exhibited a great electrocatalysis for oxygen reduction reaction (ORR) in terms of the activity, durability, methanol tolerance, and the reaction kinetics. And the excellent electrocatalytic performance stems from the effective N-doped sites that the nitrogen atom is successfully doped at the defective edges of graphene, and the annealing temperature can play significant role of the doping pattern and location of N. The research provides a new insight into the enhancement of electrocatalysis for ORR based on nonmetal carbons by using the novel N-doping method.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29547254">
                    <p>在这项工作中，通过在水热条件下将有意剥离的g-C3N4片材装配在氧化石墨烯（GO）片材上获得g-C3N4 @ GO凝胶状杂化体。首先，通过在真空中加热g-C3N4 @ GO交错杂化物以形成具有高含量吡啶-N（56.0％）和富含边缘缺陷结构的氮掺杂石墨烯纳米片（NGS），来设计特定的N掺杂工艺。制备的NGS在活性，耐久性，甲醇耐受性和反应动力学方面表现出很好的氧还原反应（ORR）电催化作用。其优异的电催化性能源于氮原子在石墨烯缺陷边缘成功掺杂的有效氮掺杂位点，退火温度对N的掺杂方式和位置起着重要作用。本研究提供了新的见解通过使用新的N掺杂方法来增强基于非金属碳的ORR的电催化。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=ACS Appl Mater Interfaces">
                Journal: ACS Appl Mater Interfaces [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29547254" target="_blank">
                PMID: 29547254
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-04
            </a>

            <a class="btn btn-sm btn-danger" title="Xiang Q, Liu Y, Zou X, Hu B, Qiang Y, Yu D, Yin W, Chen C">
                Authors: Xiang Q, Liu Y, Z...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29547254">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>258. Significant photoluminescence enhancement in WS2 monolayers through Na2S treatment.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29546899" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29546899" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29546899">
                    <p>Tungsten disulfide monolayers have attracted extensive attention in nanoelectronics and optoelectronics applications due to their remarkable electronic and optical properties. High-quality WS2 monolayers with a scalable size and uniform thickness can be synthesized by a chemical vapor deposition method (CVD). However, they commonly contain intrinsic structural defects and different populations of charge carriers, which are responsible for different contributions of excitons, trions, and biexcitons to their photoluminescence (PL) emission. Here, we adopt sodium sulphide (Na2S) solution to chemically treat CVD-grown WS2 monolayers by a simple immersing method. The results show that WS2 monolayers have a significantly enhanced PL emission by a factor of 25-fold and an obvious red-shift of the PL wavelength, resulting from the different excitonic states induced by effective n-type doping after Na2S treatment. This work provides a simple but promising chemical doping route to significantly improve the optical properties of WS2 monolayers and paves the way for the realization of practical WS2 monolayer based optoelectronic applications.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29546899">
                    <p>由于其显着的电子和光学特性，二硫化钨单层在纳米电子学和光电子学应用中引起了广泛关注。可通过化学气相沉积法（CVD）合成具有可缩放尺寸和均匀厚度的高质量WS2单层。然而，它们通常包含固有的结构缺陷和不同种类的电荷载体，它们负责激子，三价和双光子对其光致发光（PL）发射的不同贡献。在这里，我们采用硫化钠（Na2S）溶液，通过简单的浸泡法化学处理CVD生长的WS2单层。结果显示WS2单层具有显着增强的PL发射25倍以及PL波长的明显红移，这是由Na2S处理后由有效n型掺杂引起的不同激子态引起的。这项工作提供了一种简单但有前途的化学掺杂方法，可显着改善WS2单分子层的光学性质，并为实现基于WS2单层的光电子应用铺平道路。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Nanoscale">
                Journal: Nanoscale [IF=7.367]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29546899" target="_blank">
                PMID: 29546899
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-29
            </a>

            <a class="btn btn-sm btn-danger" title="Yao H, Liu L, Wang Z, Li H, Chen L, Pam ME, Chen W, Yang HY, Zhang W, Shi Y">
                Authors: Yao H, Liu L, Wan...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29546899">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>259. Molecular defects in BRAF wild-type ameloblastomas and craniopharyngiomas-differences in mutation profiles in epithelial-derived oropharyngeal neoplasms.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29546640" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29546640" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29546640">
                    <p>The aim of this study was to evaluate the mutation profile of BRAF wild-type craniopharyngiomas and ameloblastomas. Pre-screening by immunohistochemistry and pyrosequencing for identifying BRAF wild-type tumors was performed on archived specimens of ameloblastic tumors (n = 20) and craniopharyngiomas (n = 62). Subsequently, 19 BRAF wild-type tumors (nine ameloblastic tumors and ten craniopharyngiomas) were analyzed further using next-generation sequencing (NGS) targeting hot spot mutations of 22 cancer-related genes. Thereby, we found craniopharyngiomas mainly CTNNB1 mutated (8/10), including two FGFR3/CTNNB1-double mutated tumors. Ameloblastic tumors were often FGFR2 mutated (4/9; including one FGFR2/TP53/PTEN-triple mutated case) and rarely CTNNB1/TP53-double mutated (1/9) and KRAS-mutated (1/9). In the remaining samples, no mutation could be detected in the 22 genes under investigation. In conclusion, mutation profiles of BRAF wild-type craniopharyngiomas and ameloblastomas share mutations of FGFR genes and have additional mutations with potential for targeted therapy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29546640">
                    <p>本研究的目的是评估BRAF野生型颅咽管瘤和成釉细胞瘤的突变特征。在成釉细胞瘤（n = 20）和颅咽管瘤（n = 62）的标本上进行免疫组织化学和焦磷酸测序以鉴定BRAF野生型肿瘤。随后，使用针对22个癌症相关基因的热点突变的新一代测序（NGS）进一步分析了19个BRAF野生型肿瘤（9个成釉细胞瘤和10个颅咽管瘤）。因此，我们发现颅咽管瘤主要CTNNB1突变（8/10），包括两个FGFR3 / CTNNB1双重突变的肿瘤。成釉细胞瘤常常是FGFR2突变（4/9;包括一个FGFR2 / TP53 / PTEN三重突变的病例）并且很少CTNNB1 / TP53-双突变（1/9）和KRAS突变（1/9）。在剩下的样本中，在所调查的22个基因中未检测到突变。总之，BRAF野生型颅咽管瘤和成釉细胞瘤的突变谱共享FGFR基因的突变，并具有可能用于靶向治疗的额外突变。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Virchows Arch.">
                Journal: Virchows Arch. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29546640" target="_blank">
                PMID: 29546640
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-15
            </a>

            <a class="btn btn-sm btn-danger" title="Bartels S, Adisa A, Aladelusi T, Lemound J, Stucki-Koch A, Hussein S, Kreipe H, Hartmann C, Lehmann U, Hussein K">
                Authors: Bartels S, Adisa ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29546640">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>260. DNA template sequence control of bacterial RNA polymerase escape from the promoter.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29546317" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29546317" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29546317">
                    <p>Promoter escape involves breaking of the favourable contacts between RNA polymerase (RNAP) and the promoter to allow transition to an elongation complex. The sequence of DNA template that is transcribed during promoter escape (ITS; Initially Transcribed Sequence) can affect promoter escape by mechanisms that are not yet fully understood. We employed a highly parallel strategy utilizing Next Generation Sequencing (NGS) to collect data on escape properties of thousands of ITS variants. We show that ITS controls promoter escape through a combination of position-dependent effects (most prominently, sequence-directed RNAP pausing), and position-independent effects derived from sequence encoded physical properties of the template (for example, RNA/DNA duplex stability). ITS often functions as an independent unit affecting escape in the same manner regardless of the promoter from which transcription initiates. However, in some cases, a strong dependence of ITS effects on promoter context was observed suggesting that promoters may have &#39;allosteric&#39; abilities to modulate ITS effects. Large effects of ITS on promoter output and the observed interplay between promoter sequence and ITS effects suggests that the definition of bacterial promoter should include ITS sequence.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29546317">
                    <p>启动子逃逸包括破坏RNA聚合酶（RNAP）和启动子之间的有利接触，以允许过渡到延伸复合物。在启动子逃逸期间转录的DNA模板序列（ITS;最初转录序列）可以通过尚未完全理解的机制影响启动子逃逸。我们采用了高度并行的策略，利用下一代测序（NGS）来收集数千个ITS变体的逃逸特性数据。我们显示ITS控制启动子通过位置依赖效应（最显着的是，序列指导的RNAP暂停）以及源自模板的序列编码物理性质（例如RNA / DNA双链体稳定性）的位置无关效应的组合逃脱， 。无论转录起始的启动子如何，ITS通常都以相同的方式作为影响逃逸的独立单位发挥功能。然而，在某些情况下，观察到ITS对启动子背景的强烈依赖性表明启动子可能具有调节ITS效应的“变构”能力。 ITS对启动子输出的大量影响以及观察到的启动子序列与ITS效应之间的相互作用表明细菌启动子的定义应包括ITS序列。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Nucleic Acids Res.">
                Journal: Nucleic Acids Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29546317" target="_blank">
                PMID: 29546317
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-13
            </a>

            <a class="btn btn-sm btn-danger" title="Heyduk E, Heyduk T">
                Authors: Heyduk E, Heyduk T
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29546317">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>261. Next-Generation Sequencing Approaches in Genome-Wide Discovery of Single Nucleotide Polymorphism Markers Associated with Pungency and Disease Resistance in Pepper.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29546063" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29546063" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29546063">
                    <p>Pepper is an economically important horticultural plant that has been widely used for its pungency and spicy taste in worldwide cuisines. Therefore, the domestication of pepper has been carried out since antiquity. Owing to meet the growing demand for pepper with high quality, organoleptic property, nutraceutical contents, and disease tolerance, genomics assisted breeding techniques can be incorporated to develop novel pepper varieties with desired traits. The application of next-generation sequencing (NGS) approaches has reformed the plant breeding technology especially in the area of molecular marker assisted breeding. The availability of genomic information aids in the deeper understanding of several molecular mechanisms behind the vital physiological processes. In addition, the NGS methods facilitate the genome-wide discovery of DNA based markers linked to key genes involved in important biological phenomenon. Among the molecular markers, single nucleotide polymorphism (SNP) indulges various benefits in comparison with other existing DNA based markers. The present review concentrates on the impact of NGS approaches in the discovery of useful SNP markers associated with pungency and disease resistance in pepper. The information provided in the current endeavor can be utilized for the betterment of pepper breeding in future.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29546063">
                    <p>胡椒是一种经济上重要的园艺植物，在世界各地的美食中被广泛用于刺激性和辛辣味。因此，自古以来就进行了辣椒的驯化。由于满足了辣椒对高品质，感官特性，营养成分含量和疾病耐受性日益增长的需求，可以引入基因组辅助育种技术来开发具有所需特性的新型辣椒品种。新一代测序（NGS）方法的应用改变了植物育种技术，特别是在分子标记辅助育种领域。基因组信息的可用性有助于深入理解重要生理过程背后的几种分子机制。另外，NGS方法促进了与涉及重要生物学现象的关键基因相关的基于DNA的标记物的全基因组发现。在这些分子标记中，与其他现有的基于DNA的标记相比，单核苷酸多态性（SNP）具有各种益处。本综述集中于NGS方法在发现与辣椒辣味和抗病性相关的有用SNP标记方面的影响。目前努力提供的信息可用于未来改善辣椒育种。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Biomed Res Int">
                Journal: Biomed Res Int [IF=2.476]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29546063" target="_blank">
                PMID: 29546063
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Manivannan A, Kim JH, Yang EY, Ahn YK, Lee ES, Choi S, Kim DS">
                Authors: Manivannan A, Kim...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5818978" target="_blank">
                PMC5818978
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29546063">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>262. Cost-effectiveness of KRAS, EGFR and ALK testing for decision making in advanced nonsmall cell lung carcinoma: the French IFCT-PREDICT.amm study.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29545318" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29545318" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29545318">
                    <p>ALK rearrangement and EGFR/KRAS mutations constitute the primary biomarkers tested to provide targeted or nontargeted therapies in advanced nonsmall cell lung cancer (NSCLC) patients. Our objective was to assess the cost-effectiveness of biomarker testing for NSCLC.Between 2013 and 2014, 843 treatment-naive patients were prospectively recruited at 19 French hospitals into a longitudinal observational cohort study. Two testing strategies were compared, i.e. with &quot;at least one biomarker status known&quot; and &quot;at least KRAS status known&quot;, in addition to &quot;no biomarker testing&quot; as the reference strategy. The Kaplan-Meier approach was employed to assess restricted mean survival time. Direct medical costs incurred by hospitals were estimated with regard to treatment, inpatient care and biomarker testing.Compared with &quot;no biomarker testing&quot;, the &quot;at least one biomarker status known&quot; strategy yielded an incremental cost-effectiveness ratio of EUR13 230 per life-year saved, which decreased to EUR7444 per life-year saved with the &quot;at least KRAS status known&quot; testing strategy. In sensitivity analyses, biomarker testing strategies were less costly and more effective in 41% of iterations.In summary, molecular testing prior to treatment initiation proves to be cost-effective in advanced NSCLC management and may assist decision makers in defining conditions for further implementation of these innovations in general practice.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29545318">
                    <p>ALK重排和EGFR / KRAS突变构成了主要的生物标志物，用于在晚期非小细胞肺癌（NSCLC）患者中提供靶向或非靶向治疗。我们的目标是评估非小细胞肺癌生物标志物检测的成本效益。2013年至2014年，共有843名未接受治疗的患者在法国19所医院进行前瞻性招募，进行纵向观察队列研究。除了“无生物标志物测试”作为参考策略之外，还比较了两种测试策略，即“已知至少一种生物标志物状态”和“至少KRAS状态已知”。采用Kaplan-Meier方法评估受限的平均生存时间。根据治疗，住院治疗和生物标志物检测评估医院的直接医疗费用。与“无生物标志物检测”相比，“至少有一种生物标志物状态已知”策略产生的增量成本效益比为13 230欧元/节省了一年，随着“至少已知KRAS状态”测试策略的节省，每生命年减少到7444欧元。在敏感性分析中，生物标志物检测策略在41％的迭代中成本较低并且更有效。总之，开始治疗之前的分子检测证明在晚期NSCLC管理中具有成本效益，并且可以帮助决策者确定进一步实施这些创新在一般实践中。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. Respir. J.">
                Journal: Eur. Respir. J. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29545318" target="_blank">
                PMID: 29545318
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar
            </a>

            <a class="btn btn-sm btn-danger" title="Loubière S, Drezet A, Beau-Faller M, Moro-Sibilot D, Friard S, Wislez M, Blons H, Daniel C, Westeel V, Madroszyk A, Léna H, Merle P, Mazières J, Zalcman G, Lacave R, Antoine M, Morin F, Missy P, Barlesi F, Auquier P, Cadranel J">
                Authors: Loubière S, Dreze...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29545318">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>263. Comparison of whole-genome bisulfite sequencing library preparation strategies identifies sources of biases affecting DNA methylation data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29544553" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29544553" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29544553">
                    <p>Whole-genome bisulfite sequencing (WGBS) is becoming an increasingly accessible technique, used widely for both fundamental and disease-oriented research. Library preparation methods benefit from a variety of available kits, polymerases and bisulfite conversion protocols. Although some steps in the procedure, such as PCR amplification, are known to introduce biases, a systematic evaluation of biases in WGBS strategies is missing.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29544553">
                    <p>全基因组亚硫酸氢盐测序（WGBS）正在成为一种越来越容易获得的技术，广泛用于基础和疾病研究。文库制备方法受益于各种可用的试剂盒，聚合酶和亚硫酸氢盐转化方案。尽管该过程中的某些步骤（如PCR扩增）已知会引入偏倚，但缺少对WGBS策略偏倚的系统评估。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genome Biol.">
                Journal: Genome Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29544553" target="_blank">
                PMID: 29544553
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-15
            </a>

            <a class="btn btn-sm btn-danger" title="Olova N, Krueger F, Andrews S, Oxley D, Berrens RV, Branco MR, Reik W">
                Authors: Olova N, Krueger ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856372" target="_blank">
                PMC5856372
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29544553">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>264. From somatic variants towards precision oncology: Evidence-driven reporting of treatment options in molecular tumor boards.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29544535" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29544535" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29544535">
                    <p>A comprehensive understanding of cancer has been furthered with technological improvements and decreasing costs of next-generation sequencing (NGS). However, the complexity of interpreting genomic data is hindering the implementation of high-throughput technologies in the clinical context: increasing evidence on gene-drug interactions complicates the task of assigning clinical significance to genomic variants.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29544535">
                    <p>随着技术改进和降低下一代测序（NGS）的成本，对癌症的全面了解得到进一步加强。然而，解释基因组数据的复杂性阻碍了临床背景下高通量技术的实施：越来越多的基因 - 药物相互作用证据使得将基因组变异的临床意义赋予的任务变得复杂化。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Genome Med">
                Journal: Genome Med [IF=7.071]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29544535" target="_blank">
                PMID: 29544535
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-15
            </a>

            <a class="btn btn-sm btn-danger" title="Perera-Bel J, Hutter B, Heining C, Bleckmann A, Fröhlich M, Fröhling S, Glimm H, Brors B, Beißbarth T">
                Authors: Perera-Bel J, Hut...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856211" target="_blank">
                PMC5856211
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29544535">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>265. Frequency of hereditary neuropathy with liability to pressure palsies (HNPP) due to 17p11.2 deletion in a Korean newborn population.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29544507" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29544507" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29544507">
                    <p>Hereditary neuropathy with liability to pressure palsies (HNPP) is an autosomal dominant disorder mainly due to a deletion of chromosome 17p11.2 including PMP22 (PMP22 Del HNPP). The prevalence of HNPP is estimated to be 0.84 to 16 per 100,000, but could be underestimated because of the mild symptoms of HNPP. In this study, we estimated the prevalence of PMP22 Del HNPP in a Korean newborn population who underwent next-generation sequencing (NGS)-based copy number variation (CNV) analysis. Of the 11,885 newborns tested by NGS-based CNV analysis, 17p11.2 deletions were found in seven samples. The prevalence of PMP22 Del HNPP was estimated to be 58.9 per 100,000 (95% confidence interval (CI), 25.8-116.5) or 1 in 1698 (95% CI, 1/909-1/5000). Our data suggest that PMP22 Del HNPP might not be uncommon at least in the Korean population.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29544507">
                    <p>有压力麻痹（HNPP）的遗传性神经病是一种常染色体显性疾病，主要是由于包括PMP22（PMP22 Del HNPP）在内的染色体17p11.2缺失。 HNPP的患病率估计为每10万人0.84至16人，但由于HNPP症状轻微可能被低估。在这项研究中，我们估计了韩国新生儿人群中接受基于新一代测序（NGS）的拷贝数变异（CNV）分析的PMP22 Del HNPP的患病率。在通过基于NGS的CNV分析测试的11,885名新生儿中，在七个样品中发现17p11.2缺失。据估计，PMP22 Del HNPP的患病率为每10万人中有58.9（95％置信区间（CI），25.8-116.5）或1698中1人（95％CI，1 / 909-1 / 5000）。我们的数据表明，PMP22 Del HNPP可能并不罕见，至少在韩国人群中是罕见的。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Orphanet J Rare Dis">
                Journal: Orphanet J Rare Dis [IF=3.507]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29544507" target="_blank">
                PMID: 29544507
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-15
            </a>

            <a class="btn btn-sm btn-danger" title="Park JE, Noh SJ, Oh M, Cho DY, Kim SY, Ki CS">
                Authors: Park JE, Noh SJ, ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5856277" target="_blank">
                PMC5856277
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29544507">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>266. Tagmentation on Microbeads: Restore Long-Range DNA Sequence Information Using Next Generation Sequencing with Library Prepared by Surface-Immobilized Transposomes.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29543431" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29543431" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29543431">
                    <p>The next generation sequencing (NGS) technologies have been rapidly evolved and applied to various research fields, but they often suffer from losing long-range information due to short library size and read length. Here, we develop a simple, cost-efficient, and versatile NGS library preparation method, called tagmentation on microbeads (TOM). This method is capable of recovering long-range information through tagmentation mediated by microbead-immobilized transposomes. Using transposomes with DNA barcodes to identically label adjacent sequences during tagmentation, we can restore inter-read connection of each fragment from original DNA molecule by fragment-barcode linkage after sequencing. In our proof-of-principle experiment, more than 4.5% of the reads are linked with their adjacent reads, and the longest linkage is over 1112 bp. We demonstrate TOM with eight barcodes, but the number of barcodes can be scaled up by an ultrahigh complexity construction. We also show this method has low amplification bias and effectively fits the applications to identify copy number variations.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29543431">
                    <p>新一代测序（NGS）技术已经迅速发展并应用于各种研究领域，但由于库的大小和阅读时间长，它们经常会因丢失远距离信息而遭受损失。在这里，我们开发了一种简单，经济高效的多功能NGS文库制备方法，称为微珠标记（TOM）。该方法能够通过微珠固定化转座体介导的标记来恢复远距离信息。使用具有DNA条形码的转座体在标签化期间相同地标记相邻序列，我们可以在测序后通过片段 - 条形码连接恢复来自原始DNA分子的每个片段的读取间连接。在我们的原理验证实验中，超过4.5％的读数与其相邻读数连锁，最长的连锁超过1112bp。我们用八个条形码演示TOM，但是条形码的数量可以通过超高复杂度结构来扩大。我们还展示了这种方法具有低放大偏差，并且有效地适合应用程序以识别拷贝数变化。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=ACS Appl Mater Interfaces">
                Journal: ACS Appl Mater Interfaces [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29543431" target="_blank">
                PMID: 29543431
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr-11
            </a>

            <a class="btn btn-sm btn-danger" title="Chen H, Yao J, Fu Y, Pang Y, Wang J, Huang Y">
                Authors: Chen H, Yao J, Fu...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29543431">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>267. Response of the intestinal mucosal barrier of carp (Cyprinus carpio) to a bacterial challenge by Aeromonas hydrophila intubation after feeding with β-1,3/1,6-glucan.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29542825" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29542825" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29542825">
                    <p>The effect of dietary β-glucan on the bacterial community in the gut of common carp (Cyprinus carpio) was examined after oral application of Aeromonas hydrophila. Carp received either feed supplemented with 1% MacroGard® , a β-1,3/1,6-glucan, or a β-glucan-free diet. Fourteen days after feeding, half of the carp from each group were intubated with 109 colony-forming units (CFU) of a pathogenic strain of A. hydrophila. Gut samples were taken 12 hr to 7 days after application and analysed using microbiological and molecular biological techniques (NGS, RT-PCR-DGGE). The reaction of the mucosa and the microbiota to an A. hydrophila intubation differed in carp fed with β-glucan compared to carp from the control group. In β-glucan fed carp, the total bacterial amount was lower but the number of bacterial species was higher. Bacterial composition was different for carp from both treatment groups. The number of mucin filled goblet cells was reduced in carp fed the β-glucan diet. Mucus was obviously released from the goblet cells and was probably washed out of the gut together with high numbers of bacteria. This might be protective against pathogenic bacteria and, therefore, feeding with β-glucan may provide protection against infections of the gut in carp.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29542825">
                    <p>在口服应用嗜水气单胞菌后，研究了膳食β-葡聚糖对鲤鱼（Cyprinus carpio）肠道细菌群落的影响。鲤鱼接受补充1％MacroGard®，β-1,3 / 1,6-葡聚糖或不含β-葡聚糖的饮食的饲料。饲喂14天后，每组鲤鱼的一半用109菌落形成单位（CFU）的嗜水气单胞菌致病菌株插管。应用后12小时至7天取出肠样品，并使用微生物学和分子生物学技术（NGS，RT-PCR-DGGE）进行分析。与来自对照组的鲤鱼相比，在用β-葡聚糖喂养的鲤鱼中，粘膜和微生物群对嗜水气单胞菌气管插管的反应不同。在β-葡聚糖喂养的鲤鱼中，细菌总量较低，但细菌种类较多。来自两个处理组的鲤鱼​​的细菌组成不同。喂食β-葡聚糖饮食的鲤鱼中粘蛋白填充的杯状细胞数量减少。粘液明显从杯状细胞中释放出来，很可能与大量细菌一起从肠道中被冲出。这可能对致病细菌有保护作用，因此，用β-葡聚糖喂养可以防止鲤鱼肠道感染。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Fish Dis.">
                Journal: J. Fish Dis. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29542825" target="_blank">
                PMID: 29542825
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-15
            </a>

            <a class="btn btn-sm btn-danger" title="Jung-Schroers V, Adamek M, Harris S, Syakuri H, Jung A, Irnazarow I, Steinhagen D">
                Authors: Jung-Schroers V, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29542825">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>268. Quantifying tumor-infiltrating immune cells from transcriptomics data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29541787" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29541787" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29541787">
                    <p>By exerting pro- and anti-tumorigenic actions, tumor-infiltrating immune cells can profoundly influence tumor progression, as well as the success of anti-cancer therapies. Therefore, the quantification of tumor-infiltrating immune cells holds the promise to unveil the multi-faceted role of the immune system in human cancers and its involvement in tumor escape mechanisms and response to therapy. Tumor-infiltrating immune cells can be quantified from RNA sequencing data of human tumors using bioinformatics approaches. In this review, we describe state-of-the-art computational methods for the quantification of immune cells from transcriptomics data and discuss the open challenges that must be addressed to accurately quantify immune infiltrates from RNA sequencing data of human bulk tumors.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29541787">
                    <p>通过发挥促癌和抗肿瘤发生作用，肿瘤浸润免疫细胞可以深刻地影响肿瘤进展，以及抗癌治疗的成功。因此，肿瘤浸润性免疫细胞的量化有望揭示免疫系统在人类癌症中的多方面作用及其参与肿瘤逃逸机制和对治疗的反应。可以使用生物信息学方法从肿瘤的RNA测序数据定量肿瘤浸润免疫细胞。在这篇综述中，我们描述了用于从转录组学数据中量化免疫细胞的最先进的计算方法，并且讨论了必须解决的开放式挑战，以精确定量来自人体肿瘤的RNA测序数据的免疫渗透。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Immunol. Immunother.">
                Journal: Cancer Immunol. Immunother. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29541787" target="_blank">
                PMID: 29541787
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-14
            </a>

            <a class="btn btn-sm btn-danger" title="Finotello F, Trajanoski Z">
                Authors: Finotello F, Traj...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29541787">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>269. Targeted sequencing reveals distinct pathogenic variants in Chinese patients with lung adenocarcinoma brain metastases.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29541220" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29541220" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29541220">
                    <p>Lung cancer is the most common type of malignancy to metastasize to the brain, with the median survival time of patients being 6-11 months. In the present study, the aim was to compare the actionable gene mutation profiles of primary lung adenocarcinoma (LC) samples and LC brain metastasis (LCBM) samples through targeted sequencing. Next generation sequencing (NGS) of 13 formalin-fixed, paraffin-embedded LC samples and 15 LCBM samples was performed using a customized OncoAim™ cancer panel and OncoAim™ RNA fusion panel on the MiSeq platform. The OncoAim™ cancer panel pipeline and OncoAim™ RNA fusion panel pipeline were used for bioinformatic analysis. Together, 43 variants were observed in 7 genes from the 28 cancer samples. The mutated genes of LCBM were tumor protein (TP)53, epidermal growth factor receptor (EGFR), catenin β1, phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit α, mothers against decapentaplegic homolog 4, Kirsten rat sarcoma viral oncogene homolog (KRAS) and proto-oncogene B-Raf, which were exhibited in 10/15 (66.7%), 6/15 (40.0%), 3/15 (20.0%), 2/15 (13.3%), 2/15 (13.3%), 1/15 (6.7%) and 1/15 (6.7%) of samples, respectively. The mutated genes of LC were TP53, EGFR and KRAS, which were exhibited in 11/13 (84.6%), 5/13 (38.5%) and 2/13 (18.2%) of samples, respectively. echinoderm microtubule associated protein like 4-anaplastic lymphoma kinase rearrangements were present in 1 LCBM sample. For 2 LC samples and 1 LCBM sample, no genetic alterations were observed. The NGS data also revealed a novel 4-codon deletion of TP53 (p.V166_H169del) and a novel TP53 splice site mutation (7577157-63del TACTCAG). Further potentially actionable mutations were detected in LCBM, indicating a high degree of genetic heterogeneity between the LC and LCBM samples that were analyzed. The present study demonstrated that NGS provides an improved approach for the discovery of potentially actionable mutations and the understanding of the mechanisms underlying tumor progression and evolution.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29541220">
                    <p>肺癌是转移至大脑的最常见的恶性肿瘤类型，患者的中位生存时间为6-11个月。在本研究中，目的是通过靶向测序比较原发性肺腺癌（LC）样品和LC脑转移（LCBM）样品的可行性基因突变概况。在MiSeq平台上使用定制的OncoAim TM癌症组和OncoAim TM RNA融合组进行定量分析（13例福尔马林固定，石蜡包埋的LC样品和15个LCBM样品）。使用OncoAim™癌症组管道和OncoAim™RNA融合板管道进行生物信息学分析。在来自28个癌症样品的7个基因中观察到43种变体。 LCBM的突变基因是肿瘤蛋白（TP）53，表皮生长因子受体（EGFR），连环蛋白β1，磷脂酰肌醇-4,5-二磷酸3-激酶催化亚单位α，抗十倍子配体同系物4的母体，Kirsten大鼠肉瘤病毒致癌基因同系物（66.7％），6/15（40.0％），3/15（20.0％），2/15（13.3％），2/15 （13.3％），1/15（6.7％）和1/15（6.7％）。 LC突变基因分别为TP53，EGFR和KRAS，分别在11/13（84.6％），5/13（38.5％）和2/13（18.2％）样品中显示。棘皮动物微管相关蛋白如4-间变性淋巴瘤激酶重排存在于1个LCBM样品中。对于2个LC样品和1个LCBM样品，没有观察到基因改变。 NGS数据还揭示了TP53（p.V166_H169del）和新的TP53剪接位点突变（7577157-63del TACTCAG）的新密码子4-密码子缺失。在LCBM中检测到进一步潜在的可行动突变，表明LC和LCBM样品之间的高度遗传异质性被分析。目前的研究表明，NGS提供了一种改进的方法，用于发现潜在的可行动突变和对肿瘤进展和进化基础机制的理解。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncol Lett">
                Journal: Oncol Lett [IF=1.390]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29541220" target="_blank">
                PMID: 29541220
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Ma Y, Chen K, Yang Z, Guan M">
                Authors: Ma Y, Chen K, Yan...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5835849" target="_blank">
                PMC5835849
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29541220">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>270. Successful preimplantation genetic diagnosis by targeted next-generation sequencing on an ion torrent personal genome machine platform.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29541197" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29541197" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29541197">
                    <p>Hearing loss may place a heavy burden on the patient and patient&#39;s family. Given the high incidence of hearing loss among newborns and the huge cost of treatment and care (including cochlear implantation), prenatal diagnosis is strongly recommended. Termination of the fetus may be considered as an extreme outcome to the discovery of a potential deaf fetus, and therefore preimplantation genetic diagnosis has become an important option for avoiding the birth of affected children without facing the risk of abortion following prenatal diagnosis. In one case, a couple had a 7-year-old daughter affected by non-syndromic sensorineural hearing loss. The affected fetus carried a causative compound heterozygous mutation c.919-2 A&gt;G (IVS7-2 A&gt;G) and c.1707+5 G&gt;A (IVS15+5 G&gt;A) of the solute carrier family 26 member 4 gene inherited from maternal and paternal sides, respectively. The present study applied multiple displacement amplification for whole genome amplification of biopsied trophectoderm cells and next-generation sequencing (NGS)-based single nucleotide polymorphism haplotyping on an Ion Torrent Personal Genome Machine. One unaffected embryo was transferred in a frozen-thawed embryo transfer cycle and the patient was impregnated. To conclude, to the best of our knowledge, this may be the first report of NGS-based preimplantation genetic diagnosis (PGD) for non-syndromic hearing loss caused by a compound heterozygous mutation using an Ion Torrent Personal Genome Machine. NGS provides unprecedented high-throughput, highly parallel and base-pair resolution data for genetic analysis. The method meets the requirements of medium-sized diagnostics laboratories. With decreased costs compared with previous techniques (such as Sanger sequencing), this technique may have potential widespread clinical application in PGD of other types of monogenic disease.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29541197">
                    <p>听力损失可能给病人和病人的家庭带来沉重的负担。鉴于新生儿听力损失的高发生率以及治疗和护理（包括人工耳蜗植入）的巨大费用，强烈建议进行产前诊断。终止胎儿可能被认为是发现潜在的聋胎儿的极端后果，因此植入前基因诊断已成为避免受影响儿童出生而不面临产前诊断后流产风险的重要选择。在一个案例中，一对夫妇有一个7岁的女儿受非综合征型感音神经性听力损失的影响。受影响的胎儿携带溶质运载体家族26成员4的致病复合杂合突变c.919-2A&gt; G（IVS7-2A&gt; G）和c.1707 + 5G&gt; A（IVS15 + 5G&gt; A）基因分别从母系和父系遗传。本研究应用多重置换扩增技术对活检的滋养外胚层细胞进行全基因组扩增，并在Ion Torrent Personal Genome Machine上应用基于新一代测序（NGS）的单核苷酸多态性单倍型分型。一个未受影响的胚胎在冷冻 - 解冻的胚胎转移周期中转移并且患者被浸渍。总而言之，据我们所知，这可能是使用Ion Torrent个人基因组机器人针对由复杂杂合突变引起的非综合征性听力损失的基于NGS的植入前基因诊断（PGD）的首次报告。 NGS为遗传分析提供前所未有的高通量，高度并行和碱基对解析数据。该方法符合中型诊断实验室的要求。与以前的技术（如Sanger测序）相比，通过降低成本，该技术可能在其他类型的单基因疾病的PGD中具有潜在的广泛临床应用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncol Lett">
                Journal: Oncol Lett [IF=1.390]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29541197" target="_blank">
                PMID: 29541197
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Hao Y, Chen D, Zhang Z, Zhou P, Cao Y, Wei Z, Xu X, Chen B, Zou W, Lv M, Ji D, He X">
                Authors: Hao Y, Chen D, Zh...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5835955" target="_blank">
                PMC5835955
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29541197">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>271. Nimbus: A design-driven analyses suite for amplicon based NGS data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29538618" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29538618" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29538618">
                    <p>PCR-based DNA enrichment followed by massively parallel sequencing is a straightforward and cost effective method to sequence genes up to high depth. The full potential of amplicon based sequencing assays is currently not achieved as analysis methods do not take into account the source amplicons of the detected variants. Tracking the source amplicons has the potential to identify systematic biases, enhance variant calling and improve the designs of future assays.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29538618">
                    <p>基于PCR的DNA富集，然后进行大规模平行测序，是一种将基因排序到高深度的直接且经济有效的方法。由于分析方法未考虑检测到的变体的来源扩增子，目前尚未实现基于扩增子的测序分析的全部潜力。追踪源扩增子有可能识别系统性偏见，增强变异调用并改进未来化验的设计。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Bioinformatics">
                Journal: Bioinformatics [IF=7.307]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29538618" target="_blank">
                PMID: 29538618
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-10
            </a>

            <a class="btn btn-sm btn-danger" title="Brouwer RWW, van den Hout MCGN, Kockx CEM, Brosens E, Eussen B, de Klein A, Sleutels F, van IJcken WFJ">
                Authors: Brouwer RWW, van ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29538618">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>272. Investigation of MKRN3 mutation in patients with familial central precocious puberty.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29537379" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29537379" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29537379">
                    <p>Little is known about the genetic cause of idiopathic central precocious puberty (CPP). The aim of this clinical study was to determine the rate of MKRN3 mutation in cases of familial idiopathic central precocious puberty.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29537379">
                    <p>对特发性中枢性性早熟（CPP）的遗传病因知之甚少。该临床研究的目的是确定家族性特发性中枢性性早熟患者中MKRN3突变的发生率。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J Clin Res Pediatr Endocrinol">
                Journal: J Clin Res Pediatr Endocrinol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29537379" target="_blank">
                PMID: 29537379
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb-28
            </a>

            <a class="btn btn-sm btn-danger" title="Aycan Z, Savaş-Erdeve Ş, Çetinkaya S, Kurnaz E, Keskin M, Muratoğlu Şahin N, Bayramoğlu E, Ceylaner G">
                Authors: Aycan Z, Savaş-Er...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29537379">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>273. Clinical application of targeted next-generation sequencing on fetuses with congenital heart defects.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29536580" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29536580" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29536580">
                    <p>This study aimed to determine the diagnostic yield of targeted next-generation sequencing (NGS) in prenatal diagnosis of congenital heart defects (CHDs) and for investigating the possible genetic etiology of prenatal CHD cases.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29536580">
                    <p>本研究旨在确定产前诊断先天性心脏缺陷（CHDs）以及调查产前CHD病例可能的遗传病因的目标新一代测序（NGS）的诊断率。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Ultrasound Obstet Gynecol">
                Journal: Ultrasound Obstet Gynecol [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29536580" target="_blank">
                PMID: 29536580
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-13
            </a>

            <a class="btn btn-sm btn-danger" title="Hu P, Qiao F, Wang Y, Meng L, Ji X, Luo C, Xu T, Zhou R, Zhang J, Yu B, Wang L, Wang T, Pan Q, Ma D, Liang D, Xu Z">
                Authors: Hu P, Qiao F, Wan...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29536580">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>274. Bioinformatics Analysis for Cell-Free Tumor DNA Sequencing Data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29536438" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29536438" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29536438">
                    <p>As a major biomarker of liquid biopsy, cell-free tumor DNA (ctDNA), which can be extracted from blood, urine, or other circulating liquids, is able to provide comprehensive genetic information of tumor and better overcome the tumor heterogeneity problem comparing to tissue biopsy. Developed in recent years, next-generation sequencing (NGS) is a widely used technology for analyzing ctDNA. Although the technologies of processing ctDNA samples are mature, the task to detect low mutated allele frequency (MAF) variations from noisy sequencing data remains challenging. In this chapter, the authors will first explain the difficulties of analyzing ctDNA sequencing data, review related technologies, and then present some novel bioinformatics methods for analyzing ctDNA NGS data in better ways.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29536438">
                    <p>作为液体活检的主要生物标记物，可以从血液，尿液或其他循环液体中提取的无细胞肿瘤DNA（ctDNA）能够提供肿瘤的全面遗传信息，并且与组织相比可以更好地克服肿瘤异质性问题活检。新一代测序（NGS）是近年来发展起来的一种广泛用于分析ctDNA的技术。虽然处理ctDNA样本的技术已经成熟，但从噪声测序数据中检测低突变等位基因频率（MAF）变异的任务仍然具有挑战性。在本章中，作者将首先解释分析ctDNA测序数据的困难，回顾相关技术，然后提出一些新的生物信息学方法，以更好的方式分析ctDNA NGS数据。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Methods Mol. Biol.">
                Journal: Methods Mol. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29536438" target="_blank">
                PMID: 29536438
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Chen S, Liu M, Zhou Y">
                Authors: Chen S, Liu M, Zh...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29536438">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>275. Capture Hybridization of Long-Range DNA Fragments for High-Throughput Sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29536436" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29536436" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29536436">
                    <p>Capture hybridization coupled with high-throughput sequencing (HTS) has become one of the most popular approaches to address some scientific problems not only for fundamental evolution but also for ecology and human disease in recent years. However, the technical problem of limited probe capture ability affects its widespread application. Here, we propose to capture hybridize long-range DNA fragments for HTS (termed LR-LCH). We provide a case of three amphibian samples to examine LR-LCH with 2 kb libraries and comparison of standard capture hybridization with 480 bp libraries. Capture sensitivity increased from an average 13.57% of standard capture hybridization to an average 19.80% of LR-LCH; capture efficiency also increased from an average 72.56% of standard capture hybridization to an average 97.71% of LR-LCH. These indicate that longer fragments in the library generally contain both relatively variable regions and relatively conservative regions. The divergent parts of target DNA are enriched along with conservative parts of DNA sequence that effectively captured during hybridization. We present a protocol that allows users to overcome the low capture sensitivity problem for high divergent regions.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29536436">
                    <p>捕获杂交与高通量测序（HTS）相结合，近年来已成为解决一些科学问题的最受欢迎的方法之一，这些方法不仅适用于基础进化，而且适用于生态学和人类疾病。然而，探针捕获能力有限的技术问题影响了其广泛的应用。在这里，我们建议捕获杂交HTS的长程DNA片段（称为LR-LCH）。我们提供了一个三个两栖动物样本的案例来检验LR-LCH的2 kb文库和标准捕获杂交与480 bp文库的比较。捕获灵敏度从标准捕获杂交的平均13.57％增加到LR-LCH的平均19.80％;捕获效率也从标准捕获杂交的平均72.56％增加到LR-LCH的平均97.71％。这些表明文库中较长的片段通常包含相对可变的区域和相对保守的区域。目标DNA的不同部分与杂交期间有效捕获的DNA序列的保守部分一起富集。我们提出了一个协议，允许用户克服高发散区域的低捕获灵敏度问题。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Methods Mol. Biol.">
                Journal: Methods Mol. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29536436" target="_blank">
                PMID: 29536436
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Chen X, Ni G, He K, Ding ZL, Li GM, Adeola AC, Murphy RW, Wang WZ, Zhang YP">
                Authors: Chen X, Ni G, He ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29536436">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>276. Transcriptome Sequencing: RNA-Seq.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29536435" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29536435" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29536435">
                    <p>RNA sequencing (RNA-seq) can not only be used to identify the expression of common or rare transcripts but also in the identification of other abnormal events, such as alternative splicing, novel transcripts, and fusion genes. In principle, RNA-seq can be carried out by almost all of the next-generation sequencing (NGS) platforms, but the libraries of different platforms are not exactly the same; each platform has its own kit to meet the special requirements of the instrument design.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29536435">
                    <p>RNA测序（RNA-seq）不仅可用于鉴定常见或罕见转录物的表达，而且还可用于鉴定其他异常事件，如选择性剪接，新转录物和融合基因。原则上，RNA-seq可以通过几乎所有的下一代测序（NGS）平台进行，但不同平台的库不完全相同;每个平台都有自己的套件以满足仪器设计的特殊要求。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Methods Mol. Biol.">
                Journal: Methods Mol. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29536435" target="_blank">
                PMID: 29536435
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Zhang H, He L, Cai L">
                Authors: Zhang H, He L, Cai L
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29536435">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>277. Circular RNAs and exosomes in cancer: a mysterious connection.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29536331" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29536331" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29536331">
                    <p>Circular RNAs (CircRNAs) are a type of non-coding RNAs (NcRNAs) with a closed annular structure. Until next-generation sequencing (NGS) is developed, the misunderstanding of circRNAs &#39;splicing error&#39; has changed, and the mysterious veil of circRNAs has been revealed. NGS provides an approach to investigate circRNAs. Many scholars point out that circRNAs may play an important role in many diseases, especially cancer. At the same time, exosomes, as a kind of extracellular vesicles loaded with many contents, are a hotspot in recent years. They can act as &#39;messengers&#39; between cells, especially in cancer. Lately, it is interesting circRNAs are enriched and stable in exosomes, also called exo-circRNAs, and there have been several articles on circRNAs associated with exosomes. In this review, we summarize the characteristics of circRNAs, especially its main functions. Then, we briefly introduce exosomes and their function in cancer. Finally, the known relation between circRNAs and exosomes is discussed. With further researches, exo-circRNAs may be a novel pathway for cancer diagnosis and targeted therapy.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29536331">
                    <p>环状RNA（CircRNA）是一种非编码RNA（NcRNA），具有封闭的环状结构。在开发新一代测序（NGS）之前，对于circRNAs的拼接错误的误解发生了变化，揭示了circRNA的神秘面纱。 NGS提供了一种研究circRNA的方法。许多学者指出，circRNA可能在许多疾病中发挥重要作用，尤其是癌症。同时，外泌体作为一种含有大量内含物的细胞外囊泡，近年来成为热点。它们可以充当细胞之间的“信使”，特别是在癌症中。最近，令人感兴趣的是，circRNA在外来体中富集且稳定，也称为exo-circRNAs，并且已经有数篇关于与外来体相关的circRNA的文章。在这篇综述中，我们总结了circRNA的特性，特别是其主要功能。然后，我们简要介绍外来体及其在癌症中的功能。最后，讨论了circRNA与外来体之间的已知关系。随着研究的深入，exo-circRNA可能成为癌症诊断和靶向治疗的新途径。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin Transl Oncol">
                Journal: Clin Transl Oncol [IF=2.353]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29536331" target="_blank">
                PMID: 29536331
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-13
            </a>

            <a class="btn btn-sm btn-danger" title="Hou J, Jiang W, Zhu L, Zhong S, Zhang H, Li J, Zhou S, Yang S, He Y, Wang D, Chen X, Deng F, Zhang Q, Wang J, Hu J, Zhang W, Ding L, Zhao J, Tang J">
                Authors: Hou J, Jiang W, Z...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29536331">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>278. Identification and characterization of a novel adenomatous polyposis coli mutation in adult pancreatoblastoma.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29535845" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29535845" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29535845">
                    <p>During next generation sequencing (NGS) analysis, many missense mutations were found in a well-known oncogene, many of which were variant of uncertain significance mutations. We recently treated an adult patient with pancreatoblastoma by chemotherapy. Using an NGS cancer panel, we found a previously unreported missense mutation in the 1835 codon of the adenomatous polyposis coli (APC) gene. We also found a heterogeneous mutation in the 1835 codon of the APC gene in the patient&#39;s germline by Sanger sequencing. Although this patient did not have a history of familial adenomatous polyposis, functional analysis suggested the R1835G mutant APC showed attenuated repression of Wnt/β-catenin signaling activity. This is the first report showing a novel APC missense mutation involved in the onset of adult pancreatoblastoma.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29535845">
                    <p>在下一代测序（NGS）分析中，许多错义突变在众所周知的致癌基因中发现，其中许多是不确定的显着突变的变体。我们最近通过化疗治疗了一例成人胰腺母细胞瘤患者。使用NGS癌症小组，我们在腺瘤性息肉病菌（APC）基因的1835密码子中发现了以前未报道的错义突变。我们还通过Sanger测序发现患者种系中APC基因1835密码子的异质性突变。尽管该患者没有家族性腺瘤性息肉病史，但功能分析表明R1835G突变型APC显示对Wnt /β-连环蛋白信号传导活性的减弱抑制。这是第一份报告，显示了与成人胰腺母细胞瘤发病有关的新型APC错义突变。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29535845" target="_blank">
                PMID: 29535845
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb-13
            </a>

            <a class="btn btn-sm btn-danger" title="Yamaguchi S, Fujii T, Izumi Y, Fukumura Y, Han M, Yamaguchi H, Akita T, Yamashita C, Kato S, Sekiya T">
                Authors: Yamaguchi S, Fuji...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5828192" target="_blank">
                PMC5828192
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29535845">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>279. Detection of somatic mutations in cell-free DNA in plasma and correlation with overall survival in patients with solid tumors.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29535804" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29535804" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29535804">
                    <p>A suitable clinical-grade platform is required for detection of somatic mutations with high sensitivity in cell-free DNA (cfDNA) of patients with solid tumors. In this study, we evaluated in parallel ultra-deep NGS with MassARRAY and allele-specific droplet digital PCR (ddPCR) for cfDNA genotyping and correlated cfDNA yield and mutation status with overall survival (OS) of patients. We assessed plasma samples from 46 patients with various advanced metastatic solid tumors and known mutations by deep sequencing using an Ampliseq cancer hotspot panel V2 on Ion Proton. A subset of these samples with DNA availability was tested by ddPCR and UltraSEEK MassARRAY for mutation detection in 5 genes (IDH1, PIK3CA, KRAS, BRAF, and NRAS). Sixty one of 104 expected tissue mutations and 6 additional mutations not present in the tissue were detected in cfDNA. ddPCR and MassARRAY showed 83% and 77% concordance with NGS for mutation detection with 100% and 79% sensitivity, respectively. The median OS of patients with lower cfDNA yield (74 vs 50 months; P &lt; 0.03) and cfDNA negative for mutations (74.2 vs 53 months; p &lt; 0.04) was significantly longer than in patients with higher cfDNA yield and positive for mutations. A limit-of-detection of 0.1% was demonstrated for ddPCR and MassARRAY platforms using a serially diluted positive cfDNA sample. The MassARRAY and ddPCR systems enable fast and cost-effective genotyping for a targeted set of mutations and can be used for single gene testing to guide response to chemotherapy or for orthogonal validation of NGS results.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29535804">
                    <p>在实体瘤患者的无细胞DNA（cfDNA）中以高灵敏度检测体细胞突变需要合适的临床级平台。在这项研究中，我们用MassARRAY和等位基因特异性液滴数字PCR（ddPCR）平行超深NGS评估了cfDNA基因分型，并将cfDNA产量和突变状态与患者的总生存期（OS）相关联。我们使用Ion Proton上的Ampliseq癌症热点面板V2，通过深度测序评估了来自46名患有各种晚期转移性实体瘤和已知突变的患者的血浆样品。通过ddPCR和UltraSEEK MassARRAY检测这5个基因（IDH1，PIK3CA，KRAS，BRAF和NRAS）突变检测的DNA样本的子集。在cfDNA中检测到组织中未发现104个预期组织突变和6个额外突变中的60个。 ddPCR和MassARRAY分别显示与NGS的83％和77％的一致性，用于突变检测，分别具有100％和79％的敏感性。具有较低cfDNA产量（74比50个月; P &lt;0.03）和cfDNA阴性（74.2比53个月; p &lt;0.04）的患者的中位OS显着长于具有较高cfDNA产量和突变阳性的患者。使用系列稀释的阳性cfDNA样品，ddPCR和MassARRAY平台的检测极限为0.1％。 MassARRAY和ddPCR系统能够对一组有针对性的突变进行快速，经济的基因分型，并可用于单基因检测以指导对化疗的反应或NGS结果的正交验证。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Oncotarget">
                Journal: Oncotarget [IF=5.168]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29535804" target="_blank">
                PMID: 29535804
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb-13
            </a>

            <a class="btn btn-sm btn-danger" title="Mehrotra M, Singh RR, Loghavi S, Duose DY, Barkoh BA, Behrens C, Patel KP, Routbort MJ, Kopetz S, Broaddus RR, Jeffrey Medeiros L, Wistuba II, Luthra R">
                Authors: Mehrotra M, Singh...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5828199" target="_blank">
                PMC5828199
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29535804">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>280. Comparison of Burrows-Wheeler Transform-Based Mapping Algorithms Used in High-Throughput Whole-Genome Sequencing: Application to Illumina Data for Livestock Genomes.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29535759" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29535759" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29535759">
                    <p>Ongoing developments and cost decreases in next-generation sequencing (NGS) technologies have led to an increase in their application, which has greatly enhanced the fields of genetics and genomics. Mapping sequence reads onto a reference genome is a fundamental step in the analysis of NGS data. Efficient alignment of the reads onto the reference genome with high accuracy is very important because it determines the global quality of downstream analyses. In this study, we evaluate the performance of three Burrows-Wheeler transform-based mappers, BWA, Bowtie2, and HISAT2, in the context of paired-end Illumina whole-genome sequencing of livestock, using simulated sequence data sets with varying sequence read lengths, insert sizes, and levels of genomic coverage, as well as five real data sets. The mappers were evaluated based on two criteria, computational resource/time requirements and robustness of mapping. Our results show that BWA and Bowtie2 tend to be more robust than HISAT2, while HISAT2 was significantly faster and used less memory than both BWA and Bowtie2. We conclude that there is not a single mapper that is ideal in all scenarios but rather the choice of alignment tool should be driven by the application and sequencing technology.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29535759">
                    <p>新一代测序（NGS）技术的持续发展和成本降低导致其应用的增加，这大大增强了遗传学和基因组学的领域。将序列读取映射到参考基因组是分析NGS数据的基本步骤。以高准确度将读数有效地与参考基因组进行比对非常重要，因为它决定了下游分析的全局质量。在这项研究中，我们使用具有不同序列阅读长度的模拟序列数据集，评估了三种基于Burrows-Wheeler变换的图谱，BWA，Bowtie2和HISAT2在家畜双末端Illumina全基因组测序中的表现，插入大小，基因组覆盖度以及五个真实数据集。根据两个标准评估映射器，计算资源/时间要求和映射的鲁棒性。我们的结果显示BWA和Bowtie2比HISAT2更加稳健，而HISAT2比BWA和Bowtie2快得多，使用的内存也更少。我们得出结论：没有一种映射器在所有情况下都是理想的，但是对齐工具的选择应该由应用程序和排序技术来驱动。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Genet">
                Journal: Front Genet [IF=3.789]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29535759" target="_blank">
                PMID: 29535759
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Keel BN, Snelling WM">
                Authors: Keel BN, Snelling WM
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5834436" target="_blank">
                PMC5834436
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29535759">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>281. Rapid screening test of most frequent BRCA1/BRCA2 pathogenic variants in the NGS era.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29534594" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29534594" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29534594">
                    <p>The average risk of breast cancer in general Slovak population of women is 4-5% and the risk of ovarian cancer is 2%. Probability of breast/ovarian cancer development is higher in individuals carrying a causative germline DNA variant in BRCA1 or BRCA2 gene responsible for hereditary breast/ovarian cancer (HBOC). Although a major proportion of inherited breast/ovarian cancers are due to the mentioned causal mutations, a number of new genes have emerged. Here we describe a rapid, multiplex and comprehensive approach for the detection of pathogenic variants in BRCA1 and BRCA2 genes which most frequently occur in Slovak HBOC population. Analysis comprises the combination of mutation specific methods. Fluorescent PCR amplification followed by fragment analysis for detection of insertions/deletions in exon 11 of BRCA1 gene. Second method is SNaPshot analysis for detection of the most frequent missense and ins/del variants in exons 2, 5, 13, 20 of BRCA1 and exons 11, 23 and 25 of BRCA2 gene. Altogether, we have analyzed 687 samples, 86 (12.5%) in group 1, which fulfilled indication criteria based on the positive family/personal history. Group 2 involved 601 (87.5%) cases, who did not meet the indication criteria and only the screening test was recommended. Using the combined approach, we have identified 47 mutated samples (6.8%), 40 in group 1 (46.5%) and 7 in group 2 (1.1%). However, the presented screening test would not provide complex results of BRCA1/2 gene analysis, it offers testing accessible to a broader spectrum of individuals under the threshold of indication for whole gene analysis. This approach may provide valuable information even in the NGS analysis era.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29534594">
                    <p>普通斯洛伐克女性患乳腺癌的平均风险为4-5％，卵巢癌风险为2％。 BRCA1或BRCA2基因负责遗传性乳腺癌/卵巢癌（HBOC）的个体携带致病性生殖细胞DNA变异的个体患乳腺癌/卵巢癌的可能性更高。虽然遗传性乳腺癌/卵巢癌的主要部分是由于所提到的因果突变，但仍有许多新基因出现。在这里我们描述了一种快速，多重和全面的方法来检测斯洛伐克HBOC人群中最常发生的BRCA1和BRCA2基因的致病变异。分析包括突变特异性方法的组合。荧光PCR扩增，然后进行片段分析以检测BRCA1基因外显子11中的插入/缺失。第二种方法是SNaPshot分析，用于检测BRCA1外显子2,5,13,​​20和BRCA2基因外显子11,23和25中最常见的错义和ins / del变体。总的来说，我们分析了687个样本，第1组有86个（12.5％），这些样本满足了基于积极的家庭/个人历史的适应症标准。第2组涉及601例（87.5％），不符合适应症标准，仅推荐筛查试验。使用组合方法，我们确定了47个突变样本（6.8％），第1组40个（46.5％）和第2组7个（1.1％）。然而，所提供的筛查测试不会提供BRCA1 / 2基因分析的复杂结果，它提供了在整个基因分析的指示阈值下可供更广泛的个体使用的测试。即使在NGS分析时代，这种方法也可能提供有价值的信息。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Neoplasma">
                Journal: Neoplasma [IF=1.871]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29534594" target="_blank">
                PMID: 29534594
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Zidekova D, Waczulikova I, Dolesova L, Vavrova L, Hamidova O, Lohajova Behulova R, Konecny M">
                Authors: Zidekova D, Waczu...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29534594">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>282. Expanding the Utilization of Formalin-Fixed, Paraffin-Embedded Archives: Feasibility of miR-Seq for Disease Exploration and Biomarker Development from Biopsies with Clear Cell Renal Cell Carcinoma.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29534467" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29534467" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29534467">
                    <p>Novel predictive tools for clear cell renal cell carcinoma (ccRCC) are urgently needed. MicroRNAs (miRNAs) have been increasingly investigated for their predictive value, and formalin-fixed paraffin-embedded biopsy archives may potentially be a valuable source of miRNA sequencing material, as they remain an underused resource. Core biopsies of both cancerous and adjacent normal tissues were obtained from patients (n = 12) undergoing nephrectomy. After small RNA-seq, several analyses were performed, including classifier evaluation, obesity-related inquiries, survival analysis using publicly available datasets, comparisons to the current literature and ingenuity pathway analyses. In a comparison of tumour vs. normal, 182 miRNAs were found with significant differential expression; miR-155 was of particular interest as it classified all ccRCC samples correctly and correlated well with tumour size (R² = 0.83); miR-155 also predicted poor survival with hazard ratios of 2.58 and 1.81 in two different TCGA (The Cancer Genome Atlas) datasets in a univariate model. However, in a multivariate Cox regression analysis including age, sex, cancer stage and histological grade, miR-155 was not a statistically significant survival predictor. In conclusion, formalin-fixed paraffin-embedded biopsy tissues are a viable source of miRNA-sequencing material. Our results further support a role for miR-155 as a promising cancer classifier and potentially as a therapeutic target in ccRCC that merits further investigation.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29534467">
                    <p>迫切需要用于透明细胞肾细胞癌（ccRCC）的新型预测工具。 MicroRNAs（miRNA）的预测价值越来越多，福尔马林固定石蜡包埋活检档案可能成为miRNA测序材料的宝贵来源，因为它们仍然是一种未被充分利用的资源。癌性和邻近正常组织的核心活组织检查取自经历肾切除术的患者（n = 12）。在进行小RNA-seq之后，进行了一些分析，包括分类器评估，与肥胖有关的询问，使用公开可用数据集的生存分析，与当前文献和独创性途径分析的比较。在肿瘤与正常的比较中，发现182个miRNA具有显着的差异表达;由于miR-155正确地将所有ccRCC样品分类并且与肿瘤大小（R 2 = 0.83）良好相关，因此miR-155是特别感兴趣的;在一个单变量模型中，miR-155也预测了两种不同TCGA（癌症基因组图谱）数据集中存活率差的危险比为2.58和1.81。然而，在包括年龄，性别，癌症分期和组织学分级的多变量Cox回归分析中，miR-155不是统计学显着的生存预测因子。总之，福尔马林固定的石蜡包埋的活检组织是miRNA测序材料的可行来源。我们的研究结果进一步支持了miR-155作为一种有前景的癌症分类器的作用，并有可能成为ccRCC的治疗靶点，值得进一步研究。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Int J Mol Sci">
                Journal: Int J Mol Sci [IF=3.226]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29534467" target="_blank">
                PMID: 29534467
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-10
            </a>

            <a class="btn btn-sm btn-danger" title="Strauss P, Marti HP, Beisland C, Scherer A, Lysne V, Leh S, Flatberg A, Koch E, Beisvag V, Landolt L, Skogstrand T, Eikrem Ø">
                Authors: Strauss P, Marti ...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5877664" target="_blank">
                PMC5877664
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29534467">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>283. NGSphy: phylogenomic simulation of next-generation sequencing data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29534152" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29534152" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29534152">
                    <p>Advances in sequencing technologies have made it feasible to obtain massive datasets for phylogenomic inference, often consisting of large numbers of loci from multiple species and individuals. The phylogenomic analysis of next-generation sequencing (NGS) data requires a complex computational pipeline where multiple technical and methodological decisions are necessary that can influence the final tree obtained, like those related to coverage, assembly, mapping, variant calling and/or phasing.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29534152">
                    <p>测序技术的进步使得获得用于系统发生基因推断的大量数据集成为可能，其通常由来自多个物种和个体的大量位点组成。新一代测序（NGS）数据的系统发育分析需要一个复杂的计算流程，其中需要多个技术和方法决策，可以影响最终获得的树，如覆盖，组装，映射，变异调用和/或定相等。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Bioinformatics">
                Journal: Bioinformatics [IF=7.307]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29534152" target="_blank">
                PMID: 29534152
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-09
            </a>

            <a class="btn btn-sm btn-danger" title="Escalona M, Rocha S, Posada D">
                Authors: Escalona M, Rocha...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29534152">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>284. Does smoking alter the mutation profile of human papillomavirus-driven head and neck cancers?</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29533868" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29533868" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29533868">
                    <p>Human papillomavirus (HPV)-driven oropharyngeal cancer (OPC) patients are characterised by a better prognosis than their HPV-negative counterparts. However, this significant survival advantage is not homogeneous and among HPV-positive patients those with a smoking history have a significantly increased risk of oncologic failure. The reason why tobacco consumption impacts negatively the prognosis is still elusive. Tobacco might induce additional genetic alterations leading to a more aggressive phenotype. The purpose of this study was to characterise the mutational profile of HPV-positive OPCs by smoking status. We hypothesise a higher frequency of mutations affecting smokers.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29533868">
                    <p>人乳头瘤病毒（HPV）驱动的口咽癌（OPC）患者的特征在于其HPV阴性对应物的预后更好。然而，这种显着的生存优势并不均匀，在吸烟史的HPV阳性患者中，肿瘤失败的风险显着增加。烟草消费之所以对预后产生负面影响仍然难以捉摸。烟草可能诱发额外的基因改变，导致更具攻击性的表型。本研究的目的是通过吸烟状态来描述HPV阳性OPCs的突变特征。我们假设影响吸烟者的突变频率较高。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Cancer">
                Journal: Eur. J. Cancer [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29533868" target="_blank">
                PMID: 29533868
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Mirghani H, Lacroix L, Rossoni C, Sun R, Aupérin A, Casiraghi O, Villepelet A, Lacave R, Faucher G, Marty V, Ferté C, Soria JC, Even C">
                Authors: Mirghani H, Lacro...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29533868">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>285. PlasmidTron: assembling the cause of phenotypes and genotypes from NGS data.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29533742" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29533742" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29533742">
                    <p>Increasingly rich metadata are now being linked to samples that have been whole-genome sequenced. However, much of this information is ignored. This is because linking this metadata to genes, or regions of the genome, usually relies on knowing the gene sequence(s) responsible for the particular trait being measured and looking for its presence or absence in that genome. Examples of this would be the spread of antimicrobial resistance genes carried on mobile genetic elements (MGEs). However, although it is possible to routinely identify the resistance gene, identifying the unknown MGE upon which it is carried can be much more difficult if the starting point is short-read whole-genome sequence data. The reason for this is that MGEs are often full of repeats and so assemble poorly, leading to fragmented consensus sequences. Since mobile DNA, which can carry many clinically and ecologically important genes, has a different evolutionary history from the host, its distribution across the host population will, by definition, be independent of the host phylogeny. It is possible to use this phenomenon in a genome-wide association study to identify both the genes associated with the specific trait and also the DNA linked to that gene, for example the flanking sequence of the plasmid vector on which it is encoded, which follows the same patterns of distribution as the marker gene/sequence itself. We present PlasmidTron, which utilizes the phenotypic data normally available in bacterial population studies, such as antibiograms, virulence factors, or geographical information, to identify traits that are likely to be present on DNA that can randomly reassort across defined bacterial populations. It is also possible to use this methodology to associate unknown genes/sequences (e.g. plasmid backbones) with a specific molecular signature or marker (e.g. resistance gene presence or absence) using PlasmidTron. PlasmidTron uses a k-mer-based approach to identify reads associated with a phylogenetically unlinked phenotype. These reads are then assembled de novo to produce contigs in a fast and scalable-to-large manner. PlasmidTron is written in Python 3 and is available under the open source licence GNU GPL3 from https://github.com/sanger-pathogens/plasmidtron.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29533742">
                    <p>目前正在将越来越丰富的元数据与全基因组测序的样本联系起来。但是，大部分信息都被忽略。这是因为将这些元数据连接到基因或基因组区域通常依赖于了解负责测量特定性状的基因序列并寻找它在该基因组中的存在或缺失。例如，移动遗传因子（MGEs）携带的抗生素耐药基因的传播。然而，尽管可以常规鉴定抗性基因，但如果起始点是短读全基因组序列数据，鉴定其携带的未知MGE可能会困难得多。这是因为MGEs经常充满重复序列，因此组装不良，导致片段化的共有序列。由于可携带许多临床和生态学重要基因的移动DNA具有与宿主不同的进化历史，因此其在整个宿主群体中的分布将根据定义独立于宿主系统发育。在全基因组关联研究中使用这种现象是可能的，以鉴定与特定性状相关的基因以及与该基因连接的DNA，例如其所编码的质粒载体的侧翼序列，该序列如下与标记基因/序列本身相同的分布模式。我们提供PlasmidTron，它利用细菌群体研究中通常可用的表型数据，如抗生素，毒力因子或地理信息，来确定可能存在于DNA上的特征，这些特征可以随机重排在所定义的细菌种群中。也可以使用该方法使用PlasmidTron将未知基因/序列（例如质粒骨架）与特定分子标签或标记（例如抗性基因存在或不存在）相关联。 PlasmidTron使用基于k-mer的方法来鉴定与系统发育不相关表型相关的读段。然后将这些读段重新组装，以快速和可扩展的方式产生重叠群。 PlasmidTron是用Python 3编写的，可从https://github.com/sanger-pathogens/plasmidtron获得开源许可证GNU GPL3。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Microb Genom">
                Journal: Microb Genom [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29533742" target="_blank">
                PMID: 29533742
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-12
            </a>

            <a class="btn btn-sm btn-danger" title="Page AJ, Wailan A, Shao Y, Judge K, Dougan G, Klemm EJ, Thomson NR, Keane JA">
                Authors: Page AJ, Wailan A...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5885016" target="_blank">
                PMC5885016
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29533742">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>286. Pitfalls in molecular diagnosis of Friedreich ataxia.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29530802" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29530802" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29530802">
                    <p>Freidreich ataxia (FRDA) is the most common hereditary ataxia, nearly 98% of patients harbouring homozygous GAA expansions in intron 1 of the FXN gene (NM_000144.4). The remaining patients are compound heterozygous for an expansion and a point mutation or an exonic deletion. Molecular screening for FXN expansion is therefore focused on (GAA)n expansion analysis, commonly performed by triplet repeat primed PCR (PT-PCR). We report on an initial pitfall in the molecular characterization of a 15 year-old girl with Freidreich ataxia (FRDA) who carried a rare deletion in intron 1 of the FXN gene. Due to this deletion TP-PCR failed to amplify the GAA expansion. This exceptional configuration induced misinterpretation of the molecular defect in this patient, who was first reported as having no FXN expansion. NGS analysis of a panel of 212 genes involved in nuclear mitochondrial disorders further revealed an intragenic deletion encompassing exons 4-5 of the FXN gene. Modified TP-PCR analysis confirmed the presence of a classical (GAA)n expansion located in trans. This case points out the possible pitfalls in molecular diagnosis of FRDA in affected patients and their relatives: detection of the FXN expansion may be impaired by several non-pathological or pathological variants around the FXN (GAA)n repeat. We propose a new molecular strategy to accurately detect expansion by TP-PCR in FRDA patients.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29530802">
                    <p>Freidreich共济失调（FRDA）是最常见的遗传性共济失调，接近98％的患者携带FXN基因内含子1的纯合GAA扩增（NM_000144.4）。其余患者是扩展和点突变或外显子缺失的复合杂合子。因此，FXN扩增的分子筛选集中于（GAA）n扩增分析，通常通过三重重复引物PCR（PT-PCR）进行。我们报告了弗里德赖希共济失调（FRDA）15岁女孩在FXN基因内含子1中罕见缺失的分子特征的初步缺陷。由于这种缺失，TP-PCR未能扩增GAA扩增。这种特殊的配置引起了该患者分子缺陷的错误解读，该患者首次被报道没有FXN扩充。涉及核线粒体疾病的212个基因的NGS分析进一步揭示了包含FXN基因的外显子4-5的基因内缺失。改进的TP-PCR分析证实存在位于反式的经典（GAA）n扩增。该病例指出了受影响患者及其亲属的FRDA分子诊断可能存在的缺陷：检测FXN扩增可能受到FXN（GAA）n重复周围几种非病理或病理变异的影响。我们提出了一种新的分子策略，以准确检测FRDA患者TP-PCR扩增。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur J Med Genet">
                Journal: Eur J Med Genet [IF=2.137]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29530802" target="_blank">
                PMID: 29530802
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-09
            </a>

            <a class="btn btn-sm btn-danger" title="Barcia G, Rachid M, Magen M, Assouline Z, Koenig M, Funalot B, Barnerias C, Rötig A, Munnich A, Bonnefont JP, Steffann J">
                Authors: Barcia G, Rachid ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29530802">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>287. RARA and RARG Gene Downregulation Associated with EZH2 Mutation in Acute Promyelocytic-Like Morphology Leukemia.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29530751" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29530751" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29530751">
                    <p>Most Acute Promyelocytic Leukemia (APL) patients express PML-RARA fusion; in rare cases RARA is rearranged with partner genes other than PML. To date, only two patients presenting features similar to APL showing the RARG gene rearrangement have been described. We report an Acute Myeloid Leukemia (AML) patient with morphology resembling APL without involvement of the RARA gene. Molecular and Fluorescent In Situ Hybridization (FISH) analyses excluded PML-RARA fusion and variant rearrangements involving RARA and RARG loci. Targeted Next Generation Sequencing showed EZH2-D185H mutation. As this mutation involved the region of interaction with DNA methyltransferases, we speculate an epigenetic alteration of genes involved in the APL-like phenotype. Expression analysis by droplet digital Polymerase Chain Reaction (PCR) revealed downregulation of the RARA and RARG genes. We hypothesize a novel mechanism of EZH2 function alteration, which may be responsible for an AML with APL-like phenotype featuring dysregulation of the RARA and RARG genes.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29530751">
                    <p>大多数急性早幼粒细胞白血病（APL）患者表达PML-RARA融合蛋白;在极少数情况下，RARA与PML以外的伴侣基因重排。迄今为止，仅描述了两名呈现与APL相似的特征以显示RARG基因重排的患者。我们报告急性髓细胞性白血病（AML）患者的形态类似于APL而不涉及RARA基因。分子和荧光原位杂交（FISH）分析排除了涉及RARA和RARG基因座的PML-RARA融合和变异重排。靶向的下一代测序显示EZH2-D185H突变。由于该突变涉及与DNA甲基转移酶相互作用的区域，我们推测参与APL样表型的基因的表观遗传改变。通过液滴数字聚合酶链式反应（PCR）进行的表达分析揭示了RARA和RARG基因的下调。我们推测EZH2功能改变的一种新机制，其可能负责具有APL样表型的AML，其特征是RARA和RARG基因失调。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Hum. Pathol.">
                Journal: Hum. Pathol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29530751" target="_blank">
                PMID: 29530751
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-09
            </a>

            <a class="btn btn-sm btn-danger" title="Coccaro N, Zagaria A, Orsini P, Anelli L, Tota G, Casieri P, Impera L, Minervini A, Minervini CF, Cumbo C, Parciante E, Mestice A, Delia M, Brunetti C, Specchia G, Albano F">
                Authors: Coccaro N, Zagari...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29530751">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>288. High-throughput construction of multiple cas9 gene variants via assembly of high-depth tiled and sequence-verified oligonucleotides.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29529247" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29529247" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29529247">
                    <p>Selective retrieval of sequence-verified oligonucleotides (oligos) from next-generation sequencing (NGS) flow cells, termed megacloning, promises accurate and reliable gene synthesis. However, gene assembly requires a complete collection of overlapping sense and nonsense oligos, and megacloning does not typically guarantee the complete production of sequence-verified oligos. Therefore, missing oligos must be provided via repetitive rounds of megacloning, which introduces a bottleneck for scaled-up efforts at gene assembly. Here, we introduce the concept of high-depth tiled oligo design to successfully utilize megacloned oligos for gene synthesis. Using acquired oligos from a single round of the megacloning process, we assembled 72 of 81 target Cas9-coding gene variants. We further validated 62 of these cas9 constructs, and deposited the plasmids to Addgene for subsequent functional characterization by the scientific community. This study demonstrates the utility of using sequence-verified oligos for DNA assembly and provides a practical and reliable optimized method for high-throughput gene synthesis.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29529247">
                    <p>从下一代测序（NGS）流式细胞中选择性检索经序列验证的寡核苷酸（寡核苷酸），称为megacloning，保证准确和可靠的基因合成。然而，基因组装需要完整的重叠有义和无义寡核苷酸的收集，并且巨增殖通常不保证经序列验证的寡核苷酸的完整生产。因此，缺失的寡核苷酸必须通过重复轮次巨核细胞化来提供，这为基因组装上扩大的努力带来了瓶颈。在这里，我们介绍高深度平铺寡核苷酸设计的概念，以成功利用megacloned寡核苷酸进行基因合成。使用获得的寡核苷酸从一轮megacloning过程中，我们组装了81个目标Cas9编码基因变异中的72个。我们进一步验证了这些cas9构建体中的62个，并将质粒保存到Addgene以供科学界随后进行功能鉴定。该研究证实了使用经序列验证的寡核苷酸用于DNA装配的实用性，并提供了用于高通量基因合成的实用且可靠的优化方法。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Nucleic Acids Res.">
                Journal: Nucleic Acids Res. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29529247" target="_blank">
                PMID: 29529247
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Feb-26
            </a>

            <a class="btn btn-sm btn-danger" title="Cho N, Seo HN, Ryu T, Kwon E, Huh S, Noh J, Yeom H, Hwang B, Ha H, Lee JH, Kwon S, Bang D">
                Authors: Cho N, Seo HN, Ry...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29529247">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>289. Convergent Evolution of Slick Coat in Cattle through Truncation Mutations in the Prolactin Receptor.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29527221" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29527221" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29527221">
                    <p>Evolutionary adaptations are occasionally convergent solutions to the same problem. A mutation contributing to a heat tolerance adaptation in Senepol cattle, a New World breed of mostly European descent, results in the distinct phenotype known as slick, where an animal has shorter hair and lower follicle density across its coat than wild type animals. The causal variant, located in the 11th exon of prolactin receptor, produces a frameshift that results in a truncated protein. However, this mutation does not explain all cases of slick coats found in criollo breeds. Here, we obtained genome sequences from slick cattle of a geographically distinct criollo breed, namely Limonero, whose ancestors were originally brought to the Americas by the Spanish. These data were used to identify new causal alleles in the 11th exon of the prolactin receptor, two of which also encode shortened proteins that remove a highly conserved tyrosine residue. These new mutations explained almost 90% of investigated cases of animals that had slick coats, but which also did not carry the Senepol slick allele. These results demonstrate convergent evolution at the molecular level in a trait important to the adaptation of an animal to its environment.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29527221">
                    <p>进化的适应性偶尔是对同一问题的融合解决方案。 Senepol牛（一种主要是欧洲血统的新世界品种）的耐热适应性突变导致了独特的表型，称为光滑性，其中动物的毛发比其野生型动物具有更短的毛发和更低的毛囊密度。位于泌乳素受体第11外显子的致病变异体产生移码，导致截短的蛋白质。然而，这种突变并不能解释在criollo品种中发现的所有光滑外套的情况。在这里，我们从一个地理上不同的criollo品种，即Limonero，其祖先最初由西班牙人带到美洲，获得了光滑的牛的基因组序列。这些数据被用于鉴定催乳素受体第11外显子中的新因果等位基因，其中两个也编码移除高度保守的酪氨酸残基的缩短蛋白质。这些新的突变解释了近90％的调查动物皮肤滑动，但也没有携带塞内波尔浮华等位基因。这些结果表明在分子水平上趋同进化对于动物对其环境适应性的重要性。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Genet">
                Journal: Front Genet [IF=3.789]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29527221" target="_blank">
                PMID: 29527221
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Porto-Neto LR, Bickhart DM, Landaeta-Hernandez AJ, Utsunomiya YT, Pagan M, Jimenez E, Hansen PJ, Dikmen S, Schroeder SG, Kim ES, Sun J, Crespo E, Amati N, Cole JB, Null DJ, Garcia JF, Reverter A, Barendse W, Sonstegard TS">
                Authors: Porto-Neto LR, Bi...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5829098" target="_blank">
                PMC5829098
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29527221">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>290. Whole Genome Sequence Analysis of CTX-M-15 Producing Klebsiella Isolates Allowed Dissecting a Polyclonal Outbreak Scenario.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29527200" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29527200" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29527200">
                    <p>Extended-spectrum β-lactamase (ESBL) producing Klebsiella pneumoniae pose an important threat of infection with increased morbidity and mortality, especially for immunocompromised patients. Here, we use the rise of multidrug-resistant K. pneumoniae in a German neurorehabilitation center from April 2015 to April 2016 to dissect the benefit of whole genome sequencing (WGS) for outbreak analyses. In total, 53 isolates were obtained from 52 patients and examined using WGS. Two independent analysis strategies (reference-based and -free) revealed the same distinct clusters of two CTX-M-15 producing K. pneumoniae clones (ST15, n = 31; ST405, n = 7) and one CTX-M-15 producing Klebsiella quasipneumoniae strain (ST414, n = 8). Additionally, we determined sequence variations associated with antimicrobial resistance phenotypes in single isolates expressing carbapenem and colistin resistance, respectively. For rapid detection of the major K. pneumoniae outbreak clone (ST15), a selective triplex PCR was deduced from WGS data of the major outbreak strain and K. pneumoniae genome data deposited in central databases. Moreover, we introduce two novel open-source applications supporting reference genome selection (refRank; https://gitlab.com/s.fuchs/refRank) and alignment-based SNP-filtering (SNPfilter; https://gitlab.com/s.fuchs/snpfilter) in NGS analyses.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29527200">
                    <p>产超广谱β-内酰胺酶（ESBL）的肺炎克雷伯菌构成感染的重要威胁，发病率和死亡率增加，尤其是免疫功能低下的患者。在此，我们利用2015年4月至2016年4月在德国神经康复中心的多重耐药肺炎克雷伯杆菌的上升来剖析全基因组测序（WGS）在爆发分析中的益处。总共从52名患者中获得53个分离物，并使用WGS进行检查。两个独立的分析策略（基于参考和无）显示两个CTX-M-15产生的肺炎克雷伯菌克隆（ST15，n = 31; ST405，n = 7）和一个CTX-M-15产生的相同的不同簇克氏杆菌肺炎克雷伯氏菌菌株（ST414，n = 8）。另外，我们确定了分别表达碳青霉烯和粘菌素抗性的单一分离株中与抗微生物剂抗性表型相关的序列变异。为了快速检测主要的肺炎克雷伯氏菌暴发克隆（ST15），从主要暴发菌株的WGS数据和中央数据库中存储的肺炎克雷伯氏菌基因组数据推导出选择性三重PCR。此外，我们还介绍了两种支持参考基因组选择（refRank; https://gitlab.com/s.fuchs/refRank）和基于比对的SNP过滤（SNPfilter; https://gitlab.com/s）的开源应用.fuchs / snpfilter）进行NGS分析。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Front Microbiol">
                Journal: Front Microbiol [IF=4.076]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29527200" target="_blank">
                PMID: 29527200
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018
            </a>

            <a class="btn btn-sm btn-danger" title="Becker L, Fuchs S, Pfeifer Y, Semmler T, Eckmanns T, Korr G, Sissolak D, Friedrichs M, Zill E, Tung ML, Dohle C, Kaase M, Gatermann S, Rüssmann H, Steglich M, Haller S, Werner G">
                Authors: Becker L, Fuchs S...
            </a>

            
            <a class="btn btn-sm btn-primary" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5829066" target="_blank">
                PMC5829066
            </a>
            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29527200">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>291. Circular RNA expression profiles in placental villi from women with gestational diabetes mellitus.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29526755" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29526755" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29526755">
                    <p>Circular RNAs (circRNAs) have recently been shown to exert their effects on multiple pathological processes by acting as microRNA (miRNA) sponges. However, the roles of circRNAs in gestational diabetes mellitus (GDM) are largely unknown. This study aimed to identify the circRNAs involved in GDM and predict their potential biological functions. We first performed next-generation sequencing (NGS) to generate unbiased placental villi circRNA expression profiles of GDM and normal controls. In total, 48,270 circRNAs from the placental villi of the two groups were sequenced. Of these, 227 circRNAs were significantly up-regulated and 255 circRNAs were significantly down-regulated. Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) biological pathway analyses demonstrated that glycometabolism and lipometabolism processes, which are important in GDM development, were significantly enriched. Further analysis showed that most of the circRNAs harbored miRNA binding sites, and some were associated with GDM. These results showed that circRNAs are aberrantly expressed in the placental villi of GDM patients and play potential roles in the development of GDM.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29526755">
                    <p>最近证明环状RNA（circRNA）通过作为microRNA（miRNA）海绵发挥其对多种病理过程的作用。然而，circRNA在妊娠期糖尿病（GDM）中的作用在很大程度上是未知的。本研究旨在确定参与GDM的circRNA并预测其潜在的生物学功能。我们首先进行下一代测序（NGS），以产生GDM和正常对照的无偏的胎盘绒毛circRNA表达谱。总共对来自两组胎盘绒毛的48,270个circRNA进行测序。其中，227个circRNAs显着上调，255个circRNAs显着下调。基因本体论（GO）和京都基因与基因组百科全书（KEGG）生物学途径分析表明，GDM发育中重要的糖代谢和脂肪代谢过程显着增强。进一步分析表明，大多数circRNAs包含miRNA结合位点，有些与GDM相关。这些结果表明，circRNA在GDM患者的胎盘绒毛中异常表达，并在GDM的发展中起潜在作用。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Biochem. Biophys. Res. Commun.">
                Journal: Biochem. Biophys. Res. Commun. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29526755" target="_blank">
                PMID: 29526755
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-04-15
            </a>

            <a class="btn btn-sm btn-danger" title="Yan L, Feng J, Cheng F, Cui X, Gao L, Chen Y, Wang F, Zhong T, Li Y, Liu L">
                Authors: Yan L, Feng J, Ch...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29526755">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>292. Identification of novel mutations in FFPE lung adenocarcinomas using DEPArray sorting technology and next-generation sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29525983" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29525983" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29525983">
                    <p>Formalin-fixed paraffin-embedded (FFPE) tissues are utilized as the standard diagnostic method in pathology laboratories. However, admixture of unwanted tissues and shortage of normal samples, which can be used to detect somatic mutation, are considered critical factors to accurately diagnose cancer. To explore these challenges, we sorted the pure tumor cells from 22 FFPE lung adenocarcinoma tissues via Di-Electro-Phoretic Array (DEPArray) technology, a new cell sorting technology, and analyzed the variants with next-generation sequencing (NGS) for the most accurate analysis. The allele frequencies of the all gene mutations were improved by 1.2 times in cells sorted via DEPArray (tumor suppressor genes, 1.3-10.1 times; oncogenes, 1.3-2.6 times). We identified 16 novel mutations using the sequencing from sorted cells via DEPArray technology, compared to detecting 4 novel mutation by the sequencing from unsorted cells. Using this analysis, we also revealed that five genes (TP53, EGFR, PTEN, RB1, KRAS, and CTNNB1) were somatically mutated in multiple homogeneous lung adenocarcinomas. Together, we sorted pure tumor cells from 22 FFPE lung adenocarcinomas by DEPArray technology and identified 16 novel somatic mutations. We also established the precise genomic landscape for more accurate diagnosis in 22 lung adenocarcinomas with mutations detected in pure tumor cells. The results obtained in this study could offer new avenues for the treatment and the diagnosis of squamous cell lung cancers.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29525983">
                    <p>福尔马林固定石蜡包埋（FFPE）组织被用作病理学实验室的标准诊断方法。然而，可以用来检测体细胞突变的不需要的组织的混合和正常样本的短缺被认为是准确诊断癌症的关键因素。为了探索这些挑战，我们通过一种新的细胞分选技术 - 二电泳 - 阵列（DEPArray）技术对来自22个FFPE肺腺癌组织的纯肿瘤细胞进行了分类，并且使用新一代测序（NGS）准确的分析。在通过DEPArray（肿瘤抑制基因，1.3-10.1倍;癌基因，1.3-2.6倍）分选的细胞中，所有基因突变的等位基因频率提高1.2倍。我们通过DEPArray技术使用来自分选细胞的测序鉴定了16个新突变，与通过未分选细胞测序检测到4个新突变相比。使用这种分析，我们还揭示了五种基因（TP53，EGFR，PTEN，RB1，KRAS和CTNNB1）在多个均质肺腺癌中发生体细胞突变。总之，我们通过DEPArray技术对来自22个FFPE肺腺癌的纯肿瘤细胞进行了分类，并确定了16个新的体细胞突变。我们还建立了精确的基因组景观，以更准确地诊断22个肺腺癌，在纯肿瘤细胞中检测到突变。在这项研究中获得的结果可能为鳞癌的治疗和诊断提供新的途径。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=J. Appl. Genet.">
                Journal: J. Appl. Genet. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29525983" target="_blank">
                PMID: 29525983
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Mar-10
            </a>

            <a class="btn btn-sm btn-danger" title="Lee JW, Shin JY, Seo JS">
                Authors: Lee JW, Shin JY, ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29525983">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>293. Use of the LUS in sequence allele designations to facilitate probabilistic genotyping of NGS-based STR typing results.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29525576" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29525576" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29525576">
                    <p>Some of the expected advantages of next generation sequencing (NGS) for short tandem repeat (STR) typing include enhanced mixture detection and genotype resolution via sequence variation among non-homologous alleles of the same length. However, at the same time that NGS methods for forensic DNA typing have advanced in recent years, many caseworking laboratories have implemented or are transitioning to probabilistic genotyping to assist the interpretation of complex autosomal STR typing results. Current probabilistic software programs are designed for length-based data, and were not intended to accommodate sequence strings as the product input. Yet to leverage the benefits of NGS for enhanced genotyping and mixture deconvolution, the sequence variation among same-length products must be utilized in some form. Here, we propose use of the longest uninterrupted stretch (LUS) in allele designations as a simple method to represent sequence variation within the STR repeat regions and facilitate - in the nearterm - probabilistic interpretation of NGS-based typing results. An examination of published population data indicated that a reference LUS region is straightforward to define for most autosomal STR loci, and that using repeat unit plus LUS length as the allele designator can represent greater than 80% of the alleles detected by sequencing. A proof of concept study performed using a freely available probabilistic software demonstrated that the LUS length can be used in allele designations when a program does not require alleles to be integers, and that utilizing sequence information improves interpretation of both single-source and mixed contributor STR typing results as compared to using repeat unit information alone. The LUS concept for allele designation maintains the repeat-based allele nomenclature that will permit backward compatibility to extant STR databases, and the LUS lengths themselves will be concordant regardless of the NGS assay or analysis tools employed. Further, these biologically based, easy-to-derive designations uphold clear relationships between parent alleles and their stutter products, enabling analysis in fully continuous probabilistic programs that model stutter while avoiding the algorithmic complexities that come with string based searches. Though using repeat unit plus LUS length as the allele designator does not capture variation that occurs outside of the core repeat regions, this straightforward approach would permit the large majority of known STR sequence variation to be used for mixture deconvolution and, in turn, result in more informative mixture statistics in the near term. Ultimately, the method could bridge the gap from current length-based probabilistic systems to facilitate broader adoption of NGS by forensic DNA testing laboratories.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29525576">
                    <p>用于短串联重复（STR）分型的下一代测序（NGS）的一些预期优势包括通过相同长度的非同源等位基因之间的序列变异增强混合物检测和基因型解析。然而，与近年来NGS法医DNA分型方法相比，近年来，许多个案实验室已经实施或正在向概率基因分型过渡，以协助解释复杂的常染色体STR分型结果。当前的概率性软件程序是针对基于长度的数据而设计的，并且不打算容纳作为产品输入的序列字符串。然而，为了利用NGS的优势来增强基因分型和混合物解卷积，必须以某种形式利用相同长度产物之间的序列差异。在这里，我们建议在等位基因名称中使用最长的不间断延伸（LUS）作为一种简单的方法来表示STR重复区域内的序列变异，并促进 - 在近亲 - 基于NGS的分型结果的概率解释。对公布的人口数据的检查表明，参考LUS区域可以直接定义大多数常染色体STR基因座，并且使用重复单位加LUS长度作为等位基因标志可表示超过80％的通过测序检测到的等位基因。使用免费提供的概率软件进行的概念证明研究表明，当程序不要求等位基因为整数时，LUS长度可以用于等位基因的命名，并且利用序列信息可以提高对单一来源和混合来源STR的解释键入结果与单独使用重复单元信息进行比较。 LUS等位基因的概念保留了基于重复的等位基因命名法，可以向后兼容现存的STR数据库，而且无论采用哪种NGS检测或分析工具，LUS长度本身都将保持一致。此外，这些基于生物学的，易于推导的名称保持了父母等位基因与其口吃产品之间的明确关系，从而能够在完全连续的概率程序中进行分析，这些程序模拟口吃，同时避免了基于字符串的搜索带来的算法复杂性。尽管使用重复单位加LUS长度作为等位基因标识符不能捕获在核心重复区域之外发生的变异，但这种直接的方法将允许大部分已知的STR序列变异用于混合物去卷积，并且反过来导致近期内更多信息量的混合统计数据。最终，该方法可以弥补目前基于长度的概率系统的差距，促进法医DNA检测实验室更广泛地采用NGS。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Forensic Sci Int Genet">
                Journal: Forensic Sci Int Genet [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29525576" target="_blank">
                PMID: 29525576
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Just RS, Irwin JA">
                Authors: Just RS, Irwin JA
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29525576">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>294. High resolution global chromosomal aberrations from spontaneous miscarriages revealed by low coverage whole genome sequencing.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29525519" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29525519" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29525519">
                    <p>Chromosome aberrations are generally considered as one of the most substantial causative factors contributing to spontaneous miscarriages. Cytogenetic analyses like G-banded karyotype and chromosomal microarray analyses are often performed to further investigate the chromosome status of a miscarried fetus.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29525519">
                    <p>染色体畸变通常被认为是导致自发流产的最重要的致病因素之一。细胞遗传学分析，如G带染色体核型和染色体微阵列分析通常用于进一步调查流产胎儿的染色体状态。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Eur. J. Obstet. Gynecol. Reprod. Biol.">
                Journal: Eur. J. Obstet. Gynecol. Reprod. Biol. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29525519" target="_blank">
                PMID: 29525519
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Qi H, Xuan ZL, Du Y, Cai LR, Zhang H, Wen XH, Kong XD, Yang K, Mi Y, Fu XX, Cao SB, Wang J, Chen CJ, Liang JB">
                Authors: Qi H, Xuan ZL, Du...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29525519">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>295. Twenty years of molecular analyses in amyotrophic lateral sclerosis: genetic landscape of Italian patients.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29525178" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29525178" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29525178">
                    <p>Amyotrophic lateral sclerosis (ALS) is a neurodegenerative disease with a heterogeneous genetic background. Because mutation analysis by Sanger sequencing is costly and time-consuming, in recent years, next-generation sequencing (NGS) techniques have become of much interest. This study analyses the results of 20 years of molecular analyses in ALS patients in our laboratory using traditional methods and NGS. Almost 300 ALS patients underwent genetic analysis with Sanger sequencing of 7 genes or with an NGS panel of 23 genes. The C9orf72 expansion was tested by fragment size analysis. Sanger sequencing revealed mutations in 23.8% of familial and 3.8% of sporadic cases, whereas NGS detected potentially pathogenic variants in 45.5% of familial and 5.4% of sporadic cases and variants of unknown significance in 30.3% of patients. In 11.8% of patients, potentially causative mutations were found in 2 or more ALS genes. Compared to traditional methods, NGS is more effective in revealing possibly causal variants, but counseling patients becomes more complicated due to frequent variants of unknown significance and potentially oligogenic cases.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29525178">
                    <p>肌萎缩性侧索硬化（ALS）是一种具有异质遗传背景的神经退行性疾病。由于Sanger测序的突变分析成本高且耗时，近年来，新一代测序（NGS）技术引起了越来越多的关注。本研究分析了我们实验室使用传统方法和NGS在ALS患者中进行20年分子分析的结果。几乎300名ALS患者接受了7个基因的Sanger测序或者23个基因的NGS小组进行遗传分析。通过片段大小分析测试C9orf72扩增。 Sanger测序显示23.8％的家族性和3.8％的散发性病例突变，而NGS检测到45.5％的家族性和5.4％的散发性病例和30.3％的患者未知意义变异的潜在致病性变异。在11.8％的患者中，在2个或更多ALS基因中发现潜在致病突变。与传统方法相比，NGS在揭示可能的因果变异方面更有效，但由于频繁变异的未知意义和潜在的寡发性病例，咨询患者变得更加复杂。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Neurobiol. Aging">
                Journal: Neurobiol. Aging [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29525178" target="_blank">
                PMID: 29525178
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jun
            </a>

            <a class="btn btn-sm btn-danger" title="Lamp M, Origone P, Geroldi A, Verdiani S, Gotta F, Caponnetto C, Devigili G, Verriello L, Scialò C, Cabona C, Canosa A, Vanni I, Bellone E, Eleopra R, Mandich P">
                Authors: Lamp M, Origone P...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29525178">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>296. Characterization of H9N2 avian influenza viruses from the Middle East demonstrates heterogeneity at amino acid position 226 in the hemagglutinin and potential for transmission to mammals.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29524835" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29524835" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29524835">
                    <p>Next-generation sequencing (NGS) technologies are a valuable tool to monitor changes in viral genomes and determine the genetic heterogeneity of viruses. In this study, NGS was applied to clinical poultry samples from Jordan to detect eleven H9N2 low pathogenic avian influenza viruses (LPAIV). All of the viruses tested belonged to Middle East A genetic group of G1 lineage. Deep sequencing demonstrated a high degree of heterogeneity of glutamine and leucine residues at position 226 in the hemagglutinin (HA) gene, which increases specificity to either avian or mammalian-type receptors. Moreover, additional amino acid changes in PB1, PA, M1, M2, and NS1 were identified among the viruses tested. Compared to single gene amplification, application of NGS for surveillance and characterization of H9N2 LPAIV provides a complete genetic profile of emerging isolates and better understanding of the potential of zoonotic transmissions to mammals.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29524835">
                    <p>新一代测序（NGS）技术是监测病毒基因组变化并确定病毒遗传异质性的有用工具。在这项研究中，NGS被应用于来自约旦的临床家禽样本，以检测11种H9N2低致病性禽流感病毒（LPAIV）。所有测试的病毒都属于中东G1系遗传群。深度测序表明血凝素（HA）基因中第226位的谷氨酰胺和亮氨酸残基具有高度异质性，这增加了对禽类或哺乳动物类型受体的特异性。此外，在测试的病毒中鉴定出PB1，PA，M1，M2和NS1中的额外氨基酸变化。与单基因扩增相比，应用NGS监测和表征H9N2 LPAIV提供了新出现的分离株的完整遗传图谱，并更好地理解了动物传播对哺乳动物的传播潜力。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Virology">
                Journal: Virology [IF=3.353]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29524835" target="_blank">
                PMID: 29524835
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-05
            </a>

            <a class="btn btn-sm btn-danger" title="Chrzastek K, Lee DH, Gharaibeh S, Zsak A, Kapczynski DR">
                Authors: Chrzastek K, Lee ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29524835">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>297. Target capture enrichment of nuclear SNP markers for massively parallel sequencing of degraded and mixed samples.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29524767" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29524767" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29524767">
                    <p>DNA from biological forensic samples can be highly fragmented and present in limited quantity. When DNA is highly fragmented, conventional PCR based Short Tandem Repeat (STR) analysis may fail as primer binding sites may not be present on a single template molecule. Single Nucleotide Polymorphisms (SNPs) can serve as an alternative type of genetic marker for analysis of degraded samples because the targeted variation is a single base. However, conventional PCR based SNP analysis methods still require intact primer binding sites for target amplification. Recently, probe capture methods for targeted enrichment have shown success in recovering degraded DNA as well as DNA from ancient bone samples using next-generation sequencing (NGS) technologies. The goal of this study was to design and test a probe capture assay targeting forensically relevant nuclear SNP markers for clonal and massively parallel sequencing (MPS) of degraded and limited DNA samples as well as mixtures. A set of 411 polymorphic markers totaling 451 nuclear SNPs (375 SNPs and 36 microhaplotype markers) was selected for the custom probe capture panel. The SNP markers were selected for a broad range of forensic applications including human individual identification, kinship, and lineage analysis as well as for mixture analysis. Performance of the custom SNP probe capture NGS assay was characterized by analyzing read depth and heterozygote allele balance across 15 samples at 25 ng input DNA. Performance thresholds were established based on read depth ≥500X and heterozygote allele balance within ±10% deviation from 50:50, which was observed for 426 out of 451 SNPs. These 426 SNPs were analyzed in size selected samples (at ≤75 bp, ≤100 bp, ≤150 bp, ≤200 bp, and ≤250 bp) as well as mock degraded samples fragmented to an average of 150 bp. Samples selected for ≤75 bp exhibited 99-100% reportable SNPs across varied DNA amounts and as low as 0.5 ng. Mock degraded samples at 1 ng and 10 ng exhibited &gt;90% reportable SNPs. Finally, two-person male-male mixtures were tested at 10 ng in contributor varying ratios. Overall, 85-100% of alleles unique to the minor contributor were observed at all mixture ratios. Results from these studies using the SNP probe capture NGS system demonstrates proof of concept for application to forensically relevant degraded and mixed DNA samples.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29524767">
                    <p>来自生物法医样品的DNA可能高度分散并且数量有限。当DNA高度片段化时，基于常规PCR的短串联重复（STR）分析可能失败，因为引物结合位点可能不存在于单个模板分子上。单核苷酸多态性（SNPs）可以作为分析降解样品的替代类型的遗传标记，因为目标变异是单个碱基。然而，基于常规PCR的SNP分析方法仍然需要用于靶扩增的完整引物结合位点。最近，用于靶向富集的探针捕获方法已经显示使用新一代测序（NGS）技术恢复降解的DNA以及来自古代骨样品的DNA的成功。本研究的目标是设计和测试靶向法医相关核SNP标记的探针捕获测定，用于克隆和大规模并行测序（MPS）降解和限制性DNA样品以及混合物。选择总共451个核SNP（375个SNP和36个微型标记）的一组411个多态性标记用于定制探针捕获面板。 SNP标记物被选择用于广泛的法医应用，包括人类个体识别，亲属关系和谱系分析以及混合物分析。定制SNP探针捕获NGS测定的性能通过分析15ng样品在25ng输入DNA处的读取深度和杂合子等位基因平衡来表征。性能阈值基于读取深度≥500X并且杂合体等位基因平衡在距离50:50±10％偏差内，这在451个SNP中的426个中观察到。对这些426个SNPs进行了大小选择样品（≤75bp，≤100bp，≤150bp，≤200bp和≤250bp）以及模拟降解样品碎片化至平均150bp的分析。选择≤75bp的样本在不同的DNA量下呈现99-100％可报告的SNP，低至0.5ng。在1ng和10ng模拟降解样品显示&gt; 90％可报告的SNP。最后，两人男性 - 男性混合物在10ng的贡献者不同比率下进行测试。总体而言，在所有混合比率下观察到85-100％的次要贡献者独特的等位基因。使用SNP探针捕获NGS系统的这些研究结果证明了应用于法医相关降解和混合DNA样品的概念证据。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Forensic Sci Int Genet">
                Journal: Forensic Sci Int Genet [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29524767" target="_blank">
                PMID: 29524767
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Bose N, Carlberg K, Sensabaugh G, Erlich H, Calloway C">
                Authors: Bose N, Carlberg ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29524767">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>298. Novel homozygous GBA2 mutation in a patient with complicated spastic paraplegia.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29524657" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29524657" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29524657">
                    <p>Hereditary spastic paraplegias (HSPs) are a heterogeneous group of neurological disorders characterized primarily by a pyramidal syndrome with lower limb spasticity, which can manifest as pure HSP or associated with a number of neurological or non-neurological signs (i.e., complicated HSPs). The clinical variability of HSPs is associated with a wide genetic heterogeneity, with more than eighty causative genes known. Recently, next generation sequencing (NGS) has allowed increasing genetic definition in such a heterogeneous group of disorders. We report on a 56- year-old man affected by sporadic complicated HSP consisting of a pyramidal syndrome, cerebellar ataxia, congenital cataract, pes cavus, axonal sensory-motor peripheral neuropathy and cognitive decline. Brain MRI showed cerebellar atrophy and thin corpus callosum. By NGS we found a novel homozygous biallelic c.452-1G &gt; C mutation in the b-glucosidase 2 gene (GBA2), known to be causative for autosomal recessive hereditary spastic paraplegia type 46 (SPG46). The rarity of this inherited form besides reporting on a novel mutation, expands the genetic and clinical spectrum of SPG46 related HSP.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29524657">
                    <p>遗传性痉挛性截瘫（HSPs）是一组异质性的神经系统疾病，主要表现为锥体综合征伴有下肢痉挛，可表现为纯粹的HSP或与一些神经或非神经系统体征（即复杂的HSP）相关。 HSPs的临床变异性与广泛的遗传异质性有关，已知有超过八十个致病基因。最近，下一代测序（NGS）已经允许在这样一组异质性疾病中增加遗传定义。我们报告一名56岁男性患者，由散发性复杂HSP组成，包括锥体综合征，小脑性共济失调，先天性白内障，pus cavus，轴突感觉运动周围神经病变和认知功能下降。脑MRI显示小脑萎缩，胼thin体薄。通过NGS，我们在β-葡糖苷酶2基因（GBA2）中发现了一种新型的纯合biallelic c.452-1G&gt; C突变，该突变被认为是造成常染色体隐性遗传性痉挛性截瘫46型（SPG46）的原因。除了报道新突变之外，这种遗传形式的稀有性扩大了SPG46相关HSP的遗传和临床谱。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Clin Neurol Neurosurg">
                Journal: Clin Neurol Neurosurg [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29524657" target="_blank">
                PMID: 29524657
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-May
            </a>

            <a class="btn btn-sm btn-danger" title="Coarelli G, Romano S, Travaglini L, Ferraldeschi M, Nicita F, Spadaro M, Fornasiero A, Frontali M, Salvetti M, Bertini E, Ristori G">
                Authors: Coarelli G, Roman...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29524657">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>299. Primary and acquired EGFR T790M-mutant NSCLC patients identified by routine mutation testing show different characteristics but may both respond to osimertinib treatment.</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29524556" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29524556" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29524556">
                    <p>Primary EGFR T790M mutation is occasionally identified by routine mutation testing in tyrosine kinase inhibitor (TKI)-naive patients with non-small cell lung cancer (NSCLC). We herein aimed to compare the characteristics of primary and acquired T790M mutations in NSCLC patients, and their response to osimertinib. Using amplification refractory mutation system (ARMS) detection, primary T790M was identified in 0.5% (46/8723) of TKI-naive patients, whereas acquired T790M was detected in 49.7% (71/143) of TKI-relapsed patients. T790M always coexisted with a sensitizing EGFR mutation. Primary T790M more commonly coexisted with L858R, whereas acquired T790M was more likely to coexist with exon 19 deletions. Moreover, next-generation sequencing (NGS) showed that concomitant sensitizing EGFR and primary T790M mutant allele frequencies (MAFs) were highly concordant, but acquired T790M MAFs were significantly lower than the sensitizing EGFR MAFs. Sixteen acquired T790M-mutant patients received osimertinib. The median progression-free survival (PFS) was 8.1 months. Four primary T790M-mutant patients received osimertinib and the median PFS was 8.0 months. Together, our study demonstrates that primary and acquired T790M-mutant patients show distinct differences in some clinical and molecular characteristics, but may both respond to osimertinib treatment.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29524556">
                    <p>原发性EGFR T790M突变有时可通过酪氨酸激酶抑制剂（TKI）介导的非小细胞肺癌（NSCLC）患者的常规突变检测来鉴定。我们在此旨在比较原发性和获得性T790M突变在NSCLC患者中的特征及其对osimertinib的反应。使用扩增难治突变系统（ARMS）检测，初次T790M在0.5％（46/8723）TKI初治患者中鉴定，而在TKI复发患者中检测到收获T790M的49.7％（71/143）。 T790M总是与敏化性EGFR突变共存。初级T790M更常见于L858R共存，而获得性T790M更可能与外显子19缺失共存。此外，新一代测序（NGS）显示伴随致敏EGFR和初级T790M突变等位基因频率（MAF）高度一致，但获得性T790M MAF明显低于敏化EGFR MAF。收购的16例T790M突变患者接受osimertinib。中位无进展生存期（PFS）为8.1个月。四名原发性T790M突变患者接受osimertinib治疗，中位PFS为8.0个月。总之，我们的研究表明原发性和获得性T790M突变患者在某些临床和分子特征方面表现出明显差异，但可能都对osimertinib治疗有反应。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Cancer Lett.">
                Journal: Cancer Lett. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29524556" target="_blank">
                PMID: 29524556
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Jun-01
            </a>

            <a class="btn btn-sm btn-danger" title="Li W, Qiu T, Guo L, Ling Y, Gao Y, Ying J, He J">
                Authors: Li W, Qiu T, Guo ...
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29524556">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    
    <div class="panel panel-info">

        <button type="button" class="close">
            <span aria-hidden="true">&times;</span>
        </button>

        <!-- Default panel contents -->
        <div class="panel-heading">
            <b>300. Messages from the second International Conference on Clinical Metagenomics (ICCMg2).</b>
        </div>
        <div class="panel-body">

            <!-- Nav tabs -->
            <ul class="nav nav-tabs" role="tablist">
                <li role="presentation" class="active">
                    <a href="#abstract_29524500" aria-controls="abstract" role="tab" data-toggle="tab">Abstract</a>
                </li>
                <li role="presentation">
                    <a href="#abstract_cn_29524500" aria-controls="abstract_cn" role="tab" data-toggle="tab">Abstract_Chinese</a>
                </li>
            </ul>

            <!-- Tab panes -->
            <div class="tab-content well">
                <div role="tabpanel" class="tab-pane active" id="abstract_29524500">
                    <p>Clinical metagenomics (CMg) refers to as the application of metagenomic sequencing of clinical samples in order to recover clinically-relevant information. Due to the increasing access to next-generation sequencing (NGS) facilities, CMg is a gast-growing field. In this context, we held the second International Conference on Clinical Metagenomics (ICCMg2) in Geneva in October 2017. During the two days of the conference, several aspects of CMg were addressed, which we propose to summarize in the present manuscript. Besides, we also discuss the evolution of CMg from the last conference held in 2016. In brief, many technical issues related to CMg are expected to be successfully addressed in the coming years. Conversely, assessing the clinical value of CMg, implementing a quality process, storage of data and the ethics of CMg are emerging challenges.</p>
                </div>
                <div role="tabpanel" class="tab-pane" id="abstract_cn_29524500">
                    <p>临床宏基因组学（CMg）指为了恢复临床相关信息而应用临床样本的宏基因组测序。由于越来越多地使用下一代测序（NGS）设备，CMg是一个快速发展的领域。在此背景下，我们于2017年10月在日内瓦举行了第二届国际临床宏基因组学大会（ICCMg2）。在会议的两天期间，讨论了CMg的几个方面，我们建议在本手稿中对其进行总结。此外，我们还讨论了2016年举行的上次会议的CMg演变。简言之，与CMg有关的许多技术问题预计将在未来几年成功解决。相反，评估CMg的临床价值，实施质量过程，数据存储和CMg的道德操守是新出现的挑战。</p>
                </div>
            </div>

            <a class="btn btn-sm btn-info" target="_blank" href="http://www.letpub.com.cn/index.php?page=journalapp&view=search&searchsort=relevance&searchname=Microbes Infect.">
                Journal: Microbes Infect. [IF=.]
            </a>

            <a class="btn btn-sm btn-warning" href="https://www.ncbi.nlm.nih.gov/pubmed/29524500" target="_blank">
                PMID: 29524500
            </a>

            <a class="btn btn-sm btn-success">
                PubDate: 2018-Apr
            </a>

            <a class="btn btn-sm btn-danger" title="Ruppé E, Schrenzel J">
                Authors: Ruppé E, Schrenzel J
            </a>

            

            <form action="http://sci-hub.tw/" method="POST" style="display:inline" target="_blank">
                <input type="hidden" name="request" value="29524500">
                <input type="submit" value="PDF(Sci-Hub)" class="btn btn-sm btn-primary sci-hub">
            </form>

        </div>
    </div>
    

</div>



<script src="https://cdn.bootcss.com/jquery/3.3.1/jquery.min.js"></script>
<script src="https://cdn.bootcss.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>


<script>
    $(document).ready(function () {
        $('.close').click(function () {
            $(this).parent().fadeOut();
        });

        $('.panel-heading').click(function () {
            $(this).next().fadeToggle();
            $(this).next().next().fadeToggle();
        });

        //var panels = ['panel-success', 'panel-info', 'panel-warning', 'panel-danger'];
        //$('.panel').each(function () {
        //    $(this).attr('class', )
        //});
    });
</script>